\section{Abstract}
We describe a high-resolution approach for identifying microbial contributions
to soil C-cycling dynamics using nucleic acid stable isotope probing coupled
with next generation sequencing (HR-SIP). We amended series of soil microcosms
with a complex mixture of model carbon (C) substrates and inorganic nutrients
common to plant biomass. A single C constituent in the C substrate mixture was
substituted for its $^{13}$C-labeled equivalent in each microcosm series.
Specifically, in separate microcosms we used substituted $^{13}$C-xylose or
$^{13}$C-cellulose for their unlabeled equivalents. Xylose and celluose were
chosen to represent labile soluble C and polymeric insoluble C, respectively.
Incorporation of $^{13}$C into DNA was measured at 5 time points in a 30
incubation. 16S rRNA gene sequences from CsCl gradient fractions were profiled
by 454 pyrosequencing. Incorporation of $^{13}$C from xylose into microcosm DNA
was observed at days 1, 3, and 7, while incorporation of $^{13}$C from
cellulose was peaked at day 14 and was maintained through day 40. Of over 6,000
OTUs detected, a total of XX and XX unique OTUs assimilated $^{13}$C from
xylose and cellulose, respectively. Xylose assimilating OTUs are more abundant
in the microbial community than cellulose assimilating OTUs, while cellulose
OTUs demonstrate a greater substrate specificity than xylose OTUs.        
