\section{Abstract} 
We describe a novel approach for identifying microbial contributions to soil C-cycling dynamics using nucleic acid stable isotope probing coupled with next generation sequencing. Using a series of parallel soil microcosms, we amended soils with a complex mixture of model carbon (C) substrates and inorganic nutrients common to plant biomass, where a single C constituent is substituted for its \textsuperscript{13}C-labeled equivalent.

simulating soil organic matter was added to soil microcosms. In this approach, constituents of the added C mixture are systematically replaced with their 13C-labeled equivalents. Initial treatments included C mixtures that were either unlabeled or incorporated a 13C-xylose or 13C-cellulose label.  Using CsCl gradient fractionation, incorporation of 13C into DNA was measured over a period of 30 days. The 16S rRNA gene sequences from all CsCl gradient fractions were characterized by 454 pyrosequencing and classified into Operational Taxonomic Units (OTU).  OTUs that assimilated xylose and cellulose were identified and patterns of isotope incorporation were measured over time. Incorporation of 13C from xylose into OTUs was observed at days 3, 7, and 14, while notable incorporation of 13C from cellulose was observed only after day 14. We describe specific patterns of C-assimilation by thousands of discrete OTUs as a function of substrate, time, and level of isotope incorporation

We describe a nitrogenase gene sequence database that facilitates analysis of the evolution and ecology of nitrogen-fixing organisms. The database contains 32 954 aligned nitrogenase nifH sequences linked to phylogenetic trees and associated sequence metadata. The database includes 185 linked multigene entries including full-length nifH, nifD, nifK and 16S ribosomal RNA (rRNA) gene sequences. Evolutionary analyses enabled by the multigene entries support an ancient horizontal transfer of nitrogenase genes between Archaea and Bacteria and provide evidence that nifH has a different history of horizontal gene transfer from the nifDK enzyme core. Further analyses show that lineages in nitrogenase cluster I and cluster III have different rates of substitution within nifD, suggesting that nifD is under different selection pressure in these two lineages. Finally, we find that that the genetic divergence of nifH and 16S rRNA genes does not correlate well at sequence dissimilarity values used commonly to define microbial species, as stains having <3% sequence dissimilarity in their 16S rRNA genes can have up to 23% dissimilarity in nifH. The nifH database has a number of uses including phylogenetic and evolutionary analyses, the design and assessment of primers/probes and the evaluation of nitrogenase sequence diversity. Database URL: http://www.css.cornell.edu/faculty/buckley/nifh.htm.