\section{Abstract} We identify microorganisms participating in the
transformation of xylose or cellulose in soil microcosms by using nucleic acid
stable isotope probing (SIP) coupled to next generation sequencing. Microcosms
were incubated with $^{13}$C-xylose or $^{13}$C-cellulose and microcosm DNA was
interrogated for $^{13}$C-incorporation at days 1, 3, 7,
14 and 30. A total of 49 and 63 unique OTUs assimilated $^{13}$C from xylose
and cellulose into DNA, respectively. Incorporation of $^{13}$C from xylose
into microcosm DNA was observed at days 1, 3, and 7, while incorporation of
$^{13}$C from cellulose peaked at day 14 and was maintained at day 30.
Importantly, many cellulose degraders identified in this study are members of
cosmopolitan but physiologically uncharacterized soil microbial lineages
including \textit{Spartobacteria}, \textit{Chloroflexi} and
\textit{Planctomycetes}. $^{13}$C-xylose additions precipitated a temporal
cascade of $^{13}$C-incorporation activity that could be due to significant
predatory interactions among bacteria yet intra-bacteria predatory interactions
are rarely considered in soil C cycling conceptual models. Microorganisms that
assimilated $^{13}$C-xylose were faster growing and displayed less substrate
specificity than microorganisms that incorporated $^{13}$C from cellulose as
assessed by predicted \textit{rrn} gene copy number for each OTU and respective DNA
buoyant density shifts in response to $^{13}$C-labeling. Also, microorganisms
that assimilated $^{13}$C from xylose were phylogenetically overdispersed
whereas $^{13}$C-cellulose assimilating microorganisms were phylogenetically
constrained. Describing the ecology and identities of soil C cycling
microorganisms will calibrate and inform predictions of terrestrial carbon flux
in response to climate change and land management. Tuning terrestrial C flux
models with appropriate parameters for soil biomass is crucial for reconciling
contrasting predictions of soil as a future C sink and source.
