\section{Discussion}
% Fakesubsubsection:Pure culture based studies drove early
Pure culture based studies drove early soil microbial ecology research.
Important pure cultures from soil historically included nine genera
\textit{Agrobacterium}, \textit{Alcaligenes}, \textit{Arthrobacter},
\textit{Bacillus}, \textit{Flavobacterium}, \textit{Micromonospora},
\textit{Nocardia}, \textit{Pseudomonas}, and \textit{Streptomyces}
(\citep{Alexander1977} and reviewed by \citet{Janssen2006}) but
culture-independent surveys of soil microbial diversity revealed soil can
harbor 5,000 OTUs per half gram of soil \citep{Schloss2006}. We recovered almost 6,000
OTUs in this study. Although culturing techniques can produce isolates from
diverse soil lineages \citep{Janssen2002}, numerically dominant soil
microorganisms are still uncultured and we know little of their ecophysiology
\citep{Janssen2006}. In contrast DNA-SIP can characterize functional roles for
thousands of phylotypes in a single experiment. We found 104 OTUs in an
agricultural soil that can incorporate C from xylose and/or cellulose into
biomass. We also used DNA-SIP to assay substrate specificity
and temporal dynamics of C-cycling or soluble and polymeric C degraders.
Included in the $^{13}$C-xylose and $^{13}$C-cellulose responsive OTUs were
members of numerically dominant yet functionally uncharacterized soil
phylogenetic groups such as \textit{Verrucomicrobia}, \textit{Planctomycetes}
and \textit{Chloroflexi}.

\subsection{Microbial response to isotopic labels}
% Fakesubsubsection: We propose that microbial decomposition
We propose that C added to soil microcosms in this experiment followed the
following path through the microbial food web (Figure~\ref{fig:foodweb}):
First, labile C such as xylose was assimilated by fast-growing opportunistic
\textit{Firmicutes} spore formers. The remaining labile C and new biomass C was
assimilated in succession by slower growing \textit{Bacteroidetes},
\textit{Actinobacteria} and \textit{Proteobacteria} phylotypes that were either
tuned to lower C substrate concentrations, were predatory bacteria (e.g.
\textit{Agromyces}), and/or were specialized for consuming viral lysate.
C from polymeric substrates entered the bacterial community after 14 days.
Canonical cellulose degrading bacteria such as \textit{Cellvibrio} degraded
cellulose degraders but uncharacterized lineages in the \textit{Chloroflexi},
\textit{Planctomycetes} and \textit{Verrucomicrobia}, specifically the
\textit{Spartobacteria}, were also significant contributors to cellulose
decomposition. 

\subsection{Ecological strategies of soil microorganisms participating in the
decomposition of organic matter}
% Fakesubsubsection:We assessed
We assessed the ecology of $^{13}$C-responsive OTUs by estimating the
\textit{rrn} gene copy number and the BD shift upon labeling for each OTU.
\textit{rrn} gene copy number correlates positively with growth rate
\citep{11125085} and BD shift is indicative of substrate specificity (see
results). We also observed how $^{13}$C-substrate responsive OTUs changed in
relative abundance with time in the microcosms and the abundance rank of
$^{13}$C-substrate responsive OTUs in the bulk DNA. Ecological metrics show
$^{13}$C-cellulose responsive OTUs grow slower (Figure~\ref{fig:copy}), have
greater substrate specificity (Figure~\ref{fig:shift}), and are generally lower
abundance than $^{13}$C-xylose responsive OTUs (Figure~\ref{fig:shift}). The
higher abundance of xylose responders may also be in part due to of their high
\textit{rrn} gene copy number resulting in an inflated abundance relative per
genome. There are only faint ecological differences
within the $^{13}$C-cellulose responsive OTUs but the combination of
\textit{rrn} gene copy number, BD shift, abundance rank and relative abundance
change over time is consistent with phylum membership (Figure~RADVIZ).
$^{13}$C-xylose responsive OTU \textit{rrn} gene copy number correlated
inversely with the time at which the OTU was first found to incorporate
$^{13}$C into DNA (Figure~\ref{fig:copy}) suggesting that fast-growing microbes
assimilated $^{13}$C from xylose before slow growers.  

% Fakesubsubsection:Ecological metrics suggest
Ecological metrics suggest cellulose degraders are substrate specialists that
grow slow and are in low bulk abundance. Labile C responder ecological
strategies were more varied perhaps because some $^{13}$C labeled
microorganisms did not primarily assimilate xylose but became labeled via
predatory interactions and/or are saprophytes. $^{13}$C-xylose responsive OTUs
are generalists, grow faster and are more abundant when compared to
$^{13}$C-cellulose responders. $^{13}$C-xylose responders vary in growth rate
and while generally higher abundance than $^{13}$C-cellulose responders can
also be low abundance microorganisms. It's not clear whether the observed
activity succession from \textit{Firmicutes} to \textit{Bacteroidetes} and
finally \textit{Actinobacteria} in response to $^{13}$C-xylose addition marks
a trophic cascade or functional groups tuned to different resource
concentrations or both. Notably, each temporally defined response group
clustered phylogenetically suggesting a uniform ecological strategy
(Figure~\ref{fig:trees}). It's also clear that some of the
non-\textit{Firmicutes} $^{13}$C-xylose responders are closely related to known
predators (\textit{Agromyces}) and many marine predatory bacteria are members
of the \textit{Bacteroidetes} (CITE). If the temporal dynamics of
$^{13}$C-xylose incorporation are due to trophic interactions, our results
suggest that there are many predatory soil bacteria that consume fast-growing,
opportunistic, primary labile C assimilating, gram-positive spore-formers.
Hence, trophic interactions among soil bacteria may be of importance in soil
C turnover models.

\subsection{Conclusion} 
% Fakesubsubsection:SOM represents more C than the
Microorganisms sequester atmospheric carbon and respire soil organic matter
(SOM) influencing climate change on a global scale but we do not know which
microorganisms carry out soil C transformations. Molecular tools will unravel
the soil microbial food web and reveal how specific microorganisms impact soil
C flux. Our results show physiologically undefined yet cosmopolitan soil
microorganisms decompose cellulose. We also show phylogenetic groups rise and
fall and are supplanted by others in activity over 7 days in response to labile
C addition and OTUs that assimilate xylose and those that assimilate
cellulose are largely different. NEEDS WORK.
