\section{Discussion}
% Fakesubsubsection:Early soil microbial ecology was informed
Pure culture based studies drove early soil microbial ecology. Historically
important pure cultures from soil included nine genera \textit{Agrobacterium},
\textit{Alcaligenes}, \textit{Arthrobacter}, \textit{Bacillus},
\textit{Flavobacterium}, \textit{Micromonospora}, \textit{Nocardia},
\textit{Pseudomonas}, and \textit{Streptomyces} (\citep{Alexander1977} and
reviewed by \citet{Janssen2006}) but culture-independent surveys of soil
microbial diversity revealed soil can harbor 5,000 OTUs per half gram
\citep{Schloss2006}. We recovered almost 6,000 OTUs in this study. Although
culturing techniques can produce isolates from diverse soil lineages
\citep{Janssen2002}, numerically dominant soil microorganisms are still
uncultured and we know little of their ecophysiology \citep{Janssen2006}. In
contrast DNA-SIP can characterize functional roles for thousands of phylotypes
in a single experiment. We found 114 OTUs in an agricultural soil that can
incorporate C from xylose and/or cellulose into biomass. Included in the
$^{13}$C-xylose and $^{13}$C-cellulose responsive OTUs were members of
numerically dominant yet functionally uncharacterized soil phylogenetic groups
such as \textit{Verrucomicrobia}, \textit{Planctomycetes} and
\textit{Chloroflexi}. We also used DNA-SIP to assay substrate specificity and
temporal dynamics of C-cycling or soluble and polymeric C degraders.

\subsection{Microbial response to isotopic labels}
% Fakesubsubsection: We propose that microbial decomposition
We propose that organic matter in soil follows the following path
(Figure~\ref{fig:foodweb}): Labile C such as xylose is assimilated by
fast-growing opportunistic \textit{Firmicutes} spore formers. The remaining
labile C and new biomass C is assimilated in succession by slower growing
\textit{Bacteroidetes}, \textit{Actinobacteria} and \textit{Proteobacteria}
phylotypes that are either tuned to lower C substrate concentrations, are
predatory bacteria (e.g. \textit{Agromyces}), and/or are specialized for
consuming viral lysate. Polymeric C enters the bacterial community after
several weeks. Canonical cellulose degrading bacteria such as
\textit{Cellvibrio} are major cellulose degraders but uncharacterized lineages
in the \textit{Chloroflexi}, \textit{Planctomycetes} and
\textit{Verrucomicrobia}, specifically the \textit{Spartobacteria}, are
significant contributors to soil cellulose decomposition as well. 

\subsection{Ecological strategies of soil microorganisms participating in the
decomposition of organic matter}
% Fakesubsubsection:We assessed
We assessed the ecology of $^{13}$C-responsive OTUs by estimating the
\textit{rrn} gene copy number and the BD shift upon labeling for each OTU.
\textit{rrn} gene copy number correlates positively with growth rate
\citep{11125085} and BD shift is indicative of substrate specificity. We
also observed how $^{13}$C-substrate responsive OTUs changed in relative
abundance with time in the microcosms and the abundance rank of
$^{13}$C-substrate responsive OTUs in the bulk DNA. Ecological metrics show
$^{13}$C-cellulose responsive OTUs grow slower (Figure~\ref{fig:copy}),
    have greater substrate specificity (Figure~\ref{fig:shift}), and are
generally lower abundance than $^{13}$C-xylose responsive OTUs
(Figure~\ref{fig:shift}). There are only faint ecological differences within
the $^{13}$C-cellulose responsive OTUs but the combination of \textit{rrn} gene
copy number, BD shift, abundance rank and relative abundance change over time
is consistent with phylum membership (Figure~RADVIZ). $^{13}$C-xylose responsive
OTU \textit{rrn} gene copy number was negatively correlated with the time at
which the OTU was first found to incorporate $^{13}$C into DNA
(Figure~\ref{fig:copy}) suggesting that fast-growing microbes assimilated
$^{13}$C from xylose before slower growing types.  

% Fakesubsubsection:Ecological metrics suggest
Ecological metrics indicate cellulose degraders are substrate specialists that
grow slow and are in low bulk abundance. Labile C responder ecology is more
varied perhaps because some $^{13}$C labeled microorganisms did not primarily
assimilate xylose but became labeled via predatory interactions and/or are
saprophytes. $^{13}$C-xylose responsive OTUs are generalists, grow faster and
are more abundant when compared to $^{13}$C-cellulose responders.
$^{13}$C-xylose responders vary in growth rate and while generally lower
abundance than $^{13}$C-cellulose responders can also be low abundance
microorganisms. It's not clear whether the observed activity succession from
\textit{Firmicutes} to \textit{Bacteroidetes} and finally
\textit{Actinobacteria} in response to $^{13}$C-xylose addition marks a trophic
cascade or functional groups tuned to different resource concentrations or
both. Notably, each temporally defined response group clustered
phylogenetically suggesting a uniform ecological strategy (Figure~XX). It's
also clear that some of the non-\textit{Firmicutes} $^{13}$C-xylose responders
are closely related to known predators (\textit{Agromyces}) and many marine
predatory bacteria are members of the \textit{Bacteroidetes} (CITE). If the
temporal dynamics of $^{13}$C-xylose incorporation are due to trophic
interactions, our results suggest that there many predatory soil bacteria that
consume fast-growing, opportunistic, primary labile C assimilating,
gram-positive spore-formers. Hence, trophic interactions among soil bacteria
may be of importance in soil C turnover models.

% Fakesubsubsection:C substrate specificity
ARE OUR RESULTS CONSISTENT WITH SUBSTRATE SPECIFICITY STUDIES? C substrate
specificity can be assessed by measuring the BD shift of OTU DNA upon $^{13}$C
incorporation CITE. OTUs that incorporate more $^{13}$C per unit DNA have
greater specificity for the labeled substrate than OTUs that incorporate less
$^{13}$C per unit DNA. $^{13}$C-cellulose incorporating OTUs as a group
displayed greater substrate specificity than $^{13}$C-xylose incorporating OTUs
(p-value XX, Wilcoxon Rank Sum test). This suggests that polymeric C-degraders
tend to be specialists tuned to particular C-substrates such as cellulose or
lignin whereas labile C-degraders are generalists able to assimilate C from
many different soluble, labile sources. Although we observed a succession of
$^{13}$C-xylose responders (Figure~\ref{fig:l2fc} and \ref{fig:xyl_count}),
there was no discernible difference in substrate specificity between
$^{13}$C-xylose responders at days 1, 3 or 7.

\subsection{Phylogenetic affiliation of $^{13}$C-cellulose
    $^{13}$C-xylose responsive microorganisms}
% Fakesubsubsection:\textit{Verrucomicrobia} are ubiquitous in soil
\textit{Verrucomicrobia}, cosmopolitan soil microbes
\citep{Bergmann_2011}, can comprise up to 23\% of 16S rRNA gene sequences in
high-throughput DNA sequencing surveys of SSU rRNA genes in soil
\citep{Bergmann_2011} and can account for up to 9.8\% of
soil 16S rRNA \citep{Buckley_2001}. Many \textit{Verrucomicrobia} were first
isolated in the last decade \cite{Wertz_2011} but only one of the 15 most
abundant verrucomicrobial phylotypes in a global soil sample collection shared
greater than 93\% sequence identity with a cultured isolate
\citep{Bergmann_2011}. Genomic analyses and physiological profiling of
\textit{Verrucomicrobia} isolates revealed \textit{Verrucomicrobia} are capable
of methanotrophy, diazotrophy, and cellulose degradation \citep{Otsuka_2012,
Wertz_2011}. The function of soil \textit{Verrucomicrobia} in global C-cycling
remains unknown. Only two of the ten putative cellulose degrading
\textit{Verrucomicrobia} identified in this experiment shares at least 95\%
sequence identity with an isolate ("OTU.83" and "OTU.627",
Table~\ref{tab:cell}). Seven of ten $^{13}$C-cellulose responding
verrucomicrobial OTUs were classified as \textit{Spartobacteria} which are
a numerically dominant family of \textit{Verrucomicrobia} in SSU rRNA gene
surveys of 181 globally distributed soil samples \citep{Bergmann_2011}. Given
their ubiquity and abundance in soil as well as their demonstrated
incorporation of $^{13}$C from $^{13}$C-cellulose, \textit{Verrucomicrobia}
lineages, particularly \textit{Spartobacteria}, may be important contributors
to cellulose decomposition on a global scale.

% Fakesubsubsection:Soil \textit{Chloroflexi} have been found to assimilate
Cellulose degrading soil \textit{Chloroflexi} have previously been identified
in DNA-SIP studies \citep{Schellenberger_2010}. The cellulose degrading
\textit{Chloroflexi} in this study are only distantly related to isolates
\ref{tab:cell}. Chloroflexi are among the six most abundant soil phyla commonly
recovered soil microbial diversity surveys \citep{Janssen2006}.
Chloroflexi are typically not as as abundant as \textit{Verrucomicrobia} but
are roughly as abundant as \textit{Bacteroidetes} and \textit{Planctomycetes}
\citep{Janssen2006}.  Four of five $^{13}$C-cellulose responsive
\textit{Chloroflexi} identified in this study are annotated as belonging to the
\textit{Herpetosiphon} although the \textit{Herpetosiphon} SSU rRNA gene
sequences from this study from $^{13}$C-cellulose responsive OTUs all share
less than 95\% sequence identity with their closest cultured relative in the
\textit{Herpetosiphon} genus (\textit{H. geysericola}). \textit{H. geysericola}
is a predatory bacterium shown to prey upon \textit{Aerobacter} in culture and
can also digest cellulose \citep{Lewin1970}. In our study, "Herpetosiphon"
$^{13}$C-cellulose responders did not show a delayed response to
$^{13}$C-cellulose as compared to other responders but nonetheless could have
become labeled by feeding on primary $^{13}$C-cellulose degraders. The prey
specificity of predatory bacteria is not well established especially \textit{in
situ}. $^{13}$C-labeling would be positively correlated with prey specificity.
If the predator specifically preyed upon one population then it could take on
the same labeling percent as that population. Preying on multiple types would
produce a mixed and dilute labeling signature if some of the prey
were not isotopically labeled.

% Fakesubsubsection:We also observed $^{13}$C-incorporation
We also observed $^{13}$C-incorporation from cellulose by
\textit{Proteobacteria}, \textit{Planctomycetes} and \textit{Bacteroidetes}. 
Strains in Proteobacteria, Planctomycetes and Bacteroidetes have all been
previously implicated in cellulose degradation. Planctomycetes is the
least studied of the three phyla and only one Planctomycetes isolate can
grow on cellulose. None of the seven \textit{Planctomycetes} cellulose degraders
identified in this experiment are closely related to isolates.
\textit{Acidobacteria} did not pass or operational criteria for assessing
$^{13}$C incorporation from cellulose into DNA in our microcosms. 
\textit{Acidobacteria} have been found to degrade cellulose in culture CITE and
are a numerically significant soil phylum CITE. \textit{Acidobacteria} have
been shown to dominate at low nutrient availability (CITE: cederlund 2014),
which may explain why were not active in this study's  nutrient replete
microcosm conditions. The \textit{Acidobacteria} in our microcosms were mainly
annotated as belonging to the "XX" group. 

\subsection{Conclusion} 
% Fakesubsubsection:SOM represents more C than the
Microorganisms sequester atmospheric carbon and respire soil organic matter
(SOM) influencing climate change on a global scale. The specific microbial
lineages that transform different soil C components are not established,
however. Molecular tools will unravel the soil microbial food web and reveal
how specific microbial lineages impact soil C flux. Our results show
physiologically undefined yet cosmopolitan soil microorganisms decompose
cellulose. We also show phylogenetic groups rise and fall and are supplanted by
others in activity over 7 days in response to labile C addition and OTUs that
assimilate xylose and those that assimilate cellulose are largely different. 

The succession hypothesis of decomposition predicts a succession from
microbial types that use labile C to those that use recalcitrant polymeric
C over time CITE. Cellulose degraders succeeded labile C degraders as
predicted. But, in response to $^{13}$C-xylose, \textit{Firmicutes}
phylotypes were succeeded by \textit{Bacteroidetes} which were then
succeeded by \textit{Actinobacteria} representing a nested succession
(Figure~XX). We found that $^{13}$C substrate responders changed as much as
XX-fold in relative abundance over time (Figure~XX). 

The xylose responders demonstrate a smaller change in BD than the
cellulose responders suggesting that xylose responders assimilate multiple C
sources (labeled and unlabeled) consistent with a generalist response, while
cellulose responders are more heavily labeled suggesting that cellulose is
their main source of C, a response more consistent with a specialist lifestyle.
Xylose responders include many taxa, such as spore-fomers, known for the
ability to respond rapidly to an influx of new nutrients while cellulose
responders include many OTUs that are common in soil but uncultured.
 
Xylose and cellulose utilization were demonstrated across 7 phyla each
revealing a high diversity of bacteria able to utilize these substrates. The
high taxonomic diversity may enable substrate metabolism under a broad range of
environmental conditions \citep{Goldfarb_2011}. Other studies of microbial
communities have observed a positive correlation with taxonomic or phylogenetic
diversity and functional diversity
\citep{Fierer_2012,Fierer_2013,Philippot_2010,Tringe_2005,Gilbert_2010,Bryant_2012}.
The data presented here supports that specific functional attributes can be
shared among diverse taxa and closely related taxa may have very different
physiologies \citep{Fierer_2012,Philippot_2010}. This information adds to the
growing collection of data suggesting that community membership is important to
biogeochemical processes.

In the future deeper sequencing will enable us to increase coverage and assess
C use by more community members. We can expand our knowledge
of soil C use dynamics to a wide array of C substrates and increase our grasp
on specific community member contributions to the soil C cycle.
