\section{Discussion} 
% Fakesubsubsection: We highlight to key results
We highlight two key results with implications for understanding structure-function
relationships in soils, and for applying DNA-SIP in future studies of the soil-C
cycle. First, cellulose responders were members of physiologically undescribed
taxonomic groups with few exceptions. This suggests that we have much to learn
about the diversity of structural-C decomposers in soil before we can begin to
assess how they are affected by climate change and land management. Second, the
response to xylose was characterized by a succession in activity from
\textit{Paenibacillus} OTUs (day 1) to \textit{Bacteroidetes} (day 3) and finally
\textit{Micrococcales} (day 7). This activity succession was mirrored by relative
abundance profiles and may mark trophic-C exchange between these groups and/or
adaptation to different substrate concentrations. Trophic interactions will
critically influence how the global soil-C reservoir will respond to climate
change \citep{Crowther2015} but we know little of biological interactions among
soil bacteria. Often bacteria are cast as a single trophic level
\citep{Moore1988} but it may be appropriate to investigate the soil food web at
greater granularity. Additionally, our results show that DNA-SIP results can
change dramatically over time suggesting that multiple time points are
necessary to rigorously and comprehensively describe which microorganisms
consume $^{13}$C-labeled substrates in nucleic acid SIP incubations.

% Fakesubsubsection: Microorganisms that consumed
Microorganisms that consumed $^{13}$C-cellulose were rarely closely related to any
physiologically characterized cultured isolates but were members of
cosmopolitan phylogenetic groups in soil including \textit{Spartobacteria},
\textit{Planctomycetes}, and \textit{Chloroflexi}. Often cellulose responders
were less than 92\% related to their closest cultured relatives showing that we
can infer little, if anything at all, of their physiology from culture-based
studies. Notably, many \textit{Spartobacteria} were among the cellulose responder OTUs.
This is particularly interesting as \textit{Spartobacteria} are globally
dispersed and found in a variety of soil types \citep{Bergmann_2011}. These lineages may play
important roles in global cellulose turnover and would be interesting to track
in soil warming experiments to project how climate change might influence
cellulose process rates in soil (please see SI note 1 for further discussion of the
phylogenetic affiliation of cellulose responders). It should also be noted that
we amended our soil with bacterial cellulose which differs in structure than
plant biomass cellulose. These structural differences might select for different 
decomposers and caution should be taken when extrapolating our results.

% Fakesubsubsection: In addition to taxonomic identity, we quantified
In addition to taxonomic identity, we quantified four ecological properties of
microorganisms that were actively engaged in labile and structural C
decomposition in our experiment: (1) time of activity, (2) estimated
\textit{rrn} gene copy number, (3) phylogenetic clustering, and (4) density
shift in response to $^{13}$C-labeling as a rough estimate of relative
substrate specificity. Labile C was consumed before structural C by different
microorganisms. This was expected and is consistent with the degradative
succession hypothesis. Consumers of labile C had higher estimated \textit{rrn}
gene copy number than structural C consumers.  \textit{rrn} copy number is
positively correlated with the ability to resuscitate quickly in response to
nutrient influx \citep{Klappenbach_2000} which may be the advantage that
enabled xylose responders to rapidly consume xylose. Both xylose and cellulose
responders were terminally clustered phylogenetically suggesting habitat
filtering for labile and structural C consumers. Although labile C
consumption is generally considered to be a ``broad'' process, we found that
xylose responders at day 1 were mainly members of one genus,
\textit{Paenibacillus}.  It should be noted, however, the types of
$^{13}$C-labeled microorganisms changed over time in the $^{13}$C-xylose
treatment, and these microorganisms were overall more diverse than those
labeled in the $^{13}$C-cellulose treatment. It is not clear, though,  if
$^{13}$C-labeled microorganisms in the $^{13}$C-xylose treatment beyond day 1
can be considered $^{13}$C-xylose consumers as they may have consumed
$^{13}$C-labeled metabolic byproducts or $^{13}$C-labeled biomass (see below).
Regardless, most xylose-C decomposition happened by day 1 and therefore this
experiment suggests that life-history traits such as the ability to resuscitate
quickly and/or maximum growth rate may constrain the diversity of
microorganisms involved in the consumption of the lion's share of labile C.
Hence labile C decomposition may not be as ``broad'' a process as generally
thought based on studies of the genomic potential and ability in culture for
labile C consumption. The phylogenetic breadth of microbial guilds is important
to measure as the diversity of microorganisms that participate in an ecosystem
function is assumed to be positively correlated with how robust the function is
to changes in community composition \citep{Schimel_1995} (see SI note 2 for
further discussion with respect to soil-C modelling). And last, In contrast to
structural C consumers, labile C consumers showed evidence for lower substrate
specificity. We assessed substrate specificity by measuring buoyant density
shift in response to $^{13}$C-labeling (see $\Delta\hat{BD}$). This technique
could be used to assess relative substrate specificity for microorganisms that
consume a variety of substrates across a range of lability.

% Fakesubsubsection: We propose that the temporal fluctuations
We propose that the temporal fluctuations in $^{13}$C-labeling in the
$^{13}$C-xylose treatment are due to trophic exchange of $^{13}$C.
Alternatively, the temporal dynamics could be caused by microorganisms tuned to
different substrate concentrations and/or cross-feeding. Only trophic exchange,
however, can account for the precipitous drop in abundance of
\textit{Paenibacillus} with subsequent $^{13}$C-labeling of
\textit{Bacteroidetes} at day 3 followed by Micrococcales at day 7.
Furthermore, \textit{Bacteroidetes} types have been shown to become $^{13}$C
labeled after the addition of live $^{13}$C-labeled \textit{Escherichia coli}
to soil \citep{Lueders2006} and a member of the \textit{Micrococcales},
\textit{Agromyces ramosus}, is a known predator that preys
on many microorganisms including yeast and \textit{Micrococcus luteus}
\citep{16346402}. \textit{Agromyces} are abundant microorganisms in many soils
and the most abundant xylose responder in our experiment -- the fourth most
abundant OTU in our entire pooled dataset -- shares 100\% SSU rRNA gene sequence
identity to \textit{Agromyces ramosus} (see SI note 3 for further discussion of
trophic C exchange). Climate change is expected to diminish bottom-up controls 
on microbial growth increasing the importance on top-down biological
interactions for mitigating positive climate change feedbacks 
\citep{Crowther2015} though the extent of bacterial predatory activity in soil
is unknown. Elucidating the identities of bacterial predators in soil will
assist in assessing the implications of climate change on global soil-C
storage.

\subsection{Conclusion} 
% Fakesubsubsection: Microorganisms sequester atmospheric C
Microorganisms govern C-transformations in soil influencing climate change on
a global scale but we do not know the identities of microorganisms that carry
out each C transformation. In this experiment microorganisms from physiologically
uncharacterized but cosmopolitan soil lineages participated in cellulose
decomposition. Cellulose responders included members of the
\textit{Verrucomicrobia} (\textit{Spartobacteria}), \textit{Chloroflexi},
\textit{Bacteroidetes} and \textit{Planctomycetes}. \textit{Spartobacteria} in
particular are globally cosmopolitan soil microorganisms and are often the most
abundant \textit{Verrucomicrobia} order in soil \citep{Bergmann_2011}.
Fast-growing aerobic spore formers from \textit{Firmicutes} assimilated labile
C in the form of xylose. Xylose responders within the \textit{Bacteroidetes}
and \textit{Actinobacteria} likely became labeled by consuming $^{13}$C-labeled
constituents of microbial biomass either by saprotrophy or predation. Our
results suggest that cosmopolitan \textit{Spartobacteria} may degrade cellulose
on a global scale, decomposition of labile plant C may initiate trophic transfer 
within the bacterial food web, and life history traits may act
as a filter constraining the diversity of active microorganisms relative to
those with the genomic potential for a given metabolism.
