\section{Discussion} 
% Fakesubsubsection: We identified microorganisms
We identified microorganisms participating in soil C cycling using a nucleic
acid SIP approach. Specifically, we observed assimilation of $^{13}$C from
either $^{13}$C-xylose or $^{13}$C-cellulose into DNA for 104 OTUs in an 
agricultural soil. We found $^{13}$C from $^{13}$C-xylose appeared to
move into and then out of groups of related OTUs over time. By coupling nucleic
acid SIP to high throughput sequencing we could diagnose OTU activity even when
OTUs were at low relative abundance in non-fractionated DNA (e.g. on three
occasions we did not detect $^{13}$C-responders in the non-fractionated DNA).
Our results support the degradative succession hypothesis, elucidate
ecophysiological properties of soil microorganisms, reveal activity of
widespread uncultured soil bacteria, and begin to piece together the microbial
food web in soils. 

% Fakesubsubsection: The degradative succession hypothesis
The degradative succession hypothesis predicts an ecological transition in
activity during the decomposition of plant matter from microbes that decompose
labile plant biomass C to those that decompose structural more recalcitrant C.
Our results concur with the degradative succession hypothesis. Microorganisms
consumed xylose-C before cellulose-c and assimilated xylose-C into DNA faster
than to cellulose-C. Xylose is major constituent of hemicellulose and is
a labile component of fresh plant biomass. The phylogenetic composition of
xylose responders changed between days~1,~3 and~7 and few OTUs appeared
$^{13}$C-labeled in the $^{13}$C-xylose treatment after day~7. In the
$^{13}$C-cellulose treatment, few OTUs were $^{13}$C-labeled in the in the
beginning of the experiment but $^{13}$C labeled OTUs increased at day~14 and
maintained $^{13}$C-labeling through day~30. Finally, few (8~of~104) OTUs
appeared to metabolize both xylose and cellulose meaning over 30 days cellulose
responders succeeded xylose responders in activity. In addition to agreeing
with the degradative succession hypothesis, our results suggest complex
interactions between microbes occur during labile C decomposition.

% Fakesubsubsection: Correlations between community composition
Correlations between community composition and environmental characteristics
often indirectly reveal microorganisms that belong to ecologically consistent
groups \citep{Fierer2007}. In this experiment, we directly identified
ecological groups as a function of \textit{in situ} metabolism and inferred
group ecological properties through phylogenetic affiliation, temporal dynamics
of $^{13}$C-assimilation, and the extent of $^{13}$C-labeling per OTU. Xylose
responders grew faster than cellulose responders and appeared to assimilate
C from multiple sources. Xylose responders assimilated xylose-C into DNA
within~24 hours and had low $\Delta\hat{BD}$ relative to cellulose responders
suggesting xylose was not the sole C source used for growth. Xylose represented
20\% of the amendment and~3.5\% of total soil C. Xylose responders often
included the most abundant OTUs within the non-fractionated DNA and had high
estimated \textit{rrn} copy number relative to cellulose responders. However,
to some degree, high \textit{rrn} gene copy number may inflate observed xylose
responder relative abundance. Notably, the majority of xylose responder SSU
rRNA genes (86\%) matched SSU rRNA genes from cultured isolates at high
sequence identity ($>$ 97\%). 

% Fakesubsubsection: In contrast, the results
Cellulose responders, on the other hand, incorporated $^{13}$C into DNA after
xylose responders and appeared to specialize in using cellulose as a C source.
Cellulose responders grew over a span of weeks and had high $\Delta\hat{BD}$
indicating cellulose remained the dominant C source for cellulose responders
even though multiple sources of C were present (cellulose represented 6\% of
total C present in soil at the start of the experiment). Cellulose responders
were also lower in relative abundance on average within the non-fractionated
DNA and had lower estimated \textit{rrn} copy number than xylose responders.
The majority of cellulose responders were not close relatives of cultured
isolates although a number of cellulose responders shared high SSU rRNA gene
sequence identity with cultured \textit{Proteobacteria} (e.g.
\textit{Cellvibrio}), . We identified cellulose responders among phyla such as
\textit{Verrucomicrobia}, \textit{Chloroflexi}, and \textit{Planctomycetes} --
phyla whose functions within soil communities remain unknown.

% Fakesubsubsection: Verrucomicrobia comprise
\textit{Verrucomicrobia} made up 16\% of the cellulose responders.
\textit{Verrucomicrobia} are cosmopolitan soil microbes \citep{Bergmann_2011}
that can make up to 23\% of SSU rRNA gene sequences in soils
\citep{Bergmann_2011} and 9.8\% of soil SSU rRNA \citep{Buckley_2001}. Genomic
analyses and laboratory experiments show that various isolates
within the \textit{Verrucomicrobia} are capable of methanotrophy, diazotrophy,
and cellulose degradation \citep{Wertz_2011,Otsuka_2012}. Moreover,
\textit{Verrucomicrobia} have been hypothesized to degrade polysaccharides in
many environments \citep{Fierer_2013,10543821,Herlemann_2013}. However, only
one of the 15 most abundant verrucomicrobial phylotypes in globally distributed
soil samples shared $>$ 93\% SSU rRNA gene sequence identity with a cultured
isolate \citep{Bergmann_2011} and hence the role of soil
\textit{Verrucomicrobia} in global C-cycling remains unknown. The majority of
verrucomicrobial cellulose responders belonged to two clades that fall within
the \textit{Spartobacteria} (Figure~\ref{fig:tiledtree}).
\textit{Spartobacteria} outnumbered all other \textit{Verrucomicrobia}
phylotypes in SSU rRNA gene surveys of
181 globally distributed soil samples \citep{Bergmann_2011}. Given their ubiquity and abundance
in soil as well as their demonstrated incorporation of $^{13}$C from
$^{13}$C-cellulose, \textit{Verrucomicrobia} lineages, particularly
\textit{Spartobacteria}, may be important contributors to cellulose
decomposition on a global scale. 

% Fakesubsubsection: Other notable cellulose responders
Other notable cellulose responders include OTUs in the \textit{Planctomycetes}
and \textit{Chloroflexi} both of which have previously been shown to
assimilate $^{13}$C from $^{13}$C-cellulose added to soil
\citep{Schellenberger_2010}. \textit{Planctomycetes} are common in soil
\citep{Janssen2006}, comprising 4 to 7\% of bacterial cells in many soils
\citep{Zarda_1997,Chatzinotas_1998} and 7\% $\pm$ 5\% of SSU rRNA
\citep{buckley_2003}. Although soil \textit{Planctomycetes} are widespread,
their activities in soil remain uncharacterized. \textit{Plantomycetes}
represented 16\% of cellulose responders and shared $<$ 92\% SSU rRNA gene
sequence identity to their most closely related cultured isolates.
\textit{Chloroflexi} are known for metabolically dynamic lifestyles ranging
from anoxygenic phototrophy to organohalide respiration \citep{Hug_2013} and
are among the six most abundant bacterial phyla in soil \citep{Janssen2006}.
Recent studies have focused on \textit{Chloroflexi} roles in C cycling
\citep{Hug_2013,Goldfarb_2011,Cole_2013} and several \textit{Chloroflexi}
isolates use cellulose \citep{Hug_2013,Goldfarb_2011,Cole_2013}. Four of the
five \textit{Chloroflexi} cellulose responders belong to a single clade within
the \textit{Herpetosiphonales} (Figure~\ref{fig:tiledtree}). 

% Fakesubsubsection: Finally, a single
Finally, a single cellulose responder belonged to the \textit{Melainabacteria}
phylum (95\% shared SSU rRNA gene sequence identity with \textit{Vampirovibrio
chlorellavorus}). The phylogenetic position of \textit{Melainabacteria} is
debated but \textit{Melainabacteria} have been proposed to be
a non-phototrophic sister phylum to \textit{Cyanobacteria}. An analysis of
a \textit{Melainabacteria} genome \citep{Di_Rienzi_2013} suggests the
genomic capacity to degrade polysaccharides though \textit{Vampirovibrio
chlorellavorus} is an obligate predator of green alga \citep{gromov_1972}. The
\textit{Melainabacteria} cellulose responder did not respond to xylose so if it
is predatory, it may prey specifically on structural C degraders (i.e.
cellulose responders).

% Fakesubsubsection: Responders did not necessarily
Responders did not necessarily assimilate $^{13}$C into DNA directly
from $^{13}$C-xylose or $^{13}$C-cellulose. In many ways, knowledge of
secondary C degradation and/or microbial biomass turnover may be more
interesting with respect to the soil C-cycle than knowledge of primary
degradation. The response to xylose suggests xylose-C moved through different
trophic levels within the soil bacteria food web. The \textit{Bacilli} degraded
xylose first (65\% of the xylose-C had been respired by day~1) representing
84\% of day~1 xylose responders. \textit{Bacilli} also comprised about 6\% of
SSU rRNA genes present in non-fractionated DNA on day~1. However, few
\textit{Bacilli} remained $^{13}$C-labeled by day~3 and their abundance
declined reaching about 2\% of soil SSU rRNA genes by day~30. Members of the
\textit{Bacillus} \citep{Cleveland2007} and \textit{Paenibacillus} in
particular \citep{Verastegui_2014} have been previously implicated as labile
C decomposers. The decline in relative abundance of \textit{Bacilli} could be
attributed to mortality and/or sporulation coupled to mother cell lysis.
Concomitant with the decline in relative abundance and loss of $^{13}$C-label
of \textit{Bacilli}, \textit{Bacteroidetes} OTUs appeared $^{13}$C-labeled at
day~3. Finally, \textit{Actinobacteria} appeared $^{13}$C-labeled at day~7 as
\textit{Bacteroidetes} xylose responders declined in relative abundance and
became unlabeled. Hence, it seems reasonable to propose that
\textit{Bacteroidetes} and \textit{Actinobacteria} xylose responders became
$^{13}$C-labeled via the consumption $^{13}$C-labeled microbial biomass as
opposed to primary degradation of $^{13}$C-xylose. 

% Fakesubsubsection: Trophic transfer could be due
The inferred physiology of \textit{Actinobacteria} and \textit{Bacteroidetes}
xylose responders provides further evidence that the activity dynamics
represent C transfer between microbes by saprotrophy and/or predation. Most of
the \textit{Actinobacteria} xylose responders that appeared $^{13}$C-labeled at
day~7 were members of the \textit{Micrococcales} (Figure~\ref{fig:tiledtree})
and the most abundant $^{13}$C-labeled \textit{Micrococcales} OTU at day~7
(“OTU.4”, Table~\ref{tab:xyl}) is annotated as belonging in the
\textit{Agromyces}. \textit{Agromyces} are facultative predators that feed on
the gram-positive \textit{Luteobacter} in culture \citep{16346402}.
Additionally, certain types of \textit{Bacteroidetes} can assimilate $^{13}$C
from $^{13}$C-labeled \textit{Escherichia coli} added to soil
\citep{Lueders2006}. However, it is also possible that \textit{Bacilli},
\textit{Bacteroidetes}, and \textit{Actinobacteria} are adapted to use xylose
at different concentrations and that the observed activity dynamics resulted
from changes in xylose concentration over time. If trophic transfer caused the
activity dynamics, at least three different ecological groups exchanged C in~7
days. Models of the soil C cycle often exclude trophic interactions between
soil bacteria (e.g. \citep{Moore1988}), yet when soil C models do account for
predators and/or saprophytes, trophic interactions are predicted to have
significant effects on the fate of soil C \citep{Kaiser2014a}. 

\subsection{Implications for soil C cycling models}
% Fakesubsubsection: Models of soil
Functional niche characterization for soil microorganisms is necessary to
predict whether and how biogeochemical processes vary with microbial community
composition. Functional niches are defined by soil microbiologists and have
been successfully incorporated into biogeochemical process models (E.g.
\citep{wieder_2014a,Kaiser2014a}). In some C models ecological strategies such
as growth rate and substrate specificity are parameters for functional niche
behavior \citep{Kaiser2014a}. The phylogenetic breadth of a functionally
defined group is often inferred from the distribution of diagnostic genes
across genomes \citep{Berlemont2013} or from the physiology of isolates
cultured on laboratory media \citep{Martiny2013}. For instance, the wide
distribution of the glycolyis operon in microbial genomes is interpreted as
evidence that many soil microorganisms participate in glucose turnover
\citep{McGuire2010}. However, the functional niche may depend less on the
distribution of diagnostic genes across genomes and more on life history traits
that allow organisms to compete for a given substrate as it occurs in the soil.
For instance, fast growth and rapid resuscitation allow microorganisms to
compete for labile C which may often be transient in soil. Hence, life history
traits may constrain the diversity of microbes that metabolize a given C source
in the soil under a given set of conditions.

% Fakesubsubsection: Biogeochemical processes
Biogeochemical processes mediated by a broad array of taxa are assumed to
be less influenced by community change than narrow processes that involve
a single, specific chemical transformation by a narrow suite of microbial
participants \citep{Schimel_1995,McGuire2010}. In addition, the diversity of
a functionally defined group engaged in a specific C transformation is expected
to correlated with C lability \citep{McGuire2010}. However, the diversity of
active labile C and recalcitrant C decomposers in soil has not been directly
quantified. We found comparable numbers of OTUs responded to $^{13}$C-cellulose
and $^{13}$C-xylose (63 and~49, respectively). Cellulose responders were
phylogenetically clustered suggesting that the ability to degrade cellulose is
phylogenetically conserved. The clade depth of cellulose responders, 0.028 SSU
rRNA gene sequence dissimilarity, is on the same order as that observed for
glycoside hydrolases which are diagnostic enzymes for cellulose degradation
\citep{Berlemont2013}. Xylose responders clustered in terminal branches
indicating groups of closely related taxa used xylose but xylose responders
also clustered phylogenetically with respect to time of first response
(Figure~\ref{fig:tiledtree}, Figure~\ref{fig:xyl_count}). For example, xylose
responders on day~1 are dominated by members of \textit{Paenibacillus}. Thus,
microorganisms that degraded labile C and structural C were both limited in
diversity. Although the genes for xylose metabolism are likely widespread in
the soil community, it's possible limited groups of organisms had the
ecological characteristics required to degrade xylose under the experimental
conditions. Therefore it's possible few phylogenetically coherent taxa actually
participate in the metabolism of labile C-sources under a given set of
conditions, and hence changes in community composition may alter the dynamics
of structural \textit{and} labile C-transformations in soil.

% Fakesubsubsection: Broadly, we observed
Broadly, we observed labile C use by fast growing generalists and structural
C use by slow growing specialists. These results agree with the MIMICS model
which simulates leaf litter decomposition by modeling microbial decomposers
as two functionally defined groups, copiotrophs or oligotrophs
\citep{wieder_2014a}. Including these functional types improved predictions
of C storage in response to environmental change relative to models that did
not consider any microbial physiological diversity. We identified microbial
lineages engaged in labile and structural C decomposition or simply,
copiotrophs and oligotrophs. We also observed potentially greater turnover --
and at the very least rate differences in turnover -- for copiotroph biomass
relative to oligotroph biomass which may be important to consider when modeling
microbial turnover input to SOM. It's also clear that that there may be more
than two vital functional types mediating C-cycling in soil. That is, soil-C
may travel through many bacterial trophic levels where each C transfer
represents an opportunity for C stabilization in association with soil minerals
or C loss by respiration.

\subsection{Conclusion} 
% Fakesubsubsection: Microorganisms sequester atmospheric C
Microorganisms sequester atmospheric C and respire SOM influencing climate
change on a global scale but we do not know which microorganisms carry out
specific soil C transformations. In this experiment microbes from
physiologically uncharacterized but cosmopolitan soil lineages participated in
cellulose decomposition. Cellulose responders included members of the
\textit{Verrucomicrobia} (\textit{Spartobacteria}), \textit{Chloroflexi},
\textit{Bacteroidetes} and \textit{Planctomycetes}. \textit{Spartobacteria} in
particular are globally cosmopolitan soil microorganisms and are often the most
abundant \textit{Verrucomicrobia} order in soil \citep{Bergmann_2011}.
Fast-growing aerobic spore formers from \textit{Firmicutes} assimilated labile
C in the form of xylose. Xylose responders within the \textit{Bacteroidetes}
and \textit{Actinobacteria} likely became $^{13}$C-labeled by consuming
$^{13}$C-labeled biomass that was labeled itself via the primary degradation of
$^{13}$C-xylose. Our results suggest that cosmopolitan \textit{Spartobacteria}
may degrade cellulose on a global scale, labile plant biomass C may travel
through a trophic cascade within the bacteria that includes predators and/or
saprotrophs, and life history traits may act as a filter constraining the
diversity of active microorganisms relative to those with the genomic potential
for a given metabolism.
