\section{Discussion} 
% Fakesubsubsection: We highlight to key results
We highlight two key results with implications for understanding structure-function
relationships in soils, and for applying DNA-SIP in future studies of the soil-C
cycle. First, cellulose responders were members of physiologically undescribed
taxonomic groups with few exceptions. This suggests that we have much to learn
about the diversity of structural-C decomposers in soil before we can begin to
assess how they are affected by climate change and land management. Second, the
response to xylose was characterized by a succession in activity from
\textit{Paenibacillus} OTUs (day 1) to \textit{Bacteroidetes} (day 3) and finally
\textit{Micrococcales} (day 7). This activity succession was mirrored by relative
abundance profiles and may mark trophic-C exchange between these groups. While
trophic exchange has been observed previously in DNA-SIP studies
\citep{lueders2004b} most applications of DNA-SIP focus on proximal use of
labeled substrates. However, with increased sensitivity, DNA-SIP is well suited
to tracking C flows throughout microbial communities over time and is not
limited only to observing the entry point for a given substrate into the soil
C-cycle.  Trophic interactions will critically influence how the global soil-C
reservoir will respond to climate change \citep{Crowther2015} but we know
little of biological interactions among soil bacteria. Often bacteria are cast
as a single trophic level \citep{Moore1988} but it may be appropriate to
investigate the soil food web at greater granularity. Additionally, our results
show that DNA-SIP results can change dramatically over time suggesting that
multiple time points are necessary to rigorously and comprehensively describe
which microorganisms consume $^{13}$C-labeled substrates in nucleic acid SIP
incubations.

% Fakesubsubsection: Microorganisms that consumed
Microorganisms that consumed $^{13}$C-cellulose were seldom related closely to any
physiologically characterized cultured isolates but were members of
cosmopolitan phylogenetic groups in soil including \textit{Spartobacteria},
\textit{Planctomycetes}, and \textit{Chloroflexi}. Often cellulose responders
were less than 90\% related to their closest cultured relatives showing that we
can infer little, if anything at all, of their physiology from culture-based
studies. Notably, many \textit{Spartobacteria} were among the cellulose responder OTUs.
This is particularly interesting as \textit{Spartobacteria} are globally
distributed and found in a variety of soil types \citep{Bergmann_2011}. These lineages may play
important roles in global cellulose turnover (please see SI note 1 for further
discussion of the phylogenetic affiliation of cellulose responders). 

% Fakesubsubsection: In addition to taxonomic identity, we quantified
In addition to taxonomic identity, we quantified four ecological properties of
microorganisms that were actively engaged in labile and structural C
decomposition in our experiment: (1) time of activity, (2) estimated
\textit{rrn} gene copy number, (3) phylogenetic clustering, and (4) density
shift in response to $^{13}$C-labeling. Labile C was consumed before structural C and 
these substrates were consumed by different microorganisms
(Figure~\ref{fig:ord}). This was expected and is consistent with the
degradative succession hypothesis. Consumers of
labile C had higher estimated \textit{rrn}
gene copy number than structural C consumers (Figure~\ref{fig:shift}).  \textit{rrn} copy number is
positively correlated with the ability to resuscitate quickly in response to
nutrient influx \citep{Klappenbach_2000} which may be the advantage that
enabled xylose responders to rapidly consume xylose. Both xylose and cellulose
responders were terminally clustered phylogenetically suggesting that the ability
to use these substrates was phylogenetically constrained. Although labile C
consumption is generally considered to be mediated by a diverse set of 
microorganisms, we found that xylose responders at day 1 were mainly members of
one genus, \textit{Paenibacillus}. Our results suggests that life-history
traits such as the ability to resuscitate
quickly and/or grow rapidly may be more important in determining the
diversity of microorganisms that actually mediate a given process than the genomic
potential for substrate utilization (see SI note 2 for further discussion with respect to
soil-C modelling). And last, labile C consumers, in contrast to structural C consumers, 
had lower $\Delta\hat{BD}$ in response to $^{13}$C-labeling.
This result suggests that labile C consumers were generalists, assimilating C
from a variety of sources both labeled and unlabeled, while structural C
consumers were more likely to be specialists and more closely associated
with C from a single source.

% Fakesubsubsection: We propose that the temporal fluctuations
We propose that the temporal fluctuations in $^{13}$C-labeling in the
$^{13}$C-xylose treatment are due to trophic exchange of $^{13}$C.
Alternatively, the temporal dynamics could be caused by microorganisms tuned to
different substrate concentrations and/or cross-feeding. However, trophic exchange
would explain well the precipitous drop in abundance of
\textit{Paenibacillus} after day 1 with subsequent $^{13}$C-labeling of
\textit{Bacteroidetes} at day 3 as well as the precipitous drop in abundance of
\textit{Bacteroidetes} at day 3 followed by $^{13}$C-labeling of Micrococcales at day 7.
Trophic exchange could be enabled by mother cell lysis (in the case of spore formers
such as \textit{Paenibacillus}), viral lysis, and/or the direct indirect effects of
predation. \textit{Bacteroidetes} types have been shown to become
$^{13}$C-labeled after the addition of live $^{13}$C-labeled
\textit{Escherichia coli} to soil \citep{Lueders2006} indicating their ability to
assimilate C from microbial biomass. In addition, the dominant OTU labeled in the $^{13}$C-xylose
treatment from the \textit{Micrococcales} shares 100\% SSU rRNA gene sequence identity
to \textit{Agromyces ramosus} a known predator that
feeds upon on many microorganisms including yeast and \textit{Micrococcus luteus}
\citep{16346402}. \textit{Agromyces} are abundant microorganisms in many soils
and \textit{Agromyces ramosus} was the most abundant xylose responder in our
experiment -- the fourth most abundant OTU in our dataset. It is notable however,
that if \textit{Agromyces ramosus} is acting as a predator in our experiment, the
organism remains unlabeled in response to $^{13}$C-cellulose which suggests that its
activity may be specific for certain prey or for certain environmental
conditions  (see SI note 3 for further discussion of trophic C exchange).
Climate change is expected to diminish bottom-up controls on microbial growth
increasing the importance on top-down biological interactions for mitigating
positive climate change feedbacks \citep{Crowther2015}. Currently the extent
of bacterial predatory activity in soil, and its consequences for the soil C-cycle
and carbon use efficiency is largely unknown. Elucidating the identities of
bacterial predators in soil will assist in assessing the implications of
climate change on global soil-C storage.

\subsection{Conclusion} 
% Fakesubsubsection: Microorganisms sequester atmospheric C
Microorganisms govern C-transformations in soil and thereby influence global
climate but still we do not know the specific identities of microorganisms that carry out 
critical C transformations. In this experiment microorganisms from physiologically
uncharacterized but cosmopolitan soil lineages participated in cellulose
decomposition. Cellulose responders included members of the
\textit{Verrucomicrobia} (\textit{Spartobacteria}), \textit{Chloroflexi},
\textit{Bacteroidetes} and \textit{Planctomycetes}. \textit{Spartobacteria} in
particular are globally cosmopolitan soil microorganisms and are often the most
abundant \textit{Verrucomicrobia} order in soil \citep{Bergmann_2011}.
Fast-growing aerobic spore formers from \textit{Firmicutes} assimilated labile
C in the form of xylose. Xylose responders within the \textit{Bacteroidetes}
and \textit{Actinobacteria} likely became labeled by consuming $^{13}$C-labeled
constituents of microbial biomass either by saprotrophy or predation. Our
results suggest that cosmopolitan \textit{Spartobacteria} may degrade cellulose
on a global scale, decomposition of labile plant C may initiate trophic transfer 
within the bacterial food web, and life history traits may act
as a filter constraining the diversity of active microorganisms relative to
those with the genomic potential for a given metabolism.
