\section{Discussion}
% Fakesubsubsection:Pure culture based studies drove early
Pure culture based studies drove early soil microbial ecology research.
Historically important soil isolates included nine genera:
\textit{Agrobacterium}, \textit{Alcaligenes}, \textit{Arthrobacter},
\textit{Bacillus}, \textit{Flavobacterium}, \textit{Micromonospora},
\textit{Nocardia}, \textit{Pseudomonas}, and \textit{Streptomyces}
(\citep{Alexander1977} and reviewed by \citep{Janssen2006}) but
culture-independent surveys of soil microbial diversity revealed soil can
harbor 5,000 OTUs per half gram of soil \citep{Schloss2006} and that cultured
isolates did not represent \textit{in situ} numerically abundant genera. We
recovered almost 6,000 OTUs in this study. Although culturing techniques can
produce isolates from diverse soil phylogenetic lineages \citep{Janssen2002},
numerically dominant soil microorganisms are still uncultured and we know
little of their ecophysiology \citep{Janssen2006}. In contrast, DNA-SIP can
characterize functional roles for thousands of phylotypes in a single
experiment. We found 104 OTUs in an agricultural soil that can incorporate
from xylose and/or cellulose into biomass. We also used DNA-SIP to assay
substrate specificity and temporal dynamics of C-cycling or soluble and
polymeric C degraders. Included in the $^{13}$C-xylose and $^{13}$C-cellulose
responsive OTUs were members of numerically dominant yet functionally
uncharacterized soil phylogenetic groups such as \textit{Verrucomicrobia},
\textit{Planctomycetes} and \textit{Chloroflexi}.

\subsection{Microbial response to isotopic labels}
% Fakesubsubsection: We propose that microbial decomposition
We propose that C added to soil microcosms in this experiment took the
following path through the microbial food web (Figure~\ref{fig:foodweb}):
First, labile C such as xylose was assimilated by fast-growing opportunistic
\textit{Firmicutes} spore formers. The remaining labile C and new biomass C was
assimilated in succession by slower growing \textit{Bacteroidetes},
\textit{Actinobacteria} and \textit{Proteobacteria} phylotypes that were either
tuned to lower C substrate concentrations, were predatory bacteria (e.g.
\textit{Agromyces}), and/or were specialized for consuming viral lysate. C from
polymeric substrates entered the bacterial community after 14 days. Canonical
cellulose degrading bacteria such as \textit{Cellvibrio} degraded cellulose
but uncharacterized lineages in the \textit{Chloroflexi},
\textit{Planctomycetes} and \textit{Verrucomicrobia}, specifically the
\textit{Spartobacteria}, were also significant contributors to cellulose
decomposition. 

\subsection{Ecological strategies of soil microorganisms participating in the
decomposition of organic matter}
% Fakesubsubsection:We assessed the ecology of
We assessed the ecology of $^{13}$C-responsive OTUs by estimating the
\textit{rrn} gene copy number and the BD shift upon labeling for each OTU.
\textit{rrn} gene copy number correlates positively with growth rate
\citep{11125085} and BD shift is indicative of substrate specificity (see
results). We also observed how $^{13}$C-substrate responsive OTUs changed in
relative abundance with time in the microcosms and the abundance rank of
$^{13}$C-substrate responsive OTUs in the bulk DNA. Ecological metrics show
$^{13}$C-cellulose responsive OTUs grow slower (Figure~\ref{fig:shift},
Figure~\ref{fig:copy}), have greater substrate specificity
(Figure~\ref{fig:shift}), and are generally lower abundance than
$^{13}$C-xylose responsive OTUs (Figure~\ref{fig:shift}). The higher abundance
of xylose responders may also be in part due to of their high \textit{rrn} gene
copy number resulting in inflated relative abundance per genome. There are only
faint ecological differences within the $^{13}$C-cellulose responsive OTUs but
the combination of \textit{rrn} gene copy number, BD shift, abundance rank and
relative abundance change over time is consistent with phylum membership
(Figure~RADVIZ). $^{13}$C-xylose responsive OTU \textit{rrn} gene copy number
correlated inversely with the time at which the OTU was first found to
incorporate $^{13}$C into DNA (Figure~\ref{fig:shift}, Figure~\ref{fig:copy})
suggesting that fast-growing microbes assimilated $^{13}$C from xylose before
slow growers.  

% Fakesubsubsection:Ecological metrics suggest
Ecological metrics suggest cellulose degraders are substrate specialists that
grow slow and are in low bulk abundance. Labile C responder ecological
strategies were more varied perhaps because some $^{13}$C labeled
microorganisms did not primarily assimilate xylose but became labeled via
predatory interactions and/or are saprophytes. $^{13}$C-xylose responsive OTUs
are generalists, grow faster and are more abundant when compared to
$^{13}$C-cellulose responders. $^{13}$C-xylose responders vary in growth rate
and while generally higher abundance than $^{13}$C-cellulose responders can
also be low abundance microorganisms. It's not clear whether the observed
activity succession from \textit{Firmicutes} to \textit{Bacteroidetes} and
finally \textit{Actinobacteria} in response to $^{13}$C-xylose addition marks
a trophic cascade or functional groups tuned to different resource
concentrations or both. Notably, each temporally defined response group
clustered phylogenetically suggesting a uniform ecological strategy
(Figure~\ref{fig:trees}). It's also clear that some of the
non-\textit{Firmicutes} $^{13}$C-xylose responders are closely related to known
predators (\textit{Agromyces}) and many marine predatory bacteria are members
of the \textit{Bacteroidetes} (\citep{Banning2010a}). Further,
\textit{Bacteroidetes} have been implicated as soil predatory bacteria
previously \citep{Lueders2006}. If the temporal dynamics of
$^{13}$C-xylose incorporation are due to trophic interactions, our results
suggest that there are many predatory soil bacteria that consume
fast-growing, opportunistic, primary labile C assimilating, gram-positive
spore-formers. Hence, trophic interactions among soil bacteria may be of
importance in soil C turnover models.

% Fakesubsubsection:How -- or if -- phylogenetic
How -- or if -- phylogenetic composition affects SOM dynamics is an open
question \citep{Schimel_2012}. Phylogenetic composition could affect SOM
dynamics if SOM transformations were not functionally redundant traits and
if biology is rate limiting for key C transformations
\citep{Schimel_2012}. Alternatively, even with functional redundancy
resource allocation at the cell level can influence SOM fate
\citep{Schimel_2012}. It is likely that the ability to carry out soil
C transformations are redundant
within and between soil microbial communities and that in the mineral soil
abiotic factors are rate limiting \citep{Schimel_2012} Therefore phylogenetic composition
in mineral soil likely influences soil C fate as opposed to dynamics. We
demonstrate a phylogenetically coherent response to soluble C additions --
for instance, most of the initial response to xylose can be attributed to
aerobic spore formers. Assuming cellular resource allocation is consistent
with phylogeny, it follows then that phylogenetic composition can
significantly influence SOM fate. Aerobic spore-formers, for example, are
found in different proportions across soil biomes \citep{Janssen2006} and
even within regional agricultural soils CITE Berthrong. If present and
abundant, aerobic spore-formers may be primary soluble C decomposers and
allocate C in specific quantities into intra and extracellular
C components. Further, aerobic spore-formers may have a phylogenetically
coherent resistance to predation which could further affect soil C fate.
Although, not demonstrated in this study, the allocation of C from
soluble, labile pools in a soil without or under conditions not suitable
for aerobic spore-formers may be significantly different. Polymeric C, on
the other hand, did not show the same phylogenetic coherence as soluble
C decomposition in this study. This suggests that resource allocation
among cellulose degraders would not have a single phylogenetic signal and
the fate of polymeric C would not be tied to phylogenetic composition.
Though cellulose degraders as a whole likely allocate C differently than
labile C degraders. 

\subsection{Conclusion} 
% Fakesubsubsection:SOM represents more C than the
Microorganisms sequester atmospheric C and respire soil organic matter (SOM)
influencing climate change on a global scale but we do not know which
microorganisms carry out specific soil C transformations. In this experiment
microbes from uncharacterized yet ubiquitous and often abundant soil lineages
participated in cellulose decomposition. Cellulose C degraders included members
of the \textit{Verrucomicrobia} (\textit{Spartobacteria}),
\textit{Chloroflexi}, \textit{Bacteroidetes} and \textit{Planctomycetes}.
\textit{Spartobacteria} in particular are abundant microorganisms in many soil
biomes and often the most abundant \textit{Verrucomicrobia} order in soil. Our
results also suggest that members of the \textit{Bacteroidetes} and
\textit{Actinobacteria} act in the cascade of labile, soluble C through soil
trophic levels possibly as predators. Both points illustrate the
complexity of soil C dynamics and fate. The largely phylogenetically
coherent ecological groups observed in this study suggest that soil C fate
is tied to phylogenetic composition.
