\section{Discussion} 
% Fakesubsubsection: We establish ecological 
We identified microorganisms participating in soil C cycling using a nucleic
acid SIP approach. Specifically, we observed assimilation of $^{13}$C from
either $^{13}$C-xylose or $^{13}$C-cellulose into 104 OTUs from agricultural
soil samples. We found $^{13}$C appeared to move into and then out of the from
groups of related OTUs over time. By coupling nucleic acid SIP to high
throughput sequencing we diagnosed OTU activity even when OTUs were at low
relative abundance in non-fractionated DNA (as low as XX\%). Our results
support the degradative succession hypothesis, elucidate ecophysiological
properties of soil microorganisms, reveal activity of widespread uncultured
soil bacteria, and begin to piece together the microbial food web in soils. 

% Fakesubsubsection: The degradative succession hypothesis
The degradative succession hypothesis predicts an ecological transition in
activity from microbes that decompose labile plant biomass C to those that
decompose structural more recalcitrant C. The phenomena observed in our study
largely agree with the degradative succession hypothesis. Microorganisms
quickly assimilated $^{13}$C from xylose into DNA relative to cellulose. Xylose
is the major monomer of xylan and xylan itself is a major constituent of
hemicellulose. Thus, xylose represents an abundant sugar present in fresh plant
litter. The phylogenetic composition of $^{13}$C-labeled OTUS in response to
the $^{13}$C-xylose amendment changed over time and the total number of xylose
responders almost entirely diminished by the end of the incubation. In
contrast, cellulose decomposition proceeded slowly. few OTUs were
$^{13}$C-labeled by cellulose in the beginning of the experiment but cellulose
responders became active near the end when xylose responders were unlabeled.
Finally, only 8 of 104 OTUs were observed to metabolize both xylose and
cellulose demonstrating a succession in activity from xylose responders to
cellulose responders. Our results agree with the degradative succession
hypothesis and highlight additional ecological phenomena potentially associated
with plant biomass decomposition including several waves of activity in
response to labile C inputs.  

% Fakesubsubsection: Correlations between community composition
Correlations between community composition and environmental characteristics
often reveal the ecology of microorganisms \citep{Fierer2007}. In this
experiment, we similarly define microbial ecology albeit through SIP as opposed
to spatio-temporal variation in the context of environmental gradients. We
further characterized microbial ecological strategies by inferring \textit{rrn}
gene copy number. High \textit{rrn} copy number may allow microorganisms to
rapidly respond to nutrients influx \citep{Klappenbach_2000}. The xylose
responders quickly (within 24 hours) assimilated xylose-C into DNA. Xylose
responders also had relatively low
$\Delta\hat{BD}$ potentially indicating xylose was not the sole C source
used for growth. Xylose represented only 20\% of the nutrient and resource
microcosm amendment and~3.5\% of total soil C. Xylose responders often included
the most abundant OTUs within the non-fractionated DNA and had relatively high
estimated \textit{rrn} copy number. However, to some degree, high \textit{rrn}
gene copy number may inflate observed xylose responder relative abundance.
Notably, the majority of xylose responder SSU rRNA genes (86\%) matched
cultured isolate SSU rRNA genes at high sequence identity ($>$ 97\%). 

% Fakesubsubsection: In contrast, the results
Cellulose responders, on the other hand, grew slowly and appeard to specialize
in using cellulose as a C source. Cellulose responders grew over a span of weeks
and had relatively high $\Delta\hat{BD}$ indicating cellulose remained the
dominant C source for cellulose responders even though multiple sources of
C were present. Cellulose responders were also generally lower in relative
abundance within the non-fractionated DNA and had lower estimated \textit{rrn}
copy number than xylose responders. The majority of cellulose responders were
not close relatives of cultured isolates although a number of cellulose
responders shared high SSU rRNA gene sequence identity cultured
\textit{Proteobacteria} (e.g. \textit{Cellvibrio}), . We identified cellulose
responders among phyla such as \textit{Verrucomicrobia}, \textit{Chloroflexi},
and \textit{Planctomycetes}, phyla whose functions within soil communities
remain poorly characterized.

% Fakesubsubsection: Verrucomicrobia comprise
\textit{Verrucomicrobia} made up 16\% of the cellulose responders.
\textit{Verrucomicrobia} are cosmopolitan soil microbes \citep{Bergmann_2011}
that can make up to 23\% of SSU rRNA gene sequences in soils
\citep{Bergmann_2011} and 9.8\% of soil SSU rRNA \citep{Buckley_2001}. Genomic
analyses and laboratory experiments show that various isolates
within the \textit{Verrucomicrobia} are capable of methanotrophy, diazotrophy,
and cellulose degradation \citep{Wertz_2011,Otsuka_2012}. Moreover,
\textit{Verrucomicrobia} have been hypothesized to degrade polysaccharides in
many environments \citep{Fierer_2013,10543821,Herlemann_2013}. However, only
one of the 15 most abundant verrucomicrobial phylotypes observed globally in
soils shares $>$ 93\% SSU rRNA gene sequence identity with a cultured isolate
\citep{Bergmann_2011} and hence the role of soil \textit{Verrucomicrobia} in
global C-cycling remains unknown. The majority of verrucomicrobial cellulose
responders belonged to two clades that fall within the \textit{Spartobacteria}.
\textit{Spartobacteria} were the most commonly observed
\textit{Verrucomicrobia} in SSU rRNA gene surveys of
181 globally distributed soil samples \citep{Bergmann_2011}. Given their ubiquity and abundance
in soil as well as their demonstrated incorporation of $^{13}$C from
$^{13}$C-cellulose, \textit{Verrucomicrobia} lineages, particularly
\textit{Spartobacteria}, may be important contributors to cellulose
decomposition on a global scale. 

% Fakesubsubsection: Other notable cellulose responders
Other notable cellulose responders include OTUs in the \textit{Planctomycetes}
and \textit{Chloroflexi} both of which have previously been shown to
assimilate $^{13}$C from $^{13}$C-cellulose added to soil
\citep{Schellenberger_2010}. \textit{Planctomycetes} are common in soil
\citep{Jannsen2006}, comprising 4 - 7\% of bacterial cells in many soils
\citep{Zarda_1997,Chatzinotas_1998} and 7\% $+/-$ 5\% of SSU rRNA
\citep{buckley_2003}. Although soil \textit{Planctomycetes} are widespread,
their activities in soil remain poorly characterized. \textit{Plantomycetes}
represented 16\% of cellulose responders and shared $<$ 92\% SSU rRNA gene
sequence identity to their most closely related cultured isolates.

% Fakesubsubsection: Chloroflexi are among the
\textit{Chloroflexi} are known for metabolically dynamic lifestyles ranging
from anoxygenic phototrophy to organohalide respiration \citep{Hug_2013} and
are among the six most abundant bacterial phyla in soil \citep{Janssen2006}.
Recent studies have focused on \textit{Chloroflexi} roles in C cycling
\citep{Hug_2013,Goldfarb_2011,Cole_2013} and several \textit{Chloroflexi}
isolates use cellulose \citep{Hug_2013,Goldfarb_2011,Cole_2013}. Four
of the five \textit{Chloroflexi} cellulose responders belong to a single clade
within the \textit{Herpetosiphonales}. \textit{H. geysericola}, a predatory
bacterium that feeds on \textit{Aerobacter} in culture and it can also
metabolize cellulose \citep{Lewin1970},  most closely matches the \textit{Herpetosiphonales}
cellulose responders by SSU rRNA gene sequence identity although that match is
weak ($<$ 89\% sequence identity).
 

% Fakesubsubsection: Finally, a single
Finally, a single cellulose responder belonged to the \textit{Melainobacteria}
phylum (95\% shared SSU rRNA gene sequence identity with \textit{Vampirovibrio
chlorellavorus}). The phylogenetic position of \textit{Melainabacteria} is
debated but the non-phototrophic \textit{Melainabacteria} have been
hypothesized to be phylogenetically associated with Cyanobacteria. An analysis
of a ``\textit{Melainabacteria}'' genome \citep{Di_Rienzi_2013} suggests the
genomic capacity to degrade polysaccharides though \textit{Vampirovibrio
chlorellavorus} is an obligate predator of green alga \citep{gromov_1972}. It
is interesting that cellulose responders within \textit{Chloroflexi} and
\textit{Melainobacteria} are related (though distantly) to known predators. The
relatively high $\Delta\hat{BD}$ of these cellulose responders suggests that if
they are predators, they tightly associate with cellulose degrading
microorganisms. Also, these OTUs did not respond to xylose so if they are
predators, they do not prey on microorganisms like the xylose responders.

% Fakesubsubsection: We found that cellulose and xylose
We found that cellulose and xylose responders are phylogenetically clustered
and overdispersed, respectively. This suggests that the ability to degrade
cellulose is phylogenetically conserved possibly due to the degree of
biochemical complexity and/or the degree of ecological specialization required
to degrade cellulose in soil. The positive relationship between a physiological
trait’s phylogenetic depth and its complexity has been noted previously
\citep{Martiny2013} and the clade depth of cellulose responders, 0.028 SSU
rRNA gene sequence dissimilartiy, is on the same order as that observed
for glycoside hydrolases which are diagnostic enzymes for cellulose
degradation \citep{Berlemont2013}. Overdispersion, as observed for xylose
responders is indicative of a broadly distributed trait whose distribution
could might have been influenced by horizontal gene transfer or convergent
evolution. Overdispersion might alternatively indicate the xylose responders
represent multiple ecological groups which differ in their response to xylose. 

% Fakesubsubsection: We diagnosed the assimilation
Responders did not necessarily assimilate $^{13}$C into DNA directly
from $^{13}$C-xylose or $^{13}$C-cellulose. In many ways, knowledge of
secondary C degradation may be more interesting with respect to the soil
C-cycle than knowledge of primary degradation. The response to
xylose suggests xylose-C moved through different trophic levels within the soil
food web. The \textit{Bacilli} degraded xylose first (65\% of the xylose-C had
been respiredby day 1) \textit{Bacilli} represent 84\% of xylose responders,
and comprise a 6\% of SSU rRNA genes present in non-fractionated DNA. However,
in general \textit{Bacilli} were no longer $^{13}$C-labeled by day~3 and their
abundance declined reaching about 2\% of soil SSU rRNA genes by day~30. Members
of the \textit{Bacillus} \citep{Cleveland2007} and \textit{Paenibacillus} in
particular \citep{Verastegui_2014} have been previously implicated as labile
C decomposers. Concomitant with the decline in \textit{Bacilli} relative
abundance and the loss $^{13}$C-label by \textit{Bacilli} OTUs,
\textit{Bacteroidetes} OTUs appeared $^{13}$C-labeled at day~3. Subsequently,
\textit{Actinobacteria} appeared $^{13}$C-labeled at day~7 as both
\textit{Bacteroidetes} declined and \textit{Bacilli} OTUs continued to delcine
in relative abundance and both \textit{Bacilli} and \textit{Bacteroidetes} were
unlabeled. Hence, it seems reasonable to propose that \textit{Bacteroidetes}
and \textit{Actinobacteria} OTUs became $^{13}$C-labeled via the consumption
$^{13}$C-labeled microbial biomass. 

% Fakesubsubsection: Trophic transfer could be due
Trophic transfer could be due to predation and/or saprotrophy and the inferred
physiology of \textit{Actinobacteria} and \textit{Bacteroidetes} xylose
responders provides further evidence that the activity dynamics represent 
C transfer between microbes. Most of the \textit{Actinobacteria} xylose
responders that appeared $^{13}$C-labeled at day~7 were members of the
\textit{Micrococcales} (Figure XX) and the most abundant $^{13}$C-labeled
\textit{Micrococcales} OTU at day~7 (“OTU.4”, Table XX) is annotated as
belonging in the \textit{Agromyces}. \textit{Agromyces} are facultative
predators that feed on the gram-positive \textit{Luteobacter} in culture
\citep{16346402}. Additionally, soil \textit{Bacteroidetes} have been
implicated previously as predatory soil bacteria in nucleic acid SIP studies
using live \textit{Escherichia coli} to carry the isotopic label
\citep{Lueders2006}. However, it is also possible that \textit{Bacilli},
\textit{Bacteroidetes}, and \textit{Actinobacteria} are adapted to use xylose
at different concentrations and that the activity dynamics resulted from
changes in xylose concentration over time. If the activit dynamics were due to
trophic transfer, carbon was exchanged between at least three different
ecological groups in 7 days. Models of the soil C cycle rarely include trophic
interactions between soil bacteria (e.g. \citep{Moore1988}) yet when soil
C models do account for predators/saprophytes, trophic interactions are
predicted to have significant effects on the fate of soil
C \citep{Kaiser2014a}. 

\subsection{Implications for soil C cycling models}
% Fakesubsubsection: Models of soil
Models of soil C cycling rely on functional niches defined by soil
microbiologists (E.g. \citep{wieder_2014a,Kaiser2014a}). In some C models
ecological strategies such as growth rate and substrate specificity are
parameters for functional niche behavior \citep{Kaiser2014a}. The phylogenetic
breadth of an functional guilds are often inferred from the distribution of
diagnostic genes across genomes \citep{Berlemont2013} or from the physiology of
isolates cultured on laboratory media \citep{Martiny2013}. For instance, the
wide distribution of the glycolyis operon in microbial genomes is interpreted
as evidence that many soil microorganisms participate in glucose turnover (7).
Life history strategies, however, can determine which microbes use a gene from
those that have it. Xylose metabolism in soil, for instance, may depend less on
the distribution of the catabolic pathway across genomes and more on life
history strategies (e.g. growth rate) that allow organisms to compete
effectively for xylose. Phenomena such as seasonal change
\citep{Schmidt2007}, and rainfall \citep{Evans2014a} cause nutrient and
resource concentrations in soil to fluctuate. Therefore, fast growth and rapid
resuscitation allow microorganisms to favorably compete for labile C which may
often be transient. Hence, life history strategies may constrain the diversity
of microbes that are able to metabolize a given C source as it occurs in the
soil. DNA-SIP is useful for establishing \textit{in situ} (or near \textit{in
situ}) diversity of functional guilds to contrast guild diversity inferred from
genomic evidence and laboratory experiments. 

% Fakesubsubsection: Biogeochemical processes
Biogeochemical processes mediated by a broad array of taxa are assumed to
be less influenced by community change than narrow processes that involve
a single, specific chemical transformation by a narrow suite of microbial
participants \citep{Schimel_1995,McGuire2010}. In theory, the diversity of
a functional guild engaged in a specific C transformation is correlated to the
lability of the C \citep{McGuire2010}. However, the diversity of active
labile C and recalcitrant C decomposers in soil has not been directly
quantified. Notably, we found more OTUs responded to $^{13}$C-cellulose,~63,
than $^{13}$C-xylose,~49. Also, it is possible that many $^{13}$C-xylose
responders were predatory bacteria or saprophytes as opposed to primary labile
C degraders. Cellulose and xylose decomposer functional guilds were
non-overlapping in membership -- of 104 $^{13}$C-responders only 8 responded to
both cellulose and xylose -- and represented a small fraction of total soil
community diversity (Figure~\ref{fig:genspec}). While xylose use is undoubtedly
more widely distributed among microbial genomes than the ability to degrade
cellulose, the number of unique active cellulose decomposer OTUs outnumbered
the number of unique active xylose utilizer OTUs. Further, although cellulose
responders were phylogenetically clustered (NRI: XX) and xylose responders were
overdispersed (NRI: XX), it's not clear if $^{13}$C-xylose responsive OTUs in
this experiment constitute a single ecologically meaningful group or multiple
ecological groups. Notably, temporally defined $^{13}$C-xylose responder groups
were largely phylogenetically coherent (Figure~\ref{fig:tiledtree},
Figure~\ref{fig:xyl_count}). For example, most day~1 $^{13}$C-xylose responders
are members of the \textit{Paenibacillus}. Thus, active labile and structural
C associated microorganisms may be of limited diversity in soil which
implicates labile and structural C could be similarly affected by changes in
community composition. 

% Fakesubsubsection: Broadly, we observed
Broadly, we observed labile C use by fast growing generalists and structural
C use by slow growing specialists. These results agree with the MIMICS model
which simulates leaf litter decomposition modeling the microbial community as
two functional groups, copiotrophs or oligotrophs \citep{wieder_2014a}.
Including these functional types improved the predictions of C storage in
response to environmental change. Here we identified microbial
lineages engaged in labile and structural C decomposition or simply speaking
copiotrophs and oligotrophs. We also observed potentially greater turnover and
at least rate differences in turnover of copiotroph biomass relative to
oligotroph biomass which may be important to consider when modeling microbial
turnover input to SOM. It's also clear that that there may be more than two
vital functional types mediating C-cycling in soil. Specifically, there is
potentially one additional functional type consisting of saprophytes and/or
predators that feed on primary labile C degraders. These predators and/or
saprophytes are characterized by slower growth (i.e. lower \textit{rrn} copy
number) relative to primary labile C degraders. 

\subsection{Conclusion} 
% Fakesubsubsection: Microorganisms sequester atmospheric C
Microorganisms sequester atmospheric C and respire SOM influencing climate
change on a global scale but we do not know which microorganisms carry out
specific soil C transformations. In this experiment microbes from
physiologically uncharacterized but cosmopolitan soil lineages participated in
cellulose decomposition. Cellulose responders included members of the
\textit{Verrucomicrobia} (\textit{Spartobacteria}), \textit{Chloroflexi},
\textit{Bacteroidetes} and \textit{Planctomycetes}. \textit{Spartobacteria} in
particular are globally cosmopolitan soil microorganisms and are often the most
abundant \textit{Verrucomicrobia} order in soil \citep{Bergmann_2011}.
Fast-growing aerobic spore formers from Firmicutes assimilated labile C in the
form of xylose. Xylose responders within the Bacteroidetes and Actinobacteria
are likely to have assimilated xylose C as a result of dynamic trophic
exchange, mediated either by saprotrophy or predation. Our results suggest
that, cosmopolitan \textit{Spartobacteria} may degrade cellulose on a global
scale, bacterial trophic interactions can significantly impact soil C cycling,
and ecological traits are likely to act as a filter that constrains the
diversity of microorganisms that are active \textit{in situ} relative to those
that have the genomic capacity for a given process.
