\section{Discussion}
% Fakesubsubsection:Early soil microbial ecology was informed
Early soil microbial ecology was informed by studying pure cultures.  As
reviewed by \citet{Janssen2006} historically important pure cultures from soil
included nine genera \textit{Agrobacterium}, \textit{Alcaligenes},
\textit{Arthrobacter}, \textit{Bacillus}, \textit{Flavobacterium},
\textit{Micromonospora}, \textit{Nocardia}, \textit{Pseudomonas}, and
\textit{Streptomyces} but culture-independent surveys of soil microbial
diversity revealed soil may harbor as many as 5,000 OTUs per half gram
\citep{Schloss2006}. We observed almost 6,000 OTUs in this study. Although
culturing techniques are able to produce isolates from diverse soil lineages
\citep{Janssen2002}, most numerically dominant soil microorganisms are still
uncultured and we know little of their ecophysiology \citep{Janssen2006}. DNA-SIP can characterize the
functional roles of thousands of phylogenetic types in a single experiment. For
example, we assayed 5,940 OTUs for the ability to incorporate C from xylose or
cellulose into biomass and elucidated functional characteristics for 104 OTUs.
Included in the number of $^{13}$C-xylose and $^{13}$C-cellulose responders
were members of many numerically dominant yet functionally uncharacterized soil
phylogenetic groups such as \textit{Verrucomicrobia}, \textit{Planctomycetes}
and \textit{Chloroflexi}. We also show how DNA-SIP can be used to assay
substrate specificity and temporal dynamics of C-cycling including microbial
community succession during decomposition.

\subsection{Microbial response to isotopic labels}
% Fakesubsubsection: We propose that microbial decomposition
We propose that microbial decomposition of organic matter in soil follows the
following path (Figure~XX): Labile C such as xylose is assimilated by
fast-growing opportunistic \textit{Firmicutes} spore formers that may also be
stimulated by soil wetting. Leftover labile C and new biomass C is assimilated
in succession by slower growing \textit{Bacteroidetes}, \textit{Actinobacteria}
and \textit{Proteobacteria} phylotypes that are either tuned to lower C substrate
concentrations, are predatory bacteria (e.g. \textit{Agromyces}), and/or are
specialized for consuming viral lysate. Polymeric C is likely assimilated by
fungal cellulose degraders before bacterial degraders but the differences in
fungal and bacterial soil C assimilation are outside the scope of this study.
Polymeric C enters the bacterial community after several weeks. Canonical
cellulose degrading bacteria such as \textit{Cellvibrio} are major cellulose
degraders but uncharacterized lineages in the \textit{Chloroflexi} and
\textit{Verrucomicrobia}, specifically the \textit{Spartobacteria}, are
significant contributors to soil cellulose decomposition. C may also enter the bacterial community via predatory
actions of lineages, such as members of the \textit{Herpetosiphon} genus that specifically prey upon cellulose degraders; although it is not known at
this time the relative contribution, if any, of this predatory action. The
assimilation of C from polymeric substrates precipitates a smaller trophic
cascade than that for soluble, labile C.

\subsection{Phylogenetic affiliation of $^{13}$C-cellulose  responsive
microorganisms}
% Fakesubsubsection:\textit{Verrucomicrobia} are ubiquitous in soil
\textit{Verrucomicrobia}, cosmopolitan soil microbes
\citep{Bergmann_2011}, can comprise up to 23\% of 16S rRNA gene sequences in
high-throughput DNA sequencing surveys of SSU rRNA genes in soil
\citep{Bergmann_2011} and represent as high as 9.8\% of soil 16S rRNA
\citep{Buckley_2001}. Many \textit{Verrucomicrobia} were first isolated in the
last decade \cite{Wertz_2011} but only one of the 15 most abundant
verrucomicrobial phylotypes in a global soil sample collection shared greater
than 93\% sequence identity with a cultured isolate \citep{Bergmann_2011}.
Genomic analyses and physiological profiling of \textit{Verrucomicrobia}
isolates revealed \textit{Verrucomicrobia} are capable of methanotrophy,
diazotrophy, and cellulose degradation \citep{Otsuka_2012, Wertz_2011}. The function of soil \textit{Verrucomicrobia} in global C-cycling remains unknown.
For example, only two of the ten putative cellulose degrading \textit{Verrucomicrobia}
identified in this experiment shares at least 95\% sequence identity with
an isolate ("OTU.83" and "OTU.627", Table~\ref{tab:cell}). Seven of ten
$^{13}$C-cellulose responding verrucomicrobial OTUs were classified as
\textit{Spartobacteria} which are a numerically dominant
family of \textit{Verrucomicrobia} in SSU rRNA gene surveys of 181 globally
distributed soil samples \citep{Bergmann_2011}. Given their ubiquity and
abundance in soil as well as their demonstrated incorporation of $^{13}$C from
$^{13}$C-cellulose\textit{Verrucomicrobia} lineages, particularly \textit{Spartobacteria}, may be important contributors to cellulose decomposition on a global scale.

% Fakesubsubsection:Soil \textit{Chloroflexi} have been found to assimilate
Soil \textit{Chloroflexi} have previously been identified in DNA-SIP studies as assimilating $^{13}$C from cellulose and, like our findings, were also found to be underrepresented in culture collections \citep{Schellenberger_2010}. The
cellulose degrading \textit{Chloroflexi} in this study are also
only distantly related to isolates \ref{tab:cell}.  Chloroflexi are among the
six most abundant soil phyla commonly recovered from soil microbial diversity
surveys \citep{Janssen2006}. Chloroflexi are typically not as as abundant as
\textit{Verrucomicrobia} but are roughly as abundant as \textit{Bacteroidetes}
and \textit{Planctomycetes} \citep{Janssen2006}.  Four of five
$^{13}$C-cellulose responsive \textit{Chloroflexi} identified in this study are
annotated as belonging to the \textit{Herpetosiphon} although the
\textit{Herpetosiphon} SSU rRNA gene sequences from this study from 
$^{13}$C-cellulose responsive OTUs all share less
than 95\% sequence identity with their
closest cultured relative in the \textit{Herpetosiphon} genus (\textit{H.
    geysericola}). \textit{H. geysericola} is a predatory bacterium shown to
prey upon \textit{Aerobacter} in culture and can also digest cellulose
\citep{Lewin1970}. In our study, "Herpetosiphon" $^{13}$C-cellulose responders
did not show a delayed response to $^{13}$C-cellulose as compared to other
responders but nonetheless could have
become labeled by feeding on primary $^{13}$C-cellulose degraders. The prey
specificity of predatory bacteria is not well established especially \textit{in
    situ}. $^{13}$C-labeling would be positively correlated with prey
specificity. If the predator specifically preyed upon one population then it
could take on the same labeling percent as that population. Preying on multiple
types would produce a mixed and potentially diluted labeling signature if some
of the prey populations were not isotopically labeled.

% Fakesubsubsection:We also observed $^{13}$C-incorporation
We also observed $^{13}$C-incorporation from cellulose by
\textit{Proteobacteria}, \textit{Planctomycetes} and \textit{Bacteroidetes}. 
Strains in Proteobacteria, Planctomycetes and Bacteroidetes have all been
previously implicated in cellulose degradation. Planctomycetes is the
least studied of the three phyla and only one Planctomycetes isolate can
grow on cellulose. None of the seven \textit{Planctomycetes} cellulose degraders
identified in this experiment are closely related to isolates, and furthermore, the importance of their contributions to cellulose
degradation in soil is unknown. \textit{Acidobacteria} did not appear to incorporate $^{13}$C from
cellulose into DNA in our microcosms though \textit{Acidobacteria} have been
found to degrade cellulose in culture CITE and are a numerically significant
soil phylum CITE. \textit{Acidobacteria} have been shown to dominate at low
nutrient availability (CITE: cederlund 2014), which may explain why they do not
thrive under the nutrient replete experimental conditions in this study. The
\textit{Acidobacteria} in our microcosms were mainly annotated as belonging to
the "XX" group. 

\subsection{Ecological strategies of soil microorganisms participating in the
decomposition of organic matter}
% Fakesubsubsection:Ecological strategies of soil microorganisms
We observed ecological strategies among substrate responders although
boundaries between groups were not well defined. The observed ecological
strategy space appeared more like a continuum than discrete points. In general,
$^{13}$C-cellulose responders were slow-growers with low \textit{rrn} gene copy
number. $^{13}$C-cellulose responders were also typically low abundance members
of the microbial community and exhibited higher substrate specificity than
$^{13}$C-xylose responders (Figure~\ref{fig:shift}). 

%Fakesubsubsection:There was, however, as strong positive relationship
The \textit{Verrucomicrobia},
\textit{Chloroflexi} and \textit{Planctomycetes} $^{13}$C-cellulose responders exhibit 
phylum specific patterns of
substrate specificity, \textit{rrn} gene copy number, and growth over time (Figure~XX). There was a strong positive relationship between $^{13}$C-xylose
responders that responded at days 1,
3 or 7 in \textit{rrn} copy number. The estimated \textit{rrn} copy number of $^{13}$C-xylose responders declined over time, with the highest estimates at day 1. Therefore, OTUs that
grow faster assimilate C from xylose faster. With time fast growers
are replaced, with respect to xylose C assimilation, by slower growers (i.e. saprophytes and/or predators) as xylose diminishes. There was a succession in
activity with time from fast growing spore-formers to \textit{Bacteroidetes} and, finally, \textit{Actinobacteria} in our microcosms. The succession
hypothesis of decomposition groups ecological units by substrate CITE, however,
our results suggest there is a succession of microbial activity within
a single substrate. Hence, in soil, there is an ecological hierarchy coarsely
defined by the ability to assimilate C from labile or polymeric sources but
within the labile substrate degraders there are ecological subunits tuned to
specific substrate concentrations and/or trophic cascades on the temporal order
of days.

% Fakesubsubsection:HR-SIP allows us to assess C substrate specificity
C substrate specificity can be assessed by measuring the BD shift of OTU DNA
upon $^{13}$C incorporation. OTUs that incorporate more $^{13}$C per unit DNA
have greater specificity for the labeled substrate than OTUs that incorporate
less $^{13}$C per unit DNA. $^{13}$C-cellulose incorporating OTUs as a group
displayed greater substrate specificity than $^{13}$C-xylose incorporating
OTUs. This suggests that polymeric C-degraders tend to be specialists tuned to
particular C-substrates such as cellulose or lignin whereas labile C-degraders
are generalists able to assimilate C from many different labile sources.
Although we observed a succession of $^{13}$C-xylose responders
(Figure~\ref{fig:l2fc} and \ref{fig:xyl_count}), there was no discernible
difference in substrate specificity between $^{13}$C-xylose responders at days 1, 3 or 7. 

\subsection{Conclusion} 
% Fakesubsubsection:SOM represents more C than the


We found that $^{13}$C substrate responders changed as much as X-fold in
relative abundance over time (Figure~XX). This is in contrast to
a previous study CITE which suggested cellulose decomposers were found to
be consistent in relative abundance with time. Although presence in heavy
fractions can indicate $^{13}$C labeling, not all DNA in heavy fractions
is $^{13}$C-labeled. Some DNA is heavy due to high G+C.
With lower resolution fingerprinting techniques the banding pattern of SSU
rRNA gene sequences can look similar across the entire density gradient
CITE, however, high throughput sequencing of density gradient fractions
shows light and heavy fractions are statistically different even when
input DNA is entirely unlabeled (Figure~\ref{fig:time_class}, and CITE).
Hence, DNA-SIP studies that do not incorporate controls wherein amendments
contain only $^{12}$C substrates, may confuse high G+C organisms with
organisms that incorporated $^{13}$C into biomass. 

The succession hypothesis of decomposition predicts a succession from
microbial types that use labile C to those that use recalcitrant polymeric
C over time CITE. Cellulose degraders succeeded labile C degraders as
predicted. But, in response to $^{13}$C-xylose,  \textit{Firmicutes}
phylotypes were succeeded by \textit{Bacteroidetes} which were then
succeeded by \textit{Actinobacteria} representing a nested succession
(Figure~XX). 

Soluble C more robust to temperature changes because or redundancy? But we
found similar numbers of xylose and cellulose degrader
Microorganisms sequester atmospheric carbon and respire soil organic matter
(SOM) influencing climate change on a global scale but which microbial lineages
transform different soil C components are not established. Molecular tools will
unravel the soil microbial food web and reveal how specific microbial lineages
impact soil C flux. We present a cultivation independent, molecular, DNA-SIP
method to chart C use into microbial lineages. Our results show physiologically
undefined cosmopolitan microbial lineages decompose cellulose. We also show
phylogenetic groups rise and fall and are supplanted by others in activity over
7 days in response to labile C addition.  

Models of SOM turnover incorporate estimates of the carbon use efficiency
(CUE) for the microbial community where CUE is the ratio of biomass
production to biomass production plus combined respiration for growth and
 CITE.  CUE estimates from soil... 
