\section{Discussion} 
\subsection{Microbial response to isotopic labels}
% Fakesubsubsection: We propose that microbial decomposition
DNA-SIP can establish functional roles for thousands of phylotypes in a single
experiment without cultivation. We identified 104 soil OTUs that
incorporated $^{13}$C from xylose and/or cellulose into biomass and
characterized substrate specificity and C-cycling dynamics for these OTUs. We
propose xylose and cellulose C added to soil microcosms took the following path
through the microbial food web (Figure~\ref{fig:foodweb}): fast-growing
\textit{Firmicutes} spore formers first assimilated labile C followed by
\textit{Bacteroidetes}, \textit{Actinobacteria} and \textit{Proteobacteria}
phylotypes. The \textit{Bacteroidetes}, \textit{Actinobacteria} and
\textit{Proteobacteria} phylotypes may have also preyed on the rapidly
responding \textit{Firmicutes}. Canonical cellulose degrading bacteria such as
\textit{Cellvibrio} and members of cosmopolitan yet functionally
uncharacterized soil phylogenetic groups like \textit{Chloroflexi},
\textit{Planctomycetes} and \textit{Verrucomicrobia}, specifically the
\textit{Spartobacteria}, decomposed cellulose. Cellulose C incorporation into
microbial biomass peaked at day 14 and was maintained through day 30.

\subsection{Ecological strategies of soil microorganisms participating in the
decomposition of organic matter}
% Fakesubsubsection:We assessed the ecology of
We assessed $^{13}$C-responsive OTU ecological strategies by estimating each
OTU's \textit{rrn} gene copy number and BD shift due to $^{13}$C-labeling.
\textit{rrn} gene copy number correlates positively with, and indicates, growth
rate \citep{11125085,Kembel_2012}, whereas BD shift indicates substrate
specificity (see Results). $^{13}$C-cellulose responsive OTUs grow slower
(Figure~\ref{fig:shift}, Figure~\ref{fig:copy}), have greater substrate
specificity (Figure~\ref{fig:shift}), and are generally lower abundance members
of the bulk community than $^{13}$C-xylose responsive OTUs
(Figure~\ref{fig:shift}). High \textit{rrn} gene copy number may inflate
$^{13}$C-xylose-responder abundance. $^{13}$C-xylose responsive OTUs that
incorporate $^{13}$C into biomass at day one had greater \textit{rrn} gene copy
number than OTUs that responded later (Figure~\ref{fig:shift},
Figure~\ref{fig:copy}) suggesting fast-growing
microbes assimilated $^{13}$C from xylose before slow growers.

% Fakesubsubsection:NRI values have been used to assess
NRI values quantify phylogenetic clustering \citep{Webb2000} and have 
been used to assess clustering of soil OTUs that responded similarly to soil
wet up \citep{Evans2014a,Placella2012}. To our knowledge, assessing
phylogenetic clustering of OTUs found to incorporate heavy isotopes into
biomass during SIP incubations has not been attempted. We found that $^{13}$C-
cellulose and xylose responders are clustered and overdispersed, respectively.
This suggests that the ability to degrade cellulose is phylogenetically
conserved possibly reflecting the complexity of cellulose degradation
biochemistry. The positive relationship between a physiological trait's
phylogenetic depth and complexity has been noted previously
\citep{Martiny2013a} and the clustering depth of $^{13}$C-cellulose responders,
X.XX 16S rRNA gene sequence divergence, is on the same order as that observed
for glycoside hydrolases which are diagnostic enzymes for cellulose degradation
\citep{Berlemont2013}. Overdispersion, as we saw for the $^{13}$C-xylose
responsive OTUs, may be indicative of a readily horizontally transferred trait
and/or a trait that is broadly distributed phylogenetically, or, in this case
may indicate that the group comprises more than one trait. It's not clear
which $^{13}$C-xylose responsive organisms were labeled as a result of primary
xylose assimilation (see below), and therefore it's not clear if
$^{13}$C-xylose responsive OTUs in this experiment constitute a single
ecologically meaningful group or multiple ecological groups. Temporally defined
$^{13}$C-xylose responder groups, however, are phylogenetically coherent
(Figure~\ref{fug:tiledtree}, Figure~\ref{fig:xyl_count}). For example, most day
1 $^{13}$C-xylose responders are members of the \textit{Paenibacillus} (see
Supplemental~Note~XX). Notably, \textit{Paenibacillus} have been previously
implicated as labile C decomposers \citep{Verastegui_2014}.

% Fakesubsubsection:Intuitively C cycling trait diversity is inferred
Intuitively we infer C cycling functional guild diversity from the distribution
of diagnostic genes across genomes \citep{Berlemont2013} or from screening
culture collections for a particular trait \citep{Martiny2013}. For
instance, the wide distribution of the glycolysis operon in microbial
genomes is interpreted as evidence that many soil microorganisms
participate in glucose turnover \citep{McGuire2010}. \textit{In situ}
functional guild diversity, however, can vary significantly from diversity
assessed by functionally screening isolates and/or genomes. Xylose use in
soil, for instance, may be less a function of catabolic pathway
distribution among genomes and more a function of lifestyle. Phenomena
such as seasonal change \citep{Schmidt2007}, and rainfall
\citep{Placella2012} pulse deliver nutrients and resources to soil.
Therefore, fast growth and/or rapid resuscitation upon wet up
\citep{Placella2012} allow microorganisms to favorably compete for labile
C resources. Life history may limit the diversity of labile C assimilators
and life history determines growth rate and dessication resistance whereas
the ability to use labile C is phylogenetically dispersed. DNA-SIP is
useful for establishing \textit{in situ} phylogenetic clustering and
diversity of functional guilds because DNA-SIP can account for life
history strategies by targeting active microorganisms. Additionally,
snapshot estimates of community composition commonly inform soil
structure-function-relationship studies \citep{Fierer} but
labile C decomposition might not be linked to snapshot community structure, but
alternatively to community structure \textit{dynamics}. That is, fast growing
spore formers would not need to maintain high abundance to significantly
mediate cycling of pulse delivered resources. This accentuates the usefulness
of DNA-SIP for describing soil ecology as DNA-SIP assesses activity which can
be decoupled from snapshot abundance.

\subsection{Implications for soil C cycling models}
% Fakesubsubsection:Land management, climate, pollution and
Land management, climate, pollution and disturbance can influence soil
community composition \citep{McGuire2010} which in turn influences soil
biogeochemical process rates (e.g. \citep{Berlemont2014a}). Assessing
functional group diversity and establishing identities of functional group
members is necessary to predict how biogeochemical process rates will change
with community composition \citep{Schimel_1995,McGuire2010}. Aggregate
biogeochemical processes that are the sum of many subprocesses involve a broad
array of taxa and are assumed to be less influenced by community change than
narrow processes that involve a single, specific chemical transformation by
a smaller suite of microbial participants \citep{Schimel_1995,McGuire2010}.
Within an aggregate process such as C decomposition, subprocesses can be
further classified as broad or narrow \citep{McGuire2010}. In theory,
``broad'' and ``narrow'' functional guilds decompose labile and
recalcitrant C, respectively \citep{McGuire2010}. However, the diversity
of active labile C and insoluble, polymeric C decomposers in soil has not
been directly quantified. Notably, we found more OTUs responded to
$^{13}$C-cellulose, 63, than $^{13}$C-xylose, 49. Also, it is possible
that many $^{13}$C-xylose responders are predatory bacteria as opposed to
primary labile C degraders (see below). Cellulose and xylose decomposer
functional guilds were non-overlapping in membership -- of
104 $^{13}$C-responders only 8 responded to both cellulose and xylose -- and
represented a small fraction of total soil community diversity
(Figure~\ref{fig:genspec}). While xylose use is undoubtedly more widely
distributed among microbial genomes than the ability to degrade cellulose,
active cellulose decomposers outnumbered active xylose utilizers.

% Fakesubsubsection:Both $^{13}$C-cellulose and $^{13}$C-xylose
Both $^{13}$C-cellulose and $^{13}$C-xylose responders largely clustered
near the tips of the phylogenetic tree (NTI $>$ 0) at taxonomic levels broader
than the OTUs established in this study (Figure~\ref{fig:tiledtree}).
Therefore, $^{13}$C-responders distribute into fewer clades than OTUs
(Figure~\ref{fig:tiledtree}). $^{13}$C-xylose responders formed clades at X.XX
sequence divergence (consenTRAIT metric, \citep{Martiny2013}) while our OTUs
were established at
0.03 sequence divergence. 

% Fakesubsubsection:It's not clear whether the observed activity
Trophic interactions or the functional groups tuned to different resource
concentrations caused the activity succession from \textit{Firmicutes} to
\textit{Bacteroidetes} and finally \textit{Actinobacteria} in response to
xylose addition. \textit{Actinobacteria} (e.g. \textit{Agromyces}) and
\textit{Bacteroidetes} have been previously implicated as predatory soil
bacteria \citep{Lueders2006,16346402}, and, \textit{Bacteroidetes} and
\textit{Actinobacteria} activity peaked while \textit{Firmicutes}
$^{13}$C-xylose responder relative abundance plummeted in our microcosms.
Considering \textit{Agromyces} and \textit{Bacteroidetes} phylotypes are likely
soil predators, one parsimonious hypothesis for $^{13}$C-labelling of
\textit{Bacteroidetes} and \textit{Actinobacteria} with a corresponding
decrease $^{13}$C-labeled \textit{Firmicutes} abundance is that
\textit{Bacteroidetes} and \textit{Actinobacteria} fed on $^{13}$C-labeled
\textit{Firmicutes}. If the temporal dynamics of $^{13}$C-xylose incorporation
are due to trophic interactions predatory bacteria consumed, many, if not most,
fast-growing labile C degraders. Hence, soil C cycling models should include
predatory interactions between soil bacteria but rarely do (e.g.
\citep{Moore1988}).

% Fakesubsubsection:We propose two scenarios whereby community
We propose two scenarios in the context of our results whereby community
composition could affect C dynamics and fate. Genomic evidence shows cellulose
degradation is a phylogenetically conserved trait \citep{Berlemont2013}. Our
study evaluates the phylogenetic conservation of soil cellulose degradation in
active microorganisms via DNA-SIP and genomic evidence concurs with our
results. A decrease in cellulose degrader abundance would diminish cellulose
decomposition process rates as few soil microorganisms can fill the
phylogenetically conserved cellulose degradation niche. Dispersed cellulose
decomposers could renew ecosystem function, however. For labile
C decomposition, the absence fast growing spore formers would allow other
microbes to assimilate labile C provided dispersal does not enable rapid
recolonization CITE. Primary labile C degraders in this study grow fast, and
form spores and these distinct ecological strategies might indicate distinct
C use dynamics and/or resource allocation. New labile C degraders may
metabolize and allocate labile C differently thus changing labile C dynamics
and fate. Further, labile C degrader substitution could affect biomass
C turnover by predatory bacteria that feed on fast growing, spore forming
labile C decomposers. On the other hand, spore formation enables dispersal
\citep{Nicholson2000} which would allow fast growing spore formers to
continuously occupy the labile C decomposition niche. One proposed mechanism
for similar decomposition rates of labile C across soils varying in community
composition is that labile C can be used widely by microorganisms
\citep{McGuire2010}. An alternative hypothesis for consistent labile C process
rates across different soils is that labile C degraders disperse readily.
Notably, other lineages implicated in rapid labile C turnover include members
of the \textit{Actinobacteria} \citep{Placella2012} and many soil
\textit{Actinobacteria} form hyphae that facilitate dispersal
\citep{KILLHAM2007}. The two hypotheses are not mutually exclusive, but our
results and previous studies suggest that environmental conditions unfavorable
to fast-growing spore-formers and/or quickly resuscitated, hyphal
\textit{Actinobacteria} may impact labile C dynamics and fate.

\subsection{Conclusion} 
% Fakesubsubsection:Microorganisms sequester atmospheric C and respire
Microorganisms sequester atmospheric C and respire SOM influencing climate
change on a global scale but we do not know which microorganisms carry out
specific soil C transformations. In this experiment microbes from
physiologically uncharacterized but ubiquitous soil lineages participated in
cellulose decomposition. Cellulose C degraders included members of the
\textit{Verrucomicrobia} (\textit{Spartobacteria}), \textit{Chloroflexi},
\textit{Bacteroidetes} and \textit{Planctomycetes}. \textit{Spartobacteria} in
particular are globally cosmopolitan soil microorganisms and are often the most
abundant \textit{Verrucomicrobia} order in soil \citep{Bergmann_2011}.
Fast-growing \textit{Firmicutes} spore formers assimilated labile C in our
microcosms. \textit{Bacteroidetes} and \textit{Actinobacteria} phylotypes,
previously implicated as predators, may have fed on the fast growing
\textit{Firmicutes}. Our results suggest that, cosmopolitan
\textit{Spartobacteria} may degrade cellulose on a global scale, bacterial
tropic interactions can significantly impact soil C cycling, and life history
constrains functional guild diversity for labile C decomposition.
