\section{Discussion} 
% Fakesubsubsection
Nucleic-acid SIP coupled to microbiome fingerprinting
techniques has progressed from simple proof-of-concept experiments
\citep{radajewski2000stable} to studies utilizing non-DNA-sequencing microbial
community profiling methods such as DGGE, ARISA and/or tRFLP
\citep{Haichar_2007}, and currently to large experiments employing multiple
labeled substrates and high-throughput amplicon and/or shotgun DNA sequencing
\citep{Verastegui_2014}. We present a high-resolution nucleic acid SIP (HR-SIP)
approach that expands upon classical nucleic acid SIP methods in three
dimensions: 1) temporally, we sample isotopically labeled substrate amended
microcosms at multiple time points; 2) spatially, we assay more fractions along
the CsCl gradients; and 3), bioinformatically, we interrogate taxa at the level
of OTU (97\% sequence identity)  for isotope incorporation employing cutting
edge statistics for assessing differential abundance in microbiome datasets.

\subsection{Ordination of CsCl gradient fraction OTU profiles can be used to
    observe fraction-level $^{13}$C assimilation dynamics and membership
    differences}
% Fakesubsubsectionf
Many techniques are utilized to identify $^{13}$C-DNA in DNA-SIP studies
(reviewed by CITE) including the use of $^{13}$C carrier DNA, visualization of
CsCl gradient DNA with DNA intercalating dyes CITE, and screening of CsCl
gradient fractions with microbial community fingerprinting techniques or qPCR
CITE. Indeed the greatest challenge in DNA-SIP studies is teasing apart
$^{13}$C DNA from high G+C $^{12}$C-DNA \citep{Buckley_2007}. High throughput
DNA sequencing allows microbial ecologists to survey the microbial composition
of thousands of samples CITE EMP. We leveraged the high throughput nature of
next generation DNA sequencing technology to survey the microbial composition
of entire CsCl gradients.  Understanding the composition of entire gradients is
essential for robustly identifying and disentangling microbes that have
incorporated $^{13}$C into DNA from those microbes with only $^{12}$C-DNA.

% Fakesubsubsection
Each CsCl gradient fraction possesses a unique composition of SSU rRNA gene
phylogenetic types. DNA buoyant density (BD) drives differences in CsCl
gradient fraction SSU rRNA gene composition (see Figure~\ref{fig:ord}). For
instance, lighter DNA is more abundant in fractions at lighter densities so DNA
with lower G+C will be found in greater abundance at the light end of the CsCl
gradient and vice versa. Duplicate gradients receiving entirely $^{12}$C DNA
with the same bulk or non-fractionated SSU rRNA gene phylogenetic composition
would have the same overall profile of SSU rRNA gene phylogenetic types across
the density gradient. We fed microcosms identical C substrate mixtures save for
the identity of a $^{13}$C labeled substrate, and by design, DNA from all
microcosms harvested at a time point will be similar in bulk phylogenetic
composition. Therefore, SSU rRNA gene profile differences between gradients
harvested at the same time are due to $^{13}$C incorporation into bulk
community DNA. $^{13}$C-DNA shifts from its $^{12}$C BD position towards the
heavy end of the density gradient. This causes heavy fractions in gradients
that received $^{13}$C-DNA to be different in phylogenetic content than
corresponding heavy fractions from gradients that received $^{12}$C-DNA of the
same bulk phylogenetic composition.

% Fakesubsubsection
Ordination of CsCl gradient fraction phylogenetic profiles reveals differences
and similarities between gradients. It's clear that microcosms incorporated
$^{13}$C from both $^{13}$C-xylose and $^{13}$C-cellulose as gradients from
both $^{13}$C-xylose and $^{13}$C-cellulose microcosms differ from
corresponding control gradients (Figure~\ref{fig:ord}). These differences from
control gradients are focused in the heavy fractions (Figure~\ref{fig:ord}).
Analysis of SSU rRNA gene surveys has greatly benefited from utilizing
conventional methods for data exploration in ecology such as ordination
\citep{Lozupone_2008}.  SSU rRNA gene phylogenetic profiles in CsCl gradient
fractions have only recently been surveyed with high-throughput DNA sequencing
technology and subsequently explored via ordination \citep{Angel_2013,
Verastegui_2014}. Ordination of CsCl gradient fraction phylogenetic profiles
has reveled the relative influence of buoyant density and soil type on gradient
phylogenetic profile variance, however, ordination has not demonstrated isotope
incorporation.  Demonstrating isotope incorporation requires careful
comparisons between control and labeled gradients over the same buoyant density
range. By sequencing CsCl gradient fractions from both control and labeled
gradients across the full density gradient with DNA harvested from microcosms
at multiple time points, we can observe where in the density gradient $^{13}$C
isotope incorporation signal is strongest and when $^{13}$C isotope
incorporation begins (Figure~\ref{fig:ord}). $^{13}$C incorporation from
xylose and cellulose is most apparent at days 1/3/7 and days 14/30,
respectively (Figure~\ref{fig:ord}). Moreover, labeled gradient fraction
phylogenetic profiles diverge from controls most dramatically at relatively
heavy buoyant densities (Figure~\ref{fig:ord}). Also, $^{13}$C-DNA from
$^{13}$C-xylose microcosms is different in phylogenetic composition from
$^{13}$C-cellulose microcosm $^{13}$C-DNA indicating that xylose and cellulose
were assimilated by different microbial community members
(Figure~\ref{fig:ord}). Lastly, ordination indicates organisms that assimilated
$^{13}$C from $^{13}$C-xylose changed in phylogenetic type over incubation days
1, 3 and 7 (Figure~\ref{fig:ord}).

\subsection{Cellulose degraders identified from undescribed lineages and
    cosmoplitan soil taxa for which functional attributes are not established}
% Fakesubsubsection
\textit{Verrucomicrobia} are ubiquitous in soil worldwide
\citep{Bergmann_2011}.  \textit{Verrucomicrobia} can constitute 23\% of 16S
rRNA gene sequences in high-throughput DNA sequencing surveys of SSU rRNA genes
in soil \citep{Bergmann_2011} and have been shown to represent as high as 9.8\%
of soil 16S rRNA \citep{Buckley_2001}. Many \textit{Verrucomicrobia} cultivars
have been established in the last decade \cite{Wertz_2011} but only one of the
15 most abundant verrucomicrobial phylotypes in a global soil sample collection
shared greater than 93\% sequence identity with an isolate
\citep{Bergmann_2011}.  Genomic analyses and physiological profiling of
\textit{Verrucomicrobia} isolates have revealed methanotrophy and diazotrophy
\citep{Wertz_2011} within \textit{Verrucomicria} (CITE and reviewed by
\citet{Wertz_2011}). Notably, the genetic capacity to degrade cellulose and
cellulose degradation in culture have been demonstrated in
\textit{Verrucomicrobia} \citep{Otsuka_2012, Wertz_2011}.  Although, we have
learned many functional roles of \textit{Verrucomicrobia} in the environment,
the function and or global significance of soil \textit{Verrucomicrobia} in
global C-cycling is unknown. For example, only one of the putative
verrucomicrobial cellulose degraders identified in this experiment are closely
related to named cultivars (OTU.XX, Table~\ref{tab:cell}) and only XX\% of all
verrucomicrobial OTUs found in this study share at lease 97\% sequence identity
with isolates. Seven of 10 $^{13}$C-cellulose responding verrucomicrobial OTUs
were classified belonging to the \textit{Spartobacteria} order.
\textit{Spartobacteria} order was overwhelmingly the numerically dominant order
of \textit{Verrucomicrobia} in SSU rRNA gene surveys of 181 globally
distributed soil samples \citep{Bergmann_2011}. HR-SIP identifies key players
in soil C-cycling and \textit{Verrucomicrobia} lineages particularly
\textit{Spartobacteria}, given their ubiquity and abundance in soil as well as
their demonstrated incorporation of $^{13}$C from $^{13}$C-cellulose, may be
significant players in global soil cellulose respiration. 

% Fakesubsubsection
Soil \textit{Chloroflexi} have been found to assimilate cellulose before in
DNA-SIP studies with $^{13}$C-cellulose \citep{Schellenberger_2010}. Previously
identified \textit{Chloroflexi} $^{13}$C-cellulose-responders in were similarly
underrepresented in culture collections as the \textit{Chloroflexi} identified
as cellulose degraders in this study \citep{Schellenberger_2010}.
\textit{Chloroflexi} is on average XX\% abundant in soil samples screened by
the Earth Microbiome Project (EMP, CITE, Figure~XX) and is found in XX of XX
EMP soil samples (XX\%) and XX of all XX EMP samples (Figure~XX). Although not
as abundant as \textit{Spartobacteria} in EMP samples (Figure~XX),
\textit{Chloroflexi} are still a cosmopolitan soil phylum. Similarly to
\textit{Spartobacteria}, \textit{Chloroflexi} are largely uncharacterized
functionally in soil. Here we present a lineage in the \textit{Chloroflexi}
that incorporated $^{13}$C from cellulose in our microcosms.
\textit{Chloroflexi} Banfield paper

\subsection{Succession with degradation of labile C}
\label{sub:succession_with_degradation_of_labile_c}
% Fakesubsubsection
The succession hypothesis of decomposition hypothesizes a succession from
microbial types that utilize labile C to those that utilize recalcitrant
polymeric C over time CITE. We observed a \textit{sub}-succession during the 
degradation of labile C. 

\subsection{Ecological strategies of soil microorganisms participating in the
    decomposition of organic matter}
\label{sub:ecological_strategies}
% Fakesubsubsection
Ecological strategies of soil microorganisms have mostly been elucidated by
cultivation-dependent studies CITE and include characterization of growth
rates, \ldots  We observed several ecological strategies among substrate
responders although as boundaries between groups were not well defined the
observed ecological strategy space appeared more like a continuum than discrete
points. In general, $^{13}$C-cellulose responders were slow-growers with low
\textit{rrn} gene copy number. $^{13}$C-cellulose responders were also
typically low abundance members of the microbial community and exhibited higher
substrate specificity than $^{13}$C-xylose responders (Figure~\ref{fig:shift}).
Within the $^{13}$C-cellulose responders there was faint but apparent
\textit{sub}-ecological strategy groups. For instance,
\textit{Verrucomicrobia}, \textit{Chloroflexi} and \textit{Planctomycetes}
showed had distinct patterns of substrate specificity, \textit{rrn} gene copy
number and temporal dynamics in the bulk community (Figure~XX).

% Fakesubsubsection
We found that control gradients yield amplifiable DNA well into the heavy
fractions. In fact, we observe XX OTUs in the heavy fractions of control
gradients. Therefore, presence in heavy fractions of $^{13}$C gradients alone
is not evidence of $^{13}$C incorporation into DNA. Only enrichment in
$^{13}$C heavy fractions relative to control gradient fractions demonstrates
incorporation even if heavy fractions are shown to be different from controls
by fingerprinting techniques CITE Neufeld 2010.

Xylose responders change over time. Implications for DNA-SIP. Succession within
succession.

Response not consistent across phyla.

\subsection{Ecological strategies of xylose and cellulose degrading OTUs in soil}
HR-SIP allows us to assess C substrate specifity and temporal dynamics of
$^{13}$C-incorporation. C substrate specificity is assed by measureing the 
BD shift of OTU DNA upon $^{13}$C incorporation. OTUs that incorporate more
$^{13}$C per unit DNA have greater specify for the labeled substrate than
OTUs that incorporate less $^{13}$C per unit DNA. $^{13}$C-cellulose 
incorporating OTUs as a group displayed greater substrate specificity than
$^{13}$C-xylose incorporating OTUs. This suggests that polymeric C-degraders
tend to be specialists tuned to particular C-substrates such as cellulose
or lignin whereas labile C-degraders are generalists able to assimilate
C from many different labile sources. Although we observed a succession of 
$^{13}$C-xylose responders (Figure~\ref{fig:l2fc} and \ref{fig:xyl_count}), 
there was no discernible difference in substrate specificity between 
$^{13}$C-xylose responders that first responded at days 1, 3 or 7. There was,
however, a strong positive relationship between $^{13}$C-xylose responders that
first responded at days 1, 3 or 7 in \textit{rrn} copy number. $^{13}$C-xylose 
responders that first responded at day 1 had higher estimated \textit{rrn} copy 
number than responders that first responded at day 3 which had higher \textit{rrn}
copy number than responders that first responded at day 7. Therefore, OTUs that 
grow faster assimilate C from xylose faster intuitively. However, fast growers
are replaced with respect to xylose C assimilation with slower growers as
xylose diminishes. There was a succession of xylose degraders with time from
fast growing spore-formers to \textit{Bacteroidetes} types and finally 
\textit{Actinobacteria} in our microcosms. The succession hypothesis of
decomposition groups ecological units by substrate CITE, however, our results
suggest there is a succession of microbial activity for even a single
substrate. Hence, in soil, there is an ecological hierarchy coarsely defined by
the ability to assimilate C from labile or polymeric sources but within
the labile substrate degraders there are ecological subunits tuned to specific
substrate concentrations.















