\section{Discussion} 
% Fakesubsubsection: We demonstrate that 
We demonstrate that DNA-SIP can establish ecological characteristics of
microorganisms participating in soil C cycling. We distributed SSU rRNA genes
from our experimental soil into 5,940 OTUs and observed assimilation of
$^{13}$C from either $^{13}$C-xylose or $^{13}$C-cellulose into 104 OTUs.
$^{13}$C label was observed to move into and then out of groups of related OTUs
over time. The approach that we employed allowed us to interrogate OTU activity
across a wide range of natural abundances; For instance,we observed
$^{13}$C-incorporation into DNA for OTUs present at relative abundance as low
as XX\% in the non-fractionated DNA from the experimental soil. Our results
support the degradative succession hypothesis, elucidate ecophysiological
properties of soil microorganisms, reveal activity of widespread uncultured
soil bacteria, and demonstrate a microbial food web in soils. 

% Fakesubsubsection: Our results agree with
Our results agree with the degradative succession hypothesis. We observed rapid
consumption of $^{13}$C-xylose and soil microorganisms assimilated
$^{13}$C-label from $^{13}$C-xylose into DNA during days 1, 3, and 7. Xylose is
the major monomer of xylan and xylan itself is a major constituent of
hemicellulose. Thus, xylose represents an abundant sugar present in the early
phases of plant biomass degradation. The phylogenetic composition of $^{13}$C
labeled OTUS in response to the $^{13}$C-xylose amendment changed over time and
the number of $^{13}$C-labeled OTUs in response to $^{13}$C xylose
diminished almost entirely by the end of the incubation. In contrast,
degradation of cellulose proceeded more slowly and assimilation of
$^{13}$C-label into DNA from $^{13}$C-cellulose sharply increased from day~7 to
day~14 and was maintained through day~30. Finally, the vast majority
of xylose or cellulose responders had unique activity for either cellulose or
xylose as only 8 of 104 OTUs were observed to metabolize both xylose and
cellulose. 

% Fakesubsubsection: Correlations between community composition
Correlations between community composition and environmental characteristics
often reveal the ecology of microorganisms \citep{Fierer2007}. In this experiment,we utilized
DNA-SIP characterize directly the ecological characteristics of microorganisms
as by their \textit{in situ} metabolism in addition to their change in relative
abundance over time. Several lines of evidence suggest that the xylose
responders are able to grow rapidly and assimilate C from
multiple sources. The xylose responders were characterized by rapid
assimilation of xylose-C into DNA but they had relatively low $\Delta\hat{BD}$
potentially indicating that xylose was not the sole C source used for growth.
Xylose represented only 20\% of the nutrient and resource microcosm amendment
and 3.5\% of total soil C. Xylose responders were also highly
abundant within the non-fractionated DNA and had relatively high estimated
\textit{rrn} copy number. However, to some degree, high \textit{rrn} gene copy
number may inflate observed xylose responder relative abundance. High
\textit{rrn} copy number may allow microorganisms to rapidly respond to
nutrients influx \citep{Klappenbach_2000}. It is also notable that the
majority of xylose responders (86\%) are well represented among cultured
isolates. 

% Fakesubsubsection: In contrast, the results
In contrast, cellulose responders appeared to be C specialists.
Cellulose responders grew relatively slowly over a span of weeks and had
relatively high $\Delta\hat{BD}$ indicating that, although multiple sources of
C were present, cellulose remained the dominant C source for cellulose
responders. They were also present at generally lower abundance within the
non-fractionated DNA and had relatively low estimated \textit{rrn} copy number.
Also of note, while a number of cellulose responders shared high SSU rRNA gene
sequence identity to cultured isolates in the \textit{Proteobacteria} (e.g.
\textit{Cellvibrio}), the majority of cellulose responders we identified were
not close relatives of cultured isolates. We identified cellulose responders
among phyla such as \textit{Verrucomicrobia}, \textit{Chloroflexi}, and
\textit{Planctomycetes} whose functions within soil communities remain poorly
characterized.

% Fakesubsubsection: Verrucomicrobia comprise
\textit{Verrucomicrobia} made up 16\% of the cellulose responders.
\textit{Verrucomicrobia} are cosmopolitan soil microbes \citep{Bergmann_2011} that can make up
to 23\% of SSU rRNA gene sequences in soils \citep{Bergmann_2011} and 9.8\% of soil SSU rRNA
\citep{Buckley_2001}. Genomic analyses and physiological profiling has revealed that isolates
within the \textit{Verrucomicrobia} are capable of methanotrophy,
diazotrophy, and cellulose degradation \citep{Wertz_2011,Otsuka_2012} and they
have been hypothesized to degrade polysaccharides in many environments
\citep{Fierer_2013,10543821,Herlemann_2013}. However, only one of the
15 most abundant verrucomicrobial phylotypes observed globally in soils shares
$>$ 93\% SSU rRNA gene sequence identity with a cultured isolate \citep{Bergmann_2011} and hence
the role of soil \textit{Verrucomicrobia} in global C-cycling remains unknown.
The majority of verrucomicrobial cellulose responders belonged to two clades
that fall within the \textit{Spartobacteria}. \textit{Spartobacteria} were the
most commonly observed \textit{Verrucomicrobia} in SSU rRNA gene surveys of
181 globally distributed soil samples \citep{Bergmann_2011}. Given their ubiquity and abundance
in soil as well as their demonstrated incorporation of $^{13}$C from
$^{13}$C-cellulose, \textit{Verrucomicrobia} lineages, particularly
\textit{Spartobacteria}, may be important contributors to cellulose
decomposition on a global scale. 

% Fakesubsubsection: Other notable cellulose responders
Other notable cellulose responders include OTUs in the \textit{Planctomycetes}
and \textit{Chloroflexi} both of which have previously been shown to
assimilate $^{13}$C from $^{13}$C-cellulose added to soil
\citep{Schellenberger_2010}. Planctomycetes are common in soil
\citep{Jannsen2006}, comprising 4 - 7\% of bacterial cells in many soils
\citep{Zarda_1997,Chatzinotas_1998} and 7\% $+/-$ 5\% of SSU rRNA \citep{buckley_2003}       
Although \textit{Planctomycetes} are widespread in soils, their activities in
soil remain poorly characterized. \textit{Plantomycetes} represented 16\% of
cellulose responders we identified, and these responders had $<$ 92\% SSU rRNA
gene sequence identity to their most closely related cultured isolates.

% Fakesubsubsection: Chloroflexi are among the
\textit{Chloroflexi} are among the six most abundant bacterial phyla in
soil \citep{Janssen2006}. \textit{Chloroflexi} are known for
metabolically dynamic lifestyles ranging from anoxygenic phototrophy to
organohalide respiration \citep{Hug_2013}. Recent studies have focused on
\textit{Chloroflexi} roles in C cycling
\citep{Hug_2013,Goldfarb_2011,Cole_2013} and several \textit{Chloroflexi}
utilize cellulose \citep{Hug_2013,Goldfarb_2011,Cole_2013}. DNA-SIP previously
identified cellulose degrading soil \textit{Chloroflexi}
\citep{Schellenberger_2010}. Four of the five \textit{Chloroflexi}
cellulose responders belong to a single clade within the
\textit{Herpetosiphonales} although they share less than 89\% SSU gene sequence
identity with their closest cultured relative, \textit{H. geysericola}.
\textit{H. geysericola} is a predatory bacterium that feeds on
\textit{Aerobacter} in culture and it can also metabolize cellulose
\citep{Lewin1970}. 

% Fakesubsubsection: Finally, a single
Finally, a single cellulose responder was identified from
\textit{Melainobacteria} (95\% sequence identity to \textit{Vampirovibrio
chlorellavorus}). Although, the phylogenetic position of
\textit{Melainabacteria} is debated \citep{Soo_2014}, this non-phototrophic
group is closely associated with Cyanobacteria. An analysis of
a ``\textit{Melainabacteria}'' genome \citep{Di_Rienzi_2013} suggests the genomic capacity
to degrade polysaccharides though \textit{Vampirovibrio chlorellavorus} is
an obligate predator of green alga \citep{gromov_1972}. 
It is interesting that cellulose responders within \textit{Chloroflexi} and
\textit{Melainobacteria} are related (though distantly) to known predators. The
relatively high $\Delta\hat{BD}$ of these cellulose responders suggests that if
they are predators, they tightly associate with cellulose degrading
microorganisms. Also, these OTUs are not xylose responders and if they are
predators, our results indicate they prey specifically on slow growing
cellulolytic bacteria and not on the xylose responders. 

% Fakesubsubsection: We found that cellulose and xylose
We found that cellulose and xylose responders are phylogenetically clustered
and overdispersed, respectively. This suggests that the ability to degrade
cellulose is phylogenetically conserved possibly due to the degree of
biochemical complexity and/or the degree of ecological specialization required
to degrade cellulose in soil. The positive relationship between a physiological
trait’s phylogenetic depth and its complexity has been noted previously
\citep{Martiny2013} and the clade depth of cellulose responders, 0.028 SSU
rRNA gene sequence dissimilartiy, is on the same order as that observed
for glycoside hydrolases which are diagnostic enzymes for cellulose
degradation \citep{Berlemont2013}. Overdispersion, as observed for xylose
responders is indicative of a broadly distributed trait whose distribution
could be affected by horizontal gene transfer or convergent evolution, for
example by catabolic recruitment of genes normally reserved for anabolic
functions. Overdispersion could also result if the xylose responders
represent multiple distinct functional groups which differ in their
response to xylose. 

% Fakesubsubsection: We diagnosed the assimilation
$^{13}$C-responders did not necessarily assimilate $^{13}$C into DNA directly
from $^{13}$C-xylose or $^{13}$C-cellulose. In many ways, knowledge of
secondary assimilation of $^{13}$C-label may be more interesting with respect
to the soil C-cycle than knowledge of primary assimilation. In particular, the
response to xylose suggests xylose C moved through different levels within the
soil food web. The \textit{Bacilli} degraded xylose first and by day~1, 65\% of
the xylose-C had been respired, \textit{Bacilli} represent 84\% of xylose
responders, and comprise a remarkable 6\% of SSU rRNA genes present in soil.
However, \textit{Bacilli} were generally no longer $^{13}$C-labeled by day~3
and their abundance declined reaching about 2\% of soil SSU rRNA genes by
day~30. These results indicate a group of organisms with a boom-bust
lifestyle. Members of the \textit{Bacillus} \citep{Cleveland2007} and \textit{Paenibacillus} in
particular \citep{Verastegui_2014} have been previously implicated as labile C decomposers. The
decline in abundance of \textit{Bacilli} is consistent with either mortality
and/or sporulation coupled to mother cell lysis. Both mortality and sporulation
offer a mechanism for C transfer from \textit{Bacilli} to other soil
microorganisms. \textit{Bacteroidetes} are the most common xylose responders at
day~3. The assimilation of  $^{13}$C into DNA at day~3 by
\textit{Bacteroidetes} coincides with loss of $^{13}$Cr-label and decline in
relative abundance of the \textit{Bacilli}. Subsequently,
\textit{Actinobacteria} were the most common xylose responders at day~7. The
abundance of \textit{Actinobacteria} peaks at day~3 and \textit{Actinobacteria}
$^{13}$C-incorporation into DNA at day~7 corresponds with loss of
$^{13}$C-label and decline in relative abundance of the \textit{Bacteroidetes}.
Hence, it seems reasonable to propose that the observed temporal dynamics
$^{13}$C-labeling in response to $^{13}$C-xylose resulted from trophic
C transfer and that specifically \textit{Bacteroidetes} and
\textit{Actinobacteria} consumed $^{13}$C-labeled microbial biomass. 

% Fakesubsubsection: Trophic transfer could be due
Trophic transfer could be due to predation or saprotrophy and additional
evidence for C transfer by predation and/or saprotrophy comes from the
inferred physiology of the Bacteroidetes and Actinobacteria xylose responders
at days~3 and 7. Most of the \textit{Actinobacteria} xylose responders that
appeared $^{13}$C-labeled at day~7 are members of the \textit{Micrococcales} (Figure
XX) and the most abundant $^{13}$C-labeled \textit{Micrococcales} OTU at day~7
(“OTU.4”, Table XX) is annotated as belonging in the \textit{Agromyces}.
\textit{Agromyces} are facilitative predators that have been shown to feed on
the gram-positive \textit{Luteobacter} in culture \citep{16346402}.
Additionally, soil \textit{Bacteroidetes} have been implicated previously as
predatory bacteria in nucleic acid SIP studies with $^{13}$C-labeled
Eshericia coli \citep{Lueders2006}. However, it also
remains possible that the \textit{Bacilli}, \textit{Bacteroidetes}, and
\textit{Actinobacteria} are adapted to use xylose at different
concentrations and that the dynamics of assimilation result from changes
in xylose concentration over time. If the temporal dynamics we observe for
the assimilation of $^{13}$C-xylose incorporation are due to trophic
transfer, it indicates that the carbon was exchanged between at least
three different groups in 7 days. Models of the soil C cycle rarely
include trophic interactions between soil bacteria (e.g. \citep{Moore1988}).
When soil C models do account for predators/saprophytes, trophic
interactions are predicted to have significant effects on the fate of soil
C \citep{Kaiser2014a}. 

\subsection{Implications for soil C cycling models}
% Fakesubsubsection: Models of soil
Models of soil C cycling rely on functional niches defined by soil
microbiologists (E.g. \citep{wieder_2014a,Kaiser2014a}). In some C models
ecological strategies such as growth rate and substrate specificity are
parameters for functional niche behavior \citep{Kaiser2014a}. The phylogenetic
breadth of an functional guilds are often inferred from the distribution of
diagnostic genes across genomes \citep{Berlemont2013} or from the physiology of
isolates cultivated on laboratory media \citep{Martiny2013}. For instance, the
wide distribution of the glycolyis operon in microbial genomes is interpreted
as evidence that many soil microorganisms participate in glucose turnover (7).
Life history strategies, however, can determine
which microbes use a gene from those that have it. Xylose metabolism in soil,
for instance, may depend less on the distribution of the catabolic pathway
across genomes and more on life history traits that allow organisms to
compete effectively for xylose in the soil. Phenomena such as seasonal change
\citep{Schmidt2007}, and rainfall \citep{Evans2014a} cause nutrient and
resource concentrations in soil to fluctuate dynamically. Therefore, fast
growth and rapid resuscitation allow microorganisms to favorably compete for
labile C which may often be transient in soil. Hence, life history traits that
correlate with phylogeny, such as growth rate \citep{Fierer2007}, may constrain
the diversity of microbes that are actually able to metabolize a given C source
as it occurs in the soil. DNA-SIP is useful for establishing in situ diversity
of functional guilds to contrast guild diversity inferred from genomic evidence
and laboratory experiments because DNA-SIP targets active microbes. 

% Fakesubsubsection: In theory, the diversity of 
Biogeochemical processes mediated by a broad array of taxa and are assumed to
be less influenced by community change than narrow processes that involve
a single, specific chemical transformation by a narrow suite of microbial
participants \citep{Schimel_1995,McGuire2010}. In theory, the diversity of
a functional guild engaged in a specific C transformation is correlated to the
lability of the C \citep{McGuire2010}. However, the diversity of active
labile C and recalcitrant C decomposers in soil has not been directly
quantified. Notably, we found more OTUs responded to $^{13}$C-cellulose, 63,
than $^{13}$C-xylose,~49. Also, it is possible that many $^{13}$C-xylose
responders were predatory bacteria or saprophytes as opposed to primary labile
C degraders. Cellulose and xylose decomposer functional guilds were
non-overlapping in membership -- of 104 $^{13}$C-responders only 8 responded to
both cellulose and xylose -- and represented a small fraction of total soil
community diversity (Figure~\ref{fig:genspec}). While xylose use is undoubtedly
more widely distributed among microbial genomes than the ability to degrade
cellulose, the number of unique active cellulose decomposer OTUs outnumbered
the number of unique active xylose utilizer OTUs. Further, although cellulose
responders were phylogenetically clustered (NRI: XX) and xylose responders were
overdispersed (NRI: XX), it's not clear if $^{13}$C-xylose responsive OTUs in
this experiment constitute a single ecologically meaningful group or multiple
ecological groups. Notably, temporally defined $^{13}$C-xylose responder groups
were largely phylogenetically coherent (Figure~\ref{fig:tiledtree},
Figure~\ref{fig:xyl_count}). For example, most day~1 $^{13}$C-xylose responders
are members of the \textit{Paenibacillus}. Thus, active labile and structural
C associated microorganisms may be of limited diversity in soil which
implicates labile and structural C could be similarly affected by changes in
community composition. 

% Fakesubsubsection: Broadly, we observed
Broadly, we observed labile C use by fast growing generalists and structural
C use by slow growing specialists. These results agree with the MIMICS model
which simulates leaf litter decomposition incorporating the activity of two
microbial functional types characterized by copiotrophic and
oligotrophic growth strategies \citep{wieder_2014a}. The inclusion of these
different functional types improved the prediction of C response to
environmental change. We identified specific microbial members of functional
groups engaged in labile and structural C decomposition or simply speaking
copiotrophs and oligotrophs. We also observed potentially greater copiotroph
biomass turnover which may be important to consider when modeling microbial
turnover input to SOM. It's also clear that that there are more than two
functional types mediating C-cycling in soil. There is at least one more
functional type consisting of potential saprophytes and/or predators
characterized by slower growth relative to primary labile C degraders. 

\subsection{Conclusion} 
% Fakesubsubsection: Microorganisms sequester atmospheric C
Microorganisms sequester atmospheric C and respire SOM influencing climate
change on a global scale but we do not know which microorganisms carry out
specific soil C transformations. In this experiment microbes from
physiologically uncharacterized but cosmopolitan soil lineages participated in
cellulose decomposition. Cellulose responders included members of the
\textit{Verrucomicrobia} (\textit{Spartobacteria}), \textit{Chloroflexi},
\textit{Bacteroidetes} and \textit{Planctomycetes}. \textit{Spartobacteria} in
particular are globally cosmopolitan soil microorganisms and are often the most
abundant \textit{Verrucomicrobia} order in soil \citep{Bergmann_2011}.
Fast-growing aerobic spore formers from Firmicutes assimilated labile C in the
form of xylose. Xylose responders within the Bacteroidetes and Actinobacteria
are likely to have assimilated xylose C as a result of dynamic trophic
exchange, mediated either by saprotrophy or predation. Our results suggest
that, cosmopolitan \textit{Spartobacteria} may degrade cellulose on a global
scale, bacterial trophic interactions can significantly impact soil C cycling,
and ecological traits are likely to act as a filter that constrains the
diversity of microorganisms that are active in situ relative to those that have
the genomic capacity for a given process.
