\section{Discussion} 
% Fakesubsubsection: We demonstrate that 
We demonstrate that DNA-SIP can establish ecological characteristics of
microorganisms engaging in soil C cycling. We distributed SSU rRNA genes from
our experimental soil into 5,940 OTUs and observed statistically significant
assimilation of 13C from either 13C-xylose or 13C-cellulose into 104 OTUs. The
13C labeling of these OTUs was dynamic and labeled OTUs at individual time
points were phylogenetically coherent. Label was observed to move into and then
out of individual taxa and groups of related taxa over time. The approach that
we employed was sensitive with evidence of 13C- incorporation into DNA observed
for OTUs present atrelative abundance as low as XX% in the unfractionated DNA
from the soil. The data we describe supports the degradative succession
hypothesis, provides insights on the physiological ecology of soil
microorganisms, reveals the activity of widespread uncultured soil bacteria,
and provides insights into the nature of the microbial food web in soils. 

% Fakesubsubsection: The results we describe
Our results agree with the degradative succession hypothesis. We observed rapid
consumption of 13C-xylose with 13C-label assimilated into microbial DNA during
days 1, 3, and 7. Xylose is the major monomer of xylan and xylan itself a major
constituent of hemicellulose thus xylose represents an abundant sugar present
in the early phases of plant biomass degradation. The phylogenetic composition
of 13C labeled OTUS in response to the  13C-xylose amendment changed over time
and the number of OTUs labeled in response to 13C xylose diminished almost
entirely by the end of the incubation. In contrast, degradation of cellulose
proceeded more slowly and assimilation of 13C-label into DNA from 13C-cellulose
was not sharply increase from day 7 to day 14 and proceeded through day 30.
Finally, the microorganisms that metabolized xylose and cellulose were
taxonomically distinct. The vast majority of xylose or cellulose responders had
unique activity for either cellulose or xylose as only 8 of 104 OTUs were
observed to metabolize both xylose and cellulose. e. 

% Fakesubsubsection: Correlations between community composition
Correlations between community composition and environmental characteristics
often inform the ecology of microorganisms (22). In this experiment, DNA-SIP
was used to characterize directly the ecological characteristics of
microorganisms as a function of their in situ metabolism in addition their
change in abundance over time. Several lines of evidence suggest that the
xylose responders we identified are able to grow rapidly and assimilate C from
multiple sources. The xylose responders were characterized by rapid
assimilation of xylose-C into DNA but they had relatively low Δ1.66-1.76
indicating that xylose was not the sole C source used for growth. Xylose
represented only 20\% of the total C mixture added to soil and 3.5\% of total
soil C. Xylose responders were also highly abundant within the non-fractionated
DNA and had relatively high estimated rrn copy number. However, to some degree,
high rrn gene copy number may inflate observed xylose responder relative
abundance. High rrn copy number has been associated with the ability to respond
rapidly to an influx of new nutrients (17, 23). In addition, it is also notable
that the majority of xylose responders (86\%) are well represented among
cultured isolates. 

% Fakesubsubsection: In contrast, the results
In contrast, cellulose responders appeared to have a specialist
lifestyle. Cellulose responders grew relatively slowly over a span of weeks and
had relatively high indicating that, although multiple sources of
C were present, cellulose remained the dominant C source for cellulose
responders. They were also present at generally lower abundance within the
non-fractionated DNA and had relatively low estimated rrn copy number. Also of
note, while a number of cellulose responders shared high SSU rRNA gene sequence
identity to cultured isolates in the Proteobacteria (e.g. Cellvibrio) already
known to degrade cellulose, the majority of cellulose responders we identified
were not close relatives of cultured isolates. Cellulose responders were found
among phyla such as Verrucomicrobia, Chloroflexi, and Planctomycetes which have
long been found in soil, but whose functions within soil communities remain
poorly characterized.

% Fakesubsubsection: Verrucomicrobia comprise
Verrucomicrobia comprised 16\% of the cellulose responders. Verrucomicrobia are
cosmopolitan soil microbes (42) that make up to 23\% of SSU rRNA gene sequences
in soils (42) and 9.8\% of soil SSU rRNA (43) and are one of the most common
bacterial phyla in soil (Janssen 2006). Genomic analyses and physiological
profiling has revealed that diverse isolates of Verrucomicrobia are capable of
methanotrophy, diazotrophy, and cellulose degradation (44, 45), they have been
hypothesized to degrade polysaccharides in many environments (46–48). However,
only one of the 15 most abundant verrucomicrobial phylotypes observed globally
in soils shares > 93\% SSU rRNA gene sequence identity with a cultured isolate
(42) and hence the role of soil Verrucomicrobia in global C-cycling remains
unknown. The majority of verrucomicrobial cellulose responders belonged to two
clades that fall within Spartobacteria. Spartobacteria are the most commonly
observed Verrucomicrobia in SSU rRNA gene surveys of 181 globally distributed
soil samples (42). Given their ubiquity and abundance in soil as well as their
demonstrated incorporation of 13C from 13C-cellulose, Verrucomicrobia lineages,
particularly Spartobacteria, may be important contributors to cellulose
decomposition on a global scale. 

% Fakesubsubsection: Other notable cellulose responders
Other notable cellulose responders include Planctomycetes and Chloroflexi, and
both of which have previously been shown to assimilate 13C from 13C-cellulose
added to soil (Schellenberger et al. 2010 Environ Microbiol 12:845-861).
Planctomycetes are common in soil (Jannsen 2006), comprising 4 - 7\% of
bacterial cells in many soils (Chatzinotas, A., R. A. Sandaa, W. Schonhuber, R.
Amann, F. L. Daae, V. Torsvik, J. Zeyer, and D. Hahn. 1998. Analysis of
broad-scale differences in microbial community composition of two pristine
forest soils. System. Appl. Microbiol. 21:579-587; Zarda, B., D. Hahn, A.
Chatzinotas, W. Schonhuber, A. Neef, R. I. Amann, and J. Zeyer. 1997. Analysis
of bacterial community structure in bulk soil by in situ hybridization. Arch.
Microbiol. 168:185-192.) and 7\% ± 5\% of SSU (Buckley, D. H., and T. M.
Schmidt. 2003. Diversity and dynamics of microbial communities in soils from
agroecosystems. Environ. Microbiol. 5:441-452). Although Planctomycetes are
abundant and widespread in soils their activities in soil remain poorly
characterized. Plantomycetes represented 16\% of the cellulose responders we
identified, and these responders had less than 92\% SSU rRNA gene sequence
identity to the most closely related cultured isolate.

% Fakesubsubsection: Chloroflexi are among the
Chloroflexi are among the six most abundant bacterial phyla in soil (55).
Chloroflexi are known for metabolically dynamic lifestyles ranging from
anoxygenic phototrophy to organohalide respiration (51). Recent studies have
focused on Chloroflexi roles in C cycling (51–53) and several Chloroflexi
utilize cellulose (51–53). Cellulose degrading soil Chloroflexi have previously
been identified in DNA-SIP studies (54) and four of the five Chloroflexi
cellulose responders belong to a single clade within the Herpetosiphonales
although they share less than 89\% SSU gene sequence identity with their
closest cultured relative, H. geysericola. H. geysericola is a predatory
bacterium shown to prey upon Aerobacter in culture and it can also metabolize
cellulose (56). 

% Fakesubsubsection: Finally, a single
Finally, a single cellulose responder was identified from Melainobacteria (95\%
sequence identity to Vampirovibrio chlorellavorus). Although, the phylogenetic
position of Melainabacteria is debated (63), this non-phototrophic group is
closely associated with Cyanobacteria. Polysaccharide degradation is suggested
by an analysis of a “Melainabacteria” genome (62), however, Vampirovibrio
chlorellavorus is an obligate predator of green alga (Tsitologiia. 1972
Feb;14(2):256-60. Electron microscopic study of parasitism by Bdellovibrio
chlorellavorus bacteria on cells of the green alga Chlorella vulgaris). It is
interesting that cellulose responders within Chloroflexi and Melainobacteria
are related (though distantly) to known predators. The relatively high
Δ1.66-1.76 of these cellulose responders suggests that if they are predators
they are tightly associated with cellulose degrading microorganisms. Also,
these OTUs are not xylose responders and if they do have a predatory lifestyle,
our results  indicate they prey specifically on slow growing cellulolytic
bacteria and not the rapidly growing xylose responders. 

% Fakesubsubsection: We found that cellulose and xylose
We found that cellulose and xylose responders are phylogenetically clustered
and overdispersed, respectively. This suggests that the ability to degrade
cellulose is phylogenetically conserved possibly due to the degree of
biochemical complexity or the degree of ecological specialization required to
degrade cellulose in soil. The positive relationship between a physiological
trait’s phylogenetic depth and its complexity has been noted previously (25)
and the clade depth of cellulose responders, 0.028 SSU rRNA gene sequence
dissimilartiy, is on the same order as that observed for glycoside hydrolases
which are diagnostic enzymes for cellulose degradation (26). Overdispersion, as
observed for xylose responders is indicative of a broadly distributed trait
whose distribution could be affected by horizontal gene transfer or convergent
evolution, for example by catabolic recruitment of genes normally reserved for
anabolic functions. Overdispersion could also result if the xylose responders
represent multiple distinct functional groups which differ in their
relationship to xylose. 

% Fakesubsubsection: We diagnosed the assimilation
We diagnosed the assimilation of 13C into DNA but 13C-responders did not
necessarily assimilate 13C directly from 13C-xylose or 13C-cellulose. In many
ways, knowledge of secondary assimilation of 13C-label may be more interesting
with respect to the soil C-cycle than knowledge of primary assimilation. In
particular, results for the xylose responders suggest xylose C is moved through
different levels within the soil food web. The Bacilli degraded xylose first.
By day 1, 65\% of the xylose-C had already been respired, Bacilli represent
84\% of xylose responders, and comprise a remarkable 6\% of SSU rRNA genes
present in soil. However, Bacilli were generally no longer 13C-labeled by day
3 and their abundance declined reaching about 2\% of soil SSU rRNA genes by day
  30. These results indicate a group of organisms with a boom-bust lifestyle.
  Members of the Bacillus (27) and Paenibacillus in particular (28) have been
  previously implicated as labile C decomposers.  The decline in abundance of
  Bacilli is consistent with either mortality or sporulation and mother cell
  lysis and either way would represent a mechanism for 13C transfer from
  Bacilli to other soil microorganisms. Bacteroidetes are the most common
  responders at day 3. Bacteroidetes 13C- incorporation into DNA at day
  3 coincides with loss of isotopic label and decline in relative abundance of
  the Bacilli. Subsequently, Actinobacteria were the most common xylose
  responders at day 7. The abundance of Actinobacteria peaks at day
3 Actinobacteria 13C-incorporation into DNA at day 7 corresponds with loss of
label and decline in relative abundance of the Bacteroidetes. Hence, it seems
reasonable to propose that thetemporal dynamics we observe in 13C-assimilation
into DNA from 13C-xylose results from trophic C transfer and that Bacteroidetes
and Actinobacteria consumed 13C-labeled microbial biomass. 

% Fakesubsubsection: Trophic transfer could be due
Trophic transfer could be due to predation or saprotrophy. Further evidence for
C transfer by predation and/or saphrotrophy comes from the inferred physiology
of the Bacteroidetes and Actinobacteria xylose responders at days 3 and 7. Most
of the Actinobacteria xylose responders that appeared 13C labeled at day 7 are
members of the Micrococcales (Figure XX) and the most abundant Micrococcales
OTU that was labeled at day 7 (“OTU.4”, Table XX) is annotated as belonging in
the Agromyces. Agromyces are facultatively predatory soil bacteria that have
been shown to feed on the gram positive Luteobacter in culture (CITE Casida).
Additionally, soil Bacteroidetes have been implicated previously as predatory
bacteria in nucleic acid SIP studies with 13C labeled Eshericia coli (CITE
Lueders). .  However, it also remains possible that the Bacilli, Bacteroidetes,
and Actinobacteria are adapted to use xylose at different concentrations and
that the dynamics of assimilation result from changes in xylose concentration
over time. If the temporal dynamics we observe for the assimilation of
13C-xylose incorporation are due to trophic transfer, it indicates that the
carbon was exchanged between at least three different groups in 7 days. Models
of the soil C cycle rarely include trophic interactions between soil bacteria
(e.g. (35)). When soil C models do account for predators/saprophytes, trophic
interactions are predicted to have significant effects on the fate of soil
C (21). 

% Fakesubsubsection: The ecophysiology of a microooranism
The ecophysiology of a microorganism is often inferred from the distribution of
diagnostic genes across genomes (26) or from the physiology of isolates
cultivated on laboratory media (19). For instance, the wide distribution of the
glycolytic operon in microbial genomes is interpreted as evidence that many
soil microorganisms participate in glucose turnover (7). Ecological
characteristics, however, can determine which microbes use a gene from those
that have it. Xylose metabolism in soil, for instance, may depend less on the
distribution of the catabolic pathway across genomes and more on the life
history traits that allow organisms to compete effectively for xylose in the
soil. Phenomena such as seasonal change (30), and rainfall (24) cause nutrient
and resource concentrations in soil to fluctuate dynamically. Therefore, fast
growth and rapid resuscitation (24) allow microorganisms to favorably compete
for labile C which may often be transient in soil. Life history traits that
correlate with phylogeny, such as growth rate (22), may constrain the diversity
of microbes that are actually able to metabolize a given C source as it occurs
in the soil. DNA-SIP is useful for establishing in situ diversity of functional
guilds to contrast guild diversity inferred from genomic evidence and
laboratory experiments because DNA-SIP can target microbes that are active in
soil. 

\subsection{Implications for soil C cycling models}
% Fakesubsubsection: Models of soil
Models of soil C cycling rely on functional niches defined by soil
microbiologists. In these models ecological strategies such as growth rate
and substrate specificity are parameters for functional niche behavior
\citep{Kaiser2014a}. Land management, climate, pollution and disturbance can
influence soil community composition \citep{McGuire2010} which in turn
influences soil biogeochemical process rates (e.g. \citep{Berlemont2014a}).
Assessing functional group diversity and establishing identities of
functional group members is necessary to predict how biogeochemical process
rates will change with community composition
\citep{Schimel_1995,McGuire2010}. Aggregate biogeochemical processes that are
the sum of many subprocesses involve a broad array of taxa and are assumed to
be less influenced by community change than narrow processes that involve
a single, specific chemical transformation by a narrow suite of microbial
participants \citep{Schimel_1995,McGuire2010}. Within an aggregate process
such as C decomposition, subprocesses can be further classified as broad or
narrow \citep{McGuire2010}. In theory, the diversity of a functional guild
engaged in a specific C transformation is correlated to the lability of the
C at hand  \citep{McGuire2010}. However, the diversity of active labile C and
recalcitrant C decomposers in soil has not been directly quantified. Notably,
we found more OTUs responded to $^{13}$C-cellulose, 63, than $^{13}$C-xylose,
49. Also, it is possible that many $^{13}$C-xylose responders are predatory
bacteria or saprophytes as opposed to primary labile C degraders (see below).
Cellulose and xylose decomposer functional guilds were non-overlapping in
membership -- of 104 $^{13}$C-responders only 8 responded to both cellulose
and xylose -- and represented a small fraction of total soil community
diversity (Figure~\ref{fig:genspec}). While xylose use is undoubtedly more
widelydistributed among microbial genomes than the ability to degrade
cellulose, the number of unique active cellulose decomposer OTUs outnumbered
the number of unique active xylose utilizer OTUs. While cellulose responders
were phylogenetically clustered (NRI: XX) and xylose responders were
overdispersed (NRI: XX), it's not clear which $^{13}$C-xylose responsive
organisms were labeled as a result of primary xylose assimilation.
Therefore it's not clear if $^{13}$C-xylose responsive OTUs in this
experiment constitute a single ecologically meaningful group or multiple
ecological groups. Temporally defined $^{13}$C-xylose responder groups,
however, were phylogenetically coherent (Figure~\ref{fig:tiledtree},
Figure~\ref{fig:xyl_count}). For example, most day 1 $^{13}$C-xylose
responders are members of the \textit{Paenibacillus}. Both total OTU number
overall phylogenetic clustering could then be at odds with the assumed positive
relationship between substrate lability and functional guild diversity which
implicates labile and structural C could be similarly affected by changes in
community composition. 

% Fakesubsubsection: Broadly, we observed
Broadly, we observed labile C use by fast growing generalistsand structural
C use by slow growing specialists. These results agree with the MIMICS model
which evaluates the metabolism of metabolic and structural carbon by two
different microbial functional types characterized by copiotrophic and
oligotrophic growth strategies (Wieder 2014 Biogeosciences
11:3899-3917). The inclusion of these different functional types improved
significantly the prediction of C response to environmental change (Wieder 2014
Biogeosciences 11:3899-3917). We identified specific microbial members of
functional groups engaged in labile and structural C decomposition or broadly
speaking copiotrophs and oligotrophs.  Further, our results suggest the
presence of a microbial food web in soil and provide evidence for  trophic
exchange. If the temporal dynamics of xylose responders represent trophic
exchange (e.g. Firmicutes  Bacteroidetes  Actinobacteria), then C use
efficiency is regulated not just by primary metabolism but by trophic levels
where each C exchange between trophic levels is an opportunity for
C stabilization in association with soil minerals and/or C respiration.
Conclusion Microorganisms sequester atmospheric C and respire SOM influencing
climate change on a global scale but we do not know which microorganisms carry
out specific soil C transformations. In this experiment microbes from
physiologically uncharacterized but cosmopolitan soil lineages participated in
cellulose decomposition. Cellulose responders included members of the
Verrucomicrobia (Spartobacteria), Chloroflexi, Bacteroidetes and
Planctomycetes. Spartobacteria in particular are globally cosmopolitan soil
microorganisms and are often the most abundant Verrucomicrobia order in soil
(38). Fast-growing aerobic spore formers from Firmicutes assimilated labile
C in the form of xylose. Xylose responders within the Bacteroidetes and
Actinobacteria are likely to have assimilated xylose C as a result of dynamic
trophic exchange, mediated either by saprotrophy or predation. Our results
suggest that, cosmopolitan Spartobacteria may degrade cellulose on a global
scale, bacterial trophic interactions can significantly impact soil C cycling,
and ecological traits are likely to act as a filter that constrains the
diversity of microorganisms that are active in situ relative to those that have
the genomic capacity for a given process.

\subsection{Conclusion} 
% Fakesubsubsection: Microorganisms sequester atmospheric C
Microorganisms sequester atmospheric C and respire SOM influencing climate
change on a global scale but we do not know which microorganisms carry out
specific soil C transformations. In this experiment microbes from
physiologically uncharacterized but cosmopolitan soil lineages participated in
cellulose decomposition. Cellulose responders included members of the
Verrucomicrobia (Spartobacteria), Chloroflexi, Bacteroidetes and
Planctomycetes. Spartobacteria in particular are globally cosmopolitan soil
microorganisms and are often the most abundant Verrucomicrobia order in soil
(38). Fast-growing aerobic spore formers from Firmicutes assimilated labile
C in the form of xylose. Xylose responders within the Bacteroidetes and
Actinobacteria are likely to have assimilated xylose C as a result of dynamic
trophic exchange, mediated either by saprotrophy or predation. Our results
suggest that, cosmopolitan Spartobacteria may degrade cellulose on a global
scale, bacterial trophic interactions can significantly impact soil C cycling,
and ecological traits are likely to act as a filter that constrains the
diversity of microorganisms that are active in situ relative to those that have
the genomic capacity for a given process.
