\section{Discussion}
Nucleic-acid SIP coupled to microbiome fingerprinting techniques has progressed
from simple proof-of-concept experiments  CITE, to pilot studies utilizing
non-DNA-sequencing microbial community profiling methods such as DGGE CITE and
tRFLP CITE, and currently to large experiments employing multiple labeled
substrates and high-throughput amplicon and/or shotgun DNA sequencing
\citep{Verastegui_2014}. We present a high-resolution nucleic acid SIP (HR-SIP)
approach that expands upon classical nucleic acid SIP methods in three
dimensions: 1) temporally, we sample isotopically labeled substrate amended
microcosms at multiple time points; 2) spacially, we assay more fractions along
the CsCl gradients; and 3), bioinformatically, we interrogate taxa at the level
of OTU for isotope incorporation employing cutting edge statistics for
assessing differential abundance in microbiome datasets.

\subsection{Ordination of CsCl gradient fraction OTU profiles can be used to
observe fraction-level $^{13}$C assimilation dynamics and membership differences}
Each CsCl gradient fraction possesses a unique composition of SSU rRNA gene
phylogenetic types. DNA buoyant density (BD) drives differences in CsCl
gradient fraction SSU rRNA gene composition. For
instance, lighter DNA is more abundant in fractions at lighter densities so
DNA with lower G+C will be found in greater abundance at the light end of the
CsCl gradient and vice versa.  Duplicate gradients receiving only $^{12}$C DNA
with the same bulk or non-fractionated SSU rRNA gene phylogenetic composition 
would have the same overall profile of SSU rRNA gene phylogenetic types across 
the density gradient. We fed microcosms identical C substrate mixtures save 
for the identity of a $^{13}$C labeled substrate, and by design, DNA from all 
microcosms harvested at a time point will be similar in bulk phylogenetic 
composition. Therefore, SSU rRNA gene profile differences between between gradients 
harvested at the same time are due to $^{13}$C incorporation into bulk community DNA. 
$^{13}$C-DNA shifts from its $^{12}$C position towards the heavy end of the
density gradient. This causes heavy fractions in gradients that received
$^{13}$C-DNA to be different in phylogenetic content than corresponding heavy
fractions from gradients that received $^{12}$C-DNA of the same bulk
phylogenetic composition.

Ordination of CsCl gradient fraction phylogenetic profiles reveals differences
and similarities between gradients. It's clear that microcosms incorporated
$^{13}$C from both $^{13}$C-xylose and $^{13}$C-cellulose as gradients from
both $^{13}$C-xylose and $^{13}$C-cellulose microcosms differ from
corresponding control gradients (Figure~\ref{fig:ord}). These differences from
control gradients are focused in the heavy fractions (Figure~\ref{fig:ord}).
Analysis of SSU rRNA gene surveys has
greatly benefited from utilizing conventional methods for data exploration
in ecology such as ordination \citep{Lozupone_2008}.  SSU rRNA gene
phylogenetic profiles in CsCl gradient fractions have only recently been
surveyed with high-throughput DNA sequencing technology and subsequently
explored via ordination \citep{Angel_2013, Verastegui_2014}. Ordination of CsCl
gradient fraction phylogenetic profiles has reveled the relative influence of
buoyant density and soil type on gradient phylogenetic profile variance, 
however, ordination has not demonstrated isotope incorporation.  Demonstrating
isotope incorporation requires careful comparisons between control and labeled
gradients over the same buoyant density range. By sequencing CsCl gradient
fractions from both control and labeled gradients across the full density
gradient with DNA harvested from microcosms at multiple time points, we can
observe where in the density gradient $^{13}$C isotope incorporation signal is
strongest and when $^{13}$C isotope incorporation begins (Figure~\ref{fig:ord}).
$^{13}$C incorporation from xylose and cellulose is most apparent at days 1/3/7
and days 14/30, respectively (Figure~\ref{fig:ord}). Moreover, labeled gradient
fraction phylogenetic profiles diverge from controls most dramatically at
relatively heavy buoyant densities (Figure~\ref{fig:ord}). Also, $^{13}$C-DNA from $^{13}$C-xylose microcosms is different in phylogenetic composition from $^{13}$C-cellulose microcosm $^{13}$C-DNA indicating that xylose and cellulose were assimilated by different microbial community members (Figure~\ref{fig:ord}). Lastly, ordination indicates
organisms that assimilated $^{13}$C from $^{13}$C-xylose changed in phylogenetic type over incubation days 1, 3 and 7 (Figure~\ref{fig:ord}).

\textbf{Cellulose degraders identified from undescribed lineages and
cosmoplitan soil taxa for which functional attributes are not established}

\textit{Verrucomicrobia} are ubiquitous in soil worldwide
\citep{Bergmann_2011}.  \textit{Verrucomicrobia} can constitute 23\% of 16S
rRNA gene sequences in high-throughput DNA sequencing surveys of SSU rRNA genes
in soil \citep{Bergmann_2011} and have been shown to represent as high as 9.8\%
of soil 16S rRNA \citep{Buckley_2001}. Many \textit{Verrucomicrobia} cultivars
have been established in the last decade \cite{Wertz_2011} but only one of the
15 most abundant verrucomicrobial phylotypes in a global soil sample collection
shared greater than 93\% sequence identity with an isolate
\citep{Bergmann_2011}.  Genomic analyses and physiological profiling of
\textit{Verrucomicrobia} isolates have revealed methanotrophy and diazotrophy
\citep{Wertz_2011} within \textit{Verrucomicria} (CITE and reviewed by
\citet{Wertz_2011}). Notably, the genetic capacity to degrade cellulose and
cellulose degradation in culture have been demonstrated in
\textit{Verrucomicrobia} \citep{Otsuka_2012, Wertz_2011}.  Although, we have
learned many functional roles of \textit{Verrucomicrobia} in the environment,
the function and/or global significance of soil \textit{Verrucomicrobia} in
global C-cycling is unknown. For example, only one of the putative
verrucomicrobial cellulose degraders identified in this experiment are closely
related to named cultivars (OTU.XX, Table~\ref{tab:cell}) and only XX\% of all
verrucomicrobial OTUs found in this study share at lease 97\% sequence identity
with isolates. Seven of 10 $^{13}$C-cellulose responding verrucomicrobial OTUs were
classified belonging to the \textit{Spartobacteria} order.  \textit{Spartobacteria}
order was overwhelminly the numberically dominant order of \textit{Verrucomicrobia} in
SSU rRNA gene surveys of 181 globally distributed soil samples
\citep{Bergmann_2011}. HR-SIP identifies key players in soil C-cycling and
\textit{Verrucomicrobia} lineages particularly \textit{Spartobacteria}, given
their ubiquity and abundance in soil as well as their demonstrated
incorporation of $^{13}$C from $^{13}$C-celluose, may be significant players in
global soil cellulose respiration. 

is XX\% abundant in soil samples screen by the Earth Microbiom Project (EMP, CITE) and is
found in XX of XX EMP soil samples (XX\%) and XX of all XX EMP samples (FIGURE). 

\textit{Chloroflexi} Banfield paper

Xylose assimilators change over time. Implications for DNA-SIP. Succession within succession.

Response not consistent across phyla.

