\section{Discussion}
Nucleic-acid SIP coupled to microbiome fingerprinting techniques has progressed
from simple proof-of-concept experiments  CITE, to pilot studies utilizing
non-DNA-sequencing microbial community profiling methods such as DGGE CITE and
tRFLP CITE, and currently to large experiments employing multiple labeled
substrates and high-throughput amplicon and/or shotgun DNA sequencing
\citep{Verastegui_2014}. We present a high-resolution nucleic acid SIP (HR-SIP)
approach that expands upon classical nucleic acid SIP methods in three
dimensions: 1) temporally, we sample isotopically labeled substrate amended
microcosms at multiple time points; 2) spacially, we assay more fractions along
the CsCl gradients; and 3), bioinformatically, we interrogate taxa at the level
of OTU for isotope incorporation employing cutting edge statistics for
assessing differential abundance in microbiome datasets.

\textbf{Cellulose degraders identified from undescribed lineages and
cosmoplitan soil taxa for which functional attributes are not established}

\textit{Verrucomicrobia} are ubiquitous in soil worldwide
\citep{Bergmann_2011}.  \textit{Verrucomicrobia} can constitute 23\% of 16S
rRNA gene sequences in high-throughput DNA sequencing surveys of SSU rRNA genes
in soil \citep{Bergmann_2011} and have been shown to represent as high as 9.8\%
of soil 16S rRNA \citep{Buckley_2001}. Many \textit{Verrucomicrobia} cultivars
have been established in the last decade \cite{Wertz_2011} but only one of the
15 most abundant verrucomicrobial phylotypes in a global soil sample collection
shared greater than 93\% sequence identity with an isolate
\citep{Bergmann_2011}.  Genomic analyses and physiological profiling of
\textit{Verrucomicrobia} isolates have revealed methanotrophy and diazotrophy
\citep{Wertz_2011} within \textit{Verrucomicria} (CITE and reviewed by
\citet{Wertz_2011}). Notably, the genetic capacity to degrade cellulose and
cellulose degradation in culture have been demonstrated in
\textit{Verrucomicrobia} \citep{Otsuka_2012, Wertz_2011}.  Although, we have
learned many functional roles of \textit{Verrucomicrobia} in the environment,
the function and/or global significance of soil \textit{Verrucomicrobia} in
global C-cycling is unknown. For example, only one of the putative
verrucomicrobial cellulose degraders identified in this experiment are closely
related to named cultivars (OTU.XX, Table~\ref{tab:cell}) and only XX\% of all
verrucomicrobial OTUs found in this study share at lease 97\% sequence identity
with isolates. Seven of 10 $^{13}$C-cellulose responding verrucomicrobial OTUs were
classified belonging to the \textit{Spartobacteria} order.  \textit{Spartobacteria}
order was overwhelminly the numberically dominant order of \textit{Verrucomicrobia} in
SSU rRNA gene surveys of 181 globally distributed soil samples
\citep{Bergmann_2011}. HR-SIP identifies key players in soil C-cycling and
\textit{Verrucomicrobia} lineages particularly \textit{Spartobacteria}, given
their ubiquity and abundance in soil as well as their demonstrated
incorporation of $^{13}$C from $^{13}$C-celluose, may be significant players in
global soil cellulose respiration. 

is XX\% abundant in soil samples screen by the Earth Microbiom Project (EMP, CITE) and is
found in XX of XX EMP soil samples (XX\%) and XX of all XX EMP samples (FIGURE). 

\textit{Chloroflexi} Banfield paper

Xylose assimilators change over time. Implications for DNA-SIP. Succession within succession.

Response not consistent across phyla.

