

An estimated 80-90\% of C cycling in soil is mediated by microorganisms \cite{ColemanCrossley_1996,Nannipieri_2003}. Understanding microbial processing of nutrients in soils presents a special challenge due to the hetergeneous nature of soil ecosystems and methods limitations. Soils are biologically, chemically, and physically complex which affects microbial community composition, diversity, and structure \cite{Nannipieri_2003}. Confounding factors such as physical protection/aggregation, moisture content, pH, temperature, frequency and type of land disturbance, soil history, mineralogy, N quality and availability, and litter quality have all been shown to affect the ability of the soil microbial community to access and metabolize C substrates \cite{Sollins_Homann_Caldwell_1996,Kalbitz_2000}. Further, rates of metabolism are often measured without knowing the identity of the microbial species involved \cite{ndi_Pietramellara_Renella_2003} leaving the importance of community membership towards maintaining ecosystem functions unknown \cite{Allison_2008,ndi_Pietramellara_Renella_2003,Schimel_2012}. Litter bag experiments have shown that the community composition of soils can have quantitative and qualitative impacts on the breakdown of plant materials \cite{Schimel_1995}. Reciprocal exchange of litter type and microbial inocula under controlled environmental conditions reveals that differences in community composition can account for 85\% of the variation in litter carbon mineralization \cite{Strickland_2009}. In addition, assembled communities of cellulose degraders reveal that the composition of the community has significant impacts on the rate of cellulose degradation \cite{Wohl_2004}. 

An important step in understanding soil C cycling dynamics is to identify individual contributions of discrete microorganisms and to investigate the relationship between genetic diversity, community structure, and function \cite{O_Donnell_2002}. The vast majority of microorganisms continue to resist cultivation in the laboratory, and even when cultivation is achieved, the traits expressed by a microorganism in culture may not be representative of those expressed when in its natural habitat. Stable-isotope probing (SIP) provides a unique opportunity to link microbial identity to activity and has been utilized to expand our knowledge of a myriad of important biogeochemical processes \cite{Chen_Murrell_2010}. The most successful applications of this technique have identified organisms which mediate processes performed by a narrow set of functional guilds such as methanogens \cite{Lu_2005}. The technique has been less applicable to the study of soil C cycling because of limitations in resolving power as a result of simultaneous labeling of many different organisms in the community. Additionally, molecular applications such as TRFLP, DGGE, and cloning that are frequently used in conjunction with SIP provide insufficient resolution of taxon identity and depth of coverage. We have developed an approach that employs a complex mixture of substrates added to soil at a low concentration relative to soil organic matter pools along with massively parallel DNA sequencing. This greatly expands the ability of nucleic acid SIP to explore complex patterns of C-cycling in microbial communities with increased resolution. 

A temporal cascade occurs in natural microbial communities during the plant biomass degradation in which labile C degradation preceeds polymeric C \cite{Hu_1997,Rui_2009}. The aim of this study is to track the temporal dynamics of C assimilation through discrete individuals of the soil microbial community to provide greater insight into soil C-cycling. Our experimental approach employs the addition of a soil organic matter (SOM) simulant (a complex mixture of model carbon sources and inorganic nutrients common to plant biomass), where a single C constituent is substituted for its \textsuperscript{13}C-labeled equivalent, to soil. Parallel incubations of soils amended with this complex C mixture allows us to test how different C substrates cascade through discrete taxa within the soil microbial community. In this study we use \textsuperscript{13}C-xylose and \textsuperscript{13}C-cellulose as a proxy for labile and polymeric C, respectively. Using a novel approach we couple nucleic acid stable isotope probing with next generation sequencing (SIP-NGS) to elucidating soil microbial community members responsible for specific C transformations. Amplicon sequencing of 16S rRNA gene fragments from many gradient fractions and multiple gradients make it possible to track C assimilation by hundreds of different taxa. Ultimately we identify discrete microorganisms responsible for the cycling of specific C substrates. 

