\section{Introduction}
While a great deal of progress has been made in defining the flow of elements and energy through above-ground food webs, and progress has been made in characterizing detritivore food webs, we have only the most basic understanding of how matter and energy flows through microbial communities in soil. This deficiency is driven by the staggering complexity of soil microbial food webs and the opacity of these biological systems to current methods for describing the activities of soil microorganisms. Progress has been made in relating community composition to ecosystem function for certain microbial processes in soils, such as nitrification and denitrification, which are mediated by defined functional groups. Carbon-cycling processes, however, have proven more recalcitrant to study due to the wide range of organisms participating in these reactions.
 
There are 2,300 Pg of carbon (C) stored in soils worldwide, excluding plant biomass, which accounts for $\sim$80\% of the global terrestrial C pool \cite{Amundson_2001,Mendelsohn_2001,IPCC2007Synth,elsen_Ayres_Wall_Bardgett_2011,Lal_2008,BATJES_1996,Lal_2008}. When organic C from plants reaches soil it is degraded through the activity of fungi and bacteria. This C may be rapidly returned to the atmosphere as CO\textsubscript{2} or may remain in the soil as humic substances that may persist up to 2000 years \cite{yanagita1990natural}. The majority of C is respired and on an annual basis soil respiration produces 10 times more CO\textsubscript{2} than anthropogenic emissions \cite{chapin2002principles}. Global changes in atmospheric CO\textsubscript{2}, temperature, and ecosystem nitrogen inputs, are expected to impact primary production and C inputs to soils \cite{Groenigen_2006}, but it remains difficult to predict the response of soil processes to anthropogenic change \cite{DAVIDSON_2006}. Current climate change models concur on atmospheric and ocean C predictions but not terrestrial \cite{Friedlingstein_2006}. The disagreeable predictive power between models for terrestrial ecosystems reflects how little we know about belowground C cycling dynamics. 

It is estimated that 80-90\% of the C cycling in soil is mediated by microorganisms \cite{ColemanCrossley_1996,Nannipieri_2003}. Understanding microbial processing of nutrients in soils presents a special challenge due to the hetergeneous nature of soil ecosystems and our limitations in methodologies. Soils consist of an overwhelming biological, chemical, and physical complexity which affects microbial community composition, diversity, and structure (refs). Confounding factors such as physical protection/aggregation, moisture content, pH, temperature, frequency and type of land disturbance, soil history, mineralogy, N quality and availability, and litter quality have all been shown to affect the ability of the soil microbial community to access and metabolize C substrates \cite{Schlesinger_1977,dgett_Wall_Hattenschwiler_2010,Sollins_Homann_Caldwell_1996,Torn_Vitousek_Trumbore_2005,TRUMBORE_2006,Schimel_2012}. Furthermore, rates of metabolism are often measured without knowing the identity of the microbial species specifically involved \cite{ndi_Pietramellara_Renella_2003} resulting in uncertainty in importance of community diversity in maintaining ecosystem functioning \cite{Allison_2008,ndi_Pietramellara_Renella_2003,Schimel_2012}. Litter bag experiments have shown that the community composition of soils can have quantitative and qualitative impacts on the breakdown of plant materials \cite{Schimel_1995}, and reciprocal exchange of litter type and microbial inocula under controlled environmental conditions reveals that differences in community composition can account for 85\% of the variation in litter carbon mineralization \cite{Strickland_2009}. In addition, simple assembled communities of cellulose degraders reveal that the composition of the community has significant impacts on the rate of cellulose degradation \cite{Wohl_2004}. 

The first step in teasing out belowground C cycling dynamics is to identify discrete microbial contributions responsible for the measured process and understand the relationship between genetic diversity, community structure, and function \cite{O_Donnell_2002}. The vast majority of microorganisms continue to resist cultivation in the laboratory, and even when cultivation can be achieved, the traits expressed by a microorganism in culture may not be representative of those expressed when the organism is present in its natural habitat. Stable-isotope probing (SIP) provides a unique opportunity to link microbial identity to activity and has been utilized to expand our knowledge of a myriad of important biogeochemical processes and elucidate microbial activities \cite{Chen_Murrell_2010}. The most successful applications of this technique have been to identify organisms which mediate processes performed by a narrow set of organisms (ie: methanotrophs, methanogens, syntrophs, etc). The technique has been less applicable to the study of the soil C-cycle because of limitations in its resolving power as a result of simultaneous labeling of many different organisms in the community. Additionally, frequently utilized SIP-coupled molecular applications such as TRFLP, DGGE, and cloning which provide insufficient resolution of taxon identity and depth of coverage. The use of massively parallel sequencing should effectively overcome this barrier and should greatly expand the ability of nucleic acid SIP to explore complex patterns of C-cycling in microbial communities. Yet, there remain notable limitations including the need to add substrate at concentrations higher than typical in situ \cite{radajewski2000stable}, the potential for partial labeling due to label dilution \cite{radajewski2000stable,Manefield_2002,McDonald_2005}, the potential for cross-feeding and trophic cascades \cite{Morris_2002,Hutchens_2003,Lueders_2003,DeRito_2005,Mahmood_2005,McDonald_2005,Ziegler_2005} and variation in genome G+C content which causes variation in native DNA buoyant density \cite{Buckley_2007,9780408708036,HOLBEN_1995,9546163}. Several strategies have been developed to deal with these issues and each requires the collection and analysis of density gradient fractions in order to determine the degree of isotope incorporation into nucleic acids from particular microbial groups \cite{Manefield_2002,Manefield_2002,14686943}. SIP studies use single substrate experimental designs, with few exceptions \cite{Lueders_2003,Chauhan_2009}, to minimize isotope signal dilution which detracts from how microbes may experience that substrate naturally, calling into question its environmental and biological relevance.

C in plant biomass is found primarily in cellulose (30-50\%), hemicellulose (20-40\%), and lignin (15-25\%) \cite{Lynd_2002}. Hemicellulose is composed mostly of xylose with varying lesser amounts of arabinose, galactose, glucose, mannose, and rhamnose. The deposition of plant litter is hypothesized to promote a process of succession within the soil community as different plant compounds are degraded sequentially by organisms having different ecological strategies. Hemicellulose and simple sugars are the targets of initial stages of decomposition with degradation of polymeric C lagging. Microbes that respire sugars increase in abundance dramatically during the initial stages of decomposition \cite{GARRETT_1951,Alexander_1964}, and as much as 75\% of sugar C can be respired or converted into microbial biomass in the first 5 days of decomposition \cite{Engelking_2007}. Cellulose degraders take longer to respond \cite{Hu_1997} with less than 42\% of cellulose metabolized over the first 5 days of incubation \cite{Engelking_2007}. Evidence to support this succession hypothesis generally comes from observing the dynamics of CO\textsubscript{2} evolution and from observations of organisms that can be isolated from soils and litter at different stages of decomposition. Other studies have detected differences in the degradation of labile C and polymeric C substrates \cite{Engelking_2007,Anderson_1973,Stotzky_1961,Alden_2001,Furukawa_1996,Fontaine_2003,Blagodatskaya_2007,Jenkins_2010,Rui_2009,Fierer_2010}. A classic example of plant litter degradative succession is characterized by a series of stages in which sugar fungi dominate in stage one, followed by cellulolytic fungi in stage two, and lignin degrading fungi in the final stage \cite{Gessner_2010}. These studies suggest that if a complex mixture of labile and polymeric C were added to soil two waves of degradation could be observed; labile C degradation early on and subsequent polymeric C degradation. The degree to which the succession hypothesis presents an accurate model of litter decomposition has been called into question \cite{Kj_ller_1982,Frankland_1998,Osono_2005} and it seems clear that new approaches are needed to dissect soil processes and reveal the manner in which C is transformed by the soil food web.  
 





  

We propose this temporal cascade from labile C degraders preceeding the polymeric C degraders occurs in natural microbial communities, called herein 'microbial community succession'.
The aim of this study is to track the path of C added to soil as a complex C mixture to provide insight into these dynamic systems. A previous study has shown that \textsuperscript{13}C labeled plant residues enable tracking of C through microbial pathways \cite{Evershed_2006}. Utilizing this technique with single \textsuperscript{13}C labeled substrates added as a complex C mixture allows us to test how different C substrates cascade through discrete taxa within the soil microbial community. Powerful techniques such as nucleic acid stable isotope probing coupled with next generation sequencing (SIP-NGS) can then be used to parse out and identify these \textsuperscript{13}C labeled portions of the microbial community to reveal the community members that are responsible for the transformation of the labeled C. Using these methods in compilation with microcosm incubations provides a means of looking into microbial community functions while minimizing other environmental factors affecting the fate of C. Ultimately we identify discrete organisms or functional guilds that are responsible for the cycling of specific C substrates. 
It is important to understand these decomposition sequences because as environments are altered it results in measurable and functional changes in soil C \cite{Grandy_2008} which could cumulatively have large impacts at a global scale. 

and, to our knowledge, are usually conducted under the narrow scope of single substrate additions .



Tag sequencing can be used to sequence 16S rRNA gene fragments from many different gradient fractions and multiple gradients on a single machine run obtaining sequencing depth of thousands of sequences per gradient fraction. This will make it possible to follow carbon assimilation simultaneously in hundreds of different taxa.
Our experimental approach employs the addition of a complex soil organic matter simulant (a mixture of model carbon sources and inorganic nutrients common to plant biomass) to soil. The design of the SOM simulant allows for different components to be exchanged with their 13C-labeled alternatives. Soils are then incubated in parallel with addition of unlabeled SOM or different SOM mixtures in which a different individual components are 13C-labeled. Following incubation, DNA is extracted and subjected to density gradient ultracentrifugation and fractionation into gradient fractions of discrete buoyant density. DNA from bacteria in distinct gradient fractions is amplified by PCR using primers with distinct tag sequences. DNA from >200 fractions can then be run simultaneously in one sequencing reaction and deconvoluted based on their primer tags (as described in (Hamady et al. 2008) and below). As a result, by examining nucleic acids at different times it will be possible to track 13C from model substrates into the nucleic acids of different microbial groups, with the density of nucleic acids increasing over time in proportion to the amount of labeled substrate incorporated. This will provide an unprecedented view of how different types of C compounds move through the soil microbial food web over time.