\section{Introduction}
% Fakesubsubsection:Excluding plant biomass, there are 2,300 Pg of
Soils worldwide contain 2,300 Pg of carbon (C) which accounts for nearly~80\%
of the C present in the terrestrial biosphere
\citep{Amundson_2001,BATJES_1996}. Soil microorganisms drive C flux through the
terrestrial biosphere and C respiration by soil microorganisms
produces annually tenfold more CO$_{2}$ than fossil fuel emissions
\citep{chapin2002principles}. Despite the contribution of microorganisms to
global C flux, many global C models ignore the diversity of microbial
physiology \citep{Allison2010,Six2006,Treseder2011} and we still know little
about the ecophysiology of soil microorganisms. Characterizing the
ecophysiology of microbes that mediate C decomposition in soil has proven
difficult due to their overwhelming diversity. Such knowledge should assist
the development and refinement of global C models
\citep{Bradford2008,Neff_2001,McGuire2010,Wieder2013}.

% Fakesubsubsection:Though microorganisms mediate
Though microorganisms mediate 80-90\% of the soil C-cycle
\citep{ColemanCrossley_1996,Nannipieri_2003}, and microbial community
composition can account for significant variation in C mineralization
\citep{Strickland_2009}, terrestrial C-cycle models rarely consider the
community composition of soils \citep{Zak2006,Reed2007}. Variation in microbial
community composition can be linked effectively to rates of soil processes when
diagnostic genes for specific functions are available (e.g. nitrogen fixation
\citep{Hsu2009}).  However, the lack of diagnostic genes for describing soil-C
transformations has limited progress in characterizing the contributions of
individual microorganisms to decomposition. Remarkably, we still lack basic
information on the physiology and ecology of the majority of organisms that
live in soils. For example, contributions to soil processes remain
uncharacterized for cosmopolitan bacterial phyla in soil such as
\textit{Acidobacteria}, \textit{Chloroflexi}, \textit{Planctomycetes}, and
\textit{Verrucomicrobia}. These phyla combined can comprise 32\% of soil
microbial communities (based on surveys of the SSU rRNA genes in soil)
\citep{Janssen2006,Buckley2002}. 

% Fakesubsubsection: Functional niche characterization
In addition to understanding the C-cycling roles of specific microbial taxa in
soil, characterizing the functional niches soil microorganisms is necessary to
predict whether and how biogeochemical processes vary with microbial community
composition. Functional niches are defined by soil microbiologists and have
been successfully incorporated into biogeochemical process models (E.g.
\citep{wieder_2014a,Kaiser2014a}). In some C models ecological strategies such
as growth rate and substrate specificity are parameters for functional niche
behavior \citep{Kaiser2014a}. The phylogenetic breadth of a functionally
defined group is often inferred from the distribution of diagnostic genes
across genomes \citep{Berlemont2013} or from the physiology of isolates
cultured on laboratory media \citep{Martiny2013}. For instance, the wide
distribution of the glycolysis operon in microbial genomes is interpreted as
evidence that many soil microorganisms participate in glucose turnover
\citep{McGuire2010}. However, the functional niche may depend less on the
distribution of diagnostic genes across genomes and more on life history traits
that allow organisms to compete for a given substrate as it occurs in the soil.
For instance, fast growth and rapid resuscitation allow microorganisms to
compete for labile C which may often be transient in soil. Hence, life history
traits may constrain the diversity of microbes that metabolize a given C source
in the soil under a given set of conditions. Therefore, it is important to
contrast characterizations of microbial traits from genomic and/or culture
based studies with observations of active microorganisms in microcosms and
under \textit{in situ} conditions to fully understand the breadth of 
functional niches in soil.

% Fakesubsubsection:Nucleic acid SIP
Nucleic acid stable-isotope probing (SIP) links genetic identity and activity
without the need diagnostic genetic markers or cultivation and has expanded our
knowledge of microbial processes
\citep{Chen_Murrell_2010}. Nucleic acid SIP has notable complications, however,
including the need to add large amounts of labeled substrate
\citep{radajewski2000stable}, label dilution resulting in partial labeling of
nucleic acids \citep{radajewski2000stable}, the potential for cross-feeding and
secondary label incorporation \citep{DeRito2005}, and variation in genome G$+$C
content \citep{Buckley_2007}.  As a result, most applications of SIP have
targeted specialized microorganisms (for instance, methylotrophs
\citep{lueders2004b}, syntrophs \citep{lueders2004}, or microorganisms that
target pollutants \citep{derito2005}). Exploring the soil-C cycle with SIP has
proven to be more challenging because SIP has lacked the resolution necessary
to characterize the specific contributions of individual microbial groups to
the decomposition of plant biomass. High throughput DNA sequencing technology,
however, improves the resolving power of SIP \citep{Aoyagi2015}. It is now
possible to use far less isotopically labeled substrate resulting in more
environmentally realistic experimental conditions. It is also possible to
sequence rRNA genes from numerous density gradient fractions across multiple
samples thereby increasing the resolution of a typical nucleic acid SIP
experiment \citep{Verastegui_2014}. With this improved resolution the activity
of more soil microorganisms can be assessed. Further, since microbial
activities can be more comprehensively assessed, we can begin to determine the
ecological properties of functional groups defined by a specific activity in a
DNA-SIP experiment. We have employed such a high resolution DNA stable isotope
probing approach to explore the assimilation of both xylose and cellulose into
bacterial DNA in an agricultural soil. 

% Fakesubsubsection: We add a complex amendment
We added to soil a complex amendment representative of organic
matter derived from fresh plant biomass. All treatments received the same
amendment but the identity of isotopically labeled substrates was varied
between treatments. Specifically, we set up a control treatment where all
components were unlabeled, a treatment with $^{13}$C-xylose instead of
unlabeled xylose, and a treatment with $^{13}$C-cellulose instead of unlabeled
cellulose. Soil was sampled at days~1,~3,~7,~14,~and~30 and we identified
microorganisms that assimilated $^{13}$C into DNA at each point in time. We
designed the experiment to test of the degradative succession
hypothesis as it applies to soil bacteria, to identify soil bacteria that
metabolize xylose and cellulose, and to characterize temporal dynamics of
xylose and cellulose metabolism in soil. 
