\section{Introduction}
% Fakesubsubsection:Excluding plant biomass, there are 2,300 Pg of
Soils worldwide contain 2,300 Pg of carbon (C) which accounts for nearlyâ€„80\%
of the C present in the terrestrial biosphere
\citep{Amundson_2001,BATJES_1996}. Microbial soil C respiration produces
annually tenfold more CO$_{2}$ than fossil fuel emissions
\citep{chapin2002principles} yet many global C biogeochemical flux models
ignore microbial physiological diversity and its impacts on microbial activity
in soils despite the significant contribution of microbial activity to global
soil C flux. \citep{Allison2010,Six2006,Treseder2011}. It remains difficult to
predict the response of soil C to changes in climate
\citep{Nett_2001,Davidson2006a,Todd-Brown2013} but predictions of climate
change feedbacks on soil C flux improve when biogeochemical models explicitly
represent microbial physiology \citep{Wieder2013}. However, we still know
little of the ecophysiology of soil microorganisms. Efforts that describe soil
microorganism ecophysiology relevant to soil C biogeochemical fluxes should
assist the development and refinement of terrestrial C biogeochemical models
\citep{Bradford2008,Neff_2001,McGuire2010}.

%Fakesubsubsection:The decomposition
Cellulose comprises most plant C (30-50\%) followed by hemicellulose (20-40\%),
and lignin (15-25\%) \citep{Lynd2002}. Hemicellulose, being the most soluble,
degrades most easily as compared to cellulose and lignin, and is targeted in
the early stages of decomposition. Hemicellulose composition varies
considerably with xylans being the most abundant constituent, themselves
composed of differing amounts of xylose, glucose, arabinose, galactose,
mannose, and rhamnose \citep{Saha2003}. Xylose is often the most abundant sugar
in hemicellulose, comprising as much as 60-90\% of xylan in some plants (e.g
hardwoods) \citep{Spiridon2008}, wheat \citep{Sun2005}, and switchgrass
\citep{Bunnell2013}. Microbes that respire sugars proliferate during the
initial stages of decomposition \citep{Garrett1951,Alexander1964}, and
metabolize as much as 75\% of sugar C during the first 5 days of decomposition
\citep{Engelking2007}. In contrast, cellulose degradation rates increase slowly
for approximately 15 days and cellulose degradation continues for 30-90 days
\citep{Hu1997,Engelking2007}. It is hypothesized that microbes specialize in
the decomposition of specific compounds in fresh, plant-derived organic matter
and that these functionally specific guilds of microorganisms proliferate in
succession as plant C of decreasing lability is decomposed over time
\citep{Hu1997,Rui2009,AnneliseHKjoller2002,Bastian2009}. For instance, this
degradative succession hypothesis posits that rapidly growing plant sugar
decomposers respond quickly \citep{Garrett1963,Bremer1994} and are followed by
slow growing degraders of plant polymers \citep{Garrett1963}. Evidence to
support the degradative succession hypothesis comes from observing soil
respiration dynamics and characterizing microbes cultivated at different stages
of decomposition. The degree to which the succession hypothesis presents an
accurate model of litter decomposition has been called into question
\citep{AnneliseHKjoller2002,Frankland1998,osono2005} and it's clear that new
approaches are needed to dissect microbial contributions to C transformations
in soils.

% Fakesubsubsection:An important step in understanding soil C cycling
Functional guild membership and diversity establish the connections between
soil functions and community structure \citep{O_Donnell_2002}. Microorganisms
mediate an estimated 80-90\% of soil C cycling
\citep{ColemanCrossley_1996,Nannipieri_2003} but the complexity of soil
obfuscates microbial contributions to soil C cycling and the majority of
microorganisms resist cultivation in the laboratory. Stable-isotope probing
(SIP), however, links genetic identity and activity without cultivation and has
expanded our knowledge of microbial contributions to biogeochemical processes
\citep{Chen_Murrell_2010}. Successful applications of SIP have identified
organisms which mediate processes performed by functionally specialized
microorganisms of limited diversity such as methanogens \citep{Lu_2005} but SIP
has been less applicable in soil C cycling studies because simultaneous
labeling of many different organisms requires prohibitive resolving power to
successfully employ SIP. High throughput DNA sequencing technology, however,
improves the resolving power of SIP enabling exploration of complex soil
C-cycling processes.

% Fakesubsubsection:Though microorganisms mediate
Though microorganisms mediate 80-90\% of the soil C-cycle
\citep{ColemanCrossley_1996,Nannipieri_2003}, and the physiological ecology of
microorganisms can influence the dynamics of C transformations, terrestrial
C-cycle models rarely consider the community composition of soils
\citep{Zak2006,Reed2007}. Metabolic rates of soil C transformations are
measured without knowledge of the organisms that mediate these reactions
\citep{Nannipieri_2003}, leaving undefined the importance of community
membership towards maintaining ecosystem function
\citep{Nannipieri_2003,Schimel_2012,Allison_2008}. However, microbial community
composition can account for significant variation in litter C mineralization
\citep{Strickland_2009}. Variation in microbial community composition can be
linked most directly to rates of soil processes when discrete functional genes
are available to represent those processes (e.g. denitrification
\citep{Cavigelli2000}, nitrification \citep{Carney2004,Hawkes2005,Webster2005},
methanotrophy \citep{Gulledge1997}, and nitrogen fixation \citep{Hsu2009}).
However, the complexity of soil C transformations and the lack of convenient
functional genes for describing these transformations has limited progress in
characterizing the contributions of individual microbes to the soil C-cycle.
Remarkably, we still lack information on the physiology and ecology of the
majority of organisms that live in soils. For example, the bacterial phyla
Acidobacteria, Chloroflexi, Planctomycetes, and Verrucomicrobia are nearly
ubiquitous in soils, they comprise collectively up to 32\% of the SSU rRNA
genes in soil \citep{Janssen2006,Buckley2002}, but their contributions to soil
processes remain almost completely uncharacterized. 

% Fakesubsubsection:Stable-isotope probing of nucleic acids
Linking genetic identity to function in microbial ecosystems has historically
relied on microbial cultivation and surveying diagnostic marker genes, but,
most microorganisms resist cultivation in the lab by traditional methods
(Jannssen) and many processes lack suitable diagnostic genetic markers.
Additionally, laboratory cultivation rarely mimics environmental and ecological
conditions and diagnostic genes only convey the genomic potential for
a function as opposed to \textit{in situ} activity. Stable-isotope probing of
nucleic acids (SIP) links the genetic identity of microbes to specific
processes bypassing the need for diagnostic genetic markers or microbial
cultivation \citep{Chen_Murrell_2010}. However, nucleic acid SIP has notable
complications including the need to add large amounts of labeled substrate
\citep{radajewski2000stable}, label dilution resulting in partial labelling of
nucleic acids \citep{radajewski2000stable,Manefield_2002,McDonald_2005}, the
potential for cross-feeding and secondary label incorporation
\citep{Morris_2002,Hutchens2004,14686943,DeRito2005,McDonald_2005,Ziegler_2005},
and variation in genome G$+$C content
\citep{Buckley_2007,9780408708036,Holben1995,Nusslein1999}. As a result, the
most successful applications of SIP have identified functionally specialized
microbial functional guilds of limited diversity (e.g. methanotrophs
\citep{radajewski2000stable}) and the nucleic acid SIP has previously been
insufficient to follow isotopic labeling of nucleic acids from many microbes as
would be required for analysis of transformations in the soil C-cycle. 

% Fakesubsubsection:High throughput sequencing
High throughput DNA sequencing can be used to enable exploration of major
C-cycling processes in soils with nucleic acid SIP. SSU rRNA amplicons can now
readily be sequenced from numerous density gradient fractions across multiple
samples thereby increasing the resolution of a typical nucleic acid SIP
experiment \citep{Verastegui_2014}. As a consequence of this increased
resolution, it is also possible to use far less isotopically labeled
substrate resulting in conditions that are more environmentally realistic
\citep{Aoyagi2015}. We have employed such a high resolution DNA
stable isotope probing approach to explore the incorporation of $^{13}$C
labeled xylose and/or glucose into DNA by bacterial taxa over time in an
agricultural soil. Specifically, we added a mixture of nutrients and
C substrates to soil microcosms that simulated the composition of fresh organic
matter from plant biomass. All microcosms received the same C substrate mixture
where the only difference between treatments was the identity of the
isotopically labeled substrate. We added the C substrate mixture to three
series of microcosms: in one series xylose was substituted for its
$^{13}$C-equivalent, in another cellulose was substituted for its
$^{13}$C-equivalent, and in the third treatment all substrates in the mixture
were unlabeled. We harvested microcosms from each treatment at days 3, 7, 14,
and 30 and additionally harvested microcosms receiving $^{13}$C-xylose and
unlabeled substrates on day 1. The experiment was designed to provide a test of
the degradative succession hypothesis as applied to soil bacteria, to identify
the bacteria that metabolize xylose and cellulose in soils, and to characterize
the temporal dynamics of xylose and cellulose metabolism during the degradation
of fresh organic matter. 
