\section{Introduction}
% Fakesubsubsection:Excluding plant biomass, there are 2,300 Pg of
Soils worldwide contain 2,300 Pg of carbon (C) which accounts for nearly~80\%
of the C present in the terrestrial biosphere
\citep{Amundson_2001,BATJES_1996}. C respiration by soil microorganisms
produces annually tenfold more CO$_{2}$ than fossil fuel emissions
\citep{chapin2002principles}. Despite the contribution of microorganisms to
global C flux, many global C models ignore the diversity of microbial
physiology \citep{Allison2010,Six2006,Treseder2011} and we still know little
about the ecophysiology of soil microorganisms. Such knowledge should
assist the development and refinement of global C models
\citep{Bradford2008,Neff_2001,McGuire2010,Wieder2013}.

% Fakesubsubsection: Most plant C
Most plant C is comprised of cellulose (30-50\%) followed by hemicellulose
(20-40\%), and lignin (15-25\%) \citep{Lynd2002}. Hemicellulose, being the most
soluble, degrades in the early stages of decomposition. Xylans are often an
abundant component of hemicellulose, and xylans include differing
amounts of xylose, glucose, arabinose, galactose, mannose, and rhamnose
\citep{Saha2003}. Xylose is often the most abundant sugar in hemicellulose,
comprising as much as 60-90\% of xylan in some plants (e.g hardwoods
\citep{Spiridon2008}, wheat \citep{Sun2005}, and switchgrass
\citep{Bunnell2013}). Microorganisms that respire labile C in the form of sugars
proliferate during the initial stages of decomposition
\citep{Garrett1951,Alexander1964}, and metabolize as much as 75\% of sugar
C during the first 5 days \citep{Engelking2007}. In contrast,
cellulose decomposition proceeds more slowly with rates increasing for
approximately 15~days while degradation continues for 30-90~days
\citep{Hu1997,Engelking2007}. It is hypothesized that different microbial
guilds mediate the decomposition of different plant biomass components
\citep{Hu1997,Rui2009,AnneliseHKjoller2002,Bastian_2009}. The degradative
succession hypothesis posits that fast growing organisms proliferate in
response to the labile fraction of plant biomass such as sugars
\citep{Garrett1963,Bremer1994} followed by slow growing organisms targeting
structural C such as cellulose \citep{Garrett1963}. Evidence to support the
degradative succession hypothesis comes from observing soil respiration
dynamics and characterizing microorganisms cultured at different stages of
decomposition. The degree to which the succession hypothesis presents an
accurate model of litter decomposition has been questioned
\citep{AnneliseHKjoller2002,Frankland_1998,Osono_2005} and it's clear that we
need new approaches to dissect microbial contributions to C transformations in
soils.

% Fakesubsubsection:Though microorganisms mediate
Though microorganisms mediate 80-90\% of the soil C-cycle
\citep{ColemanCrossley_1996,Nannipieri_2003}, and microbial community
composition can account for significant variation in C mineralization
\citep{Strickland_2009}, terrestrial C-cycle models rarely consider the
community composition of soils \citep{Zak2006,Reed2007}. Rates of soil
C transformations are measured without knowledge of the organisms that mediate
these reactions \citep{Nannipieri_2003} leaving the importance of community
membership towards maintaining ecosystem function undefined
\citep{Nannipieri_2003,Schimel_2012,Allison_2008}. Variation in microbial
community composition can be linked effectively to rates of soil processes when
diagnostic genes for specific functions are available (e.g. denitrification
\citep{Cavigelli2000}, nitrification \citep{Carney2004,Hawkes2005,Webster2005},
methanotrophy \citep{Gulledge1997}, and nitrogen fixation \citep{Hsu2009}).
However, the lack of diagnostic genes for describing soil-C transformations has
limited progress in characterizing the contributions of individual microorganisms to
decomposition. Remarkably, we still lack basic information on the physiology
and ecology of the majority of organisms that live in soils. For example,
contributions to soil processes remain uncharacterized for 
cosmopolitan bacterial phyla in soil such as \textit{Acidobacteria},
\textit{Chloroflexi}, \textit{Planctomycetes}, and \textit{Verrucomicrobia}.
These phyla combined can comprise 32\% of soil microbial communities (based on
surveys of the SSU rRNA genes in soil) \citep{Janssen2006,Buckley2002}. 

% Fakesubsubsection:Characterizing the functions
Characterizing the functions of microbial taxa has relied historically on
culturing microorganisms and subsequently characterizing their physiology in
the laboratory, and on environmental surveys of genes diagnostic for specific
processes. But, most microorganisms are difficult to grow in culture
\citep{Janssen2006} and many biogeochemical processes lack suitable diagnostic
genes. Nucleic acid stable-isotope probing (SIP) links genetic identity and
activity without the need to grow microorganisms in culture and has expanded
our knowledge of microbial contributions to biogeochemical processes
\citep{Chen_Murrell_2010}. However, nucleic acid SIP has notable complications
including the need to add large amounts of labeled substrate
\citep{radajewski2000stable}, label dilution resulting in partial labeling of
nucleic acids \citep{radajewski2000stable,Manefield_2002,McDonald_2005}, the
potential for cross-feeding and secondary label incorporation
\citep{Morris_2002,Hutchens2004,14686943,DeRito2005,McDonald_2005,Ziegler_2005},
and variation in genome G$+$C content
\citep{Buckley_2007,9780408708036,Holben1995,Nusslein1999}. As a result, most
applications of SIP have targeted specialized microorganisms such as
methanotrophs \citep{radajewski2000stable}, methanogens \citep{lu2005},
syntrophs \citep{lueders2004}, or microorganisms that target pollutants
\citep{derito2005}. Exploring the soil-C cycle with SIP has proven to be more
challenging because SIP has lacked the resolution necessary to characterize the
specific contributions of individual microbial groups to the decomposition of
plant biomass. High throughput DNA sequencing technology, however, improves the
resolving power of SIP \citep{Aoyagi2015}. 

% Fakesubsubsection:High throughput sequencing
Coupling SIP with high throughput DNA sequencing now enables exploration of
microbial C-cycling in soils. SSU rRNA amplicons are readily sequenced from
numerous density gradient fractions across multiple samples thereby increasing
the resolution of a typical nucleic acid SIP experiment
\citep{Verastegui_2014}. It is now possible to use far less isotopically
labeled substrate resulting in more environmentally realistic experimental
conditions \citep{Aoyagi2015}. We have employed such a high resolution DNA
stable isotope probing approach to explore the assimilation of 
xylose and/or cellulose into bacterial DNA in an agricultural soil. 

% Fakesubsubsection: Specifically, we 
Specifically, we added to soil a complex amendment representative of organic
matter derived from fresh plant biomass. All treatments received the same
amendment but the identity of isotopically labeled substrates was varied
between treatments. Specifically, we set up a control treatment where all
components were unlabeled, a treatment with $^{13}$C-xylose instead of
unlabeled xylose, and a treatment with $^{13}$C-cellulose instead of unlabeled
cellulose. Soil was sampled at days~1,~3,~7,~14,~and~30 and we identified
microorganisms that assimilated $^{13}$C into DNA at each point in time. The
experiment was designed to provide a test of the degradative succession
hypothesis as it applies to soil bacteria, to identify soil bacteria that
metabolize xylose and cellulose, and to characterize temporal dynamics of
xylose and cellulose metabolism in soil. 
