\section{Introduction}
Although progress towards defining and characterizing element and energy flow through above-ground food webs and detritivore food webs has been made CITATIONS, we have only a rudimentary understanding of how matter and energy flows through soil microbial communities. This deficiency is driven by the staggering complexity of soil microbial food webs and the opacity of these biological systems to current methods for describing soil microorganism activities. Relating community composition to ecosystem function for certain microbial processes in soils, such as nitrification and denitrification, which are mediated by defined functional groups has been a useful approach. However, carbon-cycling processes have proven more recalcitrant to study due to the wide range of organisms participating in these reactions.
 
Excluding plant biomass, there are 2,300 Pg of carbon (C) stored in soils worldwide which accounts for $\sim$80\% of the global terrestrial C pool \cite{Amundson_2001,Mendelsohn_2001,IPCC2007Synth,elsen_Ayres_Wall_Bardgett_2011,Lal_2008,BATJES_1996,Lal_2008}. When organic C from plants reaches soil it is degraded by fungi, archaea and bacteria. This C is rapidly returned to the atmosphere as CO\textsubscript{2} or remains in the soil as humic substances that can persist up to 2000 years \cite{yanagita1990natural}. The majority of plant biomass C is respired and, on an annual basis, soil respiration produces 10 times more CO\textsubscript{2} than anthropogenic emissions \cite{chapin2002principles}. Global changes in atmospheric CO\textsubscript{2}, temperature, and ecosystem nitrogen inputs, are expected to impact primary production and C inputs to soils \citep{Groenigen_2006} but it remains difficult to predict the response of soil processes to anthropogenic change \cite{DAVIDSON_2006}. Current climate change models concur on atmospheric and ocean C predictions but not terrestrial \cite{Friedlingstein_2006}. Contrasting terrestrial ecosystem model predictions reflect how little is known about soil C cycling dynamics. 

An estimated 80-90\% of C cycling in soil is mediated by microorganisms \cite{ColemanCrossley_1996,Nannipieri_2003}. Understanding microbial processing of nutrients in soils presents a special challenge due to the hetergeneous nature of soil ecosystems and methods limitations. Soils are biologically, chemically, and physically complex which affects microbial community composition, diversity, and structure (refs). Confounding factors such as physical protection/aggregation, moisture content, pH, temperature, frequency and type of land disturbance, soil history, mineralogy, N quality and availability, and litter quality have all been shown to affect the ability of the soil microbial community to access and metabolize C substrates \cite{Schlesinger_1977,dgett_Wall_Hattenschwiler_2010,Sollins_Homann_Caldwell_1996,Torn_Vitousek_Trumbore_2005,TRUMBORE_2006,Schimel_2012}. Further, rates of metabolism are often measured without knowing the identity of the microbial species specifically involved \cite{ndi_Pietramellara_Renella_2003} leaving the importance of specific community members towards maintaining ecosystem functions unknown \cite{Allison_2008,ndi_Pietramellara_Renella_2003,Schimel_2012}. Litter bag experiments have shown that the community composition of soils can have quantitative and qualitative impacts on the breakdown of plant materials \cite{Schimel_1995}. Reciprocal exchange of litter type and microbial inocula under controlled environmental conditions reveals that differences in community composition can account for 85\% of the variation in litter carbon mineralization \cite{Strickland_2009}. In addition, simple, assembled communities of cellulose degraders reveal that the composition of the community has significant impacts on the rate of cellulose degradation \cite{Wohl_2004}. 

The first steps in understanding soil C cycling dynamics is to identify specific microbial community member contributions and to investigate the relationship between genetic diversity, community structure, and function \cite{O_Donnell_2002}. The vast majority of microorganisms continue to resist cultivation in the laboratory, and even when cultivation is achieved, the traits expressed by a microorganism in culture may not be representative of those expressed when the organism is present in its natural habitat. Stable-isotope probing (SIP) provides a unique opportunity to link microbial identity to activity and has been utilized to expand our knowledge of a myriad of important biogeochemical processes and elucidate microbial activities \cite{Chen_Murrell_2010}. The most successful applications of this technique have identified organisms which mediate processes performed by a narrow set of functional guilds (ie: methanotrophs, methanogens, syntrophs, etc). The technique has been less applicable to the study of soil C cycling because of limitations in resolving power as a result of simultaneous labeling of many different organisms in the community. Additionally, frequently utilized SIP-coupled molecular applications such as TRFLP, DGGE, and cloning provide insufficient resolution of taxon identity and depth of coverage. The use of massively parallel sequencing should effectively overcome this barrier and should greatly expand the ability of nucleic acid SIP to explore complex patterns of C-cycling in microbial communities. Yet, there remain notable limitations including the need to add substrate at concentrations higher than typical in situ \cite{radajewski2000stable}, the potential for partial labeling due to label dilution \cite{radajewski2000stable,Manefield_2002,McDonald_2005}, the potential for cross-feeding and trophic cascades \cite{Morris_2002,Hutchens_2003,Lueders_2003,DeRito_2005,Mahmood_2005,McDonald_2005,Ziegler_2005} and variation in genome G+C content which causes variation in native DNA buoyant density \cite{Buckley_2007,9780408708036,HOLBEN_1995,9546163}. Several strategies have been developed to deal with these issues and typically require collection and analysis of density gradient fractions to determine the degree of isotope incorporation into particular microbial groups' nucleic acids \cite{Manefield_2002,Manefield_2002,14686943}. 

To minimize isotope signal dilution, SIP studies use single substrate experimental designs with few exceptions \cite{Lueders_2003,Chauhan_2009}. This differs from how microbes experience the substrate naturally undermining its environmental and biological relevance. C in plant biomass is found primarily in cellulose (30-50\%), hemicellulose (20-40\%), and lignin (15-25\%) \cite{Lynd_2002}. Hemicellulose is composed mostly of xylose with varying lesser amounts of arabinose, galactose, glucose, mannose, and rhamnose. The deposition of plant litter is hypothesized to induce ecological succession within the soil community as different plant compounds are degraded sequentially by organisms in differenct ecological niches. Hemicellulose and simple sugars are decomposed first while polymeric C degradation lags. Microbes that respire sugars increase in abundance dramatically during the initial decompostion stages \cite{GARRETT_1951,Alexander_1964}. As much as 75\% of sugar C can be respired or converted into microbial biomass in the first 5 days of decomposition \cite{Engelking_2007}. Cellulose degraders take longer to respond \cite{Hu_1997} with less than 42\% of cellulose metabolized over the first 5 days of incubation \cite{Engelking_2007}. Evidence to support this succession hypothesis generally comes from observing the dynamics of CO\textsubscript{2} evolution and from observations of isolated soil or litter organisms at different decomposition stages. Other studies have detected differences in labile versus polymeric C degradation \cite{Engelking_2007,Anderson_1973,Stotzky_1961,Alden_2001,Furukawa_1996,Fontaine_2003,Blagodatskaya_2007,Jenkins_2010,Rui_2009,Fierer_2010}. A classic example of plant litter degradative succession is characterized by a series of stages in which sugar fungi dominate in stage one, followed by cellulolytic fungi in stage two, and lignin degrading fungi in the final stage \cite{Gessner_2010}. These studies suggest that if a complex mixture of labile and polymeric C were added to soil two waves of degradation could be observed; labile C degradation early followed by polymeric C degradation. The degree to which the succession hypothesis presents an accurate model of litter decomposition has been questioned \cite{Kj_ller_1982,Frankland_1998,Osono_2005}. New approaches are needed to reveal soil C cycling processes and the manner in which C is transformed by the soil food web.  
We propose that a temporal cascade occurs in natural microbial communities during the plant biomass degradation in which labile C degradation preceeds polymeric C, called herein 'microbial community succession'. The aim of this study is to track the path of C in a complex C mixture after it is added to soil to provide insight into these dynamic systems. Our experimental approach employs the addition of a complex soil organic matter simulant (a mixture of model carbon sources and inorganic nutrients common to plant biomass) to soil. Parallel incubations of soils amended with this complex C mixture where a single C constituent is substituted for its \textsuperscript{13}C-labeled equivalent allows us to test how different C substrates cascade through discrete taxa within the soil microbial community. A previous study has shown that \textsuperscript{13}C labeled plant residues enable tracking of C through microbial pathways \cite{Evershed_2006}. Powerful techniques such as nucleic acid stable isotope probing coupled with next generation sequencing (SIP-NGS) can then be used to identify these \textsuperscript{13}C labeled members of the microbial community elucidating the members responsible for specific C transformations. Amplicon sequencing of 16S rRNA gene fragments from many gradient fractions and multiple gradients make it possible to track carbon assimilation by hundreds of different taxa. Ultimately we identify specific organisms or functional guilds responsible for the cycling of specific C substrates. Illuminating these microbial contributions associated with decomposition in soil are important because as environments change, there are measurable and functional changes in soil C \cite{Grandy_2008} which could cumulatively have large impacts at a global scale. 
