\section{Introduction}
% Fakesubsubsection:Excluding plant biomass, there are 2,300 Pg of
Soils worldwide contain 2,300 Pg of carbon (C) which accounts for nearlyâ€„80\%
of the C present in the terrestrial biosphere
\citep{Amundson_2001,BATJES_1996}. C respiration by soil microorganisms
produces annually tenfold more CO$_{2}$ than fossil fuel emissions
\citep{chapin2002principles}. Despite the contribution of microorganisms to
global C flux, many global C models ignore microbial
physiological diversity and its impacts on microbial activity in soils.
\citep{Allison2010,Six2006,Treseder2011}. Further, predictions of climate
change feedbacks on soil C flux improve when biogeochemical models explicitly
represent microbial physiology \citep{Wieder2013}. However, we still know
little about the ecophysiology of soil microorganisms, and such knowledge
should assist the development and refinement of global C models
\citep{Bradford2008,Neff_2001,McGuire2010}.

%Fakesubsubsection:The decomposition
Cellulose comprises most plant C (30-50\%) followed by hemicellulose (20-40\%),
and lignin (15-25\%) \citep{Lynd2002}. Hemicellulose, being the most soluble,
degrades in the early stages of decomposition. Xylans are often an abundant
component of hemicellulose, and xylans themselves include differing amounts of
xylose, glucose, arabinose, galactose, mannose, and rhamnose \citep{Saha2003}.
Xylose is often the most abundant sugar in hemicellulose, comprising as much as
60-90\% of xylan in some plants (e.g hardwoods) \citep{Spiridon2008}, wheat
\citep{Sun2005}, and switchgrass \citep{Bunnell2013}. Microbes that respire
sugars proliferate during the initial stages of decomposition
\citep{Garrett1951,Alexander1964}, and metabolize as much as 75\% of sugar
C during the first 5 days of decomposition \citep{Engelking2007}. In contrast,
cellulose decomposition proceeds more slowly with rates increasing for
approximately 15 days while degradation continues for 30-90 days
\citep{Hu1997,Engelking2007}. It is hypothesized that different microbial
functional guilds mediate the decomposition of different plant biomass
components and that these guilds proliferate as they decompose their respective
C targets \citep{Hu1997,Rui2009,AnneliseHKjoller2002,Bastian2009}. For
instance, this degradative succession hypothesis posits that fast growing
organisms proliferate rapidly in response to the labile fraction of plant
biomass such as sugars \citep{Garrett1963,Bremer1994} followed by slow growing
degraders targeting structural C such as cellulose \citep{Garrett1963}.
Evidence to support the degradative succession hypothesis comes from observing
soil respiration dynamics and characterizing microbes cultured at different
stages of decomposition. The degree to which the succession hypothesis presents
an accurate model of litter decomposition has been called into question
\citep{AnneliseHKjoller2002,Frankland1998,Osono_2005} and it's clear that new
approaches are needed to dissect microbial contributions to C transformations
in soils.

% Fakesubsubsection:Though microorganisms mediate
Though microorganisms mediate 80-90\% of the soil C-cycle
\citep{ColemanCrossley_1996,Nannipieri_2003}, and microbial community
composition can account for significant variation in C mineralization
\citep{Strickland_2009} , terrestrial C-cycle models rarely consider the
community composition of soils \citep{Zak2006,Reed2007}. Metabolic rates of
soil C transformations are measured without knowledge of the organisms that
mediate these reactions \citep{Nannipieri_2003}, leaving undefined the
importance of community membership towards maintaining ecosystem function
\citep{Nannipieri_2003,Schimel_2012,Allison_2008}. Variation in microbial
community composition can be linked most directly to rates of soil processes
when diagnostic genes for specific functions are available to represent those
processes (e.g. denitrification \citep{Cavigelli2000}, nitrification
\citep{Carney2004,Hawkes2005,Webster2005}, methanotrophy \citep{Gulledge1997},
and nitrogen fixation \citep{Hsu2009}). However, the complexity of soil
C transformations and the lack of diagnostic genes for describing
these transformations has limited progress in characterizing the contributions
of individual microbes to the soil C-cycle. Remarkably, we still lack basic
information on the physiology and ecology of the majority of organisms that
live in soils. For example, contributions to soil processes remain
uncharacterized for entire bacterial phyla such as \textit{Acidobacteria},
\textit{Chloroflexi}, \textit{Planctomycetes}, and \textit{Verrucomicrobia}.
These phyla combined can comprise 32\% of soil microbial communities (based on
surveys of the SSU rRNA genes in soil) \citep{Janssen2006,Buckley2002} and they
are nearly ubiquitous in soils. 

% Fakesubsubsection:Functional guild membership
Linking genetic identity and function in microbial ecosystems has relied
historically both on culturing microorganisms subsequently characterizing
microbial physiology in the laboratory and surveying genes diagnostic for
specific biogeochemical processes. However, most
microorganisms are difficult to grow in culture \citep{Janssen2006} and many
processes lack suitable diagnostic genes. Nucleic acid stable-isotope probing
(SIP) links genetic identity and activity without the need to grow
microorganisms in culture and has expanded our knowledge of microbial
contributions to biogeochemical processes \citep{Chen_Murrell_2010}. However,
nucleic acid SIP has notable complications including the need to add large
amounts of labeled substrate \citep{radajewski2000stable}, label dilution
resulting in partial labelling of nucleic acids
\citep{radajewski2000stable,Manefield_2002,McDonald_2005}, the potential for
cross-feeding and secondary label incorporation
\citep{Morris_2002,Hutchens2004,14686943,DeRito2005,McDonald_2005,Ziegler_2005},
and variation in genome G$+$C content
\citep{Buckley_2007,9780408708036,Holben1995,Nusslein1999}. As a result, most
applications of SIP have targeted specialized microorganisms such as
methanotrophs \citep{radajewski2000stable}, methanogens \citep{lu2005},
syntrophs \citep{lueders2004}, or microbes that target pollutants
\citep{delrito2005}. SIP has generally proved less useful for exploring the
soil C-cycle because it has lacked the resolution necessary to manage
effectively the signal complexity that results from adding components of plant
biomass to microbial communities in soil. High throughput DNA sequencing
technology, however, improves the resolving power of SIP \citep{Aoyagi2015}. 

% Fakesubsubsection:High throughput sequencing
Coupling SIP with high throughput DNA sequencing now enables exploration of
microbial C-cycling in soils. SSU rRNA amplicons can be are readily sequenced
from numerous density gradient fractions across multiple samples thereby
increasing the resolution of a typical nucleic acid SIP experiment
\citep{Verastegui_2014}. It is now possible to use far less isotopically
labeled substrate resulting in conditions that are more environmentally
realistic \citep{Aoyagi2015}. We have employed such a high resolution DNA
stable isotope probing approach to explore the assimilation of $^{13}$C labeled
xylose and/or cellulose into bacterial DNA in an agricultural soil. 

Specifically, we added to soil a complex amendment that simulated organic
matter derived from fresh plant biomass. All treatments received the same
amendment but the identity of the isotopically labeled substrate was varied
between treatments. We set up a control treatment where all components were
unlabeled, a treatment with $^{13}$C-xylose, and a treatment with
$^{13}$C-cellulose. Soil was sampled at days 1, 3, 7, 14, and 30 and we
identified the microorganisms had assimilated $^{13}$C into DNA at each point
in time. The experiment was designed to provide a test of the
degradative succession hypothesis in the context of soil bacteria, to identify
soil bacteria that metabolize xylose and cellulose, and to characterize
temporal dynamics of xylose and cellulose metabolism in soil. 
