\section{Methods}
Additional information on sample collection and analytical methods is provided
in Supplemental Materials and Methods. 

Twelve soil cores (5 cm diameter x 10 cm depth) were collected from six 
sampling locations within an organically managed agricultural field in Penn
Yan, New York. Soils were sieved (2 mm), homogenized, distributed into flasks
(10 g in each 250 ml flask, n = 36) and equilibrated for 2 weeks. Soils were
amended with a mixture containing 5.3 mg C g$^{-1}$ soil dry weight (d.w.) and
brought to 50\% water holding capacity. The mixture contained 38\% cellulose,
23\% lignin, 20\% xylose, 3\% arabinose, 1\% galactose, 1\% glucose, and 0.5\%
mannose by mass, with the remaining 13.5\% mass composed of an amino acid
(in-house made replica of Teknova C0705) and macronutrient mixture (Murashige
and Skoog, Sigma Aldrich M5524).  This mixture approximates the molecular
composition of switchgrass biomass with hemicellulose replaced by its
constituent monomers \citep{Schneckenberger_2008}. Three parallel
treatments were performed which varied the isotopic composition of the mixture:
(1) unlabeled control, (2) $^{13}$C-cellulose (synthesized as described in
Supplemental Methods), (3) $^{13}$C-xylose (98 atom\% $^{13}$C, Sigma Aldrich).
A total of 12 microcosms were established per treatment. Other details relating
to substrate addition can be found in Supplemental Methods. Microcosms were
sampled destructively at days 1 (control and xylose only), 3, 7, 14, and 30 and
soils were stored at -80 $^{\circ}$C until nucleic acid extraction. The
abbreviation “13CXPS” refers to the 13C-xylose treatment ($^{13}$C Xylose Plant
Simulant), “13CCPS” refers to the $^{13}$C treatment and “12CCPS” refers to the
unlabeled control. 

Nucleic acids were extracted using a modified Griffiths protocol \cite{Griffiths_2000}
and DNA was prepared for isopycnic centrifugation as previously described
(Reference 55). A total of 5 μg of DNA was added to each 4.7 ml CsCl density
gradient composed of 1.69 g ml$^{-1}$ CsCl in15 mM Tris-HCl, 15 mM EDTA, 15 mM KCl.
Centrifugation was performed in a TLA-110 rotor at 55,000 rpm and 20°C for 66
hr. Fractions of 100 μL were collected by displacement at 3.3 $\mu$L s$^{-1}$
\cite{Manefield_2002} into Acroprep 96 filter plates (Pall Life Sciences 5035).
The refractive index of each fraction was measured immediately using a Reichart
AR200 digital refractometer as previously described
\cite{Buckley_2007}. The DNA was desalted by washing (3 x 200
$\mu$L) with TE in the Acroprep filter wells followed by resuspension in 50
$\mu$L TE. SSU rRNA genes were amplified from gradient fractions (n = 20 per
gradient) and from non-fractionated DNA. Barcoded primers consisted of:
454-specific adapter B, a 10 bp barcode \citep{Hamady_2008},
a 2 bp linker (5’-CA-3’), and 806R primer for reverse primer (BA806R); and
454-specific adapter A, a 2 bp linker (5’-TC-3’), and 515F primer for forward
primer (BA515F). Each PCR contained 1.25 U $\mu$l$^{-1}$ AmpliTaq Gold (Life
Technologies, Grand Island, NY; N8080243), 1X Buffer II (100 mM Tris-HCl, 500
mM KCl, pH~8.3), 2.5 mM MgCl2, 200 $\mu$M of each dNTP, 0.5 mg ml$^{-1}$ BSA,
0.2 $\mu$M BA515F, 0.2 $\mu$M BA806R, and 10 $\mu$L of 1:30 DNA template in 25
$\mu$l total volume). Reactions were performed in triplicate and pooled.
Amplified DNA was gel purified using the Wizard SV gel and PCR clean-up
system (Promega, Madison, WI; A9281) per manufacturer’s protocol. Samples
were normalized by SequalPrep normalization plates (Invitrogen, Carlsbad, CA;
A10510) and pooled in equimolar concentration. Amplicons were sequenced on
Roche 454 FLX system using titanium chemistry at Selah Genomics (Columbia,
SC). 

SSU rRNA gene sequences were initially screened by maximum expected errors at
a specific read length threshold  \citep{edgar2013}. Reads that had less
than~0.5 expected errors at a length of 250 nt were then aligned to the Silva
Reference Alignment as provided in the Mothur software package using the Mothur
NAST aligner \citep{DeSantis2005,schloss2009}. Reads that did not align to the
expected region of the SSU rRNA gene were discarded. After expected error and
alignment based quality control, 87\% of original reads remained. The remaining
reads were annotated using the “UClust” taxonomic annotation framework in
\citep{caparaso2010,edgar2010}. We used 97\% cluster seeds from the Silva
SSU rRNA database \citep{quast2013} as reference for taxonomic annotation
(provided at QIIME website) \citep{quast2013}. Sequences were distributed
into OTUs with a centroid based clustering algorithm (i.e. UPARSE
\citep{edgar2013}). The centroid selection also included robust chimera
screening \citep{edgar2013}. OTU centroids were established at a threshold
of 97\% sequence identity and non-centroid sequences were mapped back to
centroids. Reads that could not be mapped to an OTU centroid at greater
than or equal to 97\% sequence identity were discarded. For phylogenetic
reconstruction, alignment was performed with SSU-Align
\citep{nawrocki2009,nawrocki2013}. Columns in the alignment that were
aligned with poor confidence ($<$ 95\% of characters had posterior
probability $>$ 95\%) were not considered when building
the phylogenetic tree. FastTree \citep{price2010} was used with default
parameters to build the phylogeny. NMDS ordination was performed on
weighted Unifrac \citep{lozupone2005} distances. The Phyloseq
\citep{mcmurdie2013} wrapper for Vegan \citep{oksanen2015} (both
R packages) was used to compute sample values along NMDS axes. The
'adonis' function in Vegan was used to perform Adonis tests (default
parameters) \citep{Anderson2001a}.

We used DESeq2 (R package), an RNA-Seq differential expression statistical
framework \citep{love2014}, to identify OTUs that were enriched in high
density gradient fractions from $^{13}$C-treatments relative to corresponding
density fractions from control treatments (for review of RNA-Seq differential
expression statistics applied to microbiome OTU count data see (30)). We define
"high density gradient fractions" as gradient fractions whose density falls
between 1.7125 and 1.755 g ml$^{-1}$. Briefly, DESeq2 includes several features that
enable robust estimates of standard error in addition to reliable ranking of
logarithmic fold change (LFC) in abundance (i.e. gamma-Poisson regression
coefficients) even with low count groups where LFC can often be noisy.
Further, statistical evaluation of LFC can be performed with selected
thresholds as opposed to the often default null hypothesis that differential
abundance for an OTU is exactly zero. This enables the most biologically
interesting OTUs to be selected for subsequent analyses. We calculated LFC and
corresponding standard errors for $^{13}$C treatments and control comparisons
in high density gradients for each OTU. Subsequently, a one-sided Wald test was
used to statistically assess LFC values (using corresponding standard errors).
The user-defined null hypothesis for the Wald test was that LFC was less than
one standard deviation above the mean of all LFC values. P-values were
corrected for multiple comparisons by using the Benjamini and Hochberg (BH)
method (31). Independent filtering was performed on the basis of sparsity prior
to correcting P-values for multiple comparisons. The sparsity value that
yielded the most P-values less than 0.10 was used for independent filtering by
sparsity. Briefly, OTUs were eliminated if they failed to appear in at least
45\% of high density gradient fractions for a given $^{13}$C and control
treatment pair, these OTUs are unlikely to have sufficient data to allow for
the determination of statistical significance.
