\section{Methods}
Additional information on sample collection and analytical methods is provided in \href{https://www.authorea.com/users/3537/articles/8459/_show_article}{SI Materials and Methods}.


Twelve soil cores (5 cm diameter x 10 cm depth) were collected from six random sampling locations within an organically managed agricultural field in Penn Yan, New York. Soils were pretreated by sieving (2 mm), homogenizing sieved soil, and pre-incubating 10 g of dry soil weight in flasks for 2 weeks. Soils were amended with a 5.3 mg g soil\textsuperscript{-1} carbon mixture; representative of natural concentrations \cite{Schneckenberger_2008}. Mixture contained 38\% cellulose, 23\% lignin, 20\% xylose, 3\% arabinose, 1\% galactose, 1\% glucose, and 0.5\% mannose by mass, with the remaining 13.5\% mass composed of an amino acid (in-house made replica of Teknova C0705) and basal salt mixture (Murashige and Skoog, Sigma Aldrich M5524). Three parallel treatments were performed; (1) unlabeled control,(2)\textsuperscript{13}C-cellulose, (3)\textsuperscript{13}C-xylose (98 atom\% \textsuperscript{13}C, Sigma Aldrich). Each treatment had 2 replicates per time point (n = 4) except day 30 which had 4 replicates; total microcosms per treatment n = 12, except \textsuperscript{13}C-cellulose which was not sampled at day 1, n = 10. Other details relating to substrate addition can be found in Suppl M&M. Microcosms were sampled destructively (stored at -80{\textdegree}C until nucleic acid processing) at days 1 (control and xylose only), 3, 7, 14, and 30.


Nucleic acids were extracted using a modified Griffiths procotol \cite{Griffiths_2000}. To prepare nucleic acid extracts for isopycnic centrifugation as previously described \cite{Buckley_2007}, DNA was size selected (\textgreater4kb) using 1\% low melt agarose gel and $\beta$- agarase I enzyme extraction per manufacturers protocol (New England Biolab, M0392S). For each time point in the series isopycnic gradients were setup using a modified protocol \cite{Neufeld_2007} for a total of five \textsuperscript{12}C-control, five \textsuperscript{13}C-xylose, and four \textsuperscript{13}C-cellulose microcosms. A density gradient (average density 1.69 g mL\textsuperscript{-1}) solution of 1.762 g cesium chloride (CsCl) ml\textsuperscript{-1} in gradient buffer solution (pH 8.0 15mM Tris-HCl, 15mM EDTA, 15mM KCl) was used to separate \textsuperscript{13}C-enriched and \textsuperscript{12}C-nonenriched DNA. Each gradient was loaded with approximately 5$\mu$g of DNA and ultracentrifuged for 66 h at 55,000 rpm and room temperature (RT). Fractions of $\sim$100 $\mu$L were collected from below by displacing the DNA-CsCl-gradient buffer solution in the centrifugation tube with water using a syringe pump at a flow rate of 3.3 $\mu$L s\textsuperscript{-1} \cite{Manefield_2002} into Acroprep\textsuperscript{TM} 96 filter plate (part no. 5035, Pall Life Sciences). The refractive index of each fraction was measured using a Reichart AR200 digital refractometer modified as previously described \cite{Buckley_2007} to measure a volume of 5$\mu$L. Then buoyant density was calculated from the refractive index as previously described \cite{Buckley_2007} (see also suppl methods). The collected DNA fractions were purified by repetitive washing of Acroprep filter wells with TE. Finally, 50$\mu$L TE was added to each fraction then resuspended DNA was pipetted off the filter into a new microfuge tube. 


For every gradient, 20 fractions were chosen for sequencing between the density range 1.67-1.75 g mL\textsuperscript{-1}. Barcoded 454 primers were designed using 454-specific adapter B, 10 bp barcodes \cite{Hamady_2008}, a 2 bp linker (5'-CA-3'), and 806R primer for reverse primer (BA806R); and 454-specific adapter A, a 2 bp linker  (5'-TC-3'), and 515F primer for forward primer (BA515F). Each fraction was PCR amplified using 0.25 $\mu$L 5 U $\mu$l\textsuperscript{-1} AmpliTaq Gold (N8080243, Life Technologies, Grand Island, NY), 2.5 $\mu$L 10X Buffer II (100 mM Tris-HCl, pH 8.3, 500 mM KCl), 2.5$\mu$L 25 mM MgCl_{2}, 4 $\mu$L 5 mM dNTP, 1.25 $\mu$L 10 mg mL\textsuperscript{-1} BSA, 0.5 $\mu$L 10 $\mu$M BA515F, 1 $\mu$L 5 $\mu$M BA806R, 3 $\mu$L H$_{2}$O, 10 $\mu$L 1:30 DNA template) in triplicate. Samples were normalized either using Pico green quantification and manual calculation or by SequalPrep\textsuperscript{TM} normalization plates (A10510, Invitrogen, Carlsbad, CA), then pooled in equimolar concentrations.  Pooled DNA was gel extracted from a 1\% agarose gel using Wizard SV gel and PCR clean-up system (A9281, Promega, Madison, WI) per manufacturer's protocol.  Amplicons were sequenced on Roche 454 FLX system using titanium chemistry at Selah Genomics (formerly EnGenCore, Columbia, SC)  
