\section{Methods}
Additional information on sample collection and analytical methods is provided in \href{https://www.authorea.com/users/3537/articles/8459/_show_article}{SI Materials and Methods}.


Agricultural soil cores were collected from an organically managed farm in Penn Yan, New York. Soils were pretreated by sieving (2mm) cores, homogenizing sieved soil, and incubating 10g of dry soil weight in microcosm flasks for 2 weeks.  Soils were amended with a complex carbon mixture (38\% cellulose, 23\% lignin, 20\% xylose, 3\% arabinose, 1\% galactose, 1\% glucose, 0.5\% mannose) along with an amino acid (in-house made replica of teknova Cat#C0705) and basal salt mixture (Murashige and Skoog, Sigma M5524) at 52.6mg 10g\textsuperscript{-1} soil; representative of natural concentrations. Three parallel treatments were performed; (1) control, all \textsuperscript{12}C-carbons, (2) cellulose, all \textsuperscript{12}C-carbons except \textsuperscript{13}C-cellulose, (3) xylose, all \textsuperscript{12}C-carbons except \textsuperscript{13}C-xylose. For soil C amendments bacterial cellulose (both \textsuperscript{12}C- and \textsuperscript{13}C-) was produced using 0.1\% glucose in static liquid cultures modeled from Heo and Son \cite{Heo_Son_2002}. Replicate microcosms were harvested (stored at -80{\textdegree}C until nucleic acid processing) at days 1 (control and xylose only), 3, 7, 14, and 30.



DNA was extracted using a modified Griffiths procotol (cite). To prepare nucleic acid extracts for isopycnic centrifugation, DNA was size selected (>4kb) using 1\% low melt agarose gel and $\beta$- agarase I enzyme extraction per manufacturers protocol (New England Biolab, M0392S). Isopycnic gradients were established as described previously (35, 48) for five \textsuperscript{12}C-control, five \textsuperscript{13}C-xylose, and four \textsuperscript{13}C-cellulose microcosms; one per day harvested. A density gradient solution of 1.762 g cesium chloride (CsCl) ml\textsuperscript{-1} in gradient buffer solution (pH 8.0 15mM Tris-HCl, 15mM EDTA, 15mM KCl) was used to separate \textsuperscript{13}C-enriched and \textsuperscript{12}C-nonenriched DNA. Each gradient was loaded with approximately 5$\mu$g of DNA and utlracentrifuged for 66 h at 55,000 rpm and room temperature (RT). Fractions of $\sim$100 $\mu$L were collected into Acroprep\textsuperscript{TM} 96 filter plate (part no. 5035, Pall Life Sciences) by displacing the DNA-CsCl-gradient buffer solution in the centrifugation tube with water. The refractive index of each fraction was measured using a Reichart AR200 digital refractometer. Then buoyant density was calculated from the refractive index using the equation $\rho$=\textit{a}$\eta$-\textit{b}, where $\rho$ is the density of the CsCl (g ml\textsuperscript{-1}), $\eta$ is the measured refractive index, and \textit{a} and \textit{b} are coefficient values of 10.9276 and 13.593, respectively, for CsCl at 20{\textdegree}C \cite{9780408708036}. The collected DNA fractions were purified by repetitive washing with TE followed by centrifugation. Finally, 50$\mu$L TE was added to each fraction, incubated on filter for 5min at RT then resuspended DNA was pipetted off the filter into a new microfuge tube. For every gradient, 20 fractions were chosen for sequencing between the density range 1.67-1.75g/mL. 



Barcoded 454 primers were designed using 454-specific adapter B, 10bp barcodes \cite{Hamady_2008}, a 2bp linker (put those bp in here), and 806R primer for reverse primer (BA806R); and 454-specific adapter A, a 2bp linker, and 515F primer for forward primer (BA515F). Each fraction was PCR amplified using 0.25$\mu$L 5U/$\mu$l AmpliTaq Gold (N8080243, Life Technologies, Grand Island, NY), 2.5$\mu$L 10X Buffer II (100 mM Tris-HCl, pH 8.3, 500 mM KCl), 2.5$\mu$L 25mM MgCl_{2}, 4$\mu$L 5mM dNTP, 1.25$\mu$L 10mg/mL BSA, 0.5$\mu$L 10$\mu$M BA515F, 1$\mu$L 5$\mu$M BA806R,3$\mu$L H_{2}O, 10$\mu$L 1:30 DNA template) in triplicate. Samples were normalized either using Pico green quantification and manual calculation or by SequalPrep\textsuperscript{TM} normalization plates (A10510, Invitrogen, Carlsbad, CA), then pooled in equimolar concentrations.  Pool was gel extracted from a 1\% agarose gel using Wizard SV gel and PCR clean-up system (A9281, Promega, Madison, WI) per manufacturer's protocol.  Amplicons were sequenced on Roche 454 FLX system using titanium chemistry at Selah Genomics (formerly EnGenCore, Columbia, SC)  
