\section{Reference info I wasn't ready to part ways with}

"These results show that all of the communities were dominated by Acidobacteria, Actinobacteria, Bacteroidetes, Pro- teobacteria, and Verrucomicrobia (Fig. S1), bacterial phyla that are known to be relatively abundant and ubiquitous in soil (25). Additional phyla including Chloroflexi, Cyanobacteria, Firmicutes, and Gemmatimonadetes were also found in nearly all soils, but their relative abundances were highly variable and typically rep- resented less than 5\% of the 16S rRNA reads in any individual"\cite{Fierer_2012}

"Therefore, as with the alpha diversity patterns, the concordance in beta diversity patterns highlights that the overall functional differences between the soil microbial communities were significantly correlated with the differences in the composition of these communities. Our findings are in line with comparable studies conducted in soil (20) and other habitats that also found strong correlations between metagenome composition and taxonomic composition (34, 35, 40). Although individual functional genes may not necessarily be correlated with community structure, the overall functional attributes of soil microbial communities appear to be predictable across broad gradients in soil and biome types if one has information on the taxonomic or phylogenetic structure of the communities."\cite{Fierer_2012}

   "Very few taxa responded only to chemically recalcitrant substrates, with only 24 for cellulose (Clostridiales; termite gut clones, Deinococcus, Chloroflexi, Acetobacter, Desulfobacteraceae, Geobacteraceae, Syntrophobacteraceae, Treponema). On the other hand, cellulose users were relatively phylogenetically diverse (eight phyla), which may broaden the range of environ- mental conditions under which this metabolism may operate. Many of the taxa responding positively to cellulose have previously been shown to be capable of cellulose hydrolysis (e.g., bacteria from the families Clostridiaceae and Lachnospiraceae (Schwarz, 2001), Sphingomonas spp. (Kurakake et al., 2007), and Spirochetes (Warnecke et al., 2007), while Acetobacter aceti (Moonmangmee et al., 2002) is typically considered a cellulose producer rather than a consumer. "\cite{Goldfarb_2011}
   
   
"For bacterial succession, an increase in the proportion of Proteobacteria from winter to spring was observed, whereas that of Actinobacteria and Verrucumicrobia decreased (Figure 1b). Changes in the respective group abundances were validated by a PLFA analysis, which showed similar trends (Figure 2). A reduction of Actinobacteria was unexpected, because they are known to be involved in decomposition of organic materials, and thus are important for organic matter turnover and C cycle (Kirby, 2006). In other studies, an increase in the abundance of Actinobacteria has been shown during later stages of litter decomposition (Torres et al., 2005; Snajdr et al., 2011). The same accounts for the absence of Acidobacteria; members of this bacterial phylum can degrade various polysaccharides in- cluding cellulose and xylan (Ward et al., 2009). Based on RNA sequencing, Baldrian et al. (2012) found Acidobacteria to be the main bacterial group that was enriched in an active litter inhabiting community." 





Plant residues represent the largest proportion of C entering soil systems and are composed primarily of cellulose making it the most abundant biopolymer globally (Paul & Clark 1989, Berg & Laskowski 2005, \cite{lemm_Pautzsch_Blankenburg_2005}, Coleman & Crossley 1996, Nannipieri & Badalucco 2003). Much work has been done to study cellulose degradation by microorganisms (Beguin & Aubert 1994, Lynd et al 2002, Haicher et al 2007, Eichorst & Kuske 2012).  With the advent of culture independent techniques, we have discovered that there are far more cellulose degraders than our limited culture collection represents.  Some studies have used labeled leaf litter to identify soil decomposers, however, it is difficult to differentiate which organisms are responsible for the actual degradation of the cellulose (Evershed et al 2006). Other cellulose degradation studies add cellulose as a single substrate addition to a cultured cellulose degrader or to soil (Haichar et al 2007, Li et al 2009, Eichorst & Kuske 2012).  This method may present false positives, as an organism that might not normally consume a given C substrate may opportunistically do so under the circumstances that it is the only C source available. Additionally, organisms may behave quite differently when given a large dose of a nutrient source versus a complex C mixture which they may actually encounter in nature.


-------
" In addition, the analysis by phylogenetic and ecological groups, suggest that the communities respond in a similar trajectory of initial colonization first by heterotropic generalists and later specialists. "\cite{L_pez_Lozano_2013}
"Resource availability is also likely to be a funda- mental driver of microbial succession, but the limiting resources and environmental factors regulating succession will be more complex given the far greater physiological diversity contained within microbial communities and the breadth of environments in which succession can occur. During endoge- nous heterotrophic succession, labile substrates will be consumed first, supporting copiotrophic microbial taxa that are later replaced by more oligotrophic taxa that metabolize the remaining, more recalcitrant, organic C pools in the later stages of succession (Rui et al., 2009)."\cite{Fierer_2010}

"In correspondence with the dynamics of fatty acids, the bac- terial community showed a distinct succession during the course of residue decomposition"\cite{Rui_2009}
"dynamics of community structure seemed to be related to changes in the availability of carbon resources occurring during degradation"\cite{Bastian_2009}
Another confoundingsince competition for a limited resource typically results in the dominance of one or a few populations with the highest growth rates \cite{Fontaine_2003}.

These results revealed (i) a high diversity of bacteria able to colonize the residue as previously indicated by the considerable complexity of B-ARISA profiles and (ii) a slight increase in bacterial diversity during the residue decom- position. This might be related to the biochemical composition of the residue at the end of the degradation process (Nicolardot et al., 2007). The main components at this stage were complex substrates (cellulose, hemicelluloses, lignin) recalcitrant to degradation which might result in the stimulation of a more diverse bacterial consor- tium able to degrade them (De Boer et al., 2005).\cite{Bastian_2009}
