\section{Results and Discussion}
In this study, we couple nucleic acid SIP with next generation sequencing
(SIP-NGS) to observe C use dynamics by the soil microbial community. A series
of parallel soil microcosms all amended with a C substrate mixture were
incubated for 30 days. The substrate mixture was identical for each bottle
except in one series of bottles the cellulose was $^{13}$C-labeled in another
the xylose was $^{13}$C-labeld and in the last no sustrattes were labeled. The
C substrate mixture was designed to approximate freshly degrading plant
biomass. Xylose or cellulose carried the isotopic label so we could examine C
assimilation dynamics for labile, soluble C versus insoluble, polymeric C. 5.3
mg total mass of C substrate mixture per gram soil (including 0.42 mg xylose-C
and 0.88 mg cellulose-C g soil$^{-1}$) was added to each microcosm representing
18\% of the total soil C. Microcosms were harvested at several time points
during the incubation period and $^{13}$C assimilation was observed by
sequencing 16S rRNA gene amplicons from bulk soil DNA and CsCl gradient
fractions. Assimilation of $^{13}$-C from Xylose degradation peaked immediately, 
while cellulose $^{13}$C assimilation peaked after two weeks of incubation
(Figure~\ref{fig:ord}).

\subsection{$^{13}$C from cellulose assimilated by canonical
cellulose-degrading and uncharacterized microbial lineages in many phyla
including Chloroflexi and Verrucomicrobia} 
Only 2 and 5 OTUs were found to have incorporated $^{13}$C from labeled
cellulose at days 3 and 7, respectively. At days 14 and 30, however, 42 and 39
OTUs were found to incorporate $^{13}$C from cellulose into biomass. An average
16\% of the $^{13}$C-cellulose added was respired within the first 7 days, 38\%
by day 14, and 60\% by day 30. A \textit{Cellvibrio} and
\textit{Sandaracinaceae} OTU assimilated $^{13}$C from cellulose at day 3. Day
7 responders included the same \textit{Cellvibrio} responder as day 3, a
\textit{Verrucomicrobia} OTU and three \textit{Chloroflexi} OTUs. 50\% of Day
14 responders belong to Proteobacteria (66\% Alpha-, 19\% Gamma-, and 14\%
Beta-) followed by 17\% \textit{Planctomycetes}, 14\% \textit{Verrucomicrobia},
10\% \textit{Chloroflexi}, 7\% \textit{Actinobacteria} and 2\%
\textit{Cyanobacteria}. \textit{Bacteroidetes} OTUs begin to incoporate
$^{13}$C from cellulose at day 30 (13\% of day 30 responders). Other day 30
responding phyla include \textit{Proteobacteria} (30\% of day 30 responders;
42\% Alpha-, 42\% Delta, 8\% Gamma-, and 8\% Beta-), \textit{Planctomycetes}
(20\%), \textit{Verrucomicrobia} (20\%), \textit{Chloroflexi} (13\%) and
\textit{Cyanobacteria} (3\%). \textit{Proteobacteria},
\textit{Verrucomicrobia}, and \textit{Chloroflexi} had relatively high numbers
of responders with heavy response across multiple time points
(Figure~\ref{fig:l2fc}).

\textit{Proteobacteria} represent 46\% of all cellulose responding OTUs
identified. \textit{Cellvibrio} accounted for 3\% of all Proteobacterial
responding OTUs detected. \textit{Cellvibrio} was one of the first identified
cellulose degrading bacteria and was originally described by Winogradsky in
1929 who named it for its cellulose degrading abilities
\citep{boone2001bergeys}. All $^{13}$C-cellulose responding
\textit{Proteobacteria} share high sequence identity with 16S rRNA genes from
sequenced type straints (Table XX) except for OTU.442 (best type strain match
92\% sequence identity in the \textit{Chrondomyces} genus) and OTU.663 (best
type strain match outside \textit{Proteobacteria} entirely,
\textit{Clostridium} genus, 89\% sequence identity). Some
\textit{Proteobacteria} responders share high sequence identity with type
strains for genera known to posess cellulose degraders including
\textit{Rhizobium}, \textit{Devosia}, \textit{Stenotrophomonas} and
\textit{Cellvibrio}. One \textit{Proteobacteria} OTU shares high sequence
identity with the \textit{Brevundimonas} type strain. \textit{Brevundimonas}
has not previously been identified as a cellulose degrader, but has been shown
to degrade cellouronic acid, an oxidized form of cellulose
\citep{Tavernier_2008}.

\textit{Verrucomicrobia}, a cosmopolitan soil phylum often found in high
abundance \citep{Fierer_2013}, are implicated in polysaccaride degradation in
many environments \citep{Fierer_2013,Herlemann_2013,10543821}.
\textit{Verrucomicrobia} comprise 16\% of the total cellulose responder OTUs
detected. 40\% of \textit{Verrucomicrobia} responders belong to the uncultured
FukuN18 family originally identified in freshwater lakes \citep{Parveen_2013}.
The \textit{Verrucomicrobia} OTU with the strongest \textit{Verrucomicrobial}
response to $^{13}$C-cellulose shared high sequence identity (97\%) with an
isolate from Norway tundra soil \citep{Jiang_2011} although growth on cellulose
was not assessed for this isolate. Only one other $^{13}$C-cellulose responding
verrucomicrobium shared high DNA sequence identity with a sequenced type
strain, OTU.638 with \textit{Roseimicrobium gellanilyticum} (100\% sequence
identity) and \textit{Roseimicrobium gellanilyticum} grows on soluble celluose
\citep{Otsuka_2012}. The remaining $^{13}$C-cellulose \textit{Verrucomicrobia}
responders did not share high sequence identity with any type strains (maximum
sequence identity with any type strain 93\%). 

\textit{Chloroflexi} are traditionally known for their metabolically dynamic
lifestyles ranging from anoxygenic phototropy to organohalide respiration
\citep{Hug_2013}. Recent studies have focused on \textit{Chloroflexi} roles in C
cycling \citep{Hug_2013, Goldfarb_2011,Cole_2013} and several members of this
phylum demonstrated cellulose utilization \citep{Goldfarb_2011, Cole_2013,
Hug_2013}. Four closely related OTUs in an undescribed \textit{Chloroflexi}
lineage (closest matching type strain for all four OTUs: \textit{Herpetosiphon
geysericola}, 89\% sequence identity) responded to $^{13}$C-cellulose in this
microcosm experiment. One additional OTU also from a poorly characterized
lineage (closest type strain match a proteobacterium at 78\% sequence identity)
responded to $^{13}$C-cellulose (Figure~\ref{fig:trees}).

Other notable $^{13}$C cellulose responders include a \textit{Bacteroidetes}
OTU that shares high sequence identity (99\%) to \textit{Sporocytophaga
myxococcoides} a known cellulose degrader \citep{Vance_1980}, and three
\textit{Actinobacteria} OTUs that share high sequence identity (100\%) with
sequenced type strains. One of the three \textit{Actinobacteria}
$^{13}$C-cellulose responders is in the \textit{Streptomyces}, a genus known to
possess cellulose degraders, while the other two closely match the type strains
\textit{Allokutzneriz albata} \citep{Labeda_2008, Tomita_1993} and
\textit{Lentzea waywayandensis} \citep{LABEDA_1989, Labeda_2001} that do not
decompose cellulose in culture. Nine \textit{Plantomycetes} OTUs responded to
$^{13}$C-cellulose but none are within described genera (closest type strain
match 91\% sequence identity) (Figure~\ref{fig:trees}). Interestingly, one
responder is annotated as belonging in the \textit{Cyanobacteria}. The phylum
annotation is misleading as the OTU is not closely related to any oxygenic
phototrophs (closest type strain match \textit{Vampirovibrio chlorellavorus},
95\% sequence identity). A sister clade to the oxygenic phototrophs that does
not itself possess known phototrophs has recently been proposed to constitute
its own phylum ("Melainabacteria", \citet{Di_Rienzi_2013}) although its
phylogenetic position is debated \citep{Soo_2014}. The catalog of metabolic
capabilities associated with \textit{Cyanobacteria} (or candidate phyla
previously annotated as \textit{Cyanobacteria}) are quickly expanding
\citep{Di_Rienzi_2013, Soo_2014}.  Our findings provide evidence of cellulose
degradation within a lineage closely related to but apart from oxygenic
phototrophs. Notably, polysaccharide degradation is suggested by the analysis
of a \textit{Melainabacterial} genome \citep{Di_Rienzi_2013}. Although we
highlight $^{13}$C-cellulose responders that share high sequence identity with
described genera, by and large $^{13}$C-cellulose responders uncovered in this
experiment are not closely related to isolates (Table~XX).

\subsection{Putative spore-formers in the Firmicutes assimilate $^{13}$C from
xylose within first day after soil amendment followed by Bacteroidetes and then
Actinobacteria OTUs} 
Within the first 7 days of incubation an average 63\% of $^{13}$C-xylose was
respired and only an additional 6\% more was respired between days 7 and 30. At
the end of the 30 day experiment 30\% of the original $^{13}$C from xylose
remained in the soils. The $^{13}$C remaining in the soil from $^{13}$C-xylose
addition has likely been stabilized by assimilation into microbial biomass
and/or microbial conversion into other forms of organic matter, though it is
possible that some $^{13}$C-xylose remains unavailable to microbes due to
abiotic interactions in soil \citep{Kalbitz_2000}. All xylose
responders were first responsive in first 7 incubation days. 

At day 1, 84\% of xylose responsive OTUs belong to Firmicutes, 11\% to
\textit{Proteobacteria} and 5\% to \textit{Bacteroidetes}. At day 3 (d3),
Firmicutes responders decreased to 5\% (from 16 OTUs to 1) while
\textit{Bacteroidetes} increased to 63\% (from 1 to 12 OTUs) of day 3
responders. The remaining day 3 responders are members of the
\textit{Proteobacteria} (26\%) and the \textit{Verrucomicrobia} (5\%). Day 7
responders were 53\% Actinobacteria, 40\% Proteobacteria, and 7\% Firmicutes. A
substantial amount (75\%) of xylose responders for day 7 had not previously
been identified as responders at earlier time points. 

We observe dynamic changes in $^{13}$C-xylose assimilation with time at the
phylum level. The numerically dominant xylose responder phylum shifts from
\textit{Firmicutes} to \textit{Bacteroidetes} and then to \textit{Actinobacteria}
across days 1, 3 and 7 (Figure~\ref{fig:l2fc}). We also see strong and significant
correlation between estimated rRNA operon genome copy numbers per $^{13}$C-xylose
responder OTU genome and time (p-value, Figure~\ref{fig:copy}). $^{13}$C-xylose responder
rRNA operon geneome copy number is inversely related to time. That is, OTUs
that first respond at later time points have fewer estimated rRNA operons per
genome than OTUs that first respond earlier. rRNA operon copy number estimation
is a recent advance in microbiome science \citep{Kembel_2012} and the relationship of rRNA
operon copy number per genome with ecological strategy is well established
\citep{Klappenbach_2000}. Specifically, microorganisms with a high number of rRNA operons per
genome tend to be fast growers specialized to take advantage of boom-bust
environments whereas a low rRNA operon copy number per genome tends to occur in
microorganisms that favor slower growth under lower and more consistent
nutrient input \citep{Klappenbach_2000}. At the beginning of our incubation, OTUs with estimated
high rRNA operon copy numbers per genome or ``fast-growers'' assimilate xylose
into biomass and with time slower growers (lower rRNA operon number per genome)
begin to respond to the xylose addition. Further, $^{13}$C-xylose responders
have fewer estimated rRNA operon copy numbers per genome than
$^{13}$C-cellulose responders suggesting xylose respiring mircrobes are
generally faster growers than cellulose degraders.

All of the $^{13}$-xylose responders in the \textit{Firmicutes} phylum are closely related
(at least 99\% sequence idetity) to cultured isolates from genera that are known to form 
endospores (Table XX). Each responder is closely related to strains annotated
as members of \textit{Bacillus}, \textit{Paenibacillus} or
\textit{Lysinibacillus}. \textit{Bacteroidetes} $^{13}$C-xylose responders are
predominantly closely related to \textit{Flavobacterium} species (5 of 8 total
responders). Only one \textit{Bacteroidetes} responder is not closely related
to a cultured isolate, ``OTU.183'' (closest LTP BLAST hit, \textit{Chitinophaca
sp.}, 89.5\% sequence identity). OTU.183 shares high sequence identity with
environmental clones derived from rhizosphere samples (accession AM158371,
unpublished) and the skin microbiome (accession JF219881, CITE). Other
\textit{Bacteroidetes} responders share high sequence identities with canonical
soil genera including \textit{Dyadobacer}, \textit{Solibius} and
\textit{Terrimonas}. Six of the 8 \textit{Actinobacteria} $^{13}$C-xylose responders
are in the \{Micrococcales} order. One $^{13}$C-xylose responding \textit{Actinobacteria}
OTU shares 100\% seqeunce identity with \textit{Agromyces ramosus} (Table~XX). 
\textit{Agromyces ramosus} is a known predatotry bacterium CITE but is not dependent
on a host for growth in culture CITE. It is not possible to determine the specific origin
of assimilated $^{13}$C in a DNA-SIP experiment. The isotopically labeld C can be passed 
down through trophic levels CITE. It's possible that the $^{13}$C labeled \textit{Agromyces} 
OTU assimilating $^{13}$C by predataion on alreadly $^{13}$C labeled OTUs.

\subsection{Cellulose degrader DNA exhibits greater bouyant density shifts upon $^{13}$C incorporation than xylose degrader DNA}
\subsection{DNA BD shifts due to $^{13}$C-asimilation differ across phylogenetic types}
\textbf{Temporal dynamics of C-assimilation in soil.}  
The dynamics of $^{13}$C-xylose and $^{13}$C-cellulose assimilation varied
dramatically. Isotope incorporation increases the bouyant density (BD) of DNA
and labeled DNA is enriched in 'heavy' fractions of the density gradient.
Isotope incorporation by an OTU is revealed by enrichment of the OTU in heavy
CsCl gradient fractions containing $^{13}$C labeled DNA relative to heavy
fractions from control gradients (no $^{13}$C labeled DNA). Variation in 16S
rRNA gene amplicon pool composition in fractions of $^{13}$C-labeled samples
and their corresponding controls is readily observed in 'heavy' gradient
fractions
(\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{partitioning
along axis 2, Fig. 1}). The amplicon pool composition of 'heavy' fractions of
$^{13}$C-xylose and $^{13}$C-cellulose samples vary from corresponding controls
and from each other, indicating that the substrates were assimilated by
different members of the  microbial
community(\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig.
1A}).

The $^{13}$C-incorporation reveals temporal dynamics of C
degradation demonstrated by $^{13}$C-xylose incorporation at days
1, 3, and 7 and $^{13}$C-cellulose incorporation at days 14 and 30
(\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig.
1B}), as expected \citep{Amelung_2008}. The microbial community changed
significantly (pval) with time in the bulk community supporting the temporal
dynamics observed in the gradient fraction amplicons
(\href{https://authorea.com/users/3537/articles/8459/master/file/figures/bulk_ordination/bulk_ordination.png}{Fig.
S2}). Although within a single time point, the bulk community demonstrated no
significant difference between treatments. 

'Heavy' fraction amplicon pools from samples that received
$^{13}$C-xylose diverged from corresponding controls on days 1
through 7
(\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig.
1}). Furthermore, amplicon pool composition varied across these days
(\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig.
1B}) indicating dynamic changes in $^{13}$C-xylose assimilation
with time. At days 14 and 30 heavy fractions from $^{13}$C-xylose
labeled samples are no longer differentiated from corresponding controls
indicating that $^{13}$C is no longer detectable in DNA. The
decline in $^{13}$C-labelling of DNA is likely due to isotopic
dilution resulting from assimilation of unlabeled C and/or due to cell turnover
resulting from mortality. 

$^{13}$C-cellulose incorporation isn't detected until day 14 and
amplicon composition is consistent for both days 14 and 30
(\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig.
1}). The consistency of amplicon composition for cellulose degradation over
time compared to xylose suggests a wider array of microorganisms utilize
xylose, whereas, cellulose utilization occurs in a select few. This is
consistent with long standing notions that more microorganisms are capable of
utilizing simple carbohydrates than complex C substrates. 



\textbf{Differential C utilization by taxa.} Individual OTUs that assimilated
$^{13}$C-substrates were identified using the DESeq framework
\citep{Anders_Huber_2010} to analyze differential representation in 'heavy'
fractions
(\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/l2fc_fig1/l2fc_fig.pdf}{Fig.
2}). There were 43 and 35 unique OTUs that significantly (false discovery rate
corrected \textit{P}-values \textless 0.10,
\href{https://authorea.com/users/3537/articles/8459/_show_article}{SI})
assimilated $^{13}$C-xylose and $^{13}$C-cellulose,
respectively; herein called 'responders'
(\href{https://www.authorea.com/users/3537/articles/8459/master/file/figures/OTU_screening_schematic/OTU_screening_schematic.pdf}{Figs.
S3},
\href{https://www.authorea.com/users/3537/articles/8459/master/file/figures/l2fc_fig_pVal/l2fc_fig_pVal.png}{S4},\href{https://authorea.com/users/3537/articles/8459/master/file/figures/manhattan/manhattan.png}{S5}).






There were 6 shared responders among all unique responders identified in both
the xylose and cellulose treatments (n = 72); Stenotrophomonas, Planctomyces,
two Rhizobiaceae, Comamonadaceae, and Cellvibrio. Of these, Stenotrophomonas
and Comamonadaceae are the only taxa that are among the top ten l2fc responses
measured in both treatments. On the other hand, the only shared responder that
is not among the top ten responders for either the cellulose or xylose
treatment is Rhizobiaceae. Two of the shared responders corresponded in time
between the two treatments
(\href{https://authorea.com/users/3537/articles/8459/master/file/figures/resp_table/resp_table.png}{Table
S1}); Cellvibrio (d3) and Planctomyces (d14).  

\textit{Responder Characteristics}.  We found xylose responders were from
higher rank abundances than cellulose responders, however, cellulose responders
exhibited a greater shift in BD (i.e. assimilated more $^{13}$C)
than xylose responders in response to isotope incorporation
(\href{https://authorea.com/users/3537/articles/3612/master/file/figures/shift_and_rabund2/shift_and_rabund2.png}{Fig.
3}). 

The kernel density estimate (KDE) of BD shifts resulting from
$^{13}$C-assimilation reveal that cellulose responders exhibit a
significantly (\textit{p}\textless 0.01) greater BD shift than xylose
responders
(\href{https://authorea.com/users/3537/articles/3612/master/file/figures/shift_and_rabund2/shift_and_rabund2.png}{Fig.
3A}). A density profile for each responder is generated for the experimental
and control treatment at each of the sampling time points using relative
abundances from sequence libraries
(\href{https://authorea.com/users/3537/articles/8459/master/file/figures/xylose_resp_profiles/xylose_resp_profiles.png}{Figs.
S5}\href{https://authorea.com/users/3537/articles/8459/master/file/figures/cellulose_resp_profiles/cellulose_resp_profiles.png}{,
S6}). The difference in center of mass for each set of density profiles
(control and experimental) is measured (supp. MM) and each KDE curve represents
the collection of density shifts calculated for all responders in the
$^{13}$C-cellulose or $^{13}$C-xylose treatment
(\href{https://authorea.com/users/3537/articles/3612/master/file/figures/shift_and_rabund2/shift_and_rabund2.png}{Fig.
3A}). We observe xylose utilizers having a smaller density shift (0.008 $\pm$
0.008 g mL$^{-1}$) than cellulose utilizers (0.015 $\pm$ 0.009 g
mL$^{-1}$), with few exceptions. 

Most xylose responders are found at higher rank abundances than cellulose
responders (0.01 \textless \textit{p} \textless 0.05), which fall among the
rarer taxa in the tail of the RA curve (Fig 3B). This demonstrates that many
taxa important to cellulose cycling are present in the rarer fraction of the
overall microbial community. Yet, the transitions in abundances of responders
is difficult to discern in the bulk community abundances
(\href{https://authorea.com/users/3537/articles/8459/master/file/figures/xylose_resp_profiles/xylose_resp_profiles.png}{Figs.
S5}
\href{https://authorea.com/users/3537/articles/8459/master/file/figures/cellulose_resp_profiles/cellulose_resp_profiles.png}{,
S6}) or may not be detected with bulk community sequencing efforts. For
example, the increase in Bacteroidetes in the xylose treatment at d3 is not
observed in the bulk community abundances. Other instances may result in subtle
changes in bulk community abundance that would be difficult to differentiate
from natural variation or methodological noise.


\textbf{Patterns of carbon use vary dramatically within phylum.} Dynamic
patterns of $^{13}$C-assimilation from xylose and cellulose occur
at discrete, fine-scale taxonomic units
(\href{https://authorea.com/users/3537/articles/3612/master/file/figures/bacteria_tree/bacteria_tree.png}{Fig.
4}). Responders for xylose and cellulose are widespread across 6 and 7 phyla,
respectively
(\href{https://authorea.com/users/3537/articles/3612/master/file/figures/bacteria_tree/bacteria_tree.png}{Fig.
4}). There are 5 phyla containing responders for both treatments; of all the
responder OTUS detected within those phyla for either xylose or cellulose,
there are only six OTUs that respond to both xylose and cellulose (discussed
previously). This result suggests that phyla do not represent coherent
ecological units with respect to the soil C-cycle, that is, taxa within phyla
exhibit differences in substrate use, level of substrate specialization, and
dynamics of incorporation. 

In this study, we have identified Actinobacteria responders for both substrates
(\href{https://authorea.com/users/3537/articles/8459/master/file/figures/xylose_resp_profiles/xylose_resp_profiles.png}{Figs.
S5}
\href{https://authorea.com/users/3537/articles/8459/master/file/figures/cellulose_resp_profiles/cellulose_resp_profiles.png}{,
S6}). Although there were no shared Actinobacteria OTUs that responded to both
xylose (Microbacteriaceae, Micrococcaceae, Cellulomonadaceae, Nakamurellaceae,
Promicromonosporaceae, and Geodermatophilaceae) and cellulose
(Streptomycetaceae and Pseudonocardiaceae). This information may suggest that
while Actinobacteria exhibit an ability to utilize an array of carbon
substrates, substrate use may be more clade specific and not widespread
throughout the phylum
(\href{https://authorea.com/users/3537/articles/3612/master/file/figures/bacteria_tree/bacteria_tree.png}{Fig.
4}). Similarly, Bacteroidetes responders were identified for both substrates,
yet, at a finer taxonomic resolution there is a clear differential response for
xylose (Flavobacteriaceae and Chitinophagaceae) and cellulose (Cytophagaceae). 


Whole phylum responses were not detected for xylose or cellulose yet
utilization of these substrates spanned many phylogenetically diverse groups.
Within each phylum we observed substrate utilization at the clade or single
taxa level with each exhibiting a unique pattern of
$^{13}$C-assimilation over time
(\href{https://authorea.com/users/3537/articles/3612/master/file/figures/bacteria_tree/bacteria_tree.png}{Fig.
4, heatmap}). It has previously been suggested that all taxa within a phylum
are unlikely to share ecological characteristics \citep{Fierer_2007}, and
furthermore, within a species population
\citep{Choudoir_2012,Preheim_2011,Hunt_2008}. Habitat traits of coastal Vibrio
isolates were mapped onto microbial phylogeny revealing discrete ecological
populations based on seasonal occurrence and particulate size fractionation
\citep{Preheim_2011,Hunt_2008}. Yet, it has been proposed that the microbial
community functionality responsible for soil C cycling appear at the level of
phlya rather than species/genera \citep{Schimel_2012}. The traditional phylum
level assignment conventions could in part be due to limitations in finer scale
taxonomic identifications or methodological limitations (\textit{i.e.}
sequencing depth). Our data in concert with others
\citep{Goldfarb_2011,Fierer_2007,Choudoir_2012,Preheim_2011,Hunt_2008} would
suggest that assigning substrate utilization of a few OTUs or clades as a
phylum level response is not accurate.

\textbf{Conclusions.} We have demonstrated how next generation
sequencing-enabled SIP gives an OTU level resolution for substrate utilization.
Using this technique, we are able to resolve discrete OTUs that would otherwise
be missed using bulk community sequencing efforts. Additionally, this technique
provides greater taxonomic resolution than previous techniques (cloning, TRFLP,
ARISA) used to determine substrate utilizing community members. While we are
currently able to resolve highly responsive OTUs, there is still a need to
resolve taxa that are partially responsive which we cannot differentiate from
noise with confidence at this time. Although, if we could identify partially
responsive taxa, their contributions to the C-cycle would still be difficult to
discern. For example, a generalist utilizing many substrates including
$^{12}$C substrates and the $^{13}$C-labeled
substrate may exhibit the same partial labeling that a specialist utilizing
both the $^{13}$C-substrate and the same substrate (unlabeled)
that is inherent in the soil. Additionally, partially labeled taxa could be
further down the trophic cascade including predators or secondary consumers of
waste products from primary consumer microbes that were highly labeled.   

OTUs that assimilate xylose and those that assimilate cellulose are largely
mutually exclusive. Those OTUs that assimilate xylose are labeled within 1-7
days, while those that assimilate cellulose are labeled primarily after 2-4
weeks. The xylose responders demonstrate a smaller change in BD than the
cellulose responders suggesting that xylose responders assimilate multiple C
sources (labeled and unlabeled) consistent with a generalist response, while
cellulose responders are more heavily labeled suggesting that cellulose is
their main source of C, a response more consistent with a specialist lifestyle.
Xylose responders include many taxa, such as spore-fomers, known for the
ability to respond rapidly to an influx of new nutrients while cellulose
responders include many OTUs that are common uncultivated soil organisms.
Finally, xylose responders are more abundant in the community while cellulose
responders are, on average, more rare as indicated by their rank abundance
within the soil community. These results indicate that different bacteria in
soil have distinct physiological and ecological responses which govern their
interactions with soil C pools. 

We did not observe consistent C utilization at the phylum level although both
xylose and cellulose utilization were observed across 7 phyla each revealing a
high diversity of bacteria able to utilize these substrates. The high taxonomic
diversity may enable substrate metabolism under a broad range of environmental
conditions \citep{Goldfarb_2011}. Other studies of microbial communities have
observed a positive correlation with taxonomic or phylogenetic diversity and
functional diversity
\citep{Fierer_2012,Fierer_2013,Philippot_2010,Tringe_2005,Gilbert_2010,Bryant_2012}.
The data presented here supports that specific functional attributes can be
shared among diverse, yet distinct, taxa while closely related taxa may have
very different physiologies \citep{Fierer_2012,Philippot_2010}. This information
adds to the growing collection of data suggesting that community membership is
important to biogeochemical processes. Furthermore, it highlights a need to
examine substrate utilization by discrete microbial taxa within a whole
community context to better understand how specific community members function
within the whole. 

The sensitivity of SIP-NGS provides a means to elucidate substrate utilization by discrete microbial taxa with the hope that we can begin to construct a belowground C food web. We obtained enough information to conclusively determine isotope incorporation for 61\% of the more than 6,000 OTUs detected. For those OTUs with enough information (n = 3,825), approximately 2\% (n = 72) significantly assimilated $^{13}$C from either xylose or cellulose. In the future deeper sequencing will enable us to increase coverage and assess C use by more community members. Using the informations we gain from SIP-NGS, we can expand our knowledge of specific C-cycling OTUs by taking a targeted metagenomic approach in the nucleic acid pools of 'heavy' fractions. Furthermore, we can now expand our knowledge of soil C use dynamics to a wide array of C substrates and increase our grasp on specific community member contributions. Illuminating these microbial contributions associated with decomposition in soil are important because as environments change, there are measurable and functional changes in soil C \citep{Grandy_2008} which could cumulatively have large impacts at a global scale.
