\section{Results}
% Fakesubsubsection:We tracked the flow of C from xylose
After adding the organic matter amendment to soil, we tracked the flow of
C from xylose or C from cellulose into microbial DNA over time using DNA-SIP
(Figure~\ref{fig:setup}). The amendment consisted of various plant biomass
compounds including cellulose, lignin, sugars found in hemicellulose, amino
acids, and inorganic salts (see Supplemental Methods). The amendment was added
at 2.3 mg C g$^{-1}$ soil dry weight (d.w.), and this comprised 16\% of the
total C in the soil. The cellulose-C (0.88 mg C g$^{-1}$ soil d.w.) and
xylose-C (0.42 mg C g$^{-1}$ soil d.w.) in the amendment comprised 6\% and 3\%
of the total C in the soil, respectively. The soil microbial community respired
65\% of the xylose within one day (Figure~\ref{fig:setup}) and 29\% of the
added xylose remained in the soil at day 30 (Figure~\ref{fig:setup}). In
contrast, cellulose-C declined at a constant rate of approximately 18 $\mu$g
C d $^{-1}$ g $^{-1}$ soil d.w. and 40\% of added cellulose-C remained in the
soil at day~30 (Figure~\ref{fig:setup}).

\subsection{Soil microcosm microbial community changes with time}
% Fakesubsubsection:Changes in the soil microcosm microbial community structure
We assessed assimilation of $^{13}$C into microbial DNA by comparing the SSU
rRNA gene sequence composition of SIP density gradient fractions from
treatments including a $^{13}$C label relative to the control treatment. All
treatments used the same amendment which included xylose and cellulose, but
$^{13}$C-xylose or $^{13}$C-cellulose was substituted for its unlabeled
equivalent in two amendments. A treatment without isotopically labeled
components served as the ``control''. In the control gradient density fractions
the majority of the variance in SSU rRNA gene composition was represented by
fraction density (Figure~\ref{fig:ord}). DNA buoyant density correlates with
G$+$C content \citep{Buckley_2007} and therefore DNA G$+$C content influences
variation in the SSU rRNA gene composition of density gradient fractions. For
the $^{13}$C-cellulose treatment, SSU rRNA gene composition in gradient
fractions deviated from control at high density ($>$~1.72 g mL$^{-1}$) on days
14 and 30 (Figure~\ref{fig:ord}). For the $^{13}$C-xylose treatment, SSU rRNA
gene composition in density gradient fractions also deviated from control in
high density fractions, but in contrast to the $^{13}$C-cellulose treatment it
deviated from control on days~1,~3,~and~7 (Figure~\ref{fig:ord}). SSU rRNA gene
composition from $^{13}$C-cellulose treatment and $^{13}$C-xylose treatment
density fractions differed at high density indicating different microorganisms
assimilated C from xylose than C from cellulose
(Figure~\ref{fig:ord}). Further, the SSU rRNA gene sequence composition of high
density fractions from $^{13}$C-cellulose treatments at days 14 and 30 was
similar indicating similar microorganisms had $^{13}$C labeled DNA in
$^{13}$C-cellulose treatments at days 14 and 30. Contrastingly, in the
$^{13}$C-xylose treatment the SSU rRNA gene composition high density gradient
fractions varied between days~1,~3,~and~7 indicating that different microbes
had $^{13}$C labeled DNA on these days. In the $^{13}$C-xylose treatment, the
SSU gene composition of high density fractions was similar to control on
days~14~and~30 (Figure~\ref{fig:ord}) indicating that $^{13}$C was no longer
detectable on these days for this treatment. 

\subsection{Temporal dynamics DNA $^{13}$C incorporation of OTUs}
% Fakesubsubsection:We monitored the soil microbial community
We monitored the soil microcosm microbial community over the course of the
experiment by surveying SSU rRNA genes in non-fractionated DNA from the soil
microcosms. The SSU rRNA gene composition of the non-fractionated DNA changed
with time (Figure~\ref{fig:bulk_ord}, P-value $=$ 0.023, R$^{2}$ $=$ 0.63,
Adonis test \citep{Anderson2001a}). In contrast, the community showed no
statistical evidence for changing with treatment (P-value was 0.23)
(Figure~\ref{fig:bulk_ord}). The latter result demonstrates the substitution of
$^{13}$C-labeled substrates for unlabeled equivalents did not alter community
composition beyond what we would expect at random. The variance in
SSU rRNA gene composition among all non-fractionated DNA samples across
treatment and time was less than the variance among gradient fractions
(Figure~\ref{fig:bulk_ord}, P-value $=$ 0.003, “betadisper” function R Vegan
package \citep{oksanen2007vegan}) and therefore variance in SSU rRNA gene
composition due to the process of density gradient fractionation exceeded
variance due to temporal changes in the non-fractionated DNA.

% Fakesubsubsection:Twenty-nine OTUs
Twenty-nine OTUs exhibited sufficient statistical evidence (adjusted P-value
$<$ 0.10) to conclude they changed in relative abundance over the
course of the experiment. When SSU rRNA gene abundances were combined at
the taxonomic rank of "class", the classes that changed in abundance (P-value
$<$ 0.10) were the \textit{Bacilli} (decreased), \textit{Flavobacteria}
(decreased), \textit{Gammaproteobacteria} (decreased), and
\textit{Herpetosiphonales} (increased) (Figure~\ref{fig:time_class}). Of the 29
OTUs that changed in relative abundance over time, 14 putatively
incorporated $^{13}$C into DNA (Figure~\ref{fig:time}). OTUs that likely
assimilated $^{13}$C from $^{13}$C-cellulose into DNA tended to increase in
relative abundance with time whereas OTUs that assimilated $^{13}$C from
$^{13}$C-xylose tended to decrease over time. Those OTUs that responded to both
substrates did not exhibit a consistent relative abundance response over time
as a group (Figure~\ref{fig:time}~and~\ref{fig:babund}).

\subsection{OTUs that assimilated $^{13}$C into DNA} \label{responders}
% Fakesubsubsection:If an OTU exhibited
If an OTU exhibited strong evidence for assimilating $^{13}$C 
into DNA, we refer to that OTU as a "responder" (see Supplemental Note 1.7.4
for our operational definition of "responder"). The SSU rRNA gene sequences
produced in this study could be distributed into 5,940 OTUs and we assessed the
evidence of $^{13}$C incorporation into DNA from $^{13}$C-cellulose and
$^{13}$C-xylose for each OTU. Forty-one OTUs responded to $^{13}$C-xylose,
55 OTUs responded to $^{13}$C-cellulose, and 8 OTUs 
responded to both xylose and cellulose (Figure~\ref{fig:l2fc},
Tables~{tab:cell}~and~\ref{tab:xyl}). The number of xylose responders
peaked at days 1 and 3 and declined with time. In contrast, the number of
cellulose responders increased with time peaking at days 14 and 30
(Figure~\ref{fig:rspndr_count}). 

The phylogenetic composition of xylose responders changed with time and the
majority (86\%) of xylose responders shared > 97\% SSU rRNA gene sequence
identity with bacteria cultured in isolation
(Figure~\ref{fig:l2fc}~and~\ref{fig:xyl_count}). On day 1, \textit{Bacilli}
OTUs represented 84\% of xylose responders and the majority of these OTUs were
closely related to cultured representatives of the genus \textit{Paenibacillus}
(Table~\ref{tab:xyl}). For example, "OTU.57" (Table\ref{tab:xyl}),
annotated as \textit{Paenibacillus}, had a strong signal of $^{13}$C
assimilation from $^{13}$C-xylose into DNA at day 1 coinciding with its maximum
relative abundance in non-fractionated DNA. The relative abundance of "OTU.57"
declined until day 14 and did not appear to be $^{13}$C labeled after day~1
(Figure~\ref{fig:example}). On day 3, \textit{Bacteroidetes} OTUs comprised
63\% of xylose responders (Figure~\ref{fig:xyl_count}) and these OTUs were
closely related to cultured representatives of the \textit{Flavobacteriales}
and \textit{Sphingobacteriales} (Table~\ref{tab:xyl}). For example, ``OTU.14'',
annotated as a Flavobacterium, had a strong signal for $^{13}$C labeling in the
$^{13}$C-xylose treatment at days 1 and 3 coinciding with its maximum relative
abundance in non-fractionated DNA. The relative abundance of ``OTU.14'' then
declined until day 14 and did not show evidence of $^{13}$C labeling beyond
day~3 (Figure~\ref{fig:example}). Finally, on day~7, \textit{Actinobacteria}
OTUs represented 53\% of the xylose responders and these OTUs were closely
related to cultured representatives of \textit{Micrococcales}
(Table~\ref{tab:xyl}). For example, ``OTU.4'', annotated as \textit{Agromyces},
had signal of $^{13}$C labeling in the $^{13}$C-xylose treatment on days 1,
3 and 7 with the strongest evidence of $^{13}$C labeling at day~7 and did not
appear $^{13}$C labeled at days~14 and~30. ``OTU.4'' relative abundance in
non-fractionated DNA increased until day~3 and then declined until
day~30 (Figure\ref{fig:example}). \textit{Proteobacteria} were also common
among xylose responders at day~7 where they comprised 40\% of xylose responder
OTUs. Notably, \textit{Proteobacteria} represented the majority (6 of 8) of
OTUs that responded to both cellulose and xylose. 

%Fakesubsubsection:Cellulose responders were
The phylogenetic composition of cellulose responders did not change with time
unlike the phylogenetic composition of xylose responders. Also, in contrast to
xylose responders, cellulose responders often were not closely related ($<$
97\% SSU rRNA gene sequence identity) to cultured isolates. Both the relative
abundance and the number of cellulose responders increased over time peaking at
days 14 and 30 (Figures~\ref{fig:l2fc}, \ref{fig:rspndr_count}, and
\ref{fig:babund}). Cellulose responders belonged to the \textit{Proteobacteria}
(46\%), \textit{Verrucomicrobia} (16\%), \textit{Planctomycetes} (16\%),
\textit{Chloroflexi} (8\%), \textit{Bacteroidetes} (8\%),
\textit{Actinobacteria} (3\%), and \textit{Melainabacteria} (1 OTU)
(Table~\ref{tab:cell}). The majority (86\%) of cellulose responders in the
\textit{Proteobacteria} were closely related ($>$ 97\% identity) to bacteria
already cultured in isolation, including representatives of the genera:
\textit{Cellvibrio}, \textit{Devosia}, \textit{Rhizobium}, and
\textit{Sorangium}, which are all known for their ability to degrade cellulose
(Table~\ref{tab:cell}). Proteobacterial cellulose responders belonged to
\textit{Alpha-} (13~OTUs), \textit{Beta-} (4~OTUs), \textit{Gamma-} (5~OTUs),
and \textit{Deltaproteobacteria} (6~OTUs). 

% Fakesubsubsection:The majority (85\%) of cellulose
The majority (85\%) of cellulose responders outside of the
\textit{Proteobacteria} shared  $<$ 97\% SSU rRNA gene sequence identity to
bacteria already cultured in isolation. For example, most (70\%) of the
\textit{Verrucomicrobia} cellulose responders fell within unidentified
\textit{Spartobacteria} clades, and these shared $<$ 85\% SSU rRNA gene
sequence identity to any characterized isolate. The \textit{Spartobacteria} OTU
``OTU.2192'' exemplified many cellulose responders (Figure~\ref{:fig:example}).
``OTU.2192'' increased in non-fractionated DNA relative abundance
with time and evidence for $^{13}$C labeling of ``OTU.2192'' in the
$^{13}$C-cellulose treatment increased over time with the strongest evidence at
days~14 and~30 (Figure\ref{fig:example}). Most \textit{Choloflexi} cellulose
responders belonged to an unidentified clade within the
\textit{Herpetosiphonales} and they shared $<$ 89\% SSU rRNA gene sequence
identity to any characterized isolate. Characteristic of \textit{Chloroflexi}
cellulose responders, "OTU.64" increased in relative abundance over 30 days and
evidence for $^{13}$C labeling of ``OTU.64'' in the $^{13}$C-cellulose
treatment peaked days 14 and~30 (Figure~\ref{fig:example}). Cellulose
responders found within the \textit{Bacteroidetes} fell within the
\textit{Cytophagales} contrasting with \textit{Bacteroidetes} xylose responders
that fell instead within the \textit{Flavobacteria} or
\textit{Sphingobacteriales}. \textit{Bacteroidetes} cellulose responders
included one OTU that shared 100\% SSU rRNA gene sequence identity to species
of \textit{Sporocytophaga}, a genus that includes known cellulose degraders.

\subsection{Characteristics of cellulose and xylose responders}
% Fakesubsubsection:Cellulose responders tended
Cellulose responders, relative to xylose responders, tended to have lower
relative abundance in non-fractionated DNA, demonstrated signal consistent with
higher atom \% $^{13}$C in labeled DNA, and have lower estimated \textit{rrn} copy number. In the
non-fractionated DNA, cellulose responders had lower relative abundance
(7e$^{-4}$ (s.d. 2e$^{-3}$)) than xylose responders (2e$^{-3}$ (s.d.
4e$^{-3}$)) (Figure~\ref{fig:xyl_count}, P-value $=$ 0.00028, Wilcoxon Rank Sum
test). Six of the ten most common OTUs observed in the non-fractionated DNA
responded to xylose, and, eight of the ten most abundant responders to xylose
or cellulose in the non-fractionated DNA were xylose responders. However,
xylose and cellulose responders included OTUs at both high and low abundance
(Figure~\ref{fig:shift}).

% Fakesubsubsection:DNA buoyant density increases as the amount
DNA buoyant density (BD) increases in proportion to the atom \% $^{13}$C of the
DNA.

Hence, the extent of $^{13}$C incorporation into DNA can be evaluated as the
change in BD in enriched treatments relative to control. We calculated for each
OTU its mean BD weighted by relative abundance to determine its “center of
mass” within a given density gradient. We then quantified for each OTU the
difference in center of mass between control gradients and gradients from
13C-xylose or 13C-cellulose treatments (see supplemental methods for the
detailed calculation). 

Therefore, the amount of $^{13}$C per unit DNA can be evaluated by the change
in DNA buoyant density (BD) upon $^{13}$C labeling. In this study, we found the
density weighted average for each OTU in each gradient (i.e. mean density
weighted by relative abundance). This weighted average is the "center of mass"
of an OTU's relative abundance profile across a density gradient. We then
quantified the difference in each OTU's center of mass between control
gradients and corresponding $^{13}$C labeled gradients (see supplemental
methods for the detailed calculation). 

We refer to the change in center of mass position for an OTU in response to
$^{13}$C labeling as $\Delta\hat{BD}$. $\Delta\hat{BD}$ can be used to compare
relative differences in $^{13}$C labeling between OTUs. $\Delta\hat{BD}$
values, however, are not comparable to the BD changes observed for DNA from
pure cultures which generate molecules uniform in isotopic label, in part
because $\Delta\hat{BD}$ is based on relative abundance in density gradient
fractions (and not DNA concentration) and in part because all members of an OTU
may not uniformly respond to the isotopic label. Cellulose responder
$\Delta\hat{BD}$ (0.0163 g mL$^{-1}$ (s.d.~0.0094)) was greater than that of
xylose responders (0.0097 g mL$^{-1}$ (s.d.~0.0094)) (Figure~\ref{fig:shift},
P-value $=$
1.8610e$^{-6}$, Wilcoxon Rank Sum test). 

% Fakesubsubsection:We predicted the rrn
We predicted the \textit{rrn} gene copy number for responders as described
\citep{Kembel_2012}. The number of \textit{rrn} gene copies a microorganism has
is correlated to it's ability proliferate in response to rapid nutrient influx
\citep{Klappenbach_2000}. Cellulose responders possessed fewer estimated
\textit{rrn} copy numbers (2.7~(1.2~sd)) than xylose responders (6.2~(3.4~s.d))
(Figures~\ref{fig:shift} and \ref{fig:copy}; P = 1.878e$^{-9}$). Furthermore,
the estimated \textit{rrn} gene copy number for xylose responders was inversely
related to the day of first response (P = 2.02e$^{-15}$,
Figure~\ref{fig:copy}).

% Fakesubsubsection:We assessed phylogenetic
We assessed phylogenetic clustering of $^{13}$C-responsive OTUs with the
Nearest Taxon Index (NTI) and the Net Relatedness Index (NRI)
\citep{Webb2000}. We also quantified the average clade depth of cellulose and
xylose responders with the consenTRAIT metric \citep{Martiny2013}. Briefly, the
NRI and NTI evaluate phylogenetic clustering against a null model for the
distribution of a trait in a phylogeny. The NRI and NTI values are z-scores or
standard deviations from the mean and thus the greater the magnitude of the
NRI/NTI, the stronger the evidence for clustering (positive values) or
overdispersion (negative values). NRI assesses overall clustering whereas the
NTI assesses terminal clustering. An NRI of 1.96, for instance, would signify
overall phylogenetic clustering with a corresponding P-value of 0.05
\citep{Evans2014a}. The consenTRAIT metric is a measure of the average clade
depth for a trait in a phylogenetic tree. NRI values indicate that cellulose
responders clustered overall and at the tips of the phylogeny (NRI:~4.49,
NTI:~1.43) while xylose responders cluster terminally (NRI:~-1.33, NTI:~2.69).
The consenTRAIT clade depth for xylose and cellulose responders was~0.012
and~0.028 SSU rRNA gene sequence dissimilarity, respectively. As reference, the
average clade depth is 0.017 SSU rRNA gene sequence dissimilarity for arabinose
utilization (another five C sugar found in hemicellulose) and was~0.013
and~0.034 SSU rRNA gene sequence dissimilarity for glucosidase and cellulase
activity of isolates in culture, respectively
\citep{Martiny2013,Berlemont2013}. These results indicate xylose responders
form terminal clusters dispersed throughout the phylogeny while cellulose
responders form deep clades of terminally clustered OTUs.
