\section{Results}
In this study, we couple nucleic acid SIP with next generation sequencing
(SIP-NGS) to observe C use dynamics by the soil microbial community. A series
of parallel soil microcosms all amended with a C substrate mixture were
incubated for 30 days. The substrate mixture was identical for each bottle
except in one series of bottles the cellulose was $^{13}$C-labeled in another
the xylose was $^{13}$C-labeld and in the last no sustrattes were labeled. The
C substrate mixture was designed to approximate freshly degrading plant
biomass. Xylose or cellulose carried the isotopic label so we could examine C
assimilation dynamics for labile, soluble C versus insoluble, polymeric C. 5.3
mg total mass of C substrate mixture per gram soil (including 0.42 mg xylose-C
and 0.88 mg cellulose-C g soil$^{-1}$) was added to each microcosm representing
18\% of the total soil C. Microcosms were harvested at several time points
during the incubation period and $^{13}$C assimilation was observed by
sequencing 16S rRNA gene amplicons from bulk soil DNA and CsCl gradient
fractions. Assimilation of $^{13}$-C from Xylose peaked immediately and tapered
over the 30 day incubation whereas cellulose $^{13}$C assimilation peaked after
two weeks of incubation (Figure~\ref{fig:ord}).
\subsection{Ordination of CsCl gradient fraction OTU profiles can be used to
observe fraction-level $^{13}$C assimilation dynamics and membership}
Variation in 16S rRNA gene amplicon pool composition in fractions of
$^{13}$C-labeled samples and their corresponding controls is readily observed
in 'heavy' gradient fractions. The amplicon pool composition of 'heavy'
fractions of $^{13}$C-xylose and $^{13}$C-cellulose samples vary from
corresponding controls and from each other, indicating that the substrates were
assimilated by different members of the  microbial community.
Analysis of 16S rRNA gene surveys has greatly benefitted from utilizing
conventional methods for data exploratoration in ecology such as ordination \citep{Lozupone_2008}. 
Recently, 16S rRNA gene profiles in CsCl gradient fractions have been surveyed
with NGS technology and the gradient 16S rRNA 
phylogenetic profiles explored via ordination \citep{Angel_2013, Verastegui_2014}. Ordination of CsCl gradient fraction
phylogenetic profiles has reveled the relative influence of buoyant density and 
soil type on gradient fraction phylogenetic profile variance. However, ordination
has not been used to demonstrate isotope incorporation into DNA which requires careful
comparisons between control and labeled gradients over the same buoyant density range. By
sequencing all CsCl gradient fractions from both control and labeled gradients, we can 
observe when--as in at want time point during incubation--as well as \textit{where}--as in
at what buoyant densities along the CsCl gradients--does isotope incorporation signal 
becomes apparent (Figure~\ref{fig:ord}). Specifically, $^{13}$C incorporation from xylose
and cellulose is most apparent at days 1/3/7 and days 14/30, respectively 
(Figure~\ref{fig:ord}). Moreover, labeled gradient fraction phylogenetic profiles diverge 
from controls in relatively heavy buoyant densities (Figure~\ref{fig:ord}). Also apparent
from the ordination of CsCl gradient phylogenetic profiles is that OTUs responsive to 
cellulose are generally different than those responsive to xylose and last,
that $^{13}$C-xylose responders change in phylogenetic type over incubation
days 1, 3 and 7 (Figure~\ref{fig:ord}).

\subsection{$^{13}$C from cellulose was assimilated by canonical
cellulose-degrading and uncharacterized microbial lineages in many phyla
including Chloroflexi and Verrucomicrobia} 
Isotope incorporation by an OTU is revealed by enrichment of the OTU in heavy
CsCl gradient fractions containing $^{13}$C labeled DNA relative to heavy
fractions from control gradients (no $^{13}$C labeled DNA). Only 2 and 5 OTUs
were found to have incorporated $^{13}$C from labeled cellulose at days 3 and
7, respectively. At days 14 and 30, however, 42 and 39
OTUs were found to incorporate $^{13}$C from cellulose into biomass. An average
16\% of the $^{13}$C-cellulose added was respired within the first 7 days, 38\%
by day 14, and 60\% by day 30. A \textit{Cellvibrio} and
\textit{Sandaracinaceae} OTU assimilated $^{13}$C from cellulose at day 3. Day
7 responders included the same \textit{Cellvibrio} responder as day 3, a
\textit{Verrucomicrobia} OTU and three \textit{Chloroflexi} OTUs. 50\% of Day
14 responders belong to Proteobacteria (66\% Alpha-, 19\% Gamma-, and 14\%
Beta-) followed by 17\% \textit{Planctomycetes}, 14\% \textit{Verrucomicrobia},
10\% \textit{Chloroflexi}, 7\% \textit{Actinobacteria} and 2\%
cyanobacteria. \textit{Bacteroidetes} OTUs begin to incoporate
$^{13}$C from cellulose at day 30 (13\% of day 30 responders). Other day 30
responding phyla include \textit{Proteobacteria} (30\% of day 30 responders;
42\% Alpha-, 42\% Delta, 8\% Gamma-, and 8\% Beta-), \textit{Planctomycetes}
(20\%), \textit{Verrucomicrobia} (20\%), \textit{Chloroflexi} (13\%) and
cyanobacteria (3\%). \textit{Proteobacteria},
\textit{Verrucomicrobia}, and \textit{Chloroflexi} had relatively high numbers
of responders with heavy response across multiple time points
(Figure~\ref{fig:l2fc}).

\textit{Proteobacteria} represent 46\% of all cellulose responding OTUs
identified. \textit{Cellvibrio} accounted for 3\% of all proteobacterial
responding OTUs detected. \textit{Cellvibrio} was one of the first identified
cellulose degrading bacteria and was originally described by Winogradsky in
1929 who named it for its cellulose degrading abilities
\citep{boone2001bergeys}. All $^{13}$C-cellulose responding
\textit{Proteobacteria} share high sequence identity with 16S rRNA genes from
sequenced cultured isolatets (Table XX) except for OTU.442 (best cultured
isolate match 92\% sequence identity in the \textit{Chrondomyces} genus) and
OTU.663 (best cultured isolate match outside \textit{Proteobacteria} entirely,
\textit{Clostridium} genus, 89\% sequence identity). Some
\textit{Proteobacteria} responders share high sequence identity with type
strains for genera known to posess cellulose degraders including
\textit{Rhizobium}, \textit{Devosia}, \textit{Stenotrophomonas} and
\textit{Cellvibrio}. One \textit{Proteobacteria} OTU shares high sequence
identity with the \textit{Brevundimonas} cultured isolate.
\textit{Brevundimonas} has not previously been identified as a cellulose
degrader, but has been shown to degrade cellouronic acid, an oxidized form of
cellulose \citep{Tavernier_2008}.

\textit{Verrucomicrobia}, a cosmopolitan soil phylum often found in high
abundance \citep{Fierer_2013}, are implicated in polysaccaride degradation in
many environments \citep{Fierer_2013,Herlemann_2013,10543821}.
\textit{Verrucomicrobia} comprise 16\% of the total cellulose responder OTUs
detected. 40\% of \textit{Verrucomicrobia} responders belong to the uncultured
``FukuN18'' family originally identified in freshwater lakes
\citep{Parveen_2013}.  The \textit{Verrucomicrobia} OTU with the strongest
\textit{Verrucomicrobial} response to $^{13}$C-cellulose shared high sequence
identity (97\%) with an isolate from Norway tundra soil \citep{Jiang_2011}
although growth on cellulose was not assessed for this isolate. Only one other
$^{13}$C-cellulose responding verrucomicrobium shared high DNA sequence
identity with a sequenced type strain, ``OTU.638'' (Table XX) with
\textit{Roseimicrobium gellanilyticum} (100\% sequence identity).
\textit{Roseimicrobium gellanilyticum} grows on soluble celluose
\citep{Otsuka_2012}. The remaining $^{13}$C-cellulose \textit{Verrucomicrobia}
responders did not share high sequence identity with any cultured isolates
(maximum sequence identity with any cultured isolate 93\%). 

\textit{Chloroflexi} are traditionally known for their metabolically dynamic
lifestyles ranging from anoxygenic phototropy to organohalide respiration
\citep{Hug_2013}. Recent studies have focused on \textit{Chloroflexi} roles in
C cycling \citep{Hug_2013, Goldfarb_2011,Cole_2013} and several \textit{Chloroflexi} utilize cellulose \citep{Goldfarb_2011, Cole_2013,
Hug_2013}. Four closely related OTUs in an undescribed \textit{Chloroflexi}
lineage (closest matching cultured isolate for all four OTUs:
\textit{Herpetosiphon geysericola}, 89\% sequence identity) responded to
$^{13}$C-cellulose (Figure~\ref{fig:trees}). One additional OTU also from a
poorly characterized \textit{Chloroflexi} lineage (closest cultured isolate match a proteobacterium
at 78\% sequence identity) responded to $^{13}$C-cellulose
(Figure~\ref{fig:trees}).

Other notable $^{13}$C cellulose responders include a \textit{Bacteroidetes}
OTU that shares high sequence identity (99\%) to \textit{Sporocytophaga
myxococcoides} a known cellulose degrader \citep{Vance_1980}, and three
\textit{Actinobacteria} OTUs that share high sequence identity (100\%) with
sequenced cultured isolates. One of the three \textit{Actinobacteria}
$^{13}$C-cellulose responders is in the \textit{Streptomyces}, a genus known to
possess cellulose degraders, while the other two closely match the cultured isolates
\textit{Allokutzneriz albata} \citep{Labeda_2008, Tomita_1993} and
\textit{Lentzea waywayandensis} \citep{LABEDA_1989, Labeda_2001}, that do not
decompose cellulose in culture. Nine \textit{Plantomycetes} OTUs responded to
$^{13}$C-cellulose but none are within described genera (closest cultured isolate
match 91\% sequence identity) (Figure~\ref{fig:trees}). Interestingly, one
$^{13}$C-cellulose responder is annotated as belonging in the
cyanobacteria. The phylum annotation is misleading as the OTU is not
closely related to any oxygenic phototrophs (closest cultured isolate match
\textit{Vampirovibrio chlorellavorus}, 95\% sequence identity). A sister clade
to the oxygenic phototrophs classically annotated as ``cyanobacteria'' in
SSU rRNA gene reference databases but does not possess known phototrophs has
recently been proposed to constitute its own phylum, "Melainabacteria"
\citet{Di_Rienzi_2013}, although its phylogenetic position is debated
\citep{Soo_2014}. The catalog of metabolic capabilities associated with
cyanobacteria (or candidate phyla previously annotated as
cyanobacteria) are quickly expanding \citep{Di_Rienzi_2013, Soo_2014}.
Our findings provide evidence of cellulose degradation within a lineage closely
related to but apart from oxygenic phototrophs. Notably, polysaccharide
degradation is suggested by an analysis of a \textit{Melainabacterial} genome
\citep{Di_Rienzi_2013}. Although we highlight $^{13}$C-cellulose responders
that share high sequence identity with described genera, most
$^{13}$C-cellulose responders uncovered in this experiment are not closely
related to cultured isolates (Table~XX).

\subsection{Putative spore-formers in the Firmicutes assimilate $^{13}$C from
xylose within first day after soil amendment followed by Bacteroidetes and then
Actinobacteria OTUs} 
Within the first 7 days of incubation an average 63\% of $^{13}$C-xylose was
respired and only an additional 6\% more was respired between days 7 and 30. At
the end of the 30 day experiment 30\% of the original $^{13}$C from xylose
remained in the soils. The $^{13}$C remaining in the soil from $^{13}$C-xylose
addition has likely been stabilized by assimilation into microbial biomass
and/or microbial conversion into other forms of organic matter, though it is
possible that some $^{13}$C-xylose remains unavailable to microbes due to
abiotic interactions in soil \citep{Kalbitz_2000}. All xylose
responders were first responsive in first 7 incubation days. 

At day 1, 84\% of xylose responsive OTUs belong to Firmicutes, 11\% to
\textit{Proteobacteria} and 5\% to \textit{Bacteroidetes}. At day 3,
Firmicutes responders decreased to 5\% (from 16 OTUs to 1) while
\textit{Bacteroidetes} increased to 63\% (from 1 to 12 OTUs) of day 3
responders. The remaining day 3 responders are members of the
\textit{Proteobacteria} (26\%) and the \textit{Verrucomicrobia} (5\%). Day 7
responders were 53\% Actinobacteria, 40\% Proteobacteria, and 7\% Firmicutes. A
substantial amount (75\%) of xylose responders for day 7 had not previously
been identified as responders at earlier time points. The identities of $^{13}$C-xylose
responders change with time at the phylum level. The numerically dominant xylose
responder phylum shifts from \textit{Firmicutes} to \textit{Bacteroidetes} and
then to \textit{Actinobacteria} across days 1, 3 and 7 (Figure~\ref{fig:l2fc},
Figure~XX). 

All of the $^{13}$-xylose responders in the \textit{Firmicutes} phylum are
closely related (at least 99\% sequence idetity) to cultured isolates from
genera that are known to form endospores (Table XX). Each responder is closely
related to strains annotated as members of \textit{Bacillus},
\textit{Paenibacillus} or \textit{Lysinibacillus}. \textit{Bacteroidetes}
$^{13}$C-xylose responders are predominantly closely related to
\textit{Flavobacterium} species (5 of 8 total responders). Only one
\textit{Bacteroidetes} responder is not closely related to a cultured isolate,
``OTU.183'' (closest LTP BLAST hit, \textit{Chitinophaca sp.}, 89.5\% sequence
identity). OTU.183 shares high sequence identity with environmental clones
derived from rhizosphere samples (accession AM158371, unpublished) and the skin
microbiome (accession JF219881, CITE). Other \textit{Bacteroidetes} responders
share high sequence identities with canonical soil genera including
\textit{Dyadobacer}, \textit{Solibius} and \textit{Terrimonas}. Six of the 8
\textit{Actinobacteria} $^{13}$C-xylose responders are in the
\textit{Micrococcales} order. One $^{13}$C-xylose responding
\textit{Actinobacteria} OTU shares 100\% seqeunce identity with
\textit{Agromyces ramosus} (Table~XX).  \textit{Agromyces ramosus} is a known
predatotry bacterium CITE but is not dependent on a host for growth in culture
CITE. It is not possible to determine the specific origin of assimilated
$^{13}$C in a DNA-SIP experiment. The isotopically labeled C can be passed down
through trophic levels CITE. It's possible that the $^{13}$C labeled
\textit{Agromyces} OTU assimilating $^{13}$C by predataion on alreadly $^{13}$C
labeled OTUs.

\subsection{Cellulose degrader DNA exhibits greater bouyant density shifts upon
$^{13}$C incorporation than xylose degrader DNA} 
Cellulose responders exhibited a greater shift in BD (i.e. assimilated more
$^{13}$C per unit DNA) than xylose responders in response to isotope
incorporation (Figure~XX).  DNA buoyant density increases as it incorporates
more $^{13}$C carbons. An organism that only assimilates C into DNA from a
$^{13}$C isotopically labeled source, will have a greater $^{13}$C:$^{12}$C
ratio in its DNA than an organism utilizing a mixture of isotopically labeled
and unlabeled C sources. Upon labeling, DNA from the organism that incoporates
exclusively $^{13}$C will shift its buoyant density position further relative
to its original $^{12}$C-DNA position than the DNA buoyant density shift from
an organism that doesn't exclusively utilize isotopically labeled C. Therefore
DNA buoyant density shifts (labeled versus unlabeled DNA) indicate substrate
specificity.  We measured density shift as the change in an OTU's density
profile center of mass between corresponding contol and labeled gradients.

A density profile for each responder is generated for the experimental and
control treatment at each of the sampling time points using relative abundances
from sequence libraries . The difference in center of mass for each set of
density profiles (control and experimental) is measured (supp. MM) and each KDE
curve represents the collection of density shifts calculated for all responders
in the $^{13}$C-cellulose or $^{13}$C-xylose treatment . We observe xylose
utilizers having a smaller density shift (0.008 $\pm$ 0.008 g mL$^{-1}$) than
cellulose utilizers (0.015 $\pm$ 0.009 g mL$^{-1}$), with few exceptions. 

\subsection{Xylose responders at day 1 have more estimated rRNA operon copy numbers per genome
than xylose responders at days 3 and 7, and, Xylose responders have more rRNA operon copy numbers
than cellulose responders.}
Estimated rRNA operon genome copy numbers per $^{13}$C-xylose responder OTU
genome and and day of first response are correlated (p-value,
Figure~\ref{fig:copy}). $^{13}$C-xylose responder rRNA operon geneome copy
number is inversely related to time; that is, OTUs that first respond at later
time points have fewer estimated rRNA operons per genome than OTUs that first
respond earlier. rRNA operon copy number estimation is a recent advance in
microbiome science \citep{Kembel_2012} and the relationship of rRNA operon copy
number per genome with ecological strategy is well established
\citep{Klappenbach_2000}. Specifically, microorganisms with a high number of
rRNA operons per genome tend to be fast growers specialized to take advantage
of boom-bust environments whereas a low rRNA operon copy number per genome
tends to occur in microorganisms that favor slower growth under lower and more
consistent nutrient input \citep{Klappenbach_2000}. At the beginning of our
incubation, OTUs with estimated high rRNA operon copy numbers per genome or
``fast-growers'' assimilate xylose into biomass and with time slower growers
(lower rRNA operon number per genome) begin to respond to the xylose addition.
Further, $^{13}$C-xylose responders have fewer estimated rRNA operon copy
numbers per genome than $^{13}$C-cellulose responders suggesting xylose
respiring mircrobes are generally faster growers than cellulose degraders.


\subsection{Variation in bulk soil DNA microbial community structure is significantly less
than variation in gradient fractions} 
