\section{Results}
To observe C use dynamics by the soil microbial community, we conducted a
nucleic acid SIP experiment wherein xylose or cellulose carried the isotopic
label. CsCl gradient fractions were assayed for SSU rRNA gene content using
high-throughput DNA sequencing. We set up three soil microcosm series. All
microcosm series were amended with a C substrate mixture
that included cellulose and xylose. The C substrate mixture approximated
freshly degrading plant biomass. Each series was amended with the same
substrate mixture, however, for each series except the control, one substrate
was substituted for its $^{13}$C counterpart. In one series cellulose was
$^{13}$C-labeled in another xylose was $^{13}$C-labeled and in the control
series no sustrates were $^{13}$C labeled.  Amendments are shorthand identified
in the following figures by the following code: "13CXPS" refers to the
amendment with $^{13}$C-xylose (that is $^{13}$\textbf{C} \textbf{X}lose
\textbf{P}lant \textbf{S}imulant), "13CCPS" refers to the $^{13}$C-cellulose
amendment and "12CCPS" refers to the amendment that only contained $^{12}$C
substrates.  Xylose or cellulose were chosen to carry the isotopic label to
contrast C assimilation for labile, soluble C (xylose) versus
insoluble, polymeric C (cellulose).  5.3 mg of C substrate mixture per gram
soil was added to each microcosm representing 18\% of the total soil C. The
mixture included 0.42 mg xylose-C and 0.88 mg cellulose-C g soil$^{-1}$.
Microcosms were harvested at day 1, 3, 7, 14 and 30  during a 30 day
incubation. Assimilation of $^{13}$C from xylose peaked immediately and tapered
over the 30 day incubation whereas cellulose $^{13}$C assimilation peaked after
two weeks of incubation (Figure~\ref{fig:ord}).
\subsection{Ordination of CsCl gradient fraction OTU profiles can be used to
observe fraction-level $^{13}$C assimilation dynamics and membership}
Each CsCl gradient fraction possesses a unique composition of 16S rRNA gene
phylogenetic types. Differences in the 16S rRNA gene composition of CsCl
fractions within a single CsCl gradient is driven by buoyant density. For
instance, lighter DNA is more abundant in fractions at lighter densities so
DNA with lower G+C will be found in greater abundance at the light end of the
CsCl gradient and vice versa.  Duplicate gradients receiving only $^{12}$C DNA
with the same bulk or non-fractionated composition would have the same overall
profile of SSU rRNA gene phylogenetic types across the density gradient. As we
fed microcosms identical C substrate mixtures save for the
identity of a $^{13}$C labeled substrate, DNA from all microcosms harvested at
a time point will be similar in bulk phylogenetic composition. Therefore,
differences between between gradients harvested at the same time are due to
$^{13}$C incorporation into bulk community DNA. $^{13}$C-DNA shifts from its
$^{12}$C position towards the heavy end of the density gradient. This causes
heavy fractions in gradients that received $^{13}$C-DNA to be different in 
phylogenetic content than corresponding heavy fractions from gradients that
received $^{12}$C-DNA of the same bulk phylogenetic compositon.

Ordination of CsCl gradient fraction phylogenetic profiles reveals differences
and similarities between gradients. It's clear that microcosms incorporated
$^{13}$C from both $^{13}$C-xylose and $^{13}$C-cellulose as gradients from
both $^{13}$C-xylose and $^{13}$C-cellulose microcosms differ from
corresponding control gradients (Figure~\ref{fig:ord}). These differences from
control gradients are focused in the heavy fractions (Figure~\ref{fig:ord}).
Also, $^{13}$C-DNA from $^{13}$C-xylose microcosms is different in phylogenetic
composition from $^{13}$C-cellulose microcosm $^{13}$C-DNA indicating that
xylose and cellulose were assimilated by different microbial community members
(Figure~\ref{fig:ord}). In general analysis of SSU rRNA gene surveys has
greatly benefitted from utilizing conventional methods for data exploratoration
in ecology such as ordination \citep{Lozupone_2008}.  SSU rRNA gene
phylogenetic profiles in CsCl gradient fractions have only recently been
surveyed with high-throughput DNA sequencing technology and explored via
ordination \citep{Angel_2013, Verastegui_2014}. Ordination of CsCl gradient
fraction phylogenetic profiles has reveled the relative influence of buoyant
density and soil type on the gradient phylogenetic profile variance.
However, ordination has not demonstrated isotope incorporation.  Demonstrating
isotope incorporation requires careful comparisons between control and labeled
gradients over the same buoyant density range. By sequencing all CsCl gradient
fractions from both control and labeled gradients, we can observe when incorporation
begins in addition to the buoyant densities along the CsCl gradients where the
isotope incorporation signal is strongest (Figure~\ref{fig:ord}). $^{13}$C
incorporation from xylose and cellulose is most apparent at days 1/3/7 and days
14/30, respectively (Figure~\ref{fig:ord}). Moreover, labeled gradient fraction
phylogenetic profiles diverge from controls in relatively heavy buoyant
densities (Figure~\ref{fig:ord}). Also apparent from the ordination of CsCl
gradient phylogenetic profiles is that OTUs responsive to $^{13}$C-cellulose
are generally different taxa than those responsive to xylose and last, that
$^{13}$C-xylose responders change in phylogenetic type over incubation days 1,
3 and 7 (Figure~\ref{fig:ord}).

\subsection{$^{13}$C from cellulose was assimilated by canonical
cellulose-degrading and uncharacterized microbial lineages in many phyla
including Chloroflexi and Verrucomicrobia} 
Isotope incorporation by an OTU is revealed by enrichment of the OTU in heavy
CsCl gradient fractions containing $^{13}$C labeled DNA relative to heavy
fractions from control gradients (no $^{13}$C labeled DNA). Only 2 and 5 OTUs
were found to have incorporated $^{13}$C from labeled cellulose at days 3 and
7, respectively. At days 14 and 30, however, 42 and 39
OTUs were found to incorporate $^{13}$C from cellulose into biomass. An average
16\% of the $^{13}$C-cellulose added was respired within the first 7 days, 38\%
by day 14, and 60\% by day 30. A \textit{Cellvibrio} and
\textit{Sandaracinaceae} OTU assimilated $^{13}$C from cellulose at day 3. Day
7 $^{13}$C-cellulose responders included the same \textit{Cellvibrio} responder as day 3, a
\textit{Verrucomicrobia} OTU and three \textit{Chloroflexi} OTUs. 50\% of Day
14 responders belong to \textit{Proteobacteria} (66\% Alpha-, 19\% Gamma-, and 14\%
Beta-) followed by 17\% \textit{Planctomycetes}, 14\% \textit{Verrucomicrobia},
10\% \textit{Chloroflexi}, 7\% \textit{Actinobacteria} and 2\%
cyanobacteria. \textit{Bacteroidetes} OTUs begin to incoporate
$^{13}$C from cellulose at day 30 (13\% of day 30 responders). Other day 30
responding phyla include \textit{Proteobacteria} (30\% of day 30 responders;
42\% Alpha-, 42\% Delta, 8\% Gamma-, and 8\% Beta-), \textit{Planctomycetes}
(20\%), \textit{Verrucomicrobia} (20\%), \textit{Chloroflexi} (13\%) and
cyanobacteria (3\%). \textit{Proteobacteria},
\textit{Verrucomicrobia}, and \textit{Chloroflexi} had relatively high numbers
of responders with heavy response across multiple time points
(Figure~\ref{fig:l2fc}).

\textit{Proteobacteria} represent 46\% of all cellulose responding OTUs
identified. \textit{Cellvibrio} accounted for 3\% of all proteobacterial
responding OTUs detected. \textit{Cellvibrio} was one of the first identified
cellulose degrading bacteria and was originally described by Winogradsky in
1929 who named it for its cellulose degrading abilities
\citep{boone2001bergeys}. All $^{13}$C-cellulose responding
\textit{Proteobacteria} share high sequence identity with 16S rRNA genes from
sequenced cultured isolatets (Table~\ref{tab:cell}) except for OTU.442 (best cultured
isolate match 92\% sequence identity in the \textit{Chrondomyces} genus) and
OTU.663 (best cultured isolate match outside \textit{Proteobacteria} entirely,
\textit{Clostridium} genus, 89\% sequence identity). Some
\textit{Proteobacteria} responders share high sequence identity with type
strains for genera known to posess cellulose degraders including
\textit{Rhizobium}, \textit{Devosia}, \textit{Stenotrophomonas} and
\textit{Cellvibrio}. One \textit{Proteobacteria} OTU shares high sequence
identity with a \textit{Brevundimonas} cultured isolate.
\textit{Brevundimonas} has not previously been identified as a cellulose
degrader, but has been shown to degrade cellouronic acid, an oxidized form of
cellulose \citep{Tavernier_2008}.

\textit{Verrucomicrobia}, a cosmopolitan soil phylum often found in high
abundance \citep{Fierer_2013}, are implicated in polysaccaride degradation in
many environments \citep{Fierer_2013,Herlemann_2013,10543821}.
\textit{Verrucomicrobia} comprise 16\% of the total cellulose responder OTUs
detected. 40\% of \textit{Verrucomicrobia} responders belong to the uncultured
``FukuN18'' family originally identified in freshwater lakes
\citep{Parveen_2013}.  The \textit{Verrucomicrobia} OTU with the strongest
\textit{Verrucomicrobial} response to $^{13}$C-cellulose shared high sequence
identity (97\%) with an isolate from Norway tundra soil \citep{Jiang_2011}
although growth on cellulose was not assessed for this isolate. Only one other
$^{13}$C-cellulose responding verrucomicrobium shared high DNA sequence
identity with a sequenced type strain, ``OTU.638'' (Table~\ref{tab:cell}) with
\textit{Roseimicrobium gellanilyticum} (100\% sequence identity).
\textit{Roseimicrobium gellanilyticum} grows on soluble celluose
\citep{Otsuka_2012}. The remaining $^{13}$C-cellulose \textit{Verrucomicrobia}
responders did not share high sequence identity with any cultured isolates
(maximum sequence identity with any cultured isolate 93\%). 

\textit{Chloroflexi} are traditionally known for their metabolically dynamic
lifestyles ranging from anoxygenic phototropy to organohalide respiration
\citep{Hug_2013}. Recent studies have focused on \textit{Chloroflexi} roles in
C cycling \citep{Hug_2013, Goldfarb_2011,Cole_2013} and several \textit{Chloroflexi} utilize cellulose \citep{Goldfarb_2011, Cole_2013,
Hug_2013}. Four closely related OTUs in an undescribed \textit{Chloroflexi}
lineage (closest matching cultured isolate for all four OTUs:
\textit{Herpetosiphon geysericola}, 89\% sequence identity) responded to
$^{13}$C-cellulose (Figure~\ref{fig:trees}). One additional OTU also from a
poorly characterized \textit{Chloroflexi} lineage (closest cultured isolate match a proteobacterium
at 78\% sequence identity) responded to $^{13}$C-cellulose
(Figure~\ref{fig:trees}).

Other notable $^{13}$C cellulose responders include a \textit{Bacteroidetes}
OTU that shares high sequence identity (99\%) to \textit{Sporocytophaga
myxococcoides} a known cellulose degrader \citep{Vance_1980}, and three
\textit{Actinobacteria} OTUs that share high sequence identity (100\%) with
sequenced cultured isolates. One of the three \textit{Actinobacteria}
$^{13}$C-cellulose responders is in the \textit{Streptomyces}, a genus known to
possess cellulose degraders, while the other two closely match the cultured isolates
\textit{Allokutzneriz albata} \citep{Labeda_2008, Tomita_1993} and
\textit{Lentzea waywayandensis} \citep{LABEDA_1989, Labeda_2001}, that do not
decompose cellulose in culture. Nine \textit{Plantomycetes} OTUs responded to
$^{13}$C-cellulose but none are within described genera (closest cultured isolate
match 91\% sequence identity) (Figure~\ref{fig:trees}). Interestingly, one
$^{13}$C-cellulose responder is annotated as belonging in the
cyanobacteria. The phylum annotation is misleading as the OTU is not
closely related to any oxygenic phototrophs (closest cultured isolate match
\textit{Vampirovibrio chlorellavorus}, 95\% sequence identity, Table~\ref{tab:cell}). A sister clade
to the oxygenic phototrophs classically annotated as ``cyanobacteria'' in
SSU rRNA gene reference databases but does not possess known phototrophs has
recently been proposed to constitute its own phylum, "Melainabacteria"
\citet{Di_Rienzi_2013}, although its phylogenetic position is debated
\citep{Soo_2014}. The catalog of metabolic capabilities associated with
cyanobacteria (or candidate phyla previously annotated as
cyanobacteria) are quickly expanding \citep{Di_Rienzi_2013, Soo_2014}.
Our findings provide evidence of cellulose degradation within a lineage closely
related to but apart from oxygenic phototrophs. Notably, polysaccharide
degradation is suggested by an analysis of a ``Melainabacteria'' genome
\citep{Di_Rienzi_2013}. Although we highlight $^{13}$C-cellulose responders
that share high sequence identity with described genera, most
$^{13}$C-cellulose responders uncovered in this experiment are not closely
related to cultured isolates (Table~\ref{tab:cell}).

\subsection{Putative spore-formers in the Firmicutes assimilate $^{13}$C from
xylose within first day after soil amendment followed by Bacteroidetes and then
Actinobacteria OTUs} 
Within the first 7 days of incubation an average 63\% of $^{13}$C-xylose was
respired and only an additional 6\% more was respired between days 7 and 30. At
the end of the 30 day experiment 30\% of the original $^{13}$C from xylose
remained in the soils. The $^{13}$C remaining in the soil from $^{13}$C-xylose
addition has likely been stabilized by assimilation into microbial biomass
and/or microbial conversion into other forms of organic matter, though it is
possible that some $^{13}$C-xylose remains unavailable to microbes due to
abiotic interactions in soil \citep{Kalbitz_2000}. 

At day 1, 84\% of xylose responsive OTUs belong to Firmicutes, 11\% to
\textit{Proteobacteria} and 5\% to \textit{Bacteroidetes}. At day 3,
Firmicutes responders decreased to 5\% (from 16 OTUs to 1) while
\textit{Bacteroidetes} increased to 63\% (from 1 to 12 OTUs) of day 3
responders. The remaining day 3 responders are members of the
\textit{Proteobacteria} (26\%) and the \textit{Verrucomicrobia} (5\%). Day 7
responders were 53\% Actinobacteria, 40\% Proteobacteria, and 7\% Firmicutes. A
substantial amount (XX\%) of xylose responders for day 7 had not previously
been identified as responders at earlier time points. The identities of $^{13}$C-xylose
responders change with time at the phylum level. The numerically dominant xylose
responder phylum shifts from \textit{Firmicutes} to \textit{Bacteroidetes} and
then to \textit{Actinobacteria} across days 1, 3 and 7 (Figure~\ref{fig:l2fc},
Figure~\ref{fig:xyl_count}). 

All of the $^{13}$C-xylose responders in the \textit{Firmicutes} phylum are
closely related (at least 99\% sequence idetity) to cultured isolates from
genera that are known to form endospores (Table~\ref{tab:xyl}). Each responder is closely
related to strains annotated as members of \textit{Bacillus},
\textit{Paenibacillus} or \textit{Lysinibacillus}. \textit{Bacteroidetes}
$^{13}$C-xylose responders are predominantly closely related to
\textit{Flavobacterium} species (5 of 8 total responders). Only one
\textit{Bacteroidetes} responder is not closely related to a cultured isolate,
``OTU.183'' (closest LTP BLAST hit, \textit{Chitinophaca sp.}, 89.5\% sequence
identity). OTU.183 shares high sequence identity with environmental clones
derived from rhizosphere samples (accession AM158371, unpublished) and the skin
microbiome (accession JF219881, \citet{Kong_2012}). Other \textit{Bacteroidetes} responders
share high sequence identities with canonical soil genera including
\textit{Dyadobacer}, \textit{Solibius} and \textit{Terrimonas}. Six of the 8
\textit{Actinobacteria} $^{13}$C-xylose responders are in the
\textit{Micrococcales} order. One $^{13}$C-xylose responding
\textit{Actinobacteria} OTU shares 100\% seqeunce identity with
\textit{Agromyces ramosus} (Table~\ref{tab:xyl}).  \textit{Agromyces ramosus} is a known
predatotry bacterium but is not dependent on a host for growth in culture
\citep{16346402}. It is not possible to determine the specific origin of assimilated
$^{13}$C in a DNA-SIP experiment. The isotopically labeled C can be passed down
through trophic levels although heavy isotope representation in C pools targeted by
cross-feeders and predators would decrease with depth into the trohpic cascade.
It's possible, however, that the $^{13}$C labeled \textit{Agromyces} OTU is
assimilating $^{13}$C primarily by predatation if \textit{Agromyces} is
selective enough with respect to its prey such that it primarily attacked
$^{13}$C-xylose assimilating organisms.

\subsection{Cellulose degrader DNA exhibits greater bouyant density shifts upon
$^{13}$C incorporation than xylose degrader DNA} 
Cellulose responders exhibited a greater shift in BD (i.e. assimilated more
$^{13}$C per unit DNA) than xylose responders in response to isotope
incorporation (Figure~\ref{fig:shift}, p-value 1.86e$^{-06}$). Cellulose
responders exhibited an average shift of 0.0163 g/mL (sd 0.0094) whereas xylose
responders exhibited an average shift of 0.0097 (sd 0.0094).
One hundred percent $^{13}$C DNA has a buoyant density X.XX g/mL higher than
its $^{12}$C counterpart. DNA buoyant density increases as the
ratio of $^{13}$C carbons to $^{12}$C increases. An organism that only
assimilates C into DNA from a $^{13}$C isotopically labeled source, will have a
greater $^{13}$C:$^{12}$C ratio in its DNA than an organism utilizing a mixture
of isotopically labeled and unlabeled C sources. Upon labeling, DNA from the
organism that incoporates exclusively $^{13}$C will shift its buoyant density
position further relative to its original $^{12}$C-DNA position than the DNA
buoyant density shift from an organism that doesn't exclusively utilize
isotopically labeled C. Therefore DNA buoyant density shifts (labeled versus
unlabeled DNA) indicate substrate specificity given our experimental design (only
one substrate was labeled in each amendment).  We measured density shift as
the change in an OTU's density profile center of mass between corresponding
contol and labeled gradients. Density shifts, however, should not be evaluated
on an individual OTU basis as a small number of density shifts are observed for
each OTU and the variance of the density shift metric at the level of
individual OTUs is unknown. It is therefore more informative to compare density
shifts among substrate responder groups. Further, density shifts are based on
relative abundance profiles and would be theoretically muted in comparison to
density shifts based on absolute abundance profiles and should be interpreted
with this transformation in mind. It should also be noted that there was
overlap in observed density shifts between $^{13}$C-cellulose and
$^{13}$C-xylose responder groups suggesting that although cellulose degraders
are generally more substrate specific than xylose utilizers, some cellulose
degraders show less substrate specificity for cellulose than some xylose
utilizers for xylose (Figure~\ref{fig:shift}), and, each responder group
exhibits a range of substrate specificites (Figure~\ref{fig:shift}).

\subsection{Xylose responders at day 1 have more estimated rRNA operon copy numbers per genome
than xylose responders at days 3 and 7, and, Xylose responders have more rRNA operon copy numbers
than cellulose responders.}
Estimated rRNA operon genome copy numbers per $^{13}$C-xylose responder OTU
genome and day of first response are correlated (p-value 2.02e$^{-15}$,
Figure~\ref{fig:copy}). $^{13}$C-xylose responder rRNA operon geneome copy
number is inversely related to time; that is, OTUs that first respond at later
time points have fewer estimated rRNA operons per genome than OTUs that first
respond earlier (Figure~\ref{fig:copy}). rRNA operon copy number estimation is
a recent advance in microbiome science \citep{Kembel_2012} and the relationship
of rRNA operon copy number per genome with ecological strategy is well
established \citep{Klappenbach_2000}. Specifically, microorganisms with a high
number of rRNA operons per genome tend to be fast growers specialized to take
advantage of boom-bust environments whereas a low rRNA operon copy number per
genome tends to occur in microorganisms that favor slower growth under lower
and more consistent nutrient input \citep{Klappenbach_2000}. At the beginning
of our incubation, OTUs with estimated high rRNA operon copy numbers per genome
or ``fast-growers'' assimilate xylose into biomass and with time slower growers
(lower rRNA operon number per genome) begin to respond to the xylose addition.
Further, $^{13}$C-xylose responders have more estimated rRNA operon copy
numbers per genome than $^{13}$C-cellulose responders (p-value 1.878e$^{-09}$)
suggesting xylose respiring mircrobes are generally faster growers than
cellulose degraders.

\subsection{Xylose responders are more abundant in the soil community than cellulose
responders}
$^{13}$C-xylose responders are generally more abundant members based on
relative abundance in bulk DNA SSU rRNA gene content than $^{13}$C-cellulose
responders (Figure~\ref{fig:shift}, p-value 0.00028).  However, both $^{13}$C-xylose
and $^{13}$C-cellulose responders were found in abundant and rare OTUs
(Figure~\ref{fig:shift}). For instance, a \textit{Delftia} $^{13}$C-cellulose
responder is fairly abundant in the bulk samples (``OTU.5'',
Table~\ref{tab:cell}) with a mean bulk rank of 13 (\textit{i.e.} on average the
13th most abundant OTU) and a $^{13}$C-xylose responder (``OTU.1040'',
Table~\ref{tab:xyl}) has a mean relative abundance in bulk samples
of 2.85e$^{-05}$. Only one substrate responder ($^{13}$C-cellulose) was not
found in any bulk samples ("OTU.862", Table~\ref{tab:cell}). Of the top 10
responders sorted by descending mean rank (essentially the 10 most abundant
responders in the bulk samples), 8 are $^{13}$C-xylose responders and 5 of
these 8 have mean ranks less than 10 in bulk samples.
\subsection{Variation in bulk soil DNA microbial community structure is significantly less
than variation in gradient fractions} 
Using a distance metric that incorporates relative abundance information
(weighted Unifrac metric, \citep{Lozupone_2005}) bulk sample beta diversity was
less than gradient fraction beta diversity (p.value 0.003). Time was
significantly correlated to bulk sample phylogenetic profile variation (p-value
0.23, R$^{2}$ 0.63, Figure~\ref{fig:bulk_ord}) but treatment (\textit{i.e.} the contrast
between only $^{12}$C additions with additions that included isoptically
labeled substrates) was not (p-value 0.35).

