\section{Results and Discussion}


\textbf{Temporal microbial succession during C degradation.} With the rapid advancement and declining costs of high throughput sequencing, it has become increasingly easy to investigate microbial communities.  In this study, we couple stable-isotope probing with 454 pyrosequencing in order to better understand organic matter decomposition dynamics as a function of soil microbial community C utilization. We ran a temporal series of parallel microcosms and measured the changes in the microbial community as result of the addition of a complex carbon mixture using 454 pyrosequencing of the bulk microbial community and fractions from CsCl gradient fractionation (Fig. S1). Overall, temporal changes in microbial community composition are consistent with C decomposition being accompanied by a microbial community succession. 

Analysis of the sequenced bulk community DNA demonstrates Proteobacteria (26-35\%), Actinobacteria (19-26\%), and Acidobacteria (12-21\%) as the most dominant phyla throughout the duration of the experiment. This is consistent with previous observations \cite{Goldfarb_2011, Fierer_2007, Rui_2009}. We found trends of Proteobacteria and Actinobacteria decreasing and Acidobacteria increasing as C availability declines (TableS1). This is congruent with findings in soils sampled from a wide range of ecosystems in the US \cite{Fierer_2007}. At days 1, 3, and 7 the bulk community was composed of ~12-18\% Bacteriodetes and Firmicutes combined.  At days 14 and 30, these phyla declined to a combined 7-9\% of the whole community accompanied with an increase in Plancktomycetes, Verrucomicrobia, and Chloroflexi (2-3\% at day 1 to 5-7\% at day 30). There has been conflicting evidence about the correlation of Bacteriodetes abundance with C availability \cite{Fierer_2007, Rui_2009, Sharp_2000}, however, our data is consistent with Fierer (2007).  Additionally, the abundance of Planctomycetes, Verrucomicrobia, and Chloroflexi at later stages of decomposition are in accord with findings in wheat straw degradation \cite{Bastian_2009}. The rank abundance (RA) of the community depicts transitions we observe in high ranking phyla abundance beginning at day 7 (Fig S2A).  Despite the flucations we observe at the phylum level, the biological variability observed over time is low (FigS2B) demonstrating community stability.   

Twenty fractions from a CsCl gradient fractionation for each treatment at each time point were sequenced (Fig. S1). Using NMDS analysis from weighted unifrac distances, the relationship between microbial communities at each buoyant density from all treatments and time points are plotted (Fig 1). Each point on the NMDS represents the microbial community based on 16S sequencing from a single fraction where the size of the point is representative of the denisty of that fraction and the colors represent the treatments (Fig1A) or days (Fig1B). The high-density fractions that are differentiating from the control along NMDS2 correspond to fractions that control \textsuperscript{13}C-labeled OTUs (herein called 'responders'). The differential separation of high density fractions in the \textsuperscript{13}C-xylose treatment compared to the \textsuperscript{13}C-cellulose treatment is indicative of a difference in the responders for each of the substrates (Fig 1A). There is an observable time signature of responders at days 1, 3, and 7 for the xylose treatment and days 14 and 30 for the cellulose treatment (Fig1B). This demonstrates that different microbial community members are responsible for the consumption of these two substrates; xylose is consumed quickly, whereas, cellulose decomposition takes longer. This supports the hypothesis of a microbial community succession during the decomposition process. Furthermore, this demonstrates the sensitivity of this technique by being able to detect \textsuperscript{13}C-label incorporation in samples with low C additions (2.18mgC g\textsuperscript{-1} soil).    


\textbf{Differential taxa C utilization.} Using fractions from within a density range of 1.7125-1.755 g/ml, relative abundances of taxa in the experimental treatments were compared to their respective relative abundances in the control treatment to calculate the log\textsubscript{2} fold change (Fig2). The log\textsubscript{2} fold change demonstrates the boom and bust of taxa with time. For OTUs passing a conservative threshold of \textit{p}-value = <0.10 for log\textsubscript{2} fold change (FigSx), we measured the density shift in the experimental treatment compared to the control (Fig Sx).  Those OTUs with a center of mass shift greater than zero were considered 'responders'.   

For the \textsuperscript{13}C-xylose treatment at day 1 OTUs within Firmicutes demonstrate the strongest response. This is not surprising as it has been demonstrated that Firmicutes maintain a metabolically-ready state \cite{Jenkins_2010}\cite{Griffiths_1998}\cite{Brookes_1987}\cite{De_Nobili_2001}. Additionally, Proteobacteria, Bacteroidetes, and Actinobacteria contain responder OTUs at day 1. Genomic analysis of fast growing bacteria, specifically Proteobacteria and Firmicutes, have a higher number of total transporters enabling them to import or export a broad range of compounds \cite{Barabote_2005}. The low affinity of these transporters facilitates fast growth in times of high nutrient conditions \cite{Trivedi_2013}.  Day 3 exhibits a strong increase in Bacteroidetes response and the onset of Verrucomicrobia reponders. Notably, this pronounced response by Bacteroidetes is not captured by the bulk community abundances. Day 7 demonstrates an increased response from Proteo- and Actinobacterial OTUs.  While there is a slight increase in their abundances in the bulk community analysis at day 7, it would be difficult to differentiate that change from natural variation or methodological noise. All OTUs have a decreasing log\textsubscript{2} fold change at days 14 and 30, with only a single Firmicutes OTU passing the 'responder' criteria.     

For the \textsuperscript{13}C-cellulose treatment only one Proteobacteria passes the 'responder' criteria at day 3 and two OTUs (Proteobacteria and Chloroflexi) at day 7. This is expected since competition for a limited resource typically results in the dominance of one or a few populations with the highest growth rates \cite{Fontaine_2003}. By day 14, responders are detected in Proteobacteria, Verrucomicrobia, Chloroflexi, Planctomycetes, and Actinobacteria. The degradation of cellulose by Verrucomicrobia is consistent with recalcitrant carbon degradation by Verrucomicrobia in soil, aquatic, and anoxic rice patty soils  \cite{Fierer_2013}\cite{Herlemann_2013}\cite{10543821}. Fierer found Verrucomicrobia to be more than 50\% of their bulk community sequences which has strong implications of importance of this taxa in soil carbon cycling \cite{Fierer_2013}. 
In the \textsuperscript{13}C-cellulose treatment the same responders were detected at day 30 as with earlier time points, with the exception of Actinobacteria and the addition of Bacteriodetes. These trends are more readily observable in the bulk community abundances than discerned with xylose responders. This is likely due to the low abundance of these phlya, where changes in bulk community abundance are more pronounced and easier to detect. Comparatively, phlya of consistently high abundance mask response changes unless they present changes of grand proportions.          

Kernel density estimates (KDE) of the CsCl density shifts measured for responders in \textsuperscript{13}C-xylose were compared to those of \textsuperscript{13}C-cellulose responders (Fig 3A). An organism with 100\% 13C-labeling of DNA would exhibit a density shift of 0.04gmL\textsuperscript{-1}. Xylose utilizers have a smaller density shift (<0.02 gmL\textsuperscript{-1}) than cellulose utilizers (0.005-0.03 gmL\textsuperscript{-1}), with few exceptions. This suggests a greater substrate specificity among cellulose degraders than xylose degraders. Partial 13C-labeling (<0.04gmL\textsuperscript{-1} density shift) could be a result of various lifestyles ('trophic strategy' better word choice?) such as (1) assimilation of C from multiple substrates (both \textsuperscript{12}C and \textsuperscript{13}C in this instance) or (2) 13C-label dilution as it cascades through trophic levels via consumption of 13C-labeled organisms or waste products from organisms that are metabolizing the 13C-sunstrate.

Most xylose responders are found at higher rank abundances than cellulose responders which fall among the rare taxa in the tail of the RA curve (Fig 3B). This demonstrates that many taxa important to C-cycling are present in the rare biosphere and may be difficult or unable to detect in bulk community sequencing efforts. Responders of xylose or cellulose are wide spread across 7 phyla (Fig 4).  There are very few OTUs that utilize both cellulose and xylose (should put in a number of how many OTUs utilize both over how many responders total there were), however, at the phyla level many phylum had responders for both xylose and cellulose. 

Genomic analysis of Actinobacteria have revealed the ability to utilize a diverse array of polysaccharides which has been attributed to their resilience to stressful soil conditions \cite{Trivedi_2013}. Despite this versatile ability, they have also been demonstrated to possess high affinity transporters for specific substrates \cite{Trivedi_2013}. In this study, Actinobacteria had responders for both substrates with a peak shift of ~0.036 gmL\textsuperscript{-1} suggesting a strong substrate specificity (Figures Sx and Sz - the substrate utilization charts). Albeit, there are no OTUs within Actinobacteria that responded to both xylose and cellulose.  This information may suggest that while Actinobacteria exhibit an ability to utilize an array of carbon substrates, substrate specificity may be more clade specific for certain substrates and not widespread throughout the phylum (Figure 4).

Noteably we also identified a previously undescribed clade of Chloroflexi that exhibit a high substrate specificity based on a density shift peak at between 0.03-0.04 gmL\textsuperscript{-1}.  We saw no other cellulose utilization in Chloroflexi outside of this clade.  This phylum has previously been demonstrated to utilize cellulose (Goldfarb)  

\textbf{Carbon substrate utilization is inconsistent within phylum.} More often than not we see ecological functionality assigned at the phylum level (refs). More specifically, it has been proposed that the microbial community functionality responsible for soil C cycling appear at the level of phlya rather than species/genera \cite{Schimel_2012}. However, based on our evidence of clade or single taxa level responses for xylose and cellulose utilization, assigning phylum level functionality is not an accurate depiction of soil C utilization. Phylum level assignment conventions could in part be due to limitations in finer scale taxonomic identifications or to lack of sequencing resolution as seen here with many responders present in the rarer ranks (Figure 3B). 

Whole phylum responses were not detected for xylose or cellulose yet utilization of these substrates spanned many phylogenetically diverse groups. However, substrate utilization within each phylum was demonstrated at the clade or single taxa level. Cellulose degradation has previously been noted for spanning eight phyla in a study of forest soils amended with different carbon substrates, but was limited to a small number of taxa within each of those phyla \cite{Goldfarb_2011}. It has previously been suggested that all taxa within a phylum are unlikely to share ecological characteristics \cite{Fierer_2007}, and furthermore, within a species population \cite{Choudoir_2012}\cite{Preheim_2011}\cite{Hunt_2008}. Habitat traits of coastal Vibrio isolates were mapped onto microbial phylogeny revealing discrete ecological populations based on seasonal occurrence and particulate size fractionation \cite{Preheim_2011}\cite{Hunt_2008}. These data would suggest that portraying the response of a few OTUs or clades as a phylum level response would be overreaching.     

In a study that amended forest soils with single substrates in the presence of 3-bromodeoxyuridine (BrdU), they determined that more than 500 taxa responded to labile C but occured predominantly in two phlya (Proteobacteria and Actinobacteria)\cite{Goldfarb_2011}. On the other hand, the soils amended with more recalcitrant C such as cellulose and lignin had fewer taxa responded, yet they were wide spread across eight phyla \cite{Goldfarb_2011}. "We determinedthat simple substrateswere degradedby the same groups of organisms in both soils, and at similar rates, but pine litter was degraded by different microbial groups in the two soils, and at different rates. Thus as substrate complexity increased, the functional group responsible for its degradation became more distinct between the two soils."\cite{Waldrop_2004}
Goldfarb then functional guild literature (fierer) then propose that phyla do not serve as a single functional unit as many clades within a phyla may respond differently.    
\cite{Fierer_2012} - demonstrates that taxonomic and phylogenetic diversity does not confer functional diversity.  "There has been considera- ble debate in the field of ecology as to how the taxonomic diversity of communities relates to the observed functional, or trait-level, diversity (16); we found a strong positive correlation be- tween the functional and taxonomic diversity of soil microbial communities"\cite{Fierer_2013}
"Therefore, the abun- dant soil microbial taxa do not appear to exhibit a high degree of functional redundancy (or equiv- alence), as is often assumed (19); reductions in tax- onomic diversity were associated with decreases in the breadth of functional traits contained with- in these soil communities."\cite{Fierer_2013} 

It has been suggested that "Therefore, the abun- dant soil microbial taxa do not appear to exhibit a high degree of functional redundancy (or equiv- alence), "\cite{Fierer_2013} implying that community membership matters.  Another reason why we should study the specific microbial taxa consuming different C substrates.  
 
\textbf{Conclusions.} We have demonstrated how next generation sequencing-enabled SIP gives a taxa level resolution for substrate utilization. Using this technique, we are able to resolve discrete OTUs that would otherwise be missed using bulk community sequencing. Additionally, this technique provides greater taxonomic resolution than previous techniques (cloning, TRFLP, ARISA) used to determine substrate utilizing community members. While we are currently able to resolve highly responsive OTUs, the next step is to find better ways to resolve taxa that are partially responsive which we are not currently able to differentiate from noise with confidence. Yet, given the ability to resolve partially responsive taxa, the ecology would still be difficult to discern. For example, a generalist utilizing many substrates including the 13C-labeled substrate may exhibit the same partial labeling that a specialist utilizing both the 13C-substrate and the same substrate (unlabeled) that is inherent in the soil. Additionally, partially labeled taxa could be further down the trophic cascade including predators or secondary consumers of waste products from primary consumer microbes that were highly labeled.     
Traditionally, SIP is performed with a single time point in an attempt to minimize or eliminate cross-feeding of the substrate, as cross-feeding could cause false positives in the data set.  However, in a temporal food web mapping study, this cross-feeding is an exciting advantage of this system because it will enable us to track substrates of interest (i.e. cellulose) through many trophic cascades.  Observed organisms will exhibit a range of 13C labeling, 0-100\%, with primary consumers being the most enriched and subsequent trophic levels being less enriched (Morris 2002, McDonald 2005, Ziegler 2005).  This relationship of trophic level consumption to dilution of label will facilitate tracking C as it moves through various operational taxonomic units (OTUs) in the soil. 

Our study is consistent with carbon degradative succession has previously been demonstrated (ref). We demonstrate a rapid decrease in the labile carbon, xylose, confirmed by it’s 13C label incorporation into the microbial community DNA during the first 7 days of the experiment, after which, the label is not detectable in the DNA. Subsequently our data demonstrates a slow degradation of the more recalcitrant, polymeric carbon demonstrated by 13C-cellulose label incorporation into the microbial community DNA at 14 and 30 days.  We did not observe the 13C-cellulose signal leave the DNA within the time limits, 30 days, of our experiment.  This degradation succession is also confirmed by isotopic analysis of the soil from the microcosms (Table S1). NMDS demonstrates microbial succession and based on xylose and cellulose treatments separating away from the control, but in opposing directions indicating different microbial community members are responsible for C degradation of the two substrates. 

It is likely that the slow degradation of cellulose can be attributed to the energy-taxing process of synthesizing cellulolytic enzymes and exporting them, as cellulose is broken down externally (Schimel & Schaeffer 2012, Lynd et al 2002).  As a result, microorganisms responsible for the synthesis of cellulases preferentially shuttle energy towards enzyme synthesis rather than biomass until cellulose hydrolysis begins (Schimel & Schaeffer 2012).  This accounts for the delay in growth and ultimately the slow decomposition of cellulose (Perez et al 2002, Schimel & Schaeffer 2012).

 


 


















"Overall there was a gradual increase in the abundance of the Bacteroidetes during the succession process, which were not abundant in the first sampling (less than 0.24percent) but increased with time (around 7–9percent). Autotrophic phyla such as Clorobi, Chloroflexi and Cyanobacteria, increased in abundance the first nine months and then decrease their abundance in the last sampling (12 months) (Table 3)."\cite{L_pez_Lozano_2013}

"A total of 137 bacterial sequences at 28 days and 116 at 168 days were analysed and listed in Table 1. The Shannon diversity indices were 3.24 (S 1⁄4 43, E 1⁄4 0.86) and 4 (S 1⁄4 65, E 1⁄4 0.96), respectively, and within the range reported in other soil environ- ments (Dunbar et al.,1999; Hill et al., 2003). These results revealed (i) a high diversity of bacteria able to colonize the residue as previously indicated by the considerable complexity of B-ARISA profiles and (ii) a slight increase in bacterial diversity during the residue decom- position. This might be related to the biochemical composition of the residue at the end of the degradation process (Nicolardot et al., 2007). The main components at this stage were complex substrates (cellulose, hemicelluloses, lignin) recalcitrant to degradation which might result in the stimulation of a more diverse bacterial consor- tium able to degrade them (De Boer et al., 2005).
At 28 days of residue decomposition, the taxonomic divisions most frequently found were the Proteobacteria (69\%), the Bacter- oidetes (25\%) and the Fibrobacter (6\%). The Proteobacteria were distributed specifically in the Gammaproteobacteria (35\%), Alphaproteobacteria (26\%), and Betaproteobacteria (8\%) classes (Table 1). The significant discrimination of bacterial diversity between 28 and 168 days of incubation was mainly due to an increase in Actinobacteria (10\%), Deltaproteobacteria (9\%) and Betaproteobacteria (14\%) and a decrease in Fibrobacter (0\%) and Gammaproteobacteria (11\%). In addition, several other minor phyla were specifically identified at 168 days: Chloroflexibacteria (4\%),Verrucomicrobia (3\%), the Planctomycetes (3\%), Gemmatimona- detes (2\%), Acidobacteria (2\%), Firmicutes (1\%) and Clostridia (1\%) (Table 1).For example, Myxobacteria (and all the Deltaproteobacteria class) appear during the later stages of degradation process (Table 1).In vitro studies have revealed the ability of these bacteria to antagonize plant pathogenic as well as saprotrophic fungi (Bull, 2002) and their appearance at this stage may be partly related to their mycophagic properties.From a functional point of view, the ability to decompose cellulose is present both in bacteria (e.g. Streptomyces, Bacillus, Cytophaga, Micromonospora) and fungi (Ascomycota as well as Basidiomycota). The presence of functionally equivalent cellulolytic systems in both bacteria and fungi implies that competition for cellulose may take place in spatially limited soil niches. Some cellulose degrading bacteria are able to produce antifungal metabolites (Munimbazi and Bullerman, 1998) and other spore-associated bacteria might inhibit spore germination (Xavier and Germida, 2003; Toyota and Kimura, 1993)." \cite{Bastian_2009}

"Although we might expect an overall correlation between taxonomic com- position and the functional attributes of soil microbial communi- ties, this may not always be the case as distinct taxa can share specific functional attributes and closely related taxa may have very different physiologies and environmental tolerances (14). "\cite{Fierer_2012}

"These results show that all of the communities were dominated by Acidobacteria, Actinobacteria, Bacteroidetes, Pro- teobacteria, and Verrucomicrobia (Fig. S1), bacterial phyla that are known to be relatively abundant and ubiquitous in soil (25). Additional phyla including Chloroflexi, Cyanobacteria, Firmicutes, and Gemmatimonadetes were also found in nearly all soils, but their relative abundances were highly variable and typically rep- resented less than 5\% of the 16S rRNA reads in any individual"\cite{Fierer_2012}

"Therefore, as with the alpha diversity patterns, the concordance in beta diversity patterns highlights that the overall functional differences between the soil microbial communities were significantly correlated with the differences in the composition of these communities. Our findings are in line with comparable studies conducted in soil (20) and other habitats that also found strong correlations between metagenome composition and taxonomic composition (34, 35, 40). Although individual functional genes may not necessarily be correlated with community structure, the overall functional attributes of soil microbial communities appear to be predictable across broad gradients in soil and biome types if one has information on the taxonomic or phylogenetic structure of the communities."\cite{Fierer_2012}

   "Very few taxa responded only to chemically recalcitrant substrates, with only 24 for cellulose (Clostridiales; termite gut clones, Deinococcus, Chloroflexi, Acetobacter, Desulfobacteraceae, Geobacteraceae, Syntrophobacteraceae, Treponema). On the other hand, cellulose users were relatively phylogenetically diverse (eight phyla), which may broaden the range of environ- mental conditions under which this metabolism may operate. Many of the taxa responding positively to cellulose have previously been shown to be capable of cellulose hydrolysis (e.g., bacteria from the families Clostridiaceae and Lachnospiraceae (Schwarz, 2001), Sphingomonas spp. (Kurakake et al., 2007), and Spirochetes (Warnecke et al., 2007), while Acetobacter aceti (Moonmangmee et al., 2002) is typically considered a cellulose producer rather than a consumer. "\cite{Goldfarb_2011}
   
   
"For bacterial succession, an increase in the proportion of Proteobacteria from winter to spring was observed, whereas that of Actinobacteria and Verrucumicrobia decreased (Figure 1b). Changes in the respective group abundances were validated by a PLFA analysis, which showed similar trends (Figure 2). A reduction of Actinobacteria was unexpected, because they are known to be involved in decomposition of organic materials, and thus are important for organic matter turnover and C cycle (Kirby, 2006). In other studies, an increase in the abundance of Actinobacteria has been shown during later stages of litter decomposition (Torres et al., 2005; Snajdr et al., 2011). The same accounts for the absence of Acidobacteria; members of this bacterial phylum can degrade various polysaccharides in- cluding cellulose and xylan (Ward et al., 2009). Based on RNA sequencing, Baldrian et al. (2012) found Acidobacteria to be the main bacterial group that was enriched in an active litter inhabiting community." 





Plant residues represent the largest proportion of C entering soil systems and are composed primarily of cellulose making it the most abundant biopolymer globally (Paul & Clark 1989, Berg & Laskowski 2005, \cite{lemm_Pautzsch_Blankenburg_2005}, Coleman & Crossley 1996, Nannipieri & Badalucco 2003). Much work has been done to study cellulose degradation by microorganisms (Beguin & Aubert 1994, Lynd et al 2002, Haicher et al 2007, Eichorst & Kuske 2012).  With the advent of culture independent techniques, we have discovered that there are far more cellulose degraders than our limited culture collection represents.  Some studies have used labeled leaf litter to identify soil decomposers, however, it is difficult to differentiate which organisms are responsible for the actual degradation of the cellulose (Evershed et al 2006). Other cellulose degradation studies add cellulose as a single substrate addition to a cultured cellulose degrader or to soil (Haichar et al 2007, Li et al 2009, Eichorst & Kuske 2012).  This method may present false positives, as an organism that might not normally consume a given C substrate may opportunistically do so under the circumstances that it is the only C source available. Additionally, organisms may behave quite differently when given a large dose of a nutrient source versus a complex C mixture which they may actually encounter in nature.
