\section{Results}
% Fakesubsubsection:We tracked the flow of C from xylose
We tracked the flow of C from xylose and cellulose into microbial biomass over
time using DNA-SIP (Figure~\ref{fig:setup}). We added 3 mg C g$^{-1}$ dry
weight soil, comprising 25\% of total soil C, to each soil sample. The C
addition included 0.42 mg xylose-C and 0.88 mg cellulose-C g$^{-1}$ soil
dry weight (3.5\% and 7.3\% of total soil C, respectively). We harvested
soil samples at 1, 3, 7, 14 and 30 incubation days. The soil microbial
community respired the majority of xylose within one day. Sixty-five percent of
the $^{13}$C from $^{13}$C-xylose was respired by day 1 (Figure~\ref{fig:13C}).
Twenty-nine percent 29\% of the total added $^{13}$C-xylose remained in the
soil at day 30 (Figure~\ref{fig:13C}). In contrast, $^{13}$C from
$^{13}$C-cellulose declined at a constant rate of approximately 18 $\mu$g
C g$^{-1}$ d$^{-1}$. Forty percent of $^{13}$C added as cellulose remained in
the soil at day 30 (Figure~\ref{fig:13C}). 

\subsection{Soil microcosm microbial community changes with time}
% Fakesubsubsection:Changes in the soil microcosm microbial community structure
We assessed incorporation of $^{13}$C into microbial community DNA by comparing the
SSU rRNA gene sequence composition of SIP density gradient fractions from
either control or $^{13}$C-amended microcosm soil DNA. We set up three types of
microcosms. All microcosms received the same C mixture, but, in two microcosm
types a $^{13}$C-labeled substrate (i.e. $^{13}$C-xylose or $^{13}$C-cellulose)
was substituted for its $^{12}$C equivalent. We added only unlabeled
C substrates to the last microcosm type -- this type served as the ``control''.
The majority of the variance in SSU rRNA gene composition of control density
gradient fractions is represented by fraction density (Figure~\ref{fig:ord}.
DNA buoyant density is correlated with G$+$C content \citep{Buckley_2007} and
therefore variation in control gradient fraction SSU rRNA gene composition is
strongly influenced by DNA G$+$C content. For the $^{13}$C-cellulose amendment,
SSU rRNA gene composition in gradient fractions deviate from control fractions
at high density ($>$ 1.72 g mL$^{-1}$) at days 14 and 30
(Figure~\ref{fig:ord}). For the $^{13}$C-xylose amendment, SSU rRNA gene
composition in density gradient fractions also deviates from control in high
density fractions, but in contrast to the $^{13}$C-cellulose amendment, the SSU
rRNA gene composition of density gradient fractions deviates from control at
days 1, 3, and 7 as opposed to days 14 an 30 (Figure~\ref{fig:ord}). SSU rRNA
gene composition in high density fractions differs between $^{13}$C-cellulose
amendments and $^{13}$C-xylose amendments indicating microorganisms that
incorporated C from xylose into DNA were different from those that
incorporated C from cellulose (Figure~\ref{fig:ord}). Further, the SSU
rRNA gene sequence composition of high density fractions from
$^{13}$C-cellulose amendments at days 14 and 30 is similar indicating similar
microorganisms have $^{13}$C labeled DNA at days 14 and 30. In contrast, the
SSU rRNA gene composition of the $^{13}$C-xylose amendment high density
gradient fractions varies between days 1, 3, and
7 indicating that different microbes have $^{13}$C labeled DNA on these days.
$^{13}$C-xylose amendment gradient fraction SSU gene composition is similar
to control across he entire density gradient on days 14 and 30
(Figure~\ref{fig:ord}) indicating that $^{13}$C from $^{13}$C-xylose is no
longer detectable in DNA on days 14 and 30 for the $^{13}$C-xylose amendment. 

\subsection{Temporal dynamics of microcosm microbial community composition}
% Fakesubsubsection:We monitored the soil microbial community
We monitored the soil microbial community over the course of the experiment by
sequencing SSU rRNA gene amplicons amplified by PCR directly from
non-fractionated soil DNA. The SSU rRNA gene composition of the
non-fractionated soil DNA varied significantly with time
(Figure~\ref{fig:bulk_ord}, P-value $=$ 0.023, R$^{2}$ $=$ 0.63, Adonis test
\citet{Anderson2001a}), but did not vary significantly with respect to
amendment (Figure~\ref{fig:bulk_ord}, P-value $=$ 0.23, R$^{2}$ $=$ 0.21,
Adonis test \citet{Anderson2001a}). The latter result demonstrates the
substitution of $^{13}$C-labeled substrates for unlabeled equivalents did not
significantly alter community composition. The variance in SSU rRNA gene
composition among non-fractionated soil DNA samples was significantly less than
the variance among gradient fractions (Figure~\ref{fig:bulk_ord}, P-value $=$
0.003, “betadisper” function R Vegan package \citet{oksanen2007vegan} (15)) and
therefore the process of density gradient fractionation produced
significantly more variance in SSU rRNA gene composition than temporal
changes in the non-fractionated DNA SSU rRNA gene content.

% Fakesubsubsection:Twenty-nine OTUs
Twenty-nine OTUs changed in significantly in relative abundance with time
("BH” adjusted P-value $<$ 0.10 \citet{YBenjamini1995})). When SSU rRNA
gene abundances were combined at the taxonomic rank of "Class", only
Bacilli (decreased), Flavobacteria (decreased), Gammaproteobacteria
(decreased) and Herpetosiphonales (increased) significantly changed in
relative abundance with time (P-value $<$ 0.10, Figure~\ref{fig:time_class}). Of
the 29 OTUs that changed in relative abundance with time, 14 were found to
have incorporated $^{13}$C into DNA (Figure~\ref{fig:time} and below). Of
the 14 OTUs that were found to have both incorporated $^{13}$C into DNA and
significantly changed in relative abundance with time, OTUs that incorporated
$^{13}$C from $^{13}$C-cellulose increased with time whereas those that
incorporated $^{13}$C from $^{13}$C-xylose decreased over time and OTUs that
responded to both substrates were found to either have increased or decreased
over time (Figure~\ref{fig:time}~and~\ref{fig:babund}).

\subsection{OTUs that assimilated $^{13}$C into DNA} \label{responders}
% Fakesubsubsection:Within the first 7 days of incubation approximately 63\% 
If an OTU exhibited strong evidence for $^{13}$C incorporation into DNA we
refer to that OTU as a "responder" (see Supplemental Note 1.7.4 for our
operational definition of "responder"). We evaluated XXXX and XXXX OTUs for
evidence of $^{13}$C incorporation into DNA from $^{13}$C-cellulose and
$^{13}$C-xylose, respectively. We detected a 41 OTUs that responded to
13-xylose and 55 OTUs that responded to $^{13}$C-cellulose (Figure~\ref{fig:l2fc}, 
Tables~{tab:cell}~and~\ref{tab:xyl}). Eight OTUs responded to both xylose and
cellulose. The number of xylose responders peaked at days 1 and 3 and declined
with time. In contrast, the number of cellulose responders increased with time
peaking at days 14 and 30 (Figure~\ref{fig:rspndr_count}). 

The phylogenetic types of putatively $^{13}$C labeled OTUs (i.e. responders)
changed with time (Figure~\ref{fig:l2fc}~and~\ref{fig:xyl_count}). On day 1,
Bacilli OTUs represented 84\% of xylose responders, and the majority of these
OTUs were closely related to cultivated representatives of the genus
\textit{Paenibacillus} (n $=$ XX, Table~\ref{tab:xyl}). For example, "OTU.57"
(Table\ref{tab:xyl}), annotated as \textit{Paenibacillus}, has a strong signal
of $^{13}$C incorporation from 13C-xylose into DNA at day 1, at its maximum
relative abundance in non-fractionated soil DNA. The relative abundance of
"OTU.57" declines until day 14 and evidence of 13C labeling is not detectable
after day 1 (Figure X). On day 3 \textit{Bacteroidetes} OTUs comprised 63\% of
xylose responders (Figure~\ref{fig:xyl_count}) and were closely related to
cultivated representatives of the \textit{Flavobacteriales} and
\textit{Sphingobacteriales} (Table~\ref{tab:xyl}). For example, "OTU.14",
annotated as a Flavobacterium, has a strong signal for $^{13}$C labeling
from $^{13}$C-xylose at days 1 and 3 coinciding with its maximum relative
abundance in non-fractionated soil DNA. The relative abundance of "OTU.14" then
declines until day 14 and evidence of $^{13}$C  labeling is not significant
after day 3 (Figure X). Finally, on day 7, \textit{Actinobacteria} OTUs
represented 53\% of the xylose responders and these OTUs were closely related
to cultivated representatives of \textit{Micrococcales} (Table~\ref{tab:xyl}).
For example, "OTU.4", annotated as \textit{Agromyces}, has signal of $^{13}$C
labeling on days 1 through 7 with the strongest evidence $^{13}$C labeling
at day 7, its relative abundance in non-fractionated soil increases until day
3 and then declines gradually until day 30 and evidence of 13C labeling
declines after day 7 (Figure X). \textit{Proteobacteria} were also common
among xylose responders at day 7 where they comprised 40\% of xylose responder
OTUs. Notably, \textit{Proteobacteria} represented the majority (6 of
8) of OTUs that responded to both cellulose and xylose. The
majority (86\%) of xylose responders shared $>$ 97\% SSU rRNA gene
sequence identity to bacteria already cultivated in isolation
(Table~\ref{tab:xyl}). 

%Fakesubsubsection:Cellulose responders were
The phylogenetic types of cellulose responders did not change with time to the
same extent as the phylogenetic types of xylose responders. Also, in contrast
to xylose responders, cellulose responders often belonged to non-cultivated
microbial clades. Both the relative abundance and the number of cellulose
responders increased over time peaking at days 14 and 30
(Figures~\ref{fig:l2fc}, \ref{fig:rspndr_count}, and \ref{fig:babund}). The
phylogenetic composition of cellulose responders changed little between days 14
and 30 (Table~\ref{tab:cell}). Cellulose responders belonged to the
\textit{Proteobacteria} (46\%), \textit{Verrucomicrobia} (16\%),
\textit{Planctomycetes} (16\%), \textit{Chloroflexi} (8\%),
\textit{Bacteroidetes} (8\%), \textit{Actinobacteria} (3\%), and
\textit{Melainabacteria} (1 OTU) (Table~\ref{tab:cell}). The majority (86\%) of
cellulose responders in the \textit{Proteobacteria} were closely related ($>$
97\% identity) to bacteria already cultivated in isolation, including
representatives of the genera: \textit{Cellvibrio}, \textit{Devosia},
\textit{Rhizobium}, and \textit{Sorangium}, which are known for their ability
to degrade cellulose (Table~\ref{tab:cell}). A relatively high number of
cellulose responding OTUs belonging to the \textit{Proteobacteria} (13 OTUs)
OTUs were members of the \textit{Alphaproteobacteria}. Other proteobacterial
cellulose responders belonged to \textit{Beta-} (4 OTUs), \textit{Gamma-} (5
OTUs), and \textit{Deltaproteobacteria} (6 OTUs). 

The majority (85\%) of cellulose responders outside of the
\textit{Proteobacteria} shared  $<$ 97\% SSU rRNA gene identity to bacteria
already cultivated in isolation. For example, most (70\%) of the
\textit{Verrucomicrobia} cellulose responders fell within a few unidentified
\textit{Spartobacteria} clades, and these shared $<$ 85\% SSU rRNA gene
sequence identity to any characterized isolate. The \textit{Spartobacteria} OTU
"OTU.2192" was characteristic of many cellulose responders (Figure X). In
non-fractionated soil DNA, "OTU.2192" gradually increased in relative abundance
with time and evidence for $^{13}$C labeling of "OTU.2192" also increased
gradually over time with the strongest evidence at days 14 and 30 (Figure X).
\textit{Choloflexi} cellulose responders predominantly belonged to an
unidentified clade within the \textit{Herpetosiphonales} and these shared $<$
89\% SSU rRNA gene sequence identity to any characterized isolate.
Characteristic of other \textit{Chloroflexi} cellulose responders, "OTU.64"
increased in relative abundance over 30 days and evidence for $^{13}$C labeling
of "OTU.64" peaked days 14 and 30 (Figure X). Cellulose responders found within
the \textit{Bacteroidetes} differed from the \textit{Bacteroidetes} xylose
responders falling instead within the \textit{Cytophagales} as opposed to the
\textit{Flavobacteria} or \textit{Sphingobacteriales}. \textit{Bacteroidetes}
cellulose responders included one OTU that shared 100\% SSU rRNA gene sequence
identity to species of \textit{Sporocytophaga}, a genus that includes known
cellulose degraders.

\subsection{Characteristics of cellulose and xylose responders}
% Fakesubsubsection:Cellulose responders tended
Cellulose responders tended to have lower relative abundance in
non-fractionated soil DNA, demonstrated signal consistent with higher
$^{13}$C:12C ratios in DNA upon $^{13}$C labeling, and lower estimated
\textit{rrn} copy number than xylose responders. In the non-fractionated soil
DNA, cellulose responders had significantly lower relative abundance (7e$^{-4}$
(s.d. 2e$^{-3}$)) than xylose responders (2e$^{-3}$ (s.d. 4e$^{-3}$))
(Figure~\ref{xyl_count}, P-value $=$ 0.00028, Wilcoxon Rank Sum test). Six of
the ten most common OTUs observed in the non-fractionated soil DNA responded to
xylose, and, of the 10 most abundant $^{13}$C substrate responders in the
non-fractionated soil DNA 8 were xylose responders and 2 were cellulose
responders. However, $^{13}$C-xylose and $^{13}$C-cellulose responders included
OTUs at both high and low abundance (Figure~\ref{fig:shift}). Two
$^{13}$C-cellulose responders were not found non-fractionated soil DNA
(``OTU.862'' and ``OTU.1312'', Table~\ref{tab:cell}).

% Fakesubsubsection:Cellulose responders exhibited a greater shift in BD
DNA buoyant density increases as its ratio of $^{13}$C to $^{12}$C increases.
An organism that only assimilates C into DNA from $^{13}$C labeled sources,
will have a greater DNA $^{13}$C:$^{12}$C than an organism utilizing a mixture
of $^{13}$C labeled and unlabeled C sources (see Supplemental~Note~1.8).
Therefore, the specificity of C use can be evaluated by the change in DNA
buoyant density (BD) upon $^{13}$C labeling. In this study, we do not know the
absolute abundance of OTUs across the density gradient as our SSU rRNA gene
sequence counts are compositional in nature hence we cannot assess absolute OTU
buoyant density shifts due to $^{13}$C labeling. However, we can evaluate
relative C use specificity by quantifying and comparing the shift in the
relative abundance profile for an OTU along the density gradient in response to
$^{13}$C labeling. Specifically, in this study we calculated each OTU's
relative abundance density gradient profile center of mass shift for each
label/control DNA density gradient pair (see supplemental methods for the
detailed calculation). We refer to this metric as $\Delta\hat{BD}$. This metric
indicates relative differences in DNA 13C:12C and can be used to compare DNA
13C:12C between groups of responders. $\Delta\hat{BD}$ does not represent the
true density shift for an OTU because it is based on relative abundance and is
therefore not directly comparable to literature values for DNA density shifts
due to isotopic labeling. Cellulose responder $\Delta\hat{BD}$ (0.0163
g mL$^{-1}$ (s.d. 0.0094)) was significantly greater than that of xylose
responders (0.0097 g mL$^{-1}$ (s.d. 0.0094)) (Figure~\ref{fig:shift}, P-value
$=$ 1.8610e$^{-6}$, Wilcoxon Rank Sum test). 

% Fakesubsubsection:We assessed phylogenetic
We assessed phylogenetic clustering of $^{13}$C-responsive OTUs with the
Nearest Taxon Index (NTI) and the Net Relatedness Index (NRI)
\citep{Webb2000}. We also quantified the average clade depth of cellulose
and xylose responders with the consenTRAIT metric \citep{Martiny2013}. 
Briefly, NRI and NTI evaluate phylogenetic clustering against a null model for
the distribution of a trait in a phylogeny. The NRI and NTI values are
z-scores and thus the greater the magnitude of the NRI/NTI, the stronger
the evidence for clustering (positive values) or overdispersion (negative
values). NRI assesses overall clustering whereas the NTI assesses terminal
clustering. An NRI of 1.96, for instance, would signify overall
phylogenetic clustering with a corresponding P-value of 0.05
\citep{Evans2014a}. The consenTRAIT metric is a measure of the average
clade depth for a trait in a phylogenetic tree. NRI values indicate that
cellulose responders clustered phylogenetically (NRI: 4.49) while xylose
responders are overdispersed (NRI: -1.33). NTI values show that both
cellulose and xylose responders are terminally clustered (NTI: 1.43 and
2.69, respectively). The consenTRAIT clade depth for xylose and cellulose
responders was 0.012 and 0.028 SSU rRNA gene sequence dissimilarity,
respectively. As reference, the average clade depth is 0.017 SSU rRNA gene
sequence dissimilarity for arabinose utilization (another five C sugar
found in hemicellulose) and was 0.013 and 0.034 SSU rRNA gene sequence
dissimilarity for glucosidase and cellulase activity of isolates in
culture, respectively \citep{Martiny2013,Berlemont2013}.
