\section{Results and Discussion}

In this study, we couple DNA-SIP with high-throughput sequencing in order to better understand microbial community C use dynamics in soil. A series of parallel soil microcosms amendeded with an identical C mixture, in which the only difference is the identity of the \textsuperscript{13}C-labeled substrate, were incubated for 30 days. The C mixture was designed to approximate freshly degrading plant biomass and either xylose or cellulose were isotopically labeled to examine the dynamics of C assimilation for labile soluble C and insoluble polymeric C. A total of 5.3 mgC g\textsuperscript{-1} soil (including 0.42mg xylose-C g\textsuperscript{-1} soil and 0.88mg cellulose-C g\textsuperscript{-1} soil) was added to each microcosm and this represented 18\% of the total C present in the soils. Microcosms were harvested at discrete time points during the incubation period and the temporal and isotope assimilation dynamics of the microbial community were measured by sequencing 16S rRNA in the bulk microbial community and fractions from CsCl gradient fractionation (\href{https://www.authorea.com/users/3537/articles/8459/master/file/figures/20140708_ConceptualFig2/20140708_ConceptualFig2.pdf}{Fig. S1}). Our approach provided the sensitivity necessary to detect xylose and cellulose degradation amidst a low dose complex C amendment (5.3mgC g\textsuperscript{-1} soil) with each representing 20 and 38 percent, respectively, of the total C added. Xylose degradation was observed immediately within the first 7 days, while cellulose degradation is observed after 14 days.

\textbf{Temporal microbial and C-cycling dynamics.}  
The dynamics of \textsuperscript{13}C-cellulose and \textsuperscript{13}C-xylose assimilation varied dramatically for different microorganisms. Isotope incorporation into DNA was revealed by analyzing variation in 16S rRNA amplicons across gradient fractions (n = 20) from control samples in relation to identical experimental samples that differed by a single substitution of \textsuperscript{12}C-cellulose or \textsuperscript{12}C-xylose with their \textsuperscript{13}C equivalents (\href{https://www.authorea.com/users/3537/articles/8459/master/file/figures/20140708_ConceptualFig2/20140708_ConceptualFig2.pdf}{Fig. S1}). Isotope incorporation changes amplicon composition relative to control in the gradient fractions and this effect can be visualized in ordination by divergence of experimental samples from corresponding control points (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig. 1}). Primary variation of amplicon composition in gradient fractions along axis 1 is attributed to varying bouyant densities of genomes due to G+C content (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig. 1}). Divergence due to isotope incorporation can be seen in high-buoyant density fractions that partition along axis 2 in Figure 1. The differential divergence of high density fractions in the \textsuperscript{13}C-xylose treatment compared to the \textsuperscript{13}C-cellulose treatment is indicative of a difference in the \textsuperscript{13}C-assimilating OTUs for each of the substrates (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig. 1A}). Had the isotope incorportation from \textsuperscript{13}C-xylose and \textsuperscript{13}C-cellulose occured in the same community members, the divergence of the high-buoyant density fractions of these two treatments relative to control would have coincided in the ordination space.  

The \textsuperscript{13}C-incorporation reveals temporal dynamics of C degradation demonstrated by \textsuperscript{13}C-xylose incorporation at days 1, 3, and 7 and \textsuperscript{13}C-cellulose incorporation at days 14 and 30 (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig. 1B}). In support of this, the bulk community sequencing demonstrates significant (pval) microbial community changes over time. Although within a single time point, the bulk community demonstrated no significant difference between treatments (Fig Sx). The temporal dynamics reveal the composition of \textsuperscript{13}C-xylose assimilating amplicons are different for each of the days the label is detected based on their separate distributions for each of the time points (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig. 1B}). The disappearance of the \textsuperscript{13}C-xylose incorporation signature (relative to control) for days 14 and 30 result from loss of \textsuperscript{13}C-label in DNA over time. This occurs by dilution of \textsuperscript{13}C-label out of the DNA when a switch from \textsuperscript{13}C to \textsuperscript{12}C substrate utilization takes place during biomass turnover and/or predation. \textsuperscript{13}C-cellulose incorporation isn't detected until day 14 and amplicon composition is consistent for both days 14 and 30 (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/ordination_all1/ordination_all1.png}{Fig. 1B}). The consistency of amplicon composition for cellulose degradation over time compared to xylose suggests a wider array of microorganisms utilize xylose, whereas, cellulose utilization occurs in a select few. This is consistent with long standing notions that more microorganisms are capable of utilizing simple carbohydrates than complex C substrates. Overall patterns of C degradation observed in this study demonstrate different microbial community members are responsible for the consumption of these two substrates; xylose is consumed quickly, whereas, cellulose decomposition takes longer. This suggests a pattern of microbial community transition accompanying the decomposition process. This is consistent with \cite{Engelking_2007} who observed as much as 75\% of labile C respired or converted into microbial biomass in the first 5 days of decomposition, whereas, cellulose degraders take longer to respond \cite{Hu_1997} with less than 42\% of cellulose metabolized over the first 5 days of incubation \cite{Engelking_2007}. 

\textbf{Differential C utilization by taxa.} Individual OTUs that assimilated \textsuperscript{13}C-substrates were identified using the DESeq framework \cite{Anders_Huber_2010} to analyze differential representation in heavy fractions (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/l2fc_fig1/l2fc_fig.pdf}{Fig. 2}). There were 43 and 35 unique OTUs that significantly (\textit{p}-value \textless 0.10) assimilated \textsuperscript{13}C-xylose and \textsuperscript{13}C-cellulose, respectively; herein called 'responders' (\href{https://www.authorea.com/users/3537/articles/8459/master/file/figures/OTU_screening_schematic/OTU_screening_schematic.pdf}{Fig. S2}, \href{https://www.authorea.com/users/3537/articles/8459/master/file/figures/l2fc_fig_pVal/l2fc_fig_pVal.png}{Fig. S3}). Overall, we found xylose responders were from higher rank abundances than cellulose responders, however, cellulose responders exhibited a greater change in buoyant density (i.e. assimilated more \textsuperscript{13}C) than xylose responders in response to isotope incorporation (Figure 3). 

\textit{Xylose}. Within the first 7 days of incubation an average 63\% of \textsuperscript{13}C-xylose was respired and only an additional 6\% more was respired between days 7 and 30. Of the 60 total xylose responders 53 were responsive within the first 7 days and only 7 responders detected for days 14 and 30 (supplemental table). At day 1, 57\% of responsive OTUs belong to Firmicutes (Paenibacillaceae, Planococcaceae, and Bacillaceae) and the remaining 43\% of responders were comprised of 19\% Bacteroidetes (Flavobacteriaceae), 14\% Proteobacteria (Enterobacteriaceae, Comamonadaceae, and uncultured Gammaproteobacteria), and 10\% Actinobacteria (Micrococcaceae and Microbacteriaceae) (\href{https://www.authorea.com/users/3537/articles/3612/master/file/figures/l2fc_fig1/l2fc_fig.pdf}{Fig. 2}). At any given time soils harbor microorganisms at varying degrees of dormancy depending on nutrient availability \cite{Jones_2010}. The sudden addition of our complex C mixture would most certainly prompt dormant and non-dormant microbes back into metabolic activity, with those exhibiting higher rRNA operon copy numbers responding the fastest. The responders identified at day 1 for xylose utilization have all been noted for exhibiting some form of dormancy strategy \cite{Jones_2010, Mulyukin_2009, Darcy_2011, Sachidanandham_2008, Finkel_2006, Rittershaus_2013, Tada_2013, Lay_2013} as well as 6-14 rRNA operon copies with the exception of the Betaproteobacteria Comamonadaceae and the Actinobacterial OTUs which exhibit 1-2 copies according to representative taxa in the rrnDB v. 3.1.227 \cite{18948294,11125085}. 

Day 3 presents a decrease in Firmicutes responders (from 12 OTUs to 1) and an increase in Bacteroidetes (from 4 to 8) from day 1, as well as, the onset of Verrucomicrobia reponders. With the exception of the single Firmicutes (Paenibacillaceae), responders on day 3 possess 3-6 rRNA operon copies, less than responders on day 1. A substantial amount (75\%) of xylose responders for day 7 had not previously been identified as responders at earlier time points which attests to the wider number of taxa able to use xylose. Each of the 10 Actinobacteria responders, the most dominant response group (50\%) at day 7, belonged to a different family making it the phylum with the most wide-spread use of xylose. However, it should be noted that they were confined within two Actinobacterial Orders; Frankiales and Micrococcales. A Paenibacillus (100\% identity) was the only OTU to be identified at every time point (up to day 30) as a xylose responder. It was the the second most enriched xylose responder, log2fold change (l2fc) of 3.5, measured in the time series second only to a Gammaproteobacteria (Xanthomonadaceae; l2fc = 3.7).   

\textit{Cellulose}. In contrast with xylose responders, there were only three \textsuperscript{13}C-cellulose responders detected within the first 7 days of incubation and 46 for days 14 and 30. An average of 16\% of the \textsuperscript{13}C-cellulose added was respired within the first 7 days, 38\% by day 14, and 60\% by day 30. The earliest responders detected for \textsuperscript{13}C-cellulose assimilation were a canonical cellulose degrader, Cellvibrio, in Proteobacteria and a novel clade in Chloroflexi. At day 14, 54\% of the responders belong to Proteobacteria (65\% Alpha-, 23\% Gamma-, and 12\% Beta-), 13\% Chloroflexi, 13\% Planctomycetes, 10\% Verrucomicrobia, and 6\% Actinobacteria.

All responders for day 30 (n = 15) had been identified as responders at earlier time points in this study except two; a Deltaproteobacteria (Sandaracinaceae family) and a Bacteroidetes (Cytophagaceae family). While there are known cellulose degraders in the Bacteroidetes Cytophagaceae family, there are currently no known cellulose degraders in the Sandaracinaceae family although it's sister family Polyangiaceae has known cellulose degraders (\cite{Reichenbach_2006}, Bergey's ISBN:978-0-387-24145-6). Throughout the time series, responders with the greatest enrichment were Verrucomicrobia (Verrucomicrobiaceae), Chloroflexi, Cyanobacteria, Proteobacteria (Cellvibrio, Brevundimonas, Stenotrophomonas, Devosia), and Planctomycetes (Planctomycetaceae).      

Verrucomicrobia, a phylum found to be ubiquitous and in high abundance in soil \cite{Zhang_2008, Bergmann_2011, Fierer_2013}, have been noted for degradation of polysaccharides in soil, aquatic, and anoxic rice patty soils \cite{Fierer_2013,Herlemann_2013,10543821}. In this study, Verrucomicrobia comprise ~11\% of the total cellulose responder OTUs detected, most of which belong to the uncultured FukuN18 clade originally identified in freshwater lakes \cite{14602613, Glockner_2000, Parveen_2013}. Yet the largest enrichment measured (l2fc = 3.7) during the whole time series for \textsuperscript{13}C-cellulose assimilation was by an uncultured Verrucomicrobia in the Verrucomicrobiaceae family on day 14. 

Chloroflexi, ubitiquous across many diverse environments, are traditionally known for their metabolically dynamic lifestyles ranging from anoxygenic phototropy to organohalide respiration \cite{Yamada_2009,14527284,Hug_2013,Seshadri_2005,Tang_2011,dworkin2006the}. Yet, only recently have we shifter focus towards the metabolic functions of Chloroflexi in C cycling \cite{Hug_2013,Goldfarb_2011,Cole_2013}. In this study, we identified a previously undescribed clade within the Chloroflexi class (closest relative at a 96\% identity being Herpetosiphon) that exhibited a high substrate specificity based on a density shift peak between 0.03-0.04 gmL\textsuperscript{-1}. We observed no other cellulose utilization in Chloroflexi outside of this clade although many members of this phylum have previously been demonstrated or implicated in cellulose utilization \cite{Goldfarb_2011,Cole_2013,Hug_2013}. 

One of the most interesting findings is a single Cyanobacterial responder OTU, which exhibit the third greatest enrichment measured (l2fc = 3.35). This OTU falls into the recently described candidate phylum Melainabacteria \cite{Di_Rienzi_2013}, although its phylogenetic position is debated \cite{Soo_2014}. More importantly, the catalog of metabolic capabilities associated with Cyanobacteria are quickly expanding \cite{Di_Rienzi_2013, Soo_2014}. Our findings provide evidence of cellulose degradation for Melainabacteria, supporting hypothesized polysaccharide degradation suggested by genomic analysis \cite{Di_Rienzi_2013}.     

Proteobacteria represent 47\% of all cellulose responders identified.  Of those, Cellvibrio (named for it's cellulose degrading abilities) accounted for 5\% of all Proteobacterial responders detected. Details about max density shift (need to talk to Chuck because this data is not easy to acquire). Other prominent cellulose degrading Proteobacteria, Stenotrophomonas and Devosia have previous been demonstrated in degrading cellulose \cite{Trujillo_Cabrera_2012, Verastegui_2014}. Brevundimonas has not previously been identified as a cellulose degrader, but has been show to degrade cellouronic acid, an oxidized form of cellulose \cite{Tavernier_2008}.

Between both the xylose and cellulose treatments there were 6 shared responders of all responders (n = 72); Stenotrophomonas, Planctomyces, Rhizobiaceae, Comamonadaceae, Cellvibrio. Of these, Stenotrophomonas and Comamonadaceae are the only taxa that are among the top ten responders in both treatments. On the other hand, the only shared responder that is not among the top ten responders for either the cellulose or xylose treatment is Rhizobiaceae. Two of the shared responders corresponded in time between the two treatments (supplemental table); Cellvibrio (day 3) and Planctomyces (day 14).  

\textit{Responder Characteristics}. The kernel density estimate (KDE) of buoyant density shifts resulting from \textsuperscript{13}C-assimilation reveal that cellulose responders exhibit a significantly (wilcox rank sum; p\textless...) greater buoyant density shift than xylose responders (Figure 3A). First, a density profile for each responder is generated for the experimental and control treatment at each of the sampling time points using relative abundances from sequence libraries (Fig Sx). Then the center of mass is measured for the density profile of a responder in the control and respective experimental treatment for a single time point. A density shift is then calculated by differentially calculating the center of mass for the control from the experimental. Finally, this is repeated for all responders at each time point for both the \textsuperscript{13}C-cellulose and \textsuperscript{13}C-xylose treatments. Each KDE curve represents the collection of density shifts calculated for all responders at each time point within a treatment (Figure 3A). The ‘difference in center of mass’ is a function of both the density shift for individual DNA molecules and the heterogeneity of labeling for the population of cells within that OTU. As such, we would not expect to observe the maximum density shift (0.04gmL\textsuperscript{-1}) measured in pure cultures with 100\% \textsuperscript{13}C-labeling of DNA (cite). We observe xylose utilizers having a smaller density shift (\textless0.02 gmL\textsuperscript{-1}) than cellulose utilizers (0.005-0.03 gmL\textsuperscript{-1}), with few exceptions. This suggests greater substrate specificity among cellulose degraders than xylose degraders. Partial \textsuperscript{13}C-labeling (\textless 0.04gmL\textsuperscript{-1} density shift) could be a result of various trophic strategies such as (1) assimilation of C from multiple substrates (both \textsuperscript{12}C and \textsuperscript{13}C in this instance) or (2) \textsuperscript{13}C-label dilution as it cascades through trophic levels via consumption of \textsuperscript{13}C-labeled organisms or waste products from organisms that are metabolizing the \textsuperscript{13}C-substrate.

Most xylose responders are found at higher rank abundances than cellulose responders, which fall among the rarer taxa in the tail of the RA curve (Fig 3B). This demonstrates that many taxa important to C-cycling are present in the rarer fraction of the overall microbial community and may be difficult or unable to detect in bulk community sequencing efforts. The shift in cellulose responders is more readily observable in the bulk community abundances than discerned with xylose responders (Fig Sx and Sy). This is likely due to the initial low abundance of these phyla, where changes in bulk community abundance are more pronounced and easier to detect. Comparatively, phyla of consistently high abundance (as with xylose responders) mask response changes unless they present changes of grand proportions. For example, the increase in Bacteroidetes in the xylose treatment at day 3 is not captured by the bulk community abundances. In another instance, the increased response from Proteo- and Actinobacterial OTUs at day 7 is also observed in the bulk community analysis as a marginal increase, yet it would be difficult to differentiate that change from natural variation or methodological noise.



\textbf{Carbon substrate utilization is inconsistent within phylum.} Dynamic patterns of \textsuperscript{13}C-assimilation from xylose and cellulose occur not at a course phylum-level, but instead occur in discrete, fine-scale taxonomic units (Figure 4). Responders for xylose and cellulose are widespread across 6 and 7 phyla, respectively (Figure 4). There are 5 phyla containing responders for both treatments although within a phylum we observe utilization of both substrates by only six taxa (discussed previously). This suggests assigning phylum level functionality is not an accurate depiction of soil C utilization though ecological functionality is often assigned at the phylum level (refs).

In this study, we have identified Actinobacteria responders for both substrates with a peak shift of ~0.036 gmL\textsuperscript{-1} for cellulose and ~x gmL\textsuperscript{-1} for xylose, suggesting a strong substrate specificity (Figures Sx and Sz - the substrate utilization charts). Albeit, there are no OTUs within Actinobacteria that responded to both xylose (Microbacteriaceae, Micrococcaceae, Cellulomonadaceae, Nakamurellaceae, Promicromonosporaceae, and Geodermatophilaceae) and cellulose (Streptomycetaceae and Pseudonocardiaceae). This information may suggest that while Actinobacteria exhibit an ability to utilize an array of carbon substrates, substrate use may be more clade specific and not widespread throughout the phylum (Figure 4). In the same vein, we identified Bacteroidetes responders for both substrates, yet, at a finer taxonomic resolution there is a clear differential response for xylose (Flavobacteriaceae and Chitinophagaceae) and cellulose (Cytophagaceae). Disagreeing correlations of phylum level abundance associated with C availability has been the source of debate for the role of Bacteroidetes in the degradation process \cite{Fierer_2007,Rui_2009,Sharp_2000,L_pez_Lozano_2013,Bastian_2009}. Our results would suggest that both perspectives are correct at a phylum level, but highlights the need of greater resolution to capture the subtleties of substrate utilization.   

Whole phylum responses were not detected for xylose or cellulose yet utilization of these substrates spanned many phylogenetically diverse groups. Within each phylum we observed substrate utilization at the clade or single taxa level with each exhibiting a unique pattern of \textsuperscript{13}C-assimilation over time (Figure 4, heatmap). In a study that amended forest soils with single C substrates in the presence of 3-bromodeoxyuridine (BrdU), they determined that more than 500 taxa responded to labile C but occured predominantly in two phlya, Proteobacteria and Actinobacteria \cite{Goldfarb_2011}. On the other hand, the soils amended with polymeric C such as cellulose and lignin were noted for spanning eight phyla, but were limited to a small number of taxa within each of those phyla that were responders \cite{Goldfarb_2011}. It has previously been suggested that all taxa within a phylum are unlikely to share ecological characteristics \cite{Fierer_2007}, and furthermore, within a species population \cite{Choudoir_2012,Preheim_2011,Hunt_2008}. Habitat traits of coastal Vibrio isolates were mapped onto microbial phylogeny revealing discrete ecological populations based on seasonal occurrence and particulate size fractionation \cite{Preheim_2011,Hunt_2008}. Yet, it has been proposed that the microbial community functionality responsible for soil C cycling appear at the level of phlya rather than species/genera \cite{Schimel_2012}. The traditional phylum level assignment conventions could in part be due to limitations in finer scale taxonomic identifications or methodological limitations (ie sequencing depth), but our data in concert with others \cite{Goldfarb_2011,Fierer_2007,Choudoir_2012,Preheim_2011,Hunt_2008} would suggest that portraying the response of a few OTUs or clades as a phylum level response would be overreaching serves as a strong argument towards moving away from extrapolating substrate utilization to phylum level responses.

\textbf{Conclusions.} We have demonstrated how next generation sequencing-enabled SIP gives a taxa level resolution for substrate utilization. Using this technique, we are able to resolve discrete OTUs that would otherwise be missed using bulk community sequencing efforts. Additionally, this technique provides greater taxonomic resolution than previous techniques (cloning, TRFLP, ARISA) used to determine substrate utilizing community members. Traditionally, SIP is performed with a single time point in an attempt to minimize or eliminate cross-feeding of the substrate. However, cross-feeding is an exciting advantage of this method because it enables us to track substrates through many trophic cascades using degree of \textsuperscript{13}C-labeling in DNA as guide, with primary consumers being the most \textsuperscript{13}C-enriched and subsequent trophic levels being less enriched \cite{Morris_2002,McDonald_2005,Ziegler_2005}. This relationship of trophic level consumption to dilution of label will facilitate tracking C as it moves through various operational taxonomic units (OTUs) in the soil. While we are currently able to resolve highly responsive OTUs, there is still a need to resolve taxa that are partially responsive which we cannot differentiate from noise with confidence at this time. Yet, given the ability to resolve partially responsive taxa, the ecology would still be difficult to discern. For example, a generalist utilizing many substrates including \textsuperscript{12}C substrates and the \textsuperscript{13}C-labeled substrate may exhibit the same partial labeling that a specialist utilizing both the \textsuperscript{13}C-substrate and the same substrate (unlabeled) that is inherent in the soil. Additionally, partially labeled taxa could be further down the trophic cascade including predators or secondary consumers of waste products from primary consumer microbes that were highly labeled.   

Our study is consistent with carbon degradative succession that has previously been demonstrated \cite{Bastian_2009} (more refs). We demonstrate a rapid decrease in the labile carbon, xylose, confirmed by its \textsuperscript{13}C label incorporation into the microbial community DNA during the first 7 days of the experiment, after which, the label is not detectable in the DNA. Subsequently our data demonstrates a slow degradation of the more recalcitrant, polymeric carbon demonstrated by \textsuperscript{13}C-cellulose label incorporation into the microbial community DNA at 14 and 30 days. We did not observe the \textsuperscript{13}C-cellulose signal leave the DNA within the time limits, 30 days, of our experiment. This degradative succession is also confirmed by isotopic analysis of the soil from the microcosms (Table S1). 

We did not observe consistent C utilization at the phylum level although both xylose and cellulose utilization were observed across 7 phyla each revealing a high diversity of bacteria able to utilize these substrates. The high taxonomic diversity may enable substrate metabolism under a broad range of environmental conditions \cite{Goldfarb_2011}. Other studies of microbial communities have observed a positive correlation with taxonomic or phylogenetic diversity and functional diversity \cite{Fierer_2012,Fierer_2013,Philippot_2010,Tringe_2005,Gilbert_2010,Bryant_2012}. The data presented here supports that specific functional attributes can be shared among diverse, yet distinct, taxa while closely related taxa may have very different physiologies \cite{Fierer_2012,Philippot_2010}. This information adds to the growing collection of data suggesting that community membership is important to biogeochemical processes. Furthermore, it highlights a need to examine substrate utilization by discrete microbial taxa within a whole community context to better understand how specific community members function within the whole. The sensitivity of SIP-NGS provides a means to elucidate substrate utilization by discrete microbial taxa with the hope that we can begin to construct a belowground C food web.          
