\section{Results and Discussion}


With the rapid advancement and declining costs of high throughput sequencing, it has become increasingly easy to probe microbial communities.  In this study, we couple stable-isotope probing with 454 pyrosequencing in order to better understand organic matter decomposition dynamics as a function of soil microbial community C utilization. We ran a temporal series of parallel microcosms and measured the changes in the microbial community as result of the addition of a complex carbon mixture using 454 pyrosequencing of the bulk microbial community and fractions from SIP gradient fractionation (Fig. S1). Overall, changes in microbial community composition over time are consistent with C decomposition being accompanied by a microbial community succession. Analysis of the sequenced bulk community DNA demonstrates (insert taxa here) as the most dominant taxa at days 1, 3, and 7 followed by a transition to a (insert taxa here) dominated community at days 14 and 30. Most abundant OTUs at earlier time points were most closely related to members of Bacteriodetes, Actinomycetes, and Proteobacteria. Abundant OTUs of later time points were most closely related to members of Chloroflexi, Proteobacteria, and Verrucomicrobia.  The rank abundance of the community fluctuates minimally over time (Fig S2A) accompanied with temporally associated community shifts (FigS2B).   

Twenty fractions from a cesium chloride gradient fractionation for each treatment at each time point were sequenced (Fig. S1). Using NMDS analysis from weighted unifrac distances the relationship between all buoyant densities from all treatments and time points are plotted (Fig 1). \textsuperscript{13}C-labeled organisms are expected be to found in the higher buoyant density fractions. Each point on the NMDS represents the microbial community from a single fraction where the size of the point is representative of the density of that fraction and the colors represent the treatments (Fig1A) or days (Fig1B). The high density fractions (1.73-1.74...figure out exact densities) that are differentiating from the control along NMDS2 correspond to fractions that contain \textsuperscript{13}C-labeled OTUs (herein called 'responders'). The differential separation of high density fractions in the \textsuperscript{13}C-xylose treatment compared to the \textsuperscript{13}C-cellulose treatment is indicative of a difference in the responders for each of the substrates (Fig 1A). This data also presents an observable time signature of responders at days 1,3, and 7 for the xylose treatment and days 14 and 30 for the cellulose treatment (Fig1B). This demonstrates that different microbial community members are responsible for the consumption of these two different substrates and that xylose is consumed quickly, whereas, cellulose decomposition takes longer. This supports the hypothesis of a microbial community succession during the decomposition process. Furthermore, this demonstrates the sensitivity of this technique by being able to detect \textsuperscript{13}C-label incorporation in samples with low C additions (blah mg g\textsuperscript{-1} soil).    

Using fractions from within a denisty range of 1.7125-1.755 g/ml, relative abundances of phyla in the experimental treatments were compared to the respective relative abundances in the control treatment to calculate the log\textsubscript{2} fold change (Fig2). OTUs that exceed a user defined 0.75 log\textsubscript{2} fold change threshold are identified as potential responders for that substrate. This threshold was determined by controlling for false discovery rate based on sparsity (ref). The log\textsubscript{2} fold change demonstrates the boom and bust of phyla with time.  More notably, it shows that we should not make phylum level inferences, as only a few OTUs within a phylum are responders. To portray that as a phylum level response would be overreaching. The significance of each OTU's log\textsubscript{2} fold change in relative abundance in the treatment compared to the control (FigSx). We consider 'responders' as OTUs passing a conservative threshold of \textit{p}-value = <0.0 for log\textsubscript{2} fold change in relative abundance (FigSx).   their relative abundance in each fraction can be traced for an experimental treatment and compared to its relative abundance in the control fractions from the same time point (Fig Y).  We then generate C utilization charts to demonstration discrete OTUs in control versus treatment.  

For the \textsuperscript{13}C-xylose treatment at day 1 OTUs within Firmicutes demonstrate the strongest response. This could be because of the spore forming ready-state nature of this phylum (ref). Additionally, Proteobacteria, Bacteriodetes, and Actinobacteria contain responder OTUs at day 1. Day 3 exhibits a strong increase in Bacteriodetes response and the onset of Verrucomicrobia reponders.  Day 7 demonstrates an increased response from Proteo- and Actinobacterial OTUs.  All OTUs have a decreasing log\textsubscript{2} fold change at days 14 and 30, with only a single Firmicutes OTU passing the 'responder' criteria.     

For the \textsuperscript{13}C-cellulose treatment only one Proteobacteria passes the 'responder' criteria at day 3 and two OTUs (Proteobacteria and Chloroflexi) at day 7.  By day 14, responders are detected in Proteobacteria, Verrucomicrobia, Chloroflexi, Plancktomycetes, and Actinobacteria.  The same responders are detected at day 30 with the exception of Actinobacteria and the addition of Bacteriodetes.    

Kernel density estimates (KDE) of the CsCl density shifts measured in \textsuperscript{13}C-xylose were compared to KDE of \textsuperscript{13}C-cellulose (Fig 3A). In general, xylose utilizers have a smaller density shift (<0.02 mg/L) than cellulose utilizers (0.005-0.03), with few exceptions.  This suggests a greater substrate specificity among cellulose degraders than xylose degraders.  The vast majority of xylose degraders are found at a lower rank abundance than cellulose degraders; which fall among the rare taxa in the tail of the rank abundance curve (Fig 3B). This suggests that taxa important to C-cycling may be difficult or unable to detect in bulk community sequencing efforts. This supports evidence of functionally important taxa being in the rare biosphere. Additionally, there are few OTUs that utilize both xylose and cellulose. 

Figure 4 is a tree of 
   


 

 
Boast about what this technique can do/ how it's added to our field.
Discuss the short-comings


While we are currently able to resolve highly responsive OTUs which has provided greater taxonomic resolution than previous techniques (cloning, TRFLP), the next step is to find better ways to resolve taxa that are partially responsive which we are not able to differentiate from noise.  Yet, given the ability to resolve partially responsive taxa, the ecology would still be difficult to discern.  For example, a generalist utilizing many substrates including the 13C-labeled substrate may exhibit the same partial labeling that a specialist utilizing both the 13C-substrate and the same substrate (unlabeled) that is inherent in the soil. Additionally, partially labeled taxa could be further down the trophic cascade including predators or secondary consumers of waste products from primary consumer microbes that were highly labeled.     


Our study is consistent with carbon degradative succession has previously been demonstrated (ref). We demonstrate a rapid decrease in the labile carbon, xylose, confirmed by it’s 13C label incorporation into the microbial community DNA during the first 7 days of the experiment, after which, the label is not detectable in the DNA. Subsequently our data demonstrates a slow degradation of the more recalcitrant, polymeric carbon demonstrated by 13C-cellulose label incorporation into the microbial community DNA at 14 and 30 days.  We did not observe the 13C-cellulose signal leave the DNA within the time limits, 30 days, of our experiment.  This degradation succession is also confirmed by isotopic analysis of the soil from the microcosms.  NMDS demonstrates microbial succession and based on xylose and cellulose treatments separating away from the control, but in opposing directions indicating different microbial community members are responsible for C degradation of the two substrates. 

It is likely that the slow degradation of cellulose can be attributed to the energy-taxing process of synthesizing cellulolytic enzymes and exporting them, as cellulose is broken down externally (Schimel & Schaeffer 2012, Lynd et al 2002).  As a result, microorganisms responsible for the synthesis of cellulases preferentially shuttle energy towards enzyme synthesis rather than biomass until cellulose hydrolysis begins (Schimel & Schaeffer 2012).  This accounts for the delay in growth and ultimately the slow decomposition of cellulose (Perez et al 2002, Schimel & Schaeffer 2012).

Traditionally, SIP is performed with a single time point in an attempt to minimize or eliminate cross-feeding of the substrate, as cross-feeding could cause false positives in the data set.  However, in a temporal food web mapping study, this cross-feeding is an exciting advantage of this system because it will enable us to track substrates of interest (i.e. cellulose) through many trophic cascades.  Observed organisms will exhibit a range of 13C labeling, 0-100\%, with primary consumers being the most enriched and subsequent trophic levels being less enriched (Morris 2002, McDonald 2005, Ziegler 2005).  This relationship of trophic level consumption to dilution of label will facilitate tracking C as it moves through various operational taxonomic units (OTUs) in the soil.  


 




"The genetic potential to produce extracellular enzymes involved in the degradation of rela- tively labile forms of C, including most abundant plant structural C polymers such as cellulose and hemicellulose, seems to be less conserved among different phylum of the soil bacteria (Figure 2) [55]. However, the number of genes per genome involved in the degradation of moderately labile C seems to be higher in typical soil-inhabiting bac- teria belonging to Actinobacteria and Acidobacteria as compared to most of the Proteobacterial members (Figure 2) [56]" "Genomic information provides evidence that most soil bacteria, irrespective of their phylogenetic groupings, possess different genes in- volved in relieving stresses caused by temperature, osmot- ic potential, oxygen concentration, and metabolism (Figure 2) [34,39,58,59]" 
"Based on the information gathered from full genome sequences, we infer that bacteria belong- ing to Acidobacteria and Actinobacteria possess an impres- sive array of genes allowing breakdown, utilization, and biosynthesis of diverse structural and storage polysacchar- ides and resilience to stressful soil conditions making them truly ubiquitous in terrestrial ecosystems (Figure 2). " "Genomic analysis shows that fast growing bacteria (especially those belonging to the phyla Proteobacteria and Firmicutes) have a higher number of total transporters including ATP-binding cassettes, phosphotransferase sys- tems, and drug/metabolite transporters that could import or export a broad range of compounds (Figure 2) [61]. The presence of low affinity transporters allows fast growth in periods of feast, while enduring starvation in periods of famine. "
"However, slow growing bacteria in the Acidobacteria and Actinobacteria taxa possess a low number of total trans- porters that have high affinity to a specific substrate allowing them to thrive in low nutrient concentrations, but become saturated at high nutrient concentration lead- ing to their selective exclusion by fast growers in rich environments (Figure 2)."(\cite{Trivedi_2013}) 55 -> Zimmerman, A.E. (2013) Microdiversity of extracellular enzyme genes among sequenced prokaryotic genomes. ISME J. http://dx.doi.org/ 10.1038/ismej.2012.176
56 -> Berlemont, R. and Martiny, A.C. (2013) Phylogenetic distribution of potential cellulases in bacteria. Appl. Environ. Microbiol. 79, 1545– 1554

"Likewise, this hypothesis is consistent with recent genomic information ob- tained from Spartobacteria aquaticum, an aquatic Verrucomicrobia that is within the same class as the dominant soil Verrucomicrobia observed here, that appears to specialize on the degradation of more recalcitrant carbon compounds (28)."  Fierer found verrucos to be >50percent of their sequences, with only 5 phylotypes making up >75percent of the verrucos.  Primarily spartos.  \cite{Fierer_2013} 

"This increase in CO2 evolution (2e5 times the amount of added C) is generally accompanied by only minor shifts in microbial community structure and as such is thought to reflect the high functional redundancy of glucose catabolism in soils (Falchini et al., 2003; Landi et al., 2006). However, Blagodatskaya and Kuzyakov (2008) have argued recently that simply monitoring changes in community structure is unlikely to be sufficient for resolving the underlying mechanisms involved in such events. Key to understanding what is happening in soil in response to low level C inputs is the development of methodologies that enable the responses of individual taxa to be studied. The" \cite{Jenkins_2010}

"In contrast to the findings of Falchini et al. (2003) and Landi et al. (2006) we found that at low concentrations of glucose, [14C] RNA enrichment was limited to Bacillus, Pseudomonas, Burkholderia and b-proteobacteria (Figs. 4 and 5). This is expected since competition for a limited resource typically results in the dominance of one or a few populations with the highest growth rates (Fontaine et al., 2003). Also, these particular bacterial groups are known to be particularly adept at responding to a variety of labile C compounds entering soils, across a range of different ecosystems worldwide and have been described as opportunists, r-strategists or copio- trophs (Padmanabhan et al., 2003; Bernard et al., 2007; Cleveland et al., 2007; Fierer et al., 2007; Langenheder and Prosser, 2008)." " Bacillus and Burkholderia although present at 150 mg C g!1 soil, were more prevalent at the lower glucose concentrations (15e50 mg C g!1 soil). This may suggest that Bacillus and Burkholderia have greater growth effi- ciencies (Griffiths et al., 1999) or have evolved survival strategies such as maintaining a high adenylate energy charge ratio (Brookes et al., 1987) to remain ‘metabolically alert’ during periods of star- vation (De Nobili et al., 2001). "\cite{Jenkins_2010}

"Taxa that increase in relative abundance with labile organic substrates (i.e., glycine, sucrose) are numerous (>500), phylogenetically clustered, and occur predominantly in two phyla (Proteobacteria and Actinobacteria) including orders Actinomycetales, Enterobacteriales, Burkholderiales, Rhodocyclales, Alteromonadales, and Pseudomonadales. Taxa increasing in relative abundance with more chemically recalcitrant substrates (i.e., cellulose, lignin, or tannin–protein) are fewer (168) but more phylogenetically dispersed, occurring across eight phyla and including Clostridiales, Sphingomonadalaes, Desulfovibrionales. Just over 6percent of detected taxa, including many Burkholderiales increase in relative abundance with both labile and chemically recalcitrant substrates" \cite{Goldfarb_2011}

"Very few taxa responded only to chemically recalcitrant substrates, with only 24 for cellulose (Clostridiales; termite gut clones, Deinococcus, Chloroflexi, Acetobacter, Desulfobacteraceae, Geobacteraceae, Syntrophobacteraceae, Treponema). On the other hand, cellulose users were relatively phylogenetically diverse (eight phyla), which may broaden the range of environ- mental conditions under which this metabolism may operate. Many of the taxa responding positively to cellulose have previously been shown to be capable of cellulose hydrolysis (e.g., bacteria from the families Clostridiaceae and Lachnospiraceae (Schwarz, 2001), Sphingomonas spp. (Kurakake et al., 2007), and Spirochetes (Warnecke et al., 2007), while Acetobacter aceti (Moonmangmee et al., 2002) is typically considered a cellulose producer rather than a consumer. "\cite{Goldfarb_2011}

"In their effects on soil C cycling, the influences of microbial com- munities appear to be associated with life history patterns that are deeply rooted in microbial phylogeny – functional groups appear at the level of families or phyla rather than species or genera. "\cite{Schimel_2012}

"Overall there was a gradual increase in the abundance of the Bacteroidetes during the succession process, which were not abundant in the first sampling (less than 0.24percent) but increased with time (around 7–9percent). Autotrophic phyla such as Clorobi, Chloroflexi and Cyanobacteria, increased in abundance the first nine months and then decrease their abundance in the last sampling (12 months) (Table 3)."\cite{L_pez_Lozano_2013}

"A total of 137 bacterial sequences at 28 days and 116 at 168 days were analysed and listed in Table 1. The Shannon diversity indices were 3.24 (S 1⁄4 43, E 1⁄4 0.86) and 4 (S 1⁄4 65, E 1⁄4 0.96), respectively, and within the range reported in other soil environ- ments (Dunbar et al.,1999; Hill et al., 2003). These results revealed (i) a high diversity of bacteria able to colonize the residue as previously indicated by the considerable complexity of B-ARISA profiles and (ii) a slight increase in bacterial diversity during the residue decom- position. This might be related to the biochemical composition of the residue at the end of the degradation process (Nicolardot et al., 2007). The main components at this stage were complex substrates (cellulose, hemicelluloses, lignin) recalcitrant to degradation which might result in the stimulation of a more diverse bacterial consor- tium able to degrade them (De Boer et al., 2005).
At 28 days of residue decomposition, the taxonomic divisions most frequently found were the Proteobacteria (69\%), the Bacter- oidetes (25\%) and the Fibrobacter (6\%). The Proteobacteria were distributed specifically in the Gammaproteobacteria (35\%), Alphaproteobacteria (26\%), and Betaproteobacteria (8\%) classes (Table 1). The significant discrimination of bacterial diversity between 28 and 168 days of incubation was mainly due to an increase in Actinobacteria (10\%), Deltaproteobacteria (9\%) and Betaproteobacteria (14\%) and a decrease in Fibrobacter (0\%) and Gammaproteobacteria (11\%). In addition, several other minor phyla were specifically identified at 168 days: Chloroflexibacteria (4\%),Verrucomicrobia (3\%), the Planctomycetes (3\%), Gemmatimona- detes (2\%), Acidobacteria (2\%), Firmicutes (1\%) and Clostridia (1\%) (Table 1).For example, Myxobacteria (and all the Deltaproteobacteria class) appear during the later stages of degradation process (Table 1).In vitro studies have revealed the ability of these bacteria to antagonize plant pathogenic as well as saprotrophic fungi (Bull, 2002) and their appearance at this stage may be partly related to their mycophagic properties.From a functional point of view, the ability to decompose cellulose is present both in bacteria (e.g. Streptomyces, Bacillus, Cytophaga, Micromonospora) and fungi (Ascomycota as well as Basidiomycota). The presence of functionally equivalent cellulolytic systems in both bacteria and fungi implies that competition for cellulose may take place in spatially limited soil niches. Some cellulose degrading bacteria are able to produce antifungal metabolites (Munimbazi and Bullerman, 1998) and other spore-associated bacteria might inhibit spore germination (Xavier and Germida, 2003; Toyota and Kimura, 1993)." \cite{Bastian_2009}

"Although we might expect an overall correlation between taxonomic com- position and the functional attributes of soil microbial communi- ties, this may not always be the case as distinct taxa can share specific functional attributes and closely related taxa may have very different physiologies and environmental tolerances (14). "\cite{Fierer_2012}

"These results show that all of the communities were dominated by Acidobacteria, Actinobacteria, Bacteroidetes, Pro- teobacteria, and Verrucomicrobia (Fig. S1), bacterial phyla that are known to be relatively abundant and ubiquitous in soil (25). Additional phyla including Chloroflexi, Cyanobacteria, Firmicutes, and Gemmatimonadetes were also found in nearly all soils, but their relative abundances were highly variable and typically rep- resented less than 5\% of the 16S rRNA reads in any individual"\cite{Fierer_2012}

"Therefore, as with the alpha diversity patterns, the concordance in beta diversity patterns highlights that the overall functional differences between the soil microbial communities were significantly correlated with the differences in the composition of these communities. Our findings are in line with comparable studies conducted in soil (20) and other habitats that also found strong correlations between metagenome composition and taxonomic composition (34, 35, 40). Although individual functional genes may not necessarily be correlated with community structure, the overall functional attributes of soil microbial communities appear to be predictable across broad gradients in soil and biome types if one has information on the taxonomic or phylogenetic structure of the communities."\cite{Fierer_2012}
