\section{Results}
% Fakesubsubsection:We observed C use dynamics by the soil microbial
Our experimental design allowed us to track the flow of xylose and cellulose
C through the soil microbial community (Figure~\ref{fig:setup}).
%We observed C use dynamics in an
%agricultural soil microbial community by conducting a nucleic acid SIP
%experiment wherein xylose or cellulose carried the isotopic label. We set up
%three soil microcosm series (Figure~\ref{fig:setup}). We amended each microcosm
%with a C substrate mixture that included cellulose and xylose. The C substrate
%mixture approximated the C composition of fresh plant biomass. The same mixture
%was added to all microcosms, however, for each microcosm series except the
%control, xylose or cellulose was substituted for its $^{13}$C counterpart.
5.3 mg C substrate mixture per gram of soil was added to each
microcosm representing 18\% of the soil C. The mixture included 0.42 mg
xylose-C and 0.88 mg cellulose-C per gram soil. Microcosms were harvested at
days 1, 3, 7, 14 and 30 during a 30 day incubation. $^{13}$C-xylose
assimilation peaked immediately and tapered over the 30 day incubation whereas
$^{13}$C-cellulose assimilation peaked two weeks after amendment additions
(Figure~\ref{fig:ord}, Figure~\ref{fig:rspndr_count}). See
Supplemental~Note~1.9 for sequencing and density fractionation statistics.
Microcosm treatments (see Methods) are identified in figures by the following
code: ``13CXPS'' refers to the amendment with $^{13}$C-xylose
($^{13}$\textbf{C} \textbf{X}ylose \textbf{P}lant \textbf{S}imulant),
``13CCPS'' refers to the $^{13}$C-cellulose amendment and ``12CCPS'' refers to
the amendment that only contained $^{12}$C (i.e. control). 

\subsection{Soil microcosm microbial community changes with time}
% Fakesubsubsection:Changes in the soil microcosm microbial community structure
Changes in the bulk soil microcosm microbial community structure and membership
correlated significantly with time (Figure~\ref{fig:bulk_ord},
P-value 0.23, R$^{2}$ 0.63, Adonis test \citep{Anderson2001a}). The identity of
the $^{13}$C-labeled substrate added to the microcosms did not significantly
correlate with bulk soil community structure and membership (P-value 0.35).
Additionally, microcosm beta diversity was significantly less than gradient
fraction beta diversity (Figure~\ref{fig:bulk_ord}, P-value 0.003,
``betadisper'' function R Vegan package \citep{Anderson2006,oksanen2007vegan}).
Twenty-nine OTUs significantly changed in relative abundance with time (``BH''
adjusted P-value $<$ 0.10, \citep{YBenjamini1995}) and of these 29 OTUs, 14
were found to incorporate $^{13}$C from labeled substrates into biomass
(Figure~\ref{fig:time}). Four taxonomic classes significantly (adjusted P-value
$<$ 0.10) changed in abundance: \textit{Bacilli} (decreased),
\textit{Flavobacteria} (decreased), \textit{Gammaproteobacteria} (decreased)
and \textit{Herpetosiphonales} (increased) (Figure~\ref{fig:time_class}).
Abundances grouped by phylum for OTUs that incorporated $^{13}$C from cellulose
increased with time whereas abundances grouped by phylum of OTUs that
incorporated $^{13}$C from xylose decreased over time although
\textit{Proteobacteria} abundance spiked at day 14 (Figure~\ref{fig:babund}).

\subsection{OTUs that assimilated $^{13}$C into DNA} \label{responders}
% Fakesubsubsection:Within the first 7 days of incubation approximately 63\% 
Within the first 7 days of incubation 63\% of $^{13}$C-xylose was
respired and only 6\% more was respired from day 7 to 30. At day 30, 30\% of
the $^{13}$C from xylose remained in the soil. An average 16\% of the
$^{13}$C-cellulose added was respired within the first 7 days, 38\% by day 14,
and 60\% by day 30.   

% Fakesubsubsection:Isotope incorporation by an OTU
We refer to OTUs that putatively incorporated $^{13}$C into DNA originally from
an isotopically labeled substrate as  substrate ``responders'' (see
Supplemental~Note~1.7.4 for operational ``response'' criteria). There were 19,
19, 15, 6, and 1 $^{13}$C-xylose responders at days 1, 3, 7, 14, 30,
respectively (Figure~\ref{fig:rspndr_count}).
%At day 1, 84\% of $^{13}$C-xylose responsive OTUs belonged to
%\textit{Firmicutes}, 11\% to \textit{Proteobacteria} and 5\% to
%\textit{Bacteroidetes}. \textit{Firmicutes} responders decreased from 16 OTUs
%at day 1 to one OTU at day 3 while \textit{Bacteroidetes} responders increased
%from one OTU at day 1 to 12 OTUs at day 3. The remaining day
%3 responders are members of the \textit{Proteobacteria} (26\%) and the
%\textit{Verrucomicrobia} (5\%). Day 7 responders were 53\%
%\textit{Actinobacteria}, 40\% \textit{Proteobacteria}, and 7\%
%\textit{Firmicutes}. The identities of $^{13}$C-xylose responders changed
%with time. 
The numerically dominant $^{13}$C-xylose responder phylum shifted from
\textit{Firmicutes} to \textit{Bacteroidetes} and then to
\textit{Actinobacteria} across days 1, 3 and 7 (Figure~\ref{fig:l2fc},
Figure~\ref{fig:xyl_count}). \textit{Proteobacteria} $^{13}$C-xylose responders
were found at days 1, 3, 7 but peaked at day 7 (Figure~\ref{fig:xyl_count}). 

% Fakesubsubsection:Only 2 and 5 OTUs were found to
Only 2 and 5 OTUs responded $^{13}$C-cellulose at days 3 and 7, respectively.
At days 14 and 30, 42 and 39 OTUs responded to $^{13}$C-cellulose.
(Figure~\ref{fig:rspndr_count}). 
%A \textit{Cellvibrio} and \textit{Sandaracinaceae} OTU assimilated $^{13}$C
%from $^{13}$C-cellulose at day 3. Day 7 $^{13}$C-cellulose responders included
%the same \textit{Cellvibrio} responder as day 3, a \textit{Verrucomicrobia} OTU
%and three \textit{Chloroflexi} OTUs.  50\% of Day 14 responders belong to
%\textit{Proteobacteria} (66\% Alpha-, 19\% Gamma-, and 14\% Beta-) followed by
%17\% \textit{Planctomycetes}, 14\% \textit{Verrucomicrobia}, 10\%
%\textit{Chloroflexi}, 7\% \textit{Actinobacteria} and 2\% cyanobacteria.
%\textit{Bacteroidetes} OTUs began to incorporate $^{13}$C from cellulose at day
%30 (13\% of day 30 responders). Other day 30 responding phyla included
%\textit{Proteobacteria} (30\% of day 30 responders; 42\% Alpha-, 42\% Delta,
%8\% Gamma-, and 8\% Beta-), \textit{Planctomycetes} (20\%),
%\textit{Verrucomicrobia} (20\%), \textit{Chloroflexi} (13\%) and cyanobacteria
%(3\%). 
\textit{Proteobacteria}, \textit{Verrucomicrobia}, and \textit{Chloroflexi} had
relatively high numbers of responders with strong response across multiple time
points (Figure~\ref{fig:l2fc}). \textit{Verrucomicrobia} $^{13}$C-cellulose
responders were 70\% \textit{Spartobacteria}. \textit{Chloroflexi} responders
were annotated as members of the \textit{Herpetosiphonales} and
\textit{Anaerolineae} (Figure~\ref{fig:tiledtree}). \textit{Cellvibrio},
a canonical soil cellulose degrader, was found to respond strongly to
$^{13}$C-cellulose in the microcosms. See Supplemental~Note~2 for further
counts of $^{13}$C-responsive OTUs at greater taxonomic resolution.

\subsection{Ecological strategies of $^{13}$C responders}
% Fakesubsubsection:$^{13}$C-xylose responders are generally more abundant members based
$^{13}$C-xylose responders are generally more abundant members based on
relative abundance in bulk DNA SSU rRNA gene content than $^{13}$C-cellulose
responders (Figure~\ref{fig:shift}, P-value 0.00028, Wilcoxon Rank Sum test).
However, $^{13}$C-xylose and $^{13}$C-cellulose responders included both
abundant and rare OTUs (Figure~\ref{fig:shift}). Two
$^{13}$C-cellulose responders were not found in any bulk samples (``OTU.862''
and ``OTU.1312'', Table~\ref{tab:cell}). Of the
10 most abundant responders,
8 are $^{13}$C-xylose responders and 6 of these 8 are consistently among the 10
  most abundant OTUs in bulk samples.

% Fakesubsubsection:Cellulose responders exhibited a greater shift in BD
Cellulose-responder-DNA buoyant density (BD) shifted further along the density
gradient than xylose-responder-DNA BD in response to $^{13}$C incorporation
(Figure~\ref{fig:c1}, Figure~\ref{fig:shift}, P-value 1.8610x$^{-06}$, Wilcoxon
Rank Sum test). $^{13}$C-cellulose-responder-DNA BD shifted on average
0.0163 g mL$^{-1}$ (sd 0.0094) whereas xylose responder BD shifted on average
0.0097 g mL$^{-1}$ (sd 0.0094). For reference, 100\% $^{13}$C DNA BD is 0.04
g mL$^{-1}$ greater than the BD of its $^{12}$C counterpart. DNA BD increases
as its ratio of $^{13}$C to $^{12}$C increases. An organism that only
assimilates C into DNA from a $^{13}$C isotopically labeled source, will have
a greater $^{13}$C to $^{12}$C ratio in its DNA than an organism utilizing
a mixture of isotopically labeled and unlabeled C sources (see
Supplemental~Note~1.8). We predicted the \textit{rrn} gene copy number for each
OTU as described previously \citep{Kembel_2012}. The estimated \textit{rrn}
gene copy number for $^{13}$C-xylose responders was inversely related to the
time of first response for each OTU (P-value
2.02x10$^{-15}$, Figure~\ref{fig:copy}). OTUs that did not respond at day
1 but did respond at day 3 and/or day 7 had fewer estimated
\textit{rrn} copy number than OTUs that responded at day 1
(Figure~\ref{fig:copy}). 

%Fakesubsubsection:
We assessed phylogenetic clustering of $^{13}$C-responsive OTUs with the
Nearest Taxon Index (NTI), the Net Relatedness Index (NRI)
\citep{Webb2000}, and the consenTRAIT metric \citep{Martiny2013}.
Briefly, positive NRI and NTI with corresponding low P-values indicates
deep phylogenetic clustering whereas negative NRI with high P-values
indicates taxa are overdispersed compared to the null model
\citep{Evans2014a}. NRI and P-values for substrate responder groups
suggest $^{13}$C-xylose responders are overdispersed (NRI: -1.33, P:
0.90) while $^{13}$C-cellulose responders are clustered (NRI: 4.49, P: 0.001).
NTI values show that both $^{13}$C-cellulose and $^{13}$C-xylose responders
are clustered near the tips of the tree (NTI: 1.43 (P: 0.072), 2.69 (P:
0.001), respectively). The consenTRAIT clade depth for $^{13}$C-xylose and
$^{13}$C-cellulose responders was 0.012 and 0.028 16S rRNA sequence
dissimilarity, respectively.





