\section{Results}
% Fakesubsubsection:We observed C use dynamics by the soil microbial
We observed C use dynamics in an agricultural soil microbial community by
conducting a nucleic acid SIP experiment wherein xylose or cellulose carried
the isotopic label. We set up three soil microcosm series. Each microcosm was
amended with a C substrate mixture that included cellulose and xylose. The
C substrate mixture approximated the chemical composition of fresh
plant biomass. The same mixture was added to microcosms in each series,
however, for each series except the control, xylose or cellulose was
substituted for its $^{13}$C counterpart. Microcosm amendments are shorthand
identified in figures by the following code: "13CXPS" refers to
the amendment with $^{13}$C-xylose (that is $^{13}$\textbf{C} \textbf{X}ylose
\textbf{P}lant \textbf{S}imulant), "13CCPS" refers to the $^{13}$C-cellulose
amendment and "12CCPS" refers to the amendment that only contained $^{12}$C
substrates (i.e. control). 5.3 mg of C gram$^{-1}$ soil C substrate mixture was
added to each microcosm representing 18\% of the soil C. The mixture included
0.42 mg xylose-C and 0.88 mg cellulose-C g soil$^{-1}$. Microcosms were
harvested at days 1, 3, 7, 14 and 30 during a 30 day
incubation. $^{13}$C-xylose assimilation peaked immediately and tapered over
the 30 day incubation whereas $^{13}$C-cellulose assimilation peaked two weeks
after amendment additions(Figure~\ref{fig:ord}). 

% Fakesubsubsection:Microcosm DNA was density fractionated on
Microcosm DNA was density fractionated on CsCl density gradients. We sequenced
SSU rRNA gene amplicons from a total of 277 CsCl gradient fractions from 14
CsCl gradients and 12 bulk microcosm DNA samples. The SSU rRNA gene data set
contained 1,102,685 total sequences. The average number of sequences per sample
was 3,816 (sd 3,629) and 265 samples had over 1,000 sequences. We sequenced SSU
rRNA gene amplicons from an average of
19.8 fractions per CsCl gradient (sd 0.57). The average density between
fractions was  0.0040 g mL$^{-1}$ The sequencing effort recovered a total of
5,940 OTUs. 2,943 of the total 5,940 OTUs were observed in bulk samples. We
observed 33 unique phylum and 340 unique genus annotations.

\subsection{Soil microcosm microbial community changes with time}
% Fakesubsubsection:Changes in the soil microcosm microbial community structure
Bulk soil DNA SSU rRNA gene amplicon sequencing revealed changes in the soil
microcosm microbial community structure and membership correlated significantly
with incubation time (Figure~\ref{fig:bulk_ord}B, p-value 0.23, R$^{2}$ 0.63,
Adonis test \citet{Anderson2001a}). The identity of the $^{13}$C-labeled
substrate added to the microcosms did not significantly correlate with
community structure and membership (p-value 0.35). Additionally, microcosm beta
diversity was significantly less than gradient fraction beta diversity (p-value
0.003, "betadisper" function R Vegan package
\citet{Anderson2006,oksanen2007vegan}). Twenty-nine OTUs significantly changed
in relative abundance with time ("BH" adjusted p-value $<$0.10,
\citet{YBenjamini1995}) (Figure~\ref{fig:time}). OTUs that significantly
increased in relative abundance with time included OTUs in the
\textit{Verrucomicrobia}, \textit{Proteobacteria}, \textit{Planctomycetes},
\textit{Cyanobacteria}, \textit{Chloroflexi} and \textit{Acidobacteria}. OTUs
that significantly decreased in relative abundance with time included OTUs in
the \textit{Proteobacteria}, \textit{Firmicutes}, \textit{Bacteroidetes} and
\textit{Actinobacteria} (Figure~\ref{fig:time}). \textit{Proteobacteria} was
the only phylum that had OTUs which increased significantly and OTUs that
decreased significantly in abundance with time. If sequences were grouped by
taxonomic annotations at the class level, only four classes significantly
changed in abundance, \textit{Bacilli} (decreased), \textit{Flavobacteria}
(decreased), \textit{Gammaproteobacteria} (decreased) and
\textit{Herpetosiphonales} (increased) (Figure~\ref{fig:time_class}). Of the
29 OTUs that changed significantly in relative abundance with time, 14 are
labeled substrate responders (Figure~\ref{fig:time}).

\subsection{OTUs that assimilated $^{13}$C into DNA}
% Fakesubsubsection:Within the first 7 days of incubation approximately 63\% 
Within the first 7 days of incubation approximately 63\% of $^{13}$C-xylose was
respired and only an additional 6\% more was respired from day 7 to 30. At day
30, 30\% of the $^{13}$C from xylose remained. An average 16\% of the
$^{13}$C-cellulose added was respired within the first 7 days, 38\% by day 14,
and 60\% by day 30.   

% Fakesubsubsection:Isotope incorporation by an OTU
Isotope incorporation by an OTU is revealed by enrichment of the OTU in heavy
CsCl gradient fractions containing $^{13}$C labeled DNA relative to heavy
fractions from control gradients containing no $^{13}$C labeled DNA. We refer
to OTUs that putatively incorporated $^{13}$C into DNA originally from an
isotopically labeled substrate as  substrate ``responders''. At day 1, 84\% of
$^{13}$C-xylose responsive OTUs belonged to \textit{Firmicutes}, 11\% to
\textit{Proteobacteria} and 5\% to \textit{Bacteroidetes}. \textit{Firmicutes}
responders decreased from 16 OTUs at day 1 to one OTU at day 3 while
\textit{Bacteroidetes} responders increased from one OTU at day 1 to 12 OTUs at
day 3. The remaining day 3 responders are members of the
\textit{Proteobacteria} (26\%) and the \textit{Verrucomicrobia} (5\%). Day
7 responders were 53\% \textit{Actinobacteria}, 40\% \textit{Proteobacteria},
and 7\% \textit{Firmicutes}. The identities of $^{13}$C-xylose responders
changed with time. The numerically dominant $^{13}$C-xylose responder phylum
shifted from \textit{Firmicutes} to \textit{Bacteroidetes} and then to
\textit{Actinobacteria} across days 1, 3 and 7 (Figure~\ref{fig:l2fc},
Figure~\ref{fig:xyl_count}). 

% Fakesubsubsection:Only 2 and 5 OTUs were found to
Only 2 and 5 OTUs had incorporated $^{13}$C from
$^{13}$C-cellulose at days 3 and 7, respectively. At days 14 and 30
42 and 39 OTUs incorporated $^{13}$C from $^{13}$C-cellulose into
biomass. A \textit{Cellvibrio} and \textit{Sandaracinaceae} OTU assimilated
$^{13}$C from $^{13}$C-cellulose at day 3. Day 7 $^{13}$C-cellulose responders
included the same \textit{Cellvibrio} responder as day 3,
a \textit{Verrucomicrobia} OTU and three \textit{Chloroflexi} OTUs.  50\% of
Day 14 responders belong to \textit{Proteobacteria} (66\% Alpha-, 19\% Gamma-,
and 14\% Beta-) followed by 17\% \textit{Planctomycetes}, 14\%
\textit{Verrucomicrobia}, 10\% \textit{Chloroflexi}, 7\%
\textit{Actinobacteria} and 2\% cyanobacteria. \textit{Bacteroidetes} OTUs
began to incorporate $^{13}$C from cellulose at day
30 (13\% of day 30 responders). Other day 30 responding phyla included
\textit{Proteobacteria} (30\% of day 30 responders; 42\% Alpha-, 42\% Delta,
8\% Gamma-, and 8\% Beta-), \textit{Planctomycetes} (20\%),
\textit{Verrucomicrobia} (20\%), \textit{Chloroflexi} (13\%) and
cyanobacteria (3\%). \textit{Proteobacteria}, \textit{Verrucomicrobia}, and
\textit{Chloroflexi} had relatively high numbers of responders with strong
response across multiple time points (Figure~\ref{fig:l2fc}).

\subsection{Xylose responders are more abundant members of the soil community
than cellulose responders}
% Fakesubsubsection:$^{13}$C-xylose responders are generally more abundant members based
$^{13}$C-xylose responders are generally more abundant members based on
relative abundance in bulk DNA SSU rRNA gene content than $^{13}$C-cellulose
responders (Figure~\ref{fig:shift}, p-value 0.00028).  However, both abundant
and rare OTUs responded to $^{13}$C-xylose and $^{13}$C-cellulose
(Figure~\ref{fig:shift}). For instance, a \textit{Delftia} $^{13}$C-cellulose
responder is fairly abundant in the bulk samples (``OTU.5'',
Table~\ref{tab:cell}). OTU.5 was on average the 13th most abundant OTU in bulk
samples. A $^{13}$C-xylose responder (``OTU.1040'', Table~\ref{tab:xyl}) has a
mean relative abundance in bulk samples of 3.57x10$^{-05}$. Two
$^{13}$C-cellulose responders were not found in any bulk samples ("OTU.862" and
"OTU.1312", Table~\ref{tab:cell}). Of the 10 most abundant responders, 8 are
$^{13}$C-xylose responders and 6 of these 8 are consistently among the 10 most
abundant OTUs in bulk samples.

% Fakesubsubsection:Responder abundances summed at phylum level generally increased
Responder abundances summed at phylum level generally increased for
$^{13}$C-cellulose (Figure~\ref{fig:babund}) whereas $^{13}$C-xylose
responder abundances summed at the phylum level decreased over time for
\textit{Firmicutes}, \textit{Bacteroidetes}, \textit{Actinobacteria} and
\textit{Proteobacteria} although \textit{Proteobacteria} spiked at day 14
(Figure~\ref{fig:babund}). Bulk abundance trends are roughly consistent
with $^{13}$C assimilation.

\subsection{Cellulose degraders exhibit higher substrate specificity than
xylose utilizers} 
% Fakesubsubsection:Cellulose responders exhibited a greater shift in BD
Cellulose responders exhibited a greater shift in buoyant density (BD) than
xylose responders in response to isotope incorporation (Figure~\ref{fig:shift},
p-value 1.8610x$^{-06}$). $^{13}$C-cellulose responders shifted on average
0.0163 g mL$^{-1}$ (sd 0.0094) whereas xylose responders shifted on average
0.0097 g mL$^{-1}$ (sd 0.0094). For reference, 100\% $^{13}$C DNA BD is 0.04
g mL$^{-1}$ greater than the BD of its $^{12}$C counterpart. DNA BD increases
as its ratio of $^{13}$C to $^{12}$C increases. An organism that only
assimilates C into DNA from a $^{13}$C isotopically labeled source, will have
a greater $^{13}$C:$^{12}$C ratio in its DNA than an organism utilizing
a mixture of isotopically labeled and unlabeled C sources. Upon labeling, DNA
from an organism that incorporates exclusively $^{13}$C will increase in BD
more than DNA from an organism that does not exclusively utilize isotopically
labeled C. Therefore the magnitude DNA BD shifts indicate substrate
specificity given our experimental design as only one substrate was labeled
in each amendment. We measured density shift as the change in an OTU's
density profile center of mass between corresponding control and labeled
gradients. BD shifts, however, should not be evaluated on an individual OTU
basis as a small number of density shifts are observed for each OTU and the
variance of the density shift metric at the level of individual OTUs is
unknown. It is therefore more informative to compare density shifts among
substrate responder groups. Further, density shifts are based on relative
abundance profiles and would be distorted in comparison to density shifts
based on absolute abundance profiles and should be interpreted with this
transformation in mind. It should also be noted that there was overlap in
observed density shifts between $^{13}$C-cellulose and $^{13}$C-xylose
responder groups, suggesting that although cellulose degraders are generally
more substrate specific than xylose utilizers, each responder group exhibits
a range of substrate specificities (Figure~\ref{fig:shift}).

\subsection{Estimated rrn gene copy number in substrate responder groups}
% Fakesubsubsection:$^{13}$C-xylose responder estimated \textit{rrn} gene copy
$^{13}$C-xylose responder estimated \textit{rrn} gene copy number is inversely
related time of first response (p-value 2.02x10$^{-15}$,
Figure~\ref{fig:copy}). OTUs that first respond at later time points have fewer
estimated \textit{rrn} copy number than OTUs that first respond earlier
(Figure~\ref{fig:copy}). \textit{rrn} copy number estimation is a recent
advance in microbiome science \citep{Kembel_2012} although the relationship of
\textit{rrn} copy number per genome with ecological strategy is well
established \citep{Klappenbach_2000}.  Microorganisms with a high \textit{rrn}
copy number tend to be fast growers specialized to take advantage of boom-bust
environments whereas microorganisms with low \textit{rrn} copy number favor
slower growth under lower and more consistent nutrient input
\citep{Klappenbach_2000}. At the beginning of our incubation, OTUs with
estimated high \textit{rrn} copy number or ``fast-growers'' assimilate xylose
into biomass and with time slower growers (lower \textit{rrn} copy number)
begin to incorporate $^{13}$C from xylose.  Further, $^{13}$C-xylose responders
have more estimated rRNA operon copy numbers per genome than $^{13}$C-cellulose
responders (p-value 1.878x10$^{-09}$) suggesting xylose respiring microbes are
generally faster growers than cellulose degraders.




