\section{Results}
% Fakesubsubsection:We tracked the flow of C from xylose
After adding the organic matter amendment to soil, we tracked the flow of
$^{13}$C from $^{13}$C-xylose or $^{13}$C-cellulose into microbial DNA over
time using DNA-SIP (Figure~\ref{fig:setup}). The amendment consisted of
compounds representative of plant biomass including cellulose, lignin, sugars
found in hemicellulose, amino acids, and inorganic nutrients (see Supplemental
Information (SI)). The amendment was added at~2.9 mg C g$^{-1}$ soil dry weight
(d.w.), and this comprised 19\% of the total C in the soil. The cellulose-C
(0.88 mg C g$^{-1}$ soil d.w.) and xylose-C (0.42 mg C g$^{-1}$ soil d.w.) in
the amendment comprised 6\% and 3\% of the total C in the soil, respectively.
The soil microbial community respired 65\% of the xylose within one day and
29\% of the added xylose remained in the soil at day~30 (Figure~\ref{fig:13C}).
In contrast, cellulose-C declined at a rate of approximately 18 $\mu$g
C d $^{-1}$ g $^{-1}$ soil d.w. and 40\% of added cellulose-C remained in the
soil at day~30 (Figure~\ref{fig:13C}).

\subsection{Community-level signal of $^{13}$C-assimilation in relation to substrate and time}
% Fakesubsubsection: We assessed assimilation of
We assessed assimilation of $^{13}$C into microbial DNA by comparing the SSU
rRNA gene sequence composition of SIP density gradient fractions between
$^{13}$C treatments and the unlabeled control (see Methods and SI).  Our main
focus is to identify evidence of isotope incorporation into the DNA of specific
OTUs (as described below), but it is instructive to begin by observing overall
patterns of variance in the SSU rRNA gene sequence composition of gradient
fractions. In the unlabeled control treatment, fraction density represented the
majority of the variance in SSU rRNA gene composition (Figure~\ref{fig:ord}). This
result is expected because Genome G$+$C content correlates positively with DNA
buoyant density and influences SSU rRNA gene composition in gradient fractions
\citep{Buckley_2007}. In contrast, isotope assimilation into DNA will cause
variation in gene sequence composition between corresponding density fractions
from controls and labeled treatments. For example, the SSU rRNA gene
composition in gradient fractions from the $^{13}$C-cellulose
treatment deviated from corresponding control fractions on days~14 and~30
and this difference was observed only in the high
density fractions ($>$1.7125 g mL$^{-1}$, Figure~\ref{fig:ord}). Likewise, SSU
rRNA gene composition in gradient fractions from the $^{13}$C-xylose treatment
also deviated from corresponding control fractions but on days~1,~3,~and~7
as opposed to 14 and 30 (Figure~\ref{fig:ord}). The $^{13}$C-cellulose
and $^{13}$C-xylose treatments also differed from each other in corresponding
high density gradient fractions indicating that different
microorganisms were labeled across time these treatments (Figure~\ref{fig:ord}).
These results are generally consistent with predictions of the degradative
succession hypothesis.

We can observe further differences in the pattern of isotope incorporation over
time for each treatment. For example the SSU rRNA gene sequence
composition in the $^{13}$C-cellulose treatment was similar on days 14 and 30 in 
corresponding high density fractions indicating similar patterns of isotope
incorporation into DNA on the days. In contrast, in the $^{13}$C-xylose treatment,
the SSU rRNA gene composition varied between
days~1,~3,~and~7 in corresponding high density fractions indicating different pattenrs
of isotope incorporation into DNA on these days. In the
$^{13}$C-xylose treatment on days 14 and 40 the SSU gene composition
was similar to control on days~14~and~30 for corresponding high density
fractions (Figure~\ref{fig:ord}) indicating that $^{13}$C was no longer
detectable in bacterial DNA on these days for this treatment. These results
show that the dynamics of isotope incorporation into DNA varied considerably for
organisms that assimilated C from either xylose or cellulose.

\subsection{Temporal dynamics of OTU relative abundance in non-fractionated DNA from soil}
% Fakesubsubsection:We monitored the soil microbial community
We monitored the soil microbial community over the course of the
experiment by surveying SSU rRNA genes in non-fractionated DNA from the
soil. The SSU rRNA gene composition of the non-fractionated DNA
changed with time (Figure~\ref{fig:bulk_ord}, P-value~$=$~0.023, R$^{2}$
$=$~0.63, Adonis test \citep{Anderson2001a}). In contrast, the microbial
community could not be shown to change with treatment
 (P-value~0.23, Adonis test) (Figure~\ref{fig:bulk_ord}). The latter
result demonstrates the substitution of $^{13}$C-labeled substrates for
unlabeled equivalents could not be shown to alter the soil microbial community
composition. Twenty-nine OTUs exhibited sufficient statistical evidence
(adjusted P-value $<$ 0.10, Wald test) to conclude they changed in relative
abundance in the non-fractionated DNA over the course of the experiment
(Figure~\ref{fig:time}). When SSU rRNA gene abundances were combined at the
taxonomic rank of "class", the classes that changed in abundance (adjusted
P-value~ $<$~0.10, Wald test) were the \textit{Bacilli} (decreased),
\textit{Flavobacteria} (decreased), \textit{Gammaproteobacteria} (decreased),
and \textit{Herpetosiphonales} (increased) (Figure~\ref{fig:time_class}). Of
the~29 OTUs that changed in relative abundance over time, 14 putatively
incorporated $^{13}$C into DNA (see below and Figure~\ref{fig:time}). OTUs that likely
assimilated $^{13}$C from $^{13}$C-cellulose tended to increase in
relative abundance with time whereas OTUs that assimilated $^{13}$C from
$^{13}$C-xylose tended to decrease (Figure~\ref{fig:babund}). OTUs that
responded to both substrates did not exhibit a consistent relative abundance
response over time as a group (Figure~\ref{fig:time}~and~\ref{fig:babund}).

\subsection{Changes in the phylogenetic composition of $^{13}$C-labeled OTUs with substrate and time}
% Fakesubsubsection:If an OTU exhibited
If an OTU exhibited strong evidence for assimilating $^{13}$C into DNA, we
refer to that OTU as a ``responder'' (see Methods and SI for our operational
definition of ``responder''). The SSU rRNA gene sequences produced in this study
were binned into 5,940 OTUs and we assessed evidence of $^{13}$C-labeling from
both $^{13}$C-cellulose and $^{13}$C-xylose for each OTU. Forty-one OTUs
responded to $^{13}$C-xylose,~55 OTUs responded to $^{13}$C-cellulose, and
8 OTUs responded to both xylose and cellulose (Figure~\ref{fig:l2fc},
Figure~\ref{fig:tiledtree}, Figure~\ref{fig:genspec}, Table~\ref{tab:xyl}, and
Table~\ref{tab:cell}). The number of xylose responders peaked at days~1 and~3
and declined with time. In contrast, the number of cellulose responders
increased with time peaking at days~14 and~30 (Figure~\ref{fig:rspndr_count}). 

% Fakesubsubsection: The phylogenetic composition
The phylogenetic composition of xylose responders changed with time
(Figure~\ref{fig:l2fc}~and~Figure~\ref{fig:xyl_count}) and 86\% of xylose responders
shared $>$ 97\% SSU rRNA gene sequence identity with bacteria cultured in
isolation (Table~\ref{tab:xyl}). On day 1, \textit{Bacilli} OTUs represented
84\% of xylose responders (Figure~\ref{fig:xyl_count}) and the majority of
these OTUs were closely related to cultured representatives of the genus
\textit{Paenibacillus} (Table~\ref{tab:xyl}, Figure~\ref{fig:tiledtree}). For
example, ``OTU.57'' (Table~\ref{tab:xyl}), annotated as \textit{Paenibacillus},
had a strong signal of $^{13}$C-labeling at day~1 coinciding with its
maximum relative abundance in non-fractionated DNA. The relative abundance
of ``OTU.57'' declined until day 14 and ``OTU.57'' did not appear to be
$^{13}$C-labeled after day~1 (Figure~\ref{fig:example}). On day~3,
\textit{Bacteroidetes} OTUs comprised 63\% of xylose responders
(Figure~\ref{fig:xyl_count}) and these OTUs were closely related to
cultured representatives of the \textit{Flavobacteriales} and
\textit{Sphingobacteriales} (Table~\ref{tab:xyl},
Figure~\ref{fig:tiledtree}). For example, ``OTU.14'', annotated as
a flavobacterium, had a strong signal for $^{13}$C-labeling in the
$^{13}$C-xylose treatment at days 1 and 3 coinciding with its maximum
relative abundance in non-fractionated DNA. The relative abundance of
``OTU.14'' then declined until day 14 and did not show evidence of
$^{13}$C-labeling beyond day~3 (Figure~\ref{fig:example}). Finally, on
day~7, \textit{Actinobacteria} OTUs represented 53\% of the xylose
responders (Figure~\ref{fig:xyl_count}) and these OTUs were closely
related to cultured representatives of \textit{Micrococcales}
(Table~\ref{tab:xyl}, Figure~\ref{fig:tiledtree}). For example, ``OTU.4'',
annotated as \textit{Agromyces}, had signal for $^{13}$C-labeling in the
$^{13}$C-xylose treatment on days~1,~3~and~7 with the strongest evidence
of $^{13}$C-labeling at day~7 and did not appear $^{13}$C-labeled at
days~14 and~30. The relative abundance of ``OTU.4'' in non-fractionated
DNA increased until day~3 and then declined until day~30
(Figure~\ref{fig:example}). \textit{Proteobacteria} were also common among
xylose responders at day~7 where they comprised 40\% of xylose responder
OTUs. Notably, \textit{Proteobacteria} represented the majority (6 of 8)
of OTUs that responded to both cellulose and xylose
(Figure~\ref{fig:genspec}). 

% Fakesubsubsection: The phylogenetic composition of cellulose responders were
The phylogenetic composition of cellulose responders did not change with time
to the same extent as the xylose responders. Also, in contrast to xylose
responders, cellulose responders often were not closely related ($<$ 97\% SSU
rRNA gene sequence identity) to cultured isolates. Both the relative abundance
and the number of cellulose responders increased over time peaking at days 14
and 30 (Figure~\ref{fig:l2fc}, Figure~\ref{fig:rspndr_count}, and Figure~\ref{fig:babund}).
Cellulose responders belonged to the \textit{Proteobacteria} (46\%),
\textit{Verrucomicrobia} (16\%), \textit{Planctomycetes} (16\%),
\textit{Chloroflexi} (8\%), \textit{Bacteroidetes} (8\%),
\textit{Actinobacteria} (3\%), and \textit{Melainabacteria} (1 OTU)
(Table~\ref{tab:cell}). 

% Fakesubsubsection:The majority (85\%) of cellulose
The majority (85\%) of cellulose responders outside of the
\textit{Proteobacteria} shared  $<$ 97\% SSU rRNA gene sequence identity to
bacteria cultured in isolation. For example, 70\% of the
\textit{Verrucomicrobia} cellulose responders fell within unidentified
\textit{Spartobacteria} clades (Figure~\ref{fig:tiledtree}), and these shared $<$
85\% SSU rRNA gene sequence identity to any characterized isolate. The
\textit{Spartobacteria} OTU ``OTU.2192'' exemplified many cellulose responders
(Table~\ref{tab:cell}, Figure~\ref{fig:example}).
``OTU.2192'' increased in non-fractionated DNA relative abundance with time and
evidence for $^{13}$C-labeling of ``OTU.2192'' in the $^{13}$C-cellulose
treatment increased over time with the strongest evidence at days~14
and~30 (Figure~\ref{fig:example}). Most \textit{Chloroflexi} cellulose
responders belonged to an unidentified clade within the
\textit{Herpetosiphonales} (Figure~\ref{fig:tiledtree}) and they shared
$<$ 89\% SSU rRNA gene sequence identity to any characterized isolate.
Characteristic of \textit{Chloroflexi} cellulose responders, "OTU.64"
increased in relative abundance over 30 days and evidence for
$^{13}$C-labeling of ``OTU.64'' in the $^{13}$C-cellulose treatment peaked
days 14 and~30 (Figure~\ref{fig:example}). \textit{Bacteroidetes}
cellulose responders fell within
the \textit{Cytophagales} in contrast with \textit{Bacteroidetes} xylose
responders that belonged instead to the \textit{Flavobacteriales} or
\textit{Sphingobacteriales} (Figure~\ref{fig:tiledtree}).
\textit{Bacteroidetes} cellulose responders included one OTU that shared
100\% SSU rRNA gene sequence identity to a \textit{Sporocytophaga}
species, a genus known to include cellulose degraders. The majority (86\%)
of cellulose responders in the \textit{Proteobacteria} were closely
related ($>$ 97\% identity) to bacteria cultured in isolation, including
representatives of the genera: \textit{Cellvibrio}, \textit{Devosia},
\textit{Rhizobium}, and \textit{Sorangium}, which are all known for their
ability to degrade cellulose (Table~\ref{tab:cell}). Proteobacterial
cellulose responders belonged to \textit{Alpha} (13~OTUs), \textit{Beta}
(4~OTUs), \textit{Gamma} (5~OTUs), and \textit{Delta-proteobacteria}
(6~OTUs).

\subsection{Characteristics of cellulose and xylose responders}
% Fakesubsubsection:Cellulose responders tended
Cellulose responders, relative to xylose responders, tended to have lower
relative abundance in non-fractionated DNA, demonstrated signal consistent with
higher atom \% $^{13}$C in labeled DNA, and had lower estimated \textit{rrn}
copy number (Figure~\ref{fig:shift}). OTUs that assimilated C from either
cellulose or xylose were also clustered phylogenetically (see below) indicating
that these traits were not dispersed randomly across bacterial species. 

% Fakesubsubsection: In the non-fractionated DNA
In the non-fractionated DNA, cellulose
responders had lower relative abundance (1.2 x 10$^{-3}$ (s.d. 3.8
x 10$^{-3}$)) than xylose responders (3.5 x 10$^{-3}$ (s.d. 5.2 x 10$^{-3}$))
(Figure~\ref{fig:xyl_count}, P-value~$=$~1.12 x 10$^{-5}$, Wilcoxon Rank Sum
test). Six of the ten most common OTUs observed in the non-fractionated DNA
responded to xylose, and, seven of the ten most abundant responders to xylose
or cellulose in the non-fractionated DNA were xylose responders.

% Fakesubsubsection:DNA buoyant density increases as the amount
DNA buoyant density (BD) increases in proportion to atom \% $^{13}$C.
Hence, the extent of $^{13}$C incorporation into DNA can be evaluated by
the difference in BD between $^{13}$C-labeled and unlabeled DNA. We
calculated for each OTU its mean BD weighted by relative abundance to
determine its ``center of mass'' within a given density gradient. We then
quantified for each OTU the difference in center of mass between control
gradients and gradients from $^{13}$C-xylose or $^{13}$C-cellulose treatments
(see SI for the detailed calculation, Figure~\ref{fig:c1}). We refer to the
change in center of mass position for an OTU in response to $^{13}$C-labeling
as $\Delta\hat{BD}$. This value can be used to compare relative
differences in $^{13}$C-labeling between OTUs. $\Delta\hat{BD}$ values,
however, are not comparable to the BD changes observed for DNA from pure
cultures both because they are  based on relative abundance in density gradient
fractions (and not DNA concentration) and because isolated strains grown in
uniform conditions generate uniformly labeled molecules while OTUs composed of
heterogeneous strains in complex environmental samples do not. Cellulose
responder $\Delta\hat{BD}$ (0.0163 g mL$^{-1}$ (s.d.~0.0094)) was greater than
that of xylose responders (0.0097 g mL$^{-1}$ (s.d.~0.0094))
(Figure~\ref{fig:shift}, P-value~$=$~1.8610 x 10$^{-6}$, Wilcoxon Rank Sum
test). 

% Fakesubsubsection:We predicted the rrn
We predicted the \textit{rrn} gene copy number for responders as described
\citep{Kembel_2012}. The ability to proliferate after rapid nutrient
influx correlates positively to a microorganism's \textit{rrn} copy number
\citep{Klappenbach_2000}. Cellulose responders possessed fewer estimated
\textit{rrn} copy numbers (2.7~(1.2~s.d.)) than xylose responders
(6.2~(3.4~s.d.)) ( P~=~1.878 x 10$^{-9}$, Wilcoxon Rank Sum test,
Figure~\ref{fig:shift} and Figure~\ref{fig:copy}). Furthermore, the
estimated \textit{rrn} gene copy number for xylose responders was
inversely related to the day of first response (P~=~2.02 x 10$^{-15}$,
Wilcoxon Rank Sum test, Figure~\ref{fig:copy},Figure~\ref{fig:shift}).

% Fakesubsubsection:We assessed phylogenetic
We assessed phylogenetic clustering of $^{13}$C-responsive OTUs with the
Nearest Taxon Index (NTI) and the Net Relatedness Index (NRI)
\citep{Webb2000}. We also quantified the average clade depth of cellulose and
xylose responders with the consenTRAIT metric \citep{Martiny2013}. Briefly, the
NRI and NTI evaluate phylogenetic clustering against a null model for the
distribution of a trait in a phylogeny. The NRI and NTI values are z-scores or
standard deviations from the mean and thus the greater the magnitude of the
NRI/NTI, the stronger the evidence for clustering (positive values) or
overdispersion (negative values). NRI assesses overall clustering whereas the
NTI assesses terminal clustering \citep{Evans2014a}. The consenTRAIT metric is
a measure of the average clade depth for a trait in a phylogenetic tree. NRI
values indicate that cellulose responders clustered overall and at the tips of
the phylogeny (NRI:~4.49, NTI:~1.43) while xylose responders clustered
terminally (NRI:~-1.33, NTI:~2.69). The consenTRAIT clade depth for xylose and
cellulose responders was~0.012 and~0.028 SSU rRNA gene sequence dissimilarity,
respectively. As reference, the average clade depth as inferred from genomic
analyses or growth in culture is approximately~0.017,~0.013 and~0.034 SSU rRNA gene sequence
dissimilarity for arabinase (arabinose like xylose is a five C sugar found in
hemicellulose), glucosidase and cellulase, respectively
\citep{Martiny2013,Berlemont2013}. These results indicate xylose responders
form terminal clusters dispersed throughout the phylogeny while cellulose
responders form deep clades of terminally clustered OTUs.
