\section{Results}
To observe C use dynamics by the soil microbial community, we conducted a
nucleic acid SIP experiment wherein xylose or cellulose carried the isotopic
label, and, we assayed SSU rRNA gene content of CsCl gradient fractions using
high-throughput DNA sequencing technology. We set up three soil microcosm series. 
Microcosms in each series were were amended with a C substrate mixture
that included cellulose and xylose. The C substrate mixture approximated
freshly degrading plant biomass. The same substrate mixture was added to microcosms
in each series, however, for each series except the control, one substrate
was substituted for its $^{13}$C counterpart. In one series cellulose was
$^{13}$C-labeled in another xylose was $^{13}$C-labeled and in the control
series no sustrates were $^{13}$C labeled. Microcosm amendments are shorthand identified
in the following figures by the following code: "13CXPS" refers to the
amendment with $^{13}$C-xylose (that is $^{13}$\textbf{C} \textbf{X}lose
\textbf{P}lant \textbf{S}imulant), "13CCPS" refers to the $^{13}$C-cellulose
amendment and "12CCPS" refers to the amendment that only contained $^{12}$C
substrates. Xylose or cellulose were chosen to carry the isotopic label to
contrast C assimilation for labile, soluble C (xylose) versus
insoluble, polymeric C (cellulose).  5.3 mg of C substrate mixture per gram
soil was added to each microcosm representing 18\% of the total soil C. The
mixture included 0.42 mg xylose-C and 0.88 mg cellulose-C g soil$^{-1}$.
Microcosms were harvested at days 1, 3, 7, 14 and 30  during a 30 day
incubation. $^{13}$C-xylose assimilation peaked immediately and tapered
over the 30 day incubation whereas $^{13}$C-cellulose assimilation peaked at
two weeks of (Figure~\ref{fig:ord}).

We sequenced SSU rRNA gene amplicons from a total of 277 CsCl gradient fractions from 14 
CsCl gradients and 12 bulk microcosm DNA samples. The SSU rRNA gene data set contained 1,376,008 
total sequences. The average number of sequences per sample was 3,816 (sd 3,629) and 265 samples 
had over 1,000 sequences. We sequenced SSU rRNA gene amplicons from an average of 19.8 fractions
per CsCl gradient (sd 0.57). The average density between fractions was  0.0040 g mL$^{-1}$ 
The sequencing effort recoverd a total of 5,940 OTUs. 2,943 of the total
5,940 OTUs were observed in bulk samples. We observed 33 unique phylum and 340 unique genus
annotations.

\subsection{Soil microcosm microbial community changes with time}
Changes in the soil microcosm microbial community structure and membership
correlated with incubation time (Figure~\ref{fig:bulk_ord}B, p-value 0.23,
R$^{2}$ 0.63, Adonis test \citet{Anderson2001a}). The $^{13}$C composition
of the C-substrate addition did not significantly correlate with soil microcosm
community structure and membership (p-value 0.35). Additionally, bulk sample
beta diversity was significantly less than gradient fraction beta diversity
(p-value 0.003, \citet{Anderson2006}). Twenty-nine OTUs significantly changed
in relative abundance with time ("BH" adjusted p-value $<$ 0.10,
\citet{YBenjamini1995}).  OTUs that significantly increased in relative
abundance with time included OTUs in the \textit{Verrucomicrobia},
\textit{Proteobacteria}, \textit{Planctomycetes}, \textit{Cyanobacteria},
\textit{Chloroflexi} and \textit{Acidobacteria}. OTUs that significantly
decreased in relative abundance with time included OTUs in the
\textit{Proteobacteria}, \textit{Firmicutes}, \textit{Bacteroidetes} and
\textit{Actinobacteria} (Figure~XX).  \textit{Proteobacteria} was the only
phylum that had OTUs that significantly increased and OTUs that significantly
decreased in abundance with time. If sequences were grouped by taxonomic
annotations at the class level, only four classes significantly changed in
abundance, \textit{Bacilli} (decreased), \textit{Flavobacteria} (decreased),
\textit{Gammaproteobacteria} (decreased) and \textit{Herpetosiphonales}
(increased) (Figure~XX). Of the 29 OTUs that changed significantly in relative
abundance with time, 14 are labeled substrate responders (Figure~XX).

\subsection{OTUs that assimilated $^{13}$C from xylose}
Isotope incorporation by an OTU is revealed by enrichment of the OTU in heavy
CsCl gradient fractions containing $^{13}$C labeled DNA relative to heavy
fractions from control gradients containing no $^{13}$C labeled DNA. We refer
to OTUs that putatively incorporated $^{13}$C into DNA from an isotopically
labeled substrate as a substrate ``responder''.  Within the first 7 days of
incubation 63\% on average of $^{13}$C-xylose was respired and only an
additional 6\% more was respired from day 7 to 30. At the end of the 30 day
incubation 30\% of the $^{13}$C from added xylose remained in the soils. The
$^{13}$C remaining in the soil from $^{13}$C-xylose addition was likely
stabilized by assimilation into microbial biomass and/or microbial conversion
into other forms of organic matter. It is also possible that some
$^{13}$C-xylose remains unavailable to microbes due to abiotic interactions in
soil \citep{Kalbitz_2000}. 

At day 1, 84\% of $^{13}$C-xylose responsive OTUs belong to
\textit{Firmicutes}, 11\% to \textit{Proteobacteria} and 5\% to
\textit{Bacteroidetes}. \textit{Firmicutes} responders decreased to from 16
OTUs at day 1 to 1 OTU at day 3 while \textit{Bacteroidetes} responders
increased from 1 OTU at day 1 to 12 OTUs at day 3. The remaining day 3
responders are members of the \textit{Proteobacteria} (26\%) and the
\textit{Verrucomicrobia} (5\%). Day 7 responders were 53\%
\textit{Actinobacteria}, 40\% \textit{Proteobacteria}, and 7\%
\textit{Firmicutes}. The identities of $^{13}$C-xylose responders change with
time. The numerically dominant $^{13}$C-xylose responder phylum shifts from
\textit{Firmicutes} to \textit{Bacteroidetes} and then to
\textit{Actinobacteria} across days 1, 3 and 7 (Figure~\ref{fig:l2fc},
Figure~\ref{fig:xyl_count}). 

All of the $^{13}$C-xylose responders in the \textit{Firmicutes} phylum are
closely related (at least 99\% sequence identity) to cultured isolates from
genera that are known to form endospores (Table~\ref{tab:xyl}). Each
$^{13}$C-xylose responder is closely related to isolates annotated as members
of \textit{Bacillus}, \textit{Paenibacillus} or \textit{Lysinibacillus}.
\textit{Bacteroidetes} $^{13}$C-xylose responders are predominantly closely
related to \textit{Flavobacterium} species (5 of 8 total responders)
(Table~\ref{tab:xyl}.  Only one \textit{Bacteroidetes} $^{13}$C-xylose
responder is not closely related to a cultured isolate, ``OTU.183'' (closest
LTP BLAST hit, \textit{Chitinophaca sp.}, 89.5\% sequence identity,
Table~\ref{tab:xyl}). OTU.183 shares high sequence identity with environmental
clones derived from rhizosphere samples (accession AM158371, unpublished) and
the skin microbiome (accession JF219881, \citet{Kong_2012}). Other
\textit{Bacteroidetes} responders share high sequence identities with canonical
soil genera including \textit{Dyadobacer}, \textit{Solibius} and
\textit{Terrimonas}. Six of the 8 \textit{Actinobacteria} $^{13}$C-xylose
responders are in the \textit{Micrococcales} order. One $^{13}$C-xylose
responding \textit{Actinobacteria} OTU shares 100\% sequence identity with
\textit{Agromyces ramosus} (Table~\ref{tab:xyl}).  \textit{A. ramosus} is a
known predatory bacterium but is not dependent on a host for growth in culture
\citep{16346402}. It is not possible to determine the specific origin of
assimilated $^{13}$C in a DNA-SIP experiment. $^{13}$C can be passed down
through trophic levels although heavy isotope representation in C pools
targeted by cross-feeders and predators would be diluted with depth into the
trophic cascade.  It's possible, however, that the $^{13}$C labeled
\textit{Agromyces} OTU was assimilating $^{13}$C primarily by predation if the
\textit{Agromyces} OTU was selective enough with respect to its prey that it
primarily attacked $^{13}$C-xylose assimilating organisms. 

\subsection{$^{13}$C-cellulose incorporating OTUs} 
Only 2 and 5 OTUs were found to have incorporated $^{13}$C from
$^{13}$C-cellulose at days 3 and 7, respectively. At days 14 and 30, however,
42 and 39 OTUs were found to incorporate $^{13}$C from $^{13}$C-cellulose into
biomass. An average 16\% of the $^{13}$C-cellulose added was respired within
the first 7 days, 38\% by day 14, and 60\% by day 30.  A \textit{Cellvibrio}
and \textit{Sandaracinaceae} OTU assimilated $^{13}$C from $^{13}$C-cellulose
at day 3. Day 7 $^{13}$C-cellulose responders included the same
\textit{Cellvibrio} responder as day 3, a \textit{Verrucomicrobia} OTU and
three \textit{Chloroflexi} OTUs.  50\% of Day 14 responders belong to
\textit{Proteobacteria} (66\% Alpha-, 19\% Gamma-, and 14\% Beta-) followed by
17\% \textit{Planctomycetes}, 14\% \textit{Verrucomicrobia}, 10\%
\textit{Chloroflexi}, 7\% \textit{Actinobacteria} and 2\% cyanobacteria.
\textit{Bacteroidetes} OTUs begin to incoporate $^{13}$C from cellulose at day
30 (13\% of day 30 responders). Other day 30 responding phyla include
\textit{Proteobacteria} (30\% of day 30 responders; 42\% Alpha-, 42\% Delta,
8\% Gamma-, and 8\% Beta-), \textit{Planctomycetes} (20\%),
\textit{Verrucomicrobia} (20\%), \textit{Chloroflexi} (13\%) and cyanobacteria
(3\%). \textit{Proteobacteria}, \textit{Verrucomicrobia}, and
\textit{Chloroflexi} had relatively high numbers of responders with
strong response across multiple time points (Figure~\ref{fig:l2fc}).

\par\textit{Proteobacteria} represent 46\% of all $^{13}$C-cellulose responding
OTUs identified. \textit{Cellvibrio} accounted for 3\% of all proteobacterial
$^{13}$C-cellulose responding OTUs detected. \textit{Cellvibrio} was one of the
first identified cellulose degrading bacteria and was originally described by
Winogradsky in 1929 who named it for its cellulose degrading abilities
\citep{boone2001bergeys}. All $^{13}$C-cellulose responding
\textit{Proteobacteria} share high sequence identity with 16S rRNA genes from
sequenced cultured isolates (Table~\ref{tab:cell}) except for ``OTU.442'' (best
cultured isolate match 92\% sequence identity in the \textit{Chrondomyces}
genus, Table~\ref{tab:cell}) and ``OTU.663'' (best cultured isolate match
outside \textit{Proteobacteria} entirely, \textit{Clostridium} genus, 89\%
sequence identity, Table~\ref{tab:cell}). Some \textit{Proteobacteria}
responders share high sequence identity with type strains for genera known to
possess cellulose degraders including \textit{Rhizobium}, \textit{Devosia},
\textit{Stenotrophomonas} and \textit{Cellvibrio}. One \textit{Proteobacteria}
OTU shares high sequence identity with a \textit{Brevundimonas} cultured
isolate.  \textit{Brevundimonas} has not previously been identified as a
cellulose degrader, but has been shown to degrade cellouronic acid, an oxidized
form of cellulose \citep{Tavernier_2008}.


