\section{Results}
% Fakesubsubsection:We tracked the flow of C from xylose
After adding a C substrate mixture containing both cellulose and xylose to
soil, we tracked the flow of C from xylose or C from cellulose into microbial
biomass over time using DNA-SIP (Figure~\ref{fig:setup}). We added 3 mg
C substrate mixture as C per gram dry weight soil to experimental microcosms.
The C substrate amendment comprised 25\% of total soil C and included fresh
plant biomass components including sugars, cellulose, lignin, and amino acids.
Xylose-C and cellulose-C made up 0.42 mg and 0.88 mg per gram dry soil
and these additions represented 3.5\% and 7.3\% of total soil C, respectively.
The soil microbial community respired 65\% of the xylose within one day
(Figure~\ref{fig:13C}) and 29\% of the added xylose remained in the soil at day
30 (Figure~\ref{fig:13C}). In contrast, cellulose-C declined at a constant rate
of approximately 18 $\mu$g C g$^{-1}$ d$^{-1}$ and 40\% of added cellulose-C
remained in the soil at day 30 (Figure~\ref{fig:13C}). 

\subsection{Soil microcosm microbial community changes with time}
% Fakesubsubsection:Changes in the soil microcosm microbial community structure
We assessed incorporation of $^{13}$C into microbial community DNA by comparing the
SSU rRNA gene sequence composition of SIP density gradient fractions from
either DNA from control treatments or treatments amended with $^{13}$C-xylose
or $^{13}$C-cellulose. We set up three types of microcosms.
All microcosms received the same C mixture, but, in two microcosm types
a $^{13}$C-labeled substrate (i.e. $^{13}$C-xylose or $^{13}$C-cellulose) was
substituted for its $^{12}$C equivalent. We added only unlabeled C substrates
to the last microcosm type -- this type served as the ``control''. The majority
of the variance in SSU rRNA gene composition of control density gradient
fractions is represented by fraction density (Figure~\ref{fig:ord}. DNA buoyant
density is correlated with G$+$C content \citep{Buckley_2007} and therefore
variation in control gradient fraction SSU rRNA gene composition is strongly
influenced by DNA G$+$C content. For the $^{13}$C-cellulose treatment, SSU rRNA
gene composition in gradient fractions deviate from control fractions at high
density ($>$ 1.72 g mL$^{-1}$) at days 14 and 30 (Figure~\ref{fig:ord}). For
the $^{13}$C-xylose treatment, SSU rRNA gene composition in density gradient
fractions also deviates from control in high density fractions, but in contrast
to the $^{13}$C-cellulose treatment it deviates from control at days 1, 3, and
7 (Figure~\ref{fig:ord}). SSU rRNA gene composition in high density fractions
differs between $^{13}$C-cellulose amendments and $^{13}$C-xylose amendments
indicating different microorganisms incorporated C from xylose into DNA than
those that incorporated C from cellulose (Figure~\ref{fig:ord}). Further, the
SSU rRNA gene sequence composition of high density fractions from
$^{13}$C-cellulose treatments at days 14 and 30 is similar indicating similar
microorganisms have $^{13}$C labeled DNA at days 14 and 30. In contrast, the
SSU rRNA gene composition of the $^{13}$C-xylose treatment high density
gradient fractions varies between days 1, 3, and 7 indicating that different
microbes have $^{13}$C labeled DNA on these days. In the $^{13}$C-xylose
treatment, the SSU gene composition of high density fractions is similar to
control on days 14 and 30 (Figure~\ref{fig:ord}) indicating that $^{13}$C from
$^{13}$C-xylose is no longer detectable in DNA on these days. 

\subsection{Temporal dynamics DNA $^{13}$C incorporation of OTUs}
% Fakesubsubsection:We monitored the soil microbial community
We monitored the microcosm microbial community over the course of the
experiment by surveying SSU rRNA genes in non-fractionated DNA from soil. The
SSU rRNA gene composition of the non-fractionated DNA changed with time
(Figure~\ref{fig:bulk_ord}, P-value $=$ 0.023, R$^{2}$ $=$ 0.63, Adonis test
\citep{Anderson2001a}), but the P-value for community dissimilarity respect to
treatment was 0.23 (Adonis test \citep{Anderson2001a})
(Figure~\ref{fig:bulk_ord}). The latter result demonstrates the substitution of
$^{13}$C-labeled substrates for unlabeled equivalents did not alter community
composition beyond what we would reasonably expect at random. The variance in
SSU rRNA gene composition among all non-fractionated DNA samples across
treatment and time was less than the variance among gradient fractions
(Figure~\ref{fig:bulk_ord}, P-value $=$ 0.003, “betadisper” function R Vegan
package \citep{oksanen2007vegan}) and therefore the process of density gradient
fractionation produced more variance in SSU rRNA gene composition than temporal
changes in the non-fractionated DNA SSU rRNA gene content. That is, density
fractionation of DNA produced greater variance in SSU rRNA gene composition in
density gradient fractions than C microcosm manipulations (e.g. C additions and
incubation) in the microcosm communities.

% Fakesubsubsection:Twenty-nine OTUs
Twenty-nine OTUs exhibited sufficient statistical evidence (adjusted P-value
$<$ 0.10) to conclude they changed in relative
abundance with time. When SSU rRNA gene abundances were combined at the
taxonomic rank of "Class", \textit{Bacilli} (decreased), \textit{Flavobacteria}
(decreased), \textit{Gammaproteobacteria} (decreased) and
\textit{Herpetosiphonales} (increased) had statistically strong evidence for
changing in relative abundance with time (P-value $<$ 0.10,
Figure~\ref{fig:time_class}). Of the 29
OTUs for which relative abundance change over time was statistically
significant, 14 putatively incorporated $^{13}$C into DNA
(Figure~\ref{fig:time} and below). OTUs that likely incorporated $^{13}$C from
$^{13}$C-cellulose into DNA tended to increase in relative abundance with time
whereas OTUs that incorporated $^{13}$C from $^{13}$C-xylose tended to decrease
over time. Those OTUs that responded to both substrates did not exhibit
a consistent relative abundance response over time
(Figure~\ref{fig:time}~and~\ref{fig:babund}).

\subsection{OTUs that assimilated $^{13}$C into DNA} \label{responders}
% Fakesubsubsection:If an OTU exhibited
If an OTU exhibited strong evidence for $^{13}$C incorporation into DNA, we
refer to that OTU as a "responder" (see Supplemental Note 1.7.4 for our
operational definition of "responder"). The SSU rRNA gene sequences produced in
this study could be distributed into 5,940 OTUs and we assessed the evidence of
$^{13}$C incorporation into DNA from $^{13}$C-cellulose and $^{13}$C-xylose for
each OTU. We found a 41 OTUs that responded to 13-xylose, 55 OTUs that
responded to $^{13}$C-cellulose, and 8 OTUs that responded to both xylose and
cellulose (Figure~\ref{fig:l2fc}, Tables~{tab:cell}~and~\ref{tab:xyl}). The
number of xylose responders peaked at days 1 and 3 and declined with time. In
contrast, the number of cellulose responders increased with time peaking at
days 14 and 30 (Figure~\ref{fig:rspndr_count}). 

The phylogenetic composition of xylose responders changed with time and the
majority (86\%) of xylose responders shared > 97\% SSU rRNA gene sequence
identity with bacteria cultured in isolation
(Figure~\ref{fig:l2fc}~and~\ref{fig:xyl_count}). On day 1, \textit{Bacilli}
OTUs represented 84\% of xylose responders, and the majority of these OTUs were
closely related to cultured representatives of the genus \textit{Paenibacillus}
(n $=$ XX, Table~\ref{tab:xyl}). For example, "OTU.57" (Table\ref{tab:xyl}),
annotated as \textit{Paenibacillus}, has a strong signal of $^{13}$C
incorporation from $^{13}$C-xylose into DNA at day 1, at its maximum relative
abundance in non-fractionated DNA. The relative abundance of "OTU.57"
declined until day 14 and did not appear $^{13}$C labeled after day 1 (Figure
X). On day 3 \textit{Bacteroidetes} OTUs comprised 63\% of xylose responders
(Figure~\ref{fig:xyl_count}) and these OTUs were closely
related to cultured representatives of the \textit{Flavobacteriales} and
\textit{Sphingobacteriales} (Table~\ref{tab:xyl}). For example, "OTU.14",
annotated as a Flavobacterium, has a strong signal for $^{13}$C labeling from
$^{13}$C-xylose at days 1 and 3 coinciding with its maximum relative abundance
in non-fractionated DNA. The relative abundance of "OTU.14" then declined
until day 14 and did not showed evidence of $^{13}$C  labeling 
3 (Figure X). Finally, on day 7, \textit{Actinobacteria} OTUs represented 53\%
of the xylose responders and these OTUs were closely related to cultured
representatives of \textit{Micrococcales} (Table~\ref{tab:xyl}). For example,
``OTU.4'', annotated as \textit{Agromyces}, has signal of $^{13}$C labeling
on days 1 through 7 with the strongest evidence $^{13}$C labeling at day
7 and little evidence for $^{13}$C labeling at days 14 and 30. ``OTU.4''
relative abundance in non-fractionated DNA increased until day 3 and
then declined gradually until day 30 (Figure X). \textit{Proteobacteria} were
also common among xylose responders at day 7 where they comprised 40\% of
xylose responder OTUs. Notably, \textit{Proteobacteria} represented the
majority (6 of 8) of OTUs that responded to both cellulose and xylose. 

%Fakesubsubsection:Cellulose responders were
The phylogenetic composition of cellulose responders did not change with time
unlike phylogenetic types of xylose responders. Also, in contrast to xylose
responders, cellulose responders were often shared low ($<$ 95\%) SSU rRNA gene
sequence identity with cultured isolates. Both the relative abundance and the
number of cellulose responders increased over time peaking at days 14 and 30
(Figures~\ref{fig:l2fc}, \ref{fig:rspndr_count}, and \ref{fig:babund}). The
phylogenetic composition of cellulose responders changed little between days
14 and 30 (Table~\ref{tab:cell}). Cellulose responders belonged to the
\textit{Proteobacteria} (46\%), \textit{Verrucomicrobia} (16\%),
\textit{Planctomycetes} (16\%), \textit{Chloroflexi} (8\%),
\textit{Bacteroidetes} (8\%), \textit{Actinobacteria} (3\%), and
\textit{Melainabacteria} (1 OTU) (Table~\ref{tab:cell}). The majority (86\%)
of cellulose responders in the \textit{Proteobacteria} were closely related
($>$ 97\% identity) to bacteria already cultured in isolation, including
representatives of the genera: \textit{Cellvibrio}, \textit{Devosia},
\textit{Rhizobium}, and \textit{Sorangium}, which are all known for their
ability to degrade cellulose (Table~\ref{tab:cell}). Proteobacterial cellulose
responders belonged to \textit{Alpha-} (13 OTUs), \textit{Beta-} (4 OTUs),
\textit{Gamma-} (5 OTUs), and \textit{Deltaproteobacteria} (6 OTUs). 

% Fakesubsubsection:The majority (85\%) of cellulose
The majority (85\%) of cellulose responders outside of the
\textit{Proteobacteria} shared  $<$ 97\% SSU rRNA gene sequence identity to
bacteria already cultured in isolation. For example, most (70\%) of the
\textit{Verrucomicrobia} cellulose responders fell within unidentified
\textit{Spartobacteria} clades, and these shared $<$ 85\% SSU rRNA gene
sequence identity to any characterized isolate. The \textit{Spartobacteria} OTU
"OTU.2192" exemplified of many cellulose responders (Figure X). ``OTU.2192''
gradually increased in non-fractionated DNA relative abundance with time
and evidence for $^{13}$C labeling of "OTU.2192" also increased gradually over
time with the strongest evidence at days 14 and
30 (Figure X). \textit{Choloflexi} cellulose responders predominantly belonged
to an unidentified clade within the \textit{Herpetosiphonales} and these
shared $<$ 89\% SSU rRNA gene sequence identity to any characterized
isolate. Characteristic of other \textit{Chloroflexi} cellulose responders,
"OTU.64" increased in relative abundance over 30 days and evidence for
$^{13}$C labeling of "OTU.64" peaked days 14 and 30 (Figure X). Cellulose
responders found within the \textit{Bacteroidetes} fell within the
\textit{Cytophagales} contrasting with \textit{Bacteroidetes} xylose
responders that fell instead within the \textit{Flavobacteria} or
\textit{Sphingobacteriales}. \textit{Bacteroidetes} cellulose responders
included one OTU that shared 100\% SSU rRNA gene sequence identity to
species of \textit{Sporocytophaga}, a genus that includes known cellulose
degraders.

\subsection{Characteristics of cellulose and xylose responders}
% Fakesubsubsection:Cellulose responders tended
Cellulose responders, relative to xylose responders, tended to have lower
relative abundance in non-fractionated DNA, demonstrated signal consistent
with higher $^{13}$C incorporation per unit DNA upon $^{13}$C labeling, and
lower estimated \textit{rrn} copy number. In the
non-fractionated DNA, cellulose responders had lower
relative abundance (7e$^{-4}$ (s.d. 2e$^{-3}$)) than xylose responders
(2e$^{-3}$ (s.d. 4e$^{-3}$)) (Figure~\ref{fig:xyl_count}, P-value $=$ 0.00028,
Wilcoxon Rank Sum test). Six of the ten most common OTUs observed in the
non-fractionated DNA responded to xylose, and, eight of the ten most
abundant responders to xylose or cellulose in the non-fractionated DNA
8 were xylose responders while only 2 were cellulose responders. However,
$^{13}$C-xylose and $^{13}$C-cellulose responders included OTUs at both high
and low abundance (Figure~\ref{fig:shift}).

% Fakesubsubsection:Cellulose responders exhibited a greater shift in BD
DNA buoyant density increases as the amount of $^{13}$C per unit DNA increases.
An organism that only assimilates C into DNA from a $^{13}$C labeled source,
will have a greater DNA $^{13}$C:$^{12}$C than an
organism utilizing a mixture of $^{13}$C labeled and unlabeled C sources (see
Supplemental~Note~1.8). Therefore, the amount of $^{13}$C per unit DNA
can be evaluated by the change in DNA buoyant density (BD)
upon $^{13}$C labeling. In this study we do not know the absolute abundance of
OTU DNA across the density gradient as our SSU rRNA gene sequence counts are
compositional in nature hence we cannot assess absolute OTU DNA buoyant density
shifts due to $^{13}$C labeling. However, we can evaluate relative $^{13}$C
incorporation per unit DNA by quantifying and comparing the
shift in the relative abundance profile for an OTU along the density gradient
in response to $^{13}$C labeling. Specifically, in this study we first found
the weighted average density for each OTU's relative abundance profile (i.e.
mean density weighted by relative abundance). This weighted average is the
"center of mass" of the density profile. We then quantified the change in each
OTU profile's center of mass for a control gradient and a corresponding
$^{13}$C labeled gradient (see supplemental methods for the detailed
calculation). We refer to the change in center of mass position for an OTU in
response to $^{13}$C labeling as $\Delta\hat{BD}$. $\Delta\hat{BD}$ indicates
relative differences $^{13}$C incorporation per unit DNA between OTUs.
$\Delta\hat{BD}$ does not represent the true density shift for an OTU because
it is based on relative abundance and is therefore not directly comparable to
literature values for DNA density shifts due to isotopic labeling. Cellulose
responder $\Delta\hat{BD}$ (0.0163 g mL$^{-1}$ (s.d.
0.0094)) was greater than that of xylose responders (0.0097
g mL$^{-1}$ (s.d. 0.0094)) (Figure~\ref{fig:shift}, P-value $=$ 1.8610e$^{-6}$,
Wilcoxon Rank Sum test). 

% Fakesubsubsection:We predicted the rrn
We predicted the \textit{rrn} gene copy number for responders as described
\citep{Kembel_2012}. The number of \textit{rrn} gene copies
a microorganism has is correlated to it's ability to increase growth
rapidly in response to nutrient influx \citep{Klappenbach_2000}. Cellulose
responders possessed fewer estimated \textit{rrn} copy numbers (X) than  xylose
responders (X) (Figures~\ref{fig:shift} and \ref{fig:copy}; P = 1.878e$^{-9}$).
Furthermore, xylose responder estimated \textit{rrn} gene copy number related
inversely to the day of first response (P = 2.02e$^{-15}$,
Figure~\ref{fig:copy}).

% Fakesubsubsection:We assessed phylogenetic
We assessed phylogenetic clustering of $^{13}$C-responsive OTUs with the
Nearest Taxon Index (NTI) and the Net Relatedness Index (NRI)
\citep{Webb2000}. We also quantified the average clade depth of cellulose and
xylose responders with the consenTRAIT metric \citep{Martiny2013}. Briefly, the
NRI and NTI evaluate phylogenetic clustering against a null model for the
distribution of a trait in a phylogeny. The NRI and NTI values are z-scores or
standard deviations from the mean and thus the greater the magnitude of the
NRI/NTI, the stronger the evidence for clustering (positive values) or
overdispersion (negative values). NRI assesses overall clustering whereas the
NTI assesses terminal clustering. An NRI of 1.96, for instance, would signify
overall phylogenetic clustering with a corresponding P-value of 0.05
\citep{Evans2014a}. The consenTRAIT metric is a measure of the average clade
depth for a trait in a phylogenetic tree. NRI values indicate that cellulose
responders clustered phylogenetically (NRI: 4.49) while xylose responders are
overdispersed although statistical support for the xylose responder NRI is not
strong (NRI: -1.33). NTI values show that both cellulose and xylose responders
are terminally clustered (NTI: 1.43 and 2.69, respectively). The consenTRAIT
clade depth for xylose and cellulose responders was 0.012 and 0.028 SSU rRNA
gene sequence dissimilarity, respectively. As reference, the average clade
depth is
0.017 SSU rRNA gene sequence dissimilarity for arabinose utilization (another
five C sugar found in hemicellulose) and was 0.013 and 0.034 SSU rRNA gene
sequence dissimilarity for glucosidase and cellulase activity of isolates in
culture, respectively \citep{Martiny2013,Berlemont2013}. These results
indicate that there are terminal clusters of xylose responders distributed
throughout the phylogeny while cellulose responders are clustered terminally
and deeply with the phylogeny.
