\subsection{Figure S1}
Percentage of added $^{13}$C remaining in soil over time.     \subsection{Figure S2}
The experimental design.  A carbon mixture, in addition to inorganic salts and amino acids (not shown here), was added to each soil microcosm where the only difference between treatments is the $^{13}$C-labeled isotope (in red). At days 1, 3, 7, 14, and 30 replicate microcosms were destructively harvested for downstream molecular applications. Bulk DNA from each treatment and time point (n = 14) was CsCl density separated by centrifuged and fractionated (orange tubes wherein each arrow represents a fraction from the density gradient). Fractions were 16S gene sequenced using next generation sequencing technology.\subsection{Figure S3}
Counts of responders to each isotopically labeled substrate (cellulose and xylose) over time.\subsection{Figure S4}
Ordination of bulk gradient fraction phylogenetic profiles.\subsection{Figure S5}
Fold change time$^{-1}$ for OTUs that changed significantly in abundance over time. One panel per phylum (phyla indicated on the right). Taxonomic class indicated on the left.\subsection{Figure S6}
Relative abundance versus day for classes that changed significantly in relative abundance with time.\subsection{Figure S7}
Sum of bulk abundances with selected phylum for responder OTUs.
    \subsection{Figure S8}
Left column shows counts of $^{13}$C-xylose responders in the \textit{Actinobacteria, Bacteroidetes, Firmicutes} and \textit{Proteobacteria} at days 1, 3, 7 and 30. Right panel shows OTU fold enrichment in heavy gradient fractions for $^{13}$C-xylose amendment DNA relative to corresponding control fractions. 
    \subsection{Figure S9}
Phylum specific trees. Heatmap indicates fold change between heavy fractions of control gradients versus labeled gradients. Dots indicate the position of "responders" to $^{13}$C-xylose (green) or $^{13}$C-cellulose (blue).\subsection{Figure S10}
Density profile for a single $^{13}$C-cellulose "responder" OTU in the labeled gradient, blue, and the control gradient, orange. Vertical lines show center of mass for each density profile and arrow denotes the magnitude and direction of the BD shift upon labeling. Panel at right shows relative abundance values in the heavy fractions for each gradient. \subsection{Figure S11}
Estimated rRNA operon copy number per genome for $^{13}$C responding OTUS. Panel titles indicate which labeled substrate(s) are depicted.\subsection{Figure S12}
Density profile for a single $^{13}$C-cellulose "non-responder" OTU in the labeled gradient, blue, and the control gradient, orange. Vertical line shows where "heavy" fractions begin as defined in our analysis. Panel at right shows relative abundance values in the heavy fractions for each gradient.