\subsection{Figure S1}
An organic matter enrichment including C components and nutrients commonly found in plant biomass was added to soil microcosms. At days 1, 3, 7, 14, and 30 replicate microcosms were destructively harvested.  Bulk DNA from each treatment and time point (n = 14) was subjected to CsCl density gradient centrifugation and density gradients were fractionated (orange tubes wherein each arrow represents a fraction from the density gradient). SSU rRNA genes were PCR amplified and sequenced from gradient fractions and from non-fractionated DNA (representing the bulk soil microbial community).  \subsection{Figure S2}
Percentage of added $^{13}$C remaining in soil over time. \subsection{Figure S3}
NMDS analysis of SSU rRNA gene composition differences between  non-fractionated DNA alone (right panel) and in the context of SIP gradient fractions (left panel). Non-fractionated DNA SSU rRNA gene composition changed with time but not with treatment (right panel) and variance of non-fractionated DNA SSU rRNA gene composition was less than variance introduced by density fractionation (left panel). Distance in SSU rRNA gene composition was quantified with the weighted UniFrac metric.  \subsection{Figure S4}
Change in non-fractionated DNA relative abundance versus time (expressed as LFC) for OTUs that changed significantly (P-value $<$ 0.10, Wald test). Each panel shows one phylum (labeled on the right). The taxonomic class is indicated on the left. Colors represent whether OTUs responded to just xylose (green), just cellulose (blue), or both xylose and cellulose (red).  \subsection{Figure S5}
Relative abundance in non-fractionated DNA versus time for classes that changed significantly. \subsection{Figure S6}
Change in relative abundance in non-fractionated DNA over time for xylose
responders (13CXPS) and cellulose responders (13CCPS). Each panel represents
a phylum except for the lower right panel which shows all reponders to both
xylose and celluose.    
\subsection{Figure S7}
Maximum enrichment at any point in time in heavy fractions of $^{13}$C-treatments relative to control (expressed as LFC) shown for $^{13}$C-cellulose versus $^{13}$C-xylose treatments. Each point represents an OTU. Blue points are cellulose responders, green xylose responders, red are responders to both xylose and cellulose, and gray points are OTUs that did not repspond to either substrate. Line indicates a slope of one. \subsection{Figure S8}
Counts of xylose responders and cellulose responders over time. \subsection{Figure S9}
Raw data from example responders highlighted in the main text (see Results). The left column shows DNA-SIP density fraction relative abundances for $^{13}$C-xylose or $^{13}$C-cellulose gradients in addition to control gradients for each of the chosen OTUs. Time is indicated by the color of the relative abundance profile (see legend). Gradient profiles are shaded by treatment where orange represents ``control'' profles, blue ``$^{13}$C-cellulose'', and green ``$^{13}$C-xylose.'' The right column shows the relative abundance of each OTU in non-fractionated DNA (i.e. the DNA that was subsequently fractionated on the density gradient). Enrichment in the heavy end of the gradient in $^{13}$C-treatments indicates an OTU has $^{13}$C-labeled DNA. \subsection{Figure S10}
Estimated \textit{rrn} copy number for xylose and cellulose responders. The leftmost panel contrasts estimated \textit{rrn} copy number for cellulose (13CCPS) and xylose (13CXPS) responders. The right panel shows estimated \textit{rrn} copy number versus time of first response for xylose responders. Colors denote the phylum of the OTUs (see legend).  \subsection{Figure S11}
Density profile for a single cellulose responder in the $^{13}$C-cellulose treatment (blue) and control (orange). Vertical lines show center of mass for each density profile and the arrow denotes the magnitude and direction of $\Delta\hat{BD}$. Right panel shows relative abundance values in the high density fractions (The boxplot line is the median value. The box spans one interquartile range (IR) about the median, whiskers extend 1.5 times the IR and the dots indicate outlier values beyond 1.5 times the IR).   \subsection{Figure S12}
Density profile for a single non-responder OTU. The $^{13}$C-cellulose treatment is in blue and the control treatment is in orange. The vertical line shows where "heavy" fractions begin as defined in our analysis. The right panel shows relative abundance values in the heavy fractions for each gradient (The boxplot line is the median value. The box spans one interquartile range (IR) about the median, whiskers extend 1.5 times the IR and the dots indicate outlier values beyond 1.5 times the IR).  