\section{Supplemental Notes}
\subsection{Phylogenetic affiliation of $^{13}$C-cellulose and
    $^{13}$C-xylose responsive microorganisms}
% Fakesubsubsection:\textit{Proteobacteria} represent 46\% of all
\textit{Proteobacteria} represent 46\% of all $^{13}$C-cellulose responding
OTUs identified. \textit{Cellvibrio} accounted for 3\% of all proteobacterial
$^{13}$C-cellulose responding OTUs detected. \textit{Cellvibrio} was one of the
first identified cellulose degrading bacteria and was originally described by
Winogradsky in 1929 who named it for its cellulose degrading abilities
\citep{boone2001bergeys}. All $^{13}$C-cellulose responding
\textit{Proteobacteria} share high sequence identity with 16S rRNA genes from
sequenced cultured isolates (Table~\ref{tab:cell}) except for ``OTU.442'' (best
cultured isolate match 92\% sequence identity in the \textit{Chrondomyces}
genus, Table~\ref{tab:cell}) and ``OTU.663'' (best cultured isolate match
outside \textit{Proteobacteria} entirely, \textit{Clostridium} genus, 89\%
sequence identity, Table~\ref{tab:cell}). Some \textit{Proteobacteria}
responders share high sequence identity with isolates in genera known to
possess cellulose degraders including \textit{Rhizobium}, \textit{Devosia},
\textit{Stenotrophomonas} and \textit{Cellvibrio}. One \textit{Proteobacteria}
OTU shares high sequence identity (100\%) with a \textit{Brevundimonas} cultured
isolate.  \textit{Brevundimonas} has not previously been identified as a
cellulose degrader, but has been shown to degrade cellouronic acid, an oxidized
form of cellulose \citep{Tavernier_2008}.

% Fakesubsubsection:\textit{Verrucomicrobia}, a cosmopolitan soil phylum 
\textit{Verrucomicrobia}, a cosmopolitan soil phylum often found in high
abundance \citep{Fierer_2013}, are hypothesized to degrade polysaccharides in
many environments \citep{Fierer_2013,Herlemann_2013,10543821}.
\textit{Verrucomicrobia} comprise 16\% of the total $^{13}$C-cellulose
responder OTUs detected. 40\% of \textit{Verrucomicrobia} $^{13}$C-cellulose
responders belong to the uncultured ``FukuN18'' family originally identified in
freshwater lakes \citep{Parveen_2013}.  The strongest \textit{Verrucomicrobial}
responder OTU to $^{13}$C-cellulose shared high sequence identity (97\%) with
an isolate from Norway tundra soil \citep{Jiang_2011} although growth on
cellulose was not assessed for this isolate. Only one other $^{13}$C-cellulose
responding verrucomicrobium shared high DNA sequence identity with an
isolate, ``OTU.638'' (Table~\ref{tab:cell}) with \textit{Roseimicrobium
gellanilyticum} (100\% sequence identity) which has been shown to grow on
soluble cellulose \citep{Otsuka_2012}. The remaining $^{13}$C-cellulose
\textit{Verrucomicrobia} responders did not share high sequence identity with
any isolates (maximum sequence identity with any isolate
93\%).

% Fakesubsubsection:\textit{Chloroflexi} are traditionally known for
\textit{Chloroflexi} are known for metabolically dynamic
lifestyles ranging from anoxygenic phototrophy to organohalide respiration
\citep{Hug_2013}. Recent studies have focused on \textit{Chloroflexi} roles in
C cycling \citep{Hug_2013, Goldfarb_2011,Cole_2013} and several
\textit{Chloroflexi} utilize cellulose \citep{Goldfarb_2011, Cole_2013,
Hug_2013}. Four closely related OTUs in an undescribed \textit{Chloroflexi}
lineage (closest matching isolate for all four OTUs:
\textit{Herpetosiphon geysericola}, 89\% sequence identity,
Table~\ref{tab:cell}) responded to $^{13}$C-cellulose (Figure~\ref{fig:trees}).
One additional OTU also from a poorly characterized \textit{Chloroflexi}
lineage (closest cultured isolate matched a proteobacterium at 78\% sequence
identity) responded to $^{13}$C-cellulose (Figure~\ref{fig:trees}).

% Fakesubsubsection:Other notable $^{13}$C-cellulose responders
Other notable $^{13}$C-cellulose responders include a \textit{Bacteroidetes}
OTU that shares high sequence identity (99\%) to \textit{Sporocytophaga
myxococcoides} a known cellulose degrader \citep{Vance_1980}, and three
\textit{Actinobacteria} OTUs that share high sequence identity (100\%) with
isolates. One of the three \textit{Actinobacteria}
$^{13}$C-cellulose responders is in the \textit{Streptomyces}, a genus known to
possess cellulose degraders, while the other two share high sequence identity
to cultured isolates \textit{Allokutzneriz albata} \citep{Labeda_2008,
Tomita_1993} and \textit{Lentzea waywayandensis} \citep{LABEDA_1989,
Labeda_2001}; neither isolate decomposes cellulose in culture. Nine
\textit{Planctomycetes} OTUs responded to $^{13}$C-cellulose but none are within
described genera (closest cultured isolate match 91\% sequence identity,
Table~\ref{tab:cell}) (Figure~\ref{fig:trees}). One
$^{13}$C-cellulose responder is annotated as ``cyanobacteria''.
The cyanobacteria phylum annotation is misleading as the OTU is not closely
related to any oxygenic phototrophs (closest cultured isolate match
\textit{Vampirovibrio chlorellavorus}, 95\% sequence identity,
Table~\ref{tab:cell}). A sister clade to the oxygenic phototrophs classically
annotated as ``cyanobacteria'' in SSU rRNA gene reference databases, but does
not possess any known phototrophs, has recently been proposed to constitute its own
phylum, "Melainabacteria" \citet{Di_Rienzi_2013}; although, the phylogenetic
position of ``Melainabacteria'' is debated \citep{Soo_2014}. The catalog of
metabolic capabilities associated with cyanobacteria (or candidate phyla
previously annotated as cyanobacteria) are quickly expanding
\citep{Di_Rienzi_2013, Soo_2014}. Our findings provide evidence for cellulose
degradation within a lineage closely related to but apart from oxygenic
phototrophs. Notably, polysaccharide degradation is suggested by an analysis of
a ``Melainabacteria'' genome \citep{Di_Rienzi_2013}. Although we highlight
$^{13}$C-cellulose responders that share high sequence identity with described
genera, most $^{13}$C-cellulose responders uncovered in this experiment are not
closely related to cultured isolates (Table~\ref{tab:cell}).

% Fakesubsubsection:\textit{Verrucomicrobia} are ubiquitous in soil
\textit{Verrucomicrobia}, cosmopolitan soil microbes
\citep{Bergmann_2011}, can comprise up to 23\% of 16S rRNA gene sequences in
high-throughput DNA sequencing surveys of SSU rRNA genes in soil
\citep{Bergmann_2011} and can account for up to 9.8\% of
soil 16S rRNA \citep{Buckley_2001}. Many \textit{Verrucomicrobia} were first
isolated in the last decade \cite{Wertz_2011} but only one of the 15 most
abundant verrucomicrobial phylotypes in a global soil sample collection shared
greater than 93\% sequence identity with a cultured isolate
\citep{Bergmann_2011}. Genomic analyses and physiological profiling of
\textit{Verrucomicrobia} isolates revealed \textit{Verrucomicrobia} are capable
of methanotrophy, diazotrophy, and cellulose degradation \citep{Otsuka_2012,
Wertz_2011}, yet the function of soil \textit{Verrucomicrobia} in global C-cycling
remains unknown. Only two of the ten putative cellulose degrading
\textit{Verrucomicrobia} identified in this experiment share at least 95\%
sequence identity with an isolate ("OTU.83" and "OTU.627",
Table~\ref{tab:cell}). Seven of ten $^{13}$C-cellulose responding
verrucomicrobial OTUs were classified as \textit{Spartobacteria} which are
a numerically dominant family of \textit{Verrucomicrobia} in SSU rRNA gene
surveys of 181 globally distributed soil samples \citep{Bergmann_2011}. Given
their ubiquity and abundance in soil as well as their demonstrated
incorporation of $^{13}$C from $^{13}$C-cellulose, \textit{Verrucomicrobia}
lineages, particularly \textit{Spartobacteria}, may be important contributors
to cellulose decomposition on a global scale.

% Fakesubsubsection:Soil \textit{Chloroflexi} have been found to assimilate
Cellulose degrading soil \textit{Chloroflexi} have previously been identified
in DNA-SIP studies \citep{Schellenberger_2010}. The cellulose degrading
\textit{Chloroflexi} in this study are only distantly related to isolates
\ref{tab:cell}. Chloroflexi are among the six most abundant soil phyla commonly
recovered soil microbial diversity surveys \citep{Janssen2006}.
Chloroflexi are typically not as as abundant as \textit{Verrucomicrobia} but
are roughly as abundant as \textit{Bacteroidetes} and \textit{Planctomycetes}
\citep{Janssen2006}.  Four of five $^{13}$C-cellulose responsive
\textit{Chloroflexi} identified in this study are annotated as belonging to the
\textit{Herpetosiphon} although they share less than 95\% sequence identity with their closest cultured relative in the
\textit{Herpetosiphon} genus (\textit{H. geysericola}). \textit{H. geysericola}
is a predatory bacterium shown to prey upon \textit{Aerobacter} in culture and
can also digest cellulose \citep{Lewin1970}. In our study, "Herpetosiphon"
$^{13}$C-cellulose responders did not show a delayed response to
$^{13}$C-cellulose as compared to other responders but nonetheless could have
become labeled by feeding on primary $^{13}$C-cellulose degraders. The prey
specificity of predatory bacteria is not well established especially \textit{in
situ}. $^{13}$C-labeling would be positively correlated with prey specificity.
If the predator specifically preyed upon one population then it could take on
the same labeling percent as that population given enough generations. Preying on multiple types would
produce a mixed and dilute labeling signature if some of the prey
were not isotopically labeled.

% Fakesubsubsection:We also observed $^{13}$C-incorporation
We also observed $^{13}$C-incorporation from cellulose by
\textit{Proteobacteria}, \textit{Planctomycetes} and \textit{Bacteroidetes}. 
Strains in \textit{Proteobacteria}, \textit{Planctomycetes} and \textit{Bacteroidetes} have all been
previously implicated in cellulose degradation. \textit{Planctomycetes} is the
least studied of the three phyla and only one \textit{Planctomycetes} isolate can
grow on cellulose. None of the seven \textit{Planctomycetes} cellulose degraders
identified in this experiment are closely related to isolates.
\textit{Acidobacteria} did not pass or operational criteria for assessing
$^{13}$C incorporation from cellulose into DNA in our microcosms. 
\textit{Acidobacteria} have been found to degrade cellulose in culture CITE and
are a numerically significant soil phylum CITE. \textit{Acidobacteria} have
been shown to dominate at low nutrient availability (CITE: cederlund 2014),
which may explain why they were not active upon nutrient additions. 
The \textit{Acidobacteria} in our microcosms were mainly annotated as belonging
to candidate orders in the Silva taxonomic nomenclature. The highest relative
abundance for any \textit{Acidobacteria} order in the bulk samples was 0.20
(order "DA023") and the highest relative abundance of the Acidobacteria phylum
was 0.23.


% Fakesubsubsection:All of the $^{13}$C-xylose responders in the \textit{Firmicutes}
All of the $^{13}$C-xylose responders in the \textit{Firmicutes} phylum are
closely related (at least 99\% sequence identity) to cultured isolates from
genera that are known to form endospores (Table~\ref{tab:xyl}). Each
$^{13}$C-xylose responder is closely related to isolates annotated as members
of \textit{Bacillus}, \textit{Paenibacillus} or \textit{Lysinibacillus}.
\textit{Bacteroidetes} $^{13}$C-xylose responders are predominantly closely
related to \textit{Flavobacterium} species (5 of 8 total responders)
(Table~\ref{tab:xyl}).  Only one \textit{Bacteroidetes} $^{13}$C-xylose
responder is not closely related to a cultured isolate, ``OTU.183'' (closest
LTP BLAST hit, \textit{Chitinophaca sp.}, 89.5\% sequence identity,
Table~\ref{tab:xyl}). OTU.183 shares high sequence identity with environmental
clones derived from rhizosphere samples (accession AM158371, unpublished) and
the skin microbiome (accession JF219881, \citet{Kong_2012}). Other
\textit{Bacteroidetes} responders share high sequence identities with canonical
soil genera including \textit{Dyadobacer}, \textit{Solibius} and
\textit{Terrimonas}. Six of the 8 \textit{Actinobacteria} $^{13}$C-xylose
responders are in the \textit{Micrococcales} order. One $^{13}$C-xylose
responding \textit{Actinobacteria} OTU shares 100\% sequence identity with
\textit{Agromyces ramosus} (Table~\ref{tab:xyl}).  \textit{A. ramosus} is a
known predatory bacterium but is not dependent on a host for growth in culture
\citep{16346402}. It is not possible to determine the specific origin of
assimilated $^{13}$C in a DNA-SIP experiment. $^{13}$C can be passed down
through trophic levels although heavy isotope representation in C pools
targeted by cross-feeders and predators would be diluted with depth into the
trophic cascade. It is possible, however, that the $^{13}$C labeled
\textit{Agromyces} OTU was assimilating $^{13}$C primarily by predation if the
\textit{Agromyces} OTU was selective enough with respect to its prey that it
primarily attacked $^{13}$C-xylose assimilating organisms. 

\subsection{Phylogenetic breadth of ecological strategies}
% Fakesubsubsection:It has been proposed that ecological
It has been proposed that ecological strategies of soil microorganisms
correlate to broad taxonomic groupings CITE (Evans, Schimel, Fierer). Our
results suggest that while substrate utilization is not defined at the level of
OTU, it is also not defined at the phylum or even family level. Ecology
captured at broad phylogenetic breadth would indicate that phylogenetic
composition would change in response to changing conditions whereas if ecology
is defined by high resolution groups, it suggests that community ecology can
change without a corresponding change in composition or specifically that
extant taxa would rapidly adapt to environmental change CITE. We found that
substrate utilization is captured by phylogenetic levels somewhere between the
genus and phylum level (Figure~\ref{fig:trees}). Our measurements did define
differences between and within substrate responder groups suggesting there are
ecological strategy sub-groups within larger groups defined by an affinity
for a particular substrate (Figure~\ref{fig:shift},
Figure~\ref{fig:copy}). Hence, compositional changes could occur at
different phylogenetic scales in response to environmental perturbation. 

\subsection{Buoyant density shift estimates}
Upon labeling, DNA from an organism that incorporates exclusively $^{13}$C will
increase in BD more than DNA from an organism that does not exclusively utilize
isotopically labeled C. Therefore the magnitude DNA BD shifts indicate
substrate specificity given our experimental design as only one substrate was
labeled in each amendment. We measured density shift as the change in an OTU's
density profile center of mass between corresponding control and labeled
gradients. BD shifts, however, should not be evaluated on an individual OTU
basis as a small number of density shifts are observed for each OTU and the
variance of the density shift metric at the level of individual OTUs is
unknown. It is therefore more informative to compare density shifts among
substrate responder groups. Further, density shifts are based on relative
abundance profiles and would be distorted in comparison to density shifts based
on absolute abundance profiles and should be interpreted with this
transformation in mind. It should also be noted that there was overlap in
observed density shifts between $^{13}$C-cellulose and $^{13}$C-xylose
responder groups, suggesting that although cellulose degraders are generally
more substrate specific than xylose utilizers, each responder group
exhibits a range of substrate specificities (Figure~\ref{fig:shift}). 

\subsection{\textit{rrn} gene copy number}  
\textit{rrn} copy number estimation is a recent advance in microbiome
science \citep{Kembel_2012} although the relationship of \textit{rrn} copy
number per genome with ecological strategy is well established
\citep{Klappenbach_2000}. Microorganisms with a high \textit{rrn} copy
number tend to be fast growers specialized to take advantage of boom-bust
environments whereas microorganisms with low \textit{rrn} copy number
favor slower growth under lower and more consistent nutrient input
\citep{Klappenbach_2000}. At the beginning of our incubation, OTUs with
estimated high \textit{rrn} copy number or ``fast-growers'' assimilate
xylose into biomass and with time slower growers (lower \textit{rrn} copy
number) begin to incorporate $^{13}$C from xylose.  Further,
$^{13}$C-xylose responders have more estimated rRNA operon copy numbers
per genome than $^{13}$C-cellulose responders (p-value
1.878x10$^{-09}$) suggesting xylose respiring microbes are generally
faster growers than cellulose degraders.

\subsection{Sequencing statistics and density fractionation}
% Fakesubsubsection:Microcosm DNA was density fractionated on
Microcosm DNA was density fractionated on CsCl density gradients. We sequenced
SSU rRNA gene amplicons from a total of 277 CsCl gradient fractions from 14
CsCl gradients and 12 bulk microcosm DNA samples. The SSU rRNA gene data set
contained 1,102,685 total sequences. The average number of sequences per sample
was 3,816 (sd 3,629) and 265 samples had over 1,000 sequences. We sequenced SSU
rRNA gene amplicons from an average of
19.8 fractions per CsCl gradient (sd 0.57). The average density between
fractions was  0.0040 g mL$^{-1}$ The sequencing effort recovered a total of
5,940 OTUs. 2,943 of the total 5,940 OTUs were observed in bulk samples. We
observed 33 unique phylum and 340 unique genus annotations.


