\section{Supplemental Methods}


\textbf{Soil Collection and Preparation}



Soils were collected from an organic farm in Penn Yan, New York.  These soils are characterized as Honoeye/Lima, a silty clay loam on calcareous bedrock. To get a field average, 10cm cores were collected (in duplicate) from six different sampling locations around the field using a slide hammer bulk density sampler (coordinates: (1) N 42° 40.288’ W 77° 02.438’, (2) N 42° 40.296’ W 77° 02.438’, (3) N 42° 40.309’ W 77° 02.445’, (4) N 42° 40.333’ W 77° 02.425’, (5) N 42° 40.340’ W 77° 02.420’, (6) N 42° 40.353’ W 77° 02.417’) on November 21, 2011.   Cores were all sieved through a 2mm sieve, homogenized by mixing, and stored at 4°C until use (within 1-2 week of collection).  Carbon and nitrogen content , as well as, water holding capacity were previously measured for these soils as .  
50\% water holding capacity measured. 



\textbf{Cellulose production}



\textit{Gluconoacetobacter xylinus} was grown on Heo and Son 0.1\% glucose agar plates (using \textsuperscript{12}C-glucose) at 30°C without inositol \cite{Heo_Son_2002}.  Heo and Son liquid minimal media was made with 0.1\% glucose (one batch with \textsuperscript{12}C- and another with \textsuperscript{13}C-glucose).  To increase surface area for cellulose growth, 100mL of the media were added to individual 1L Erlenmeyer flasks.  Each aliquot of media was inoculated with three isolated colonies of \textit{Gluconoacetobacter xylinus} from the Heo and Son plate previously grown.  Flasks were kept static in the dark at 30°C for 2-3 weeks until thick cellulose pellicule had formed.  To wash culture from cellulose, two parts 1\% alconox was added to each flask and autoclaved.  Cellulose pellicules were rinsed with deionized (DI) water (1L) until no suds were produced ($\sim$10 rinses).  Each pellicule was put in 1L of DI water for overnight dialysis.  Pellicules were rinsed (3 washes) with DI water twice a day for 2-3 days to ensure no detergent or culture media remained associated with cellulose.  Harvested pellicules were dried overnight (60°C) and then cut into pieces and ground using ball grinder until desired size range (53um - 250um) was achieved (checked by sieve).  Size range was based on particulate organic matter to emulate how microbes may experience cellulose in the environment and for even distribution in microcosms.                    



\textbf{Soil microcosms}

A subset of soil was dried at 105°C overnight to determine soil moisture content. Microcosms (35 total) were created with the equivalent of ten grams dry soil weight of the sieved soil in a 250mL Erlenmeyer flask capped with a butyl rubber stopper to preventing drying of soils.  Microcosms were kept at 25°C for 2 weeks for bottle conditioning (better phrasing: pre-incubation?) and stoppers were removed for 10min every 3 days to prevent anoxia of soils. A complex carbon mixture (38\% cellulose, 23\% lignin, 20\% xylose, 3\% arabinose, 1\% galactose, 1\% glucose, 0.5\% mannose) was designed based on switch grass biomass composition.  This complex carbon mixture was added to microcosms along with an amino acid (in-house made replica of teknova Cat#C0705) and basal salt mixture (Murashige and Skoog, Sigma M5524) at 52.6mg 10g\textsuperscript{-1} soil; representative of natural concentrations.  Cellulose (20mg) and lignin (12mg) were sprinkled over the surface of the soil in each microcosm as dry additions due to their insolubility to ensure same concentration in every flask. The remaining carbons, amino acids, and basal salts were added by pipetting evenly over soil surface as a liquid addition until 50\% water holding capacity is achieved.  Three parallel treatments were performed; (1) control, all \textsuperscript{12}C-carbons, (2) cellulose, all \textsuperscript{12}C-carbons except \textsuperscript{13}C-cellulose, (3) xylose, all \textsuperscript{12}C-carbons except \textsuperscript{13}C-xylose.  Replicate microcosms were harvested (stored at -80°C until nucleic acid processing) at days 1 (control and xylose only), 3, 7, 14, and 30.  Microcosms remained stoppered throughout incubation, except for every third day when flasks were flushed with fresh air for 2 minutes.  A subset of microcosm soil for each treatment and time point were isotopically analyzed at Cornell University Stable Isotope Laboratory to determine amount of \textsuperscript{13}C that remained at each time point.                       



\textbf{Nucleic acid extraction}

DNA was extracted from 0.25g soil using a modified Griffiths procotol (Griffiths 2000 Rapid Method for Coextraction of DNA and RNA from Natural Environments for Analysis of Ribosomal DNA- and rRNA-Based Microbial Community Composition, 10.1128/AEM.66.12.5488-5491.2000).  Soils were bead beat in 2mL lysis tubes containing 0.5g silica/zirconia beads (treated at 300°C for 4 hours to remove RNases), 0.5mL extraction buffer (240 mM Phosphate buffer
0.5\% N-lauryl sarcosine), and 0.5mL phenol-chloroform-isoamyl alcohol (25:24:1) for 1 min at 5.5 ms\textsuperscript{-1}. After beat beading, 85uL 5M NaCl and 60uL 10\% hexadecyltriammonium bromide (CTAB)/0.7M NaCl were added to lysis tube, vortexed, chilled for 1min on ice, and centrifuged at 16,000 x g for 5min at 4°C.  The aqueous layer was transferred to a new tube and reserved on ice.  To increase DNA recovery, a double extraction was performed by adding 85uL 5M NaCl and 0.5mL extraction buffer to same lysis tube, vortexed, and centrifuged same as before.  New aqueous layer was added to previously collected aqueous layer and washed with 0.5mL chloroform:isoamyl alcohol (24:1), vortexed, and centrifuged same as before.  Aqueous layer was transferred to a new tube and nucleic acids were precipitated using 2 volumes polyethylene glycol solution (30\% PEG 8000, 1.6M NaCl) on ice for 2hrs, followed by centrifugation at 16,000 x g, 4°C for 30min.  Supernatant was discarded and pellets were washed with 1mL ice cold 70\% EtOH; each wash consisting of a vortex followed by centrifugation at 16,000 x g, 4°C for 10min.  Pellets were air dried, resuspended in 50uL TE, replicate extractions pooled, and stored at -20°C. To prepare nucleic acid extracts for isopycnic centrifugation, DNA was size selected (>4kb) using 1\% low melt agarose gel and $\beta$- agarase I enzyme extraction per manufacturers protocol (New England Biolab, M0392S).  Final resuspension of DNA pellet was in 50$\mu$L TE.   


\textbf{Isopycnic centrifugation and fractionation}

 Isopycnic gradients were established as described previously (35, 48) for five \textsuperscript{12}C-control, five \textsuperscript{13}C-xylose, and four \textsuperscript{13}C-cellulose microcosms; one per day harvested. A density gradient solution of 1.762 g cesium chloride (CsCl) ml\textsuperscript{-1} in gradient buffer solution (pH 8.0 15mM Tris-HCl, 15mM EDTA, 15mM KCl) was used to separate \textsuperscript{13}C-enriched and \textsuperscript{12}C-nonenriched DNA. Each gradient was loaded with approximately 5$\mu$g of DNA into a Beckman OptiSeal\textsuperscript{TM} polyallomer centrifuge tube (part no. 361621; Beckman, Palo Alto, CA) containing the cesium chloride-gradient buffer solution. Samples were centrifuged on a Beckman Coulter Optima\textsuperscript{TM} MAX-E ultracentrifuge using a TLA-110 fixed-angle rotor for 66 h at 55,000 rpm and room temperature (RT). Fractions of $\sim$100 $\mu$L were collected into Acroprep\textsuperscript{TM} 96 filter plate (part no. 5035, Pall Life Sciences) by displacing the DNA-CsCl-gradient buffer solution in the centrifugation tube with water. The refractive index of each fraction was measured using a Reichart AR200 digital refractometer. Then buoyant density was calculated from the refractive index using the equation $\rho$=\textit{a}$\eta$-\textit{b}, where $\rho$ is the density of the CsCl (g ml\textsuperscript{-1}), $\eta$ is the measured refractive index, and \textit{a} and \textit{b} are coefficient values of 10.9276 and 13.593, respectively, for CsCl at 20°C \cite{9780408708036}. The collected DNA fractions were purified by washing each with 200$\mu$L TE, centrifuging for 10min at 500 x g and discarding flow through. This was repeated five more times except using 300$\mu$L of TE in subsequent washes and centrifuging for 20min on the final wash. Finally, 50$\mu$L TE was added to each fraction, incubated on filter for 5min at RT then resuspended DNA was pipetted off the filter into a new microfuge tube.   Fractions were quantitated by qPCR (BioRad) using 12.5$\mu$L QuantiFast  SYBR greenPCR  master mix (204056, Qiagen, Valencia, CA), 1.25$\mu$L 10$\mu$M 515F (5'-GTGCCAGCMGCCGCGGTAA -3'), 1.25$\mu$L 10$\mu$M 806R (5'-GGACTACHVGGGTWTCTAAT-3'), and 1:100 dilution of template (include PCR cycles? or are they previously published?). For every gradient, 20 fractions were chosen for sequencing between the density range 1.67-1.75g/mL (supplementary?)     


\textbf{Sequencing}  Barcoded 454 primers were designed using 454-specific adapter B, 10bp barcodes \cite{Hamady_2008}, a 2bp linker (put those bp in here), and 806R primer for reverse primer (BA806R); and 454-specific adapter A, a 2bp linker, and 515F primer for forward primer (BA515F). Each fraction was PCR amplified using 0.25$\mu$L 5U/$\mu$l AmpliTaq Gold (N8080243, Life Technologies, Grand Island, NY), 2.5$\mu$L 10X Buffer II (100 mM Tris-HCl, pH 8.3, 500 mM KCl), 2.5$\mu$L 25mM MgCl_{2}, 4$\mu$L 5mM dNTP, 1.25$\mu$L 10mg/mL BSA, 0.5$\mu$L 10$\mu$M BA515F, 1$\mu$L 5$\mu$M BA806R,3$\mu$L H_{2}O, 10$\mu$L 1:30 DNA template) in triplicate and checked by 1\% agarose gel. Replicate reactions were then pooled, normalized either using Pico green quantification and manual calculation or by SequalPrep\textsuperscript{TM} normalization plates (A10510, Invitrogen, Carlsbad, CA), then pooled in equimolar concentrations.  Overall pool was gel extracted from a 1\% agarose gel using Wizard SV gel and PCR clean-up system (A9281, Promega, Madison, WI) per manufacturer's protocol.  Amplicons were sequenced on Roche 454 FLX system using titanium chemistry at Selah Genomics (formerly EnGenCore, Columbia, SC)    

\textbf{Post-Sequencing Analysis}
 