\subsection{Figure S1}
The experimental design.  A carbon mixture, in addition to inorganic salts and amino acids (not shown here), was added to each soil microcosm where the only difference between treatments is the $^{13}$C-labeled isotope (in red). At days 1, 3, 7, 14, and 30 replicate microcosms were destructively harvested for downstream molecular applications. Bulk DNA from each treatment and time point (n = 14) was CsCl density separated by centrifuged and fractionated (orange tubes wherein each arrow represents a fraction from the density gradient). Fractions were 16S gene sequenced using next generation sequencing technology.\subsection{Figure S2}
Percentage of added $^{13}$C remaining in soil over time.     \subsection{Figure S3}
Ordination of bulk gradient fraction phylogenetic profiles.\subsection{Figure S4}
Fold change time$^{-1}$ for OTUs that changed significantly in abundance over time. One panel per phylum (phyla indicated on the right). Taxonomic class indicated on the left.\subsection{Figure S5}
Relative abundance versus day for classes that changed significantly in relative abundance with time.\subsection{Figure S6}
Sum of bulk abundances with selected phylum for responder OTUs.
    \subsection{Figure S7}
Maximum log$_{2}$ fold change in response to labeled substrate incorporation for each substrate and all OTUs that pass sparsity filtering in both substrate analyses. Points colored by response. Line has slope of 1 with intercept at the origin. OTUs falling in the top-right quadrant responded to both substrates while those in the top-left and bottom-right responded to $^{13}$C-xylose and $^{13}$C-cellulose, respectively.  \subsection{Figure S8}
Counts of responders to each isotopically labeled substrate (cellulose and xylose) over time.\subsection{Figure S9}
The left column shows DNA-SIP density fraction relative abundances for all  gradients for each of the OTUs. Gradient profiles are shaded by treatment where orange represents ``control'' profles, blue ``$^{13}$C-cellulose'', and green ``$^{13}$C-xylose.'' The right column shows the abundance of each OTU in non-fractionated DNA (i.e. the DNA that was subsequently fractionated on the density gradient). Enrichment in the heavy end of the gradient in $^{13}$C treatments indicates an OTU has $^{13}$C-labeled DNA that is greater in buoyant density than it would be unlabeled. \subsection{Figure S10}
Estimated rRNA operon copy number per genome for $^{13}$C responding OTUS. Panel titles indicate which labeled substrate(s) are depicted.\subsection{Figure S11}
Density profile for a single $^{13}$C-cellulose "responder" OTU in the labeled gradient, blue, and the control gradient, orange. Vertical lines show center of mass for each density profile and arrow denotes the magnitude and direction of the BD shift upon labeling. Panel at right shows relative abundance values in the heavy fractions for each gradient. \subsection{Figure S12}
Density profile for a single $^{13}$C-cellulose "non-responder" OTU in the labeled gradient, blue, and the control gradient, orange. Vertical line shows where "heavy" fractions begin as defined in our analysis. Panel at right shows relative abundance values in the heavy fractions for each gradient.