\documentclass{article}
\usepackage{amsfonts,amsmath,amssymb}
\usepackage{graphicx}
\usepackage{color}
\usepackage{url}

\usepackage[numbers,sort&compress]{natbib}

\usepackage{geometry}

\usepackage{multirow, array, booktabs, longtable, threeparttablex}
\newcolumntype{P}[1]{>{\raggedright\arraybackslash}p{#1}}

\usepackage{caption}

\usepackage{hyperref}
\hypersetup{
    colorlinks,
    citecolor=black,
    filecolor=black,
    linkcolor=black,
    urlcolor=black
}

\newcommand{\beginsupplement}{%
\setcounter{table}{0} \renewcommand{\thetable}{S\arabic{table}}%
\setcounter{figure}{0} \renewcommand{\thefigure}{S\arabic{figure}}% 
}

\bibliographystyle{pnas}

\title{Supplemental Information}
\date{\vspace{-5ex}}

\begin{document}

\maketitle

\tableofcontents

%\listoffigures 

%\listoftables

\section{Supplemental Methods}\label{SI} 

\subsection{Soil Collection and Preparation} 
% Fakesubsubsection: We collected soils from
We collected soils from an organic farm in Penn Yan, New York. Soils were
Honoeye/Lima, a silty clay loam on calcareous bedrock. To get a field average,
cores (5 cm diameter x 10 cm depth) were collected in duplicate from six
different sampling locations around the field using a slide hammer bulk density
sampler (coordinates: (1) N 42$^{\circ}$ 40.288’ W 77$^{\circ}$ 02.438’, (2)
N 42° 40.296’ W 77$^{\circ}$ 02.438’, (3) N 42$^{\circ}$ 40.309’ W 77$^{\circ}$
02.445’, (4) N 42$^{\circ}$ 40.333’ W 77$^{\circ}$ 02.425’, (5) N 42$^{\circ}$
   40.340’ W 77$^{\circ}$ 02.420’, (6) N 42$^{\circ}$ 40.353’ W 77$^{\circ}$
      02.417’) on November 21,
2011. Soil cores were sieved (2mm), homogenized by mixing, and
stored at 4°C until preincubation (within 1-2 week of collection).
Carbon and nitrogen content were previously measured for these soils
\citep{Berthrong_2013}. Reported soil C values for the organic field were
12.15 ($\pm$ s.d. 0.78) mg C g$^{-1}$ dry soil and 1.16 ($\pm$ s.d. 0.13)
mg N g$^{-1}$ dry soil. 

\subsection{Cellulose production}
% Fakesubsubsection: Bacterial cellulose was produced by
Bacterial cellulose was produced by \textit{Gluconoacetobacter xylinus} grown
in Heo and Son \citep{Heo_2002} minimal media (HS medium) made with 0.1\%
glucose and without inositol.  For the production of 13C-cellulose,
$^{13}$C$_{6}$-D-glucose, 99 atom \% 13C (Isotec) was used. Cellulose was
produced in 1L Erlenmeyer flasks containing 100 mL HS medium inoculated with
three colonies of \textit{Gluconoacetobacter xylinus} grown on HS agar plates.
Flasks were incubated statically in the dark at 30$^{\circ}$C for 2-3 weeks.
Cellulose pellicules were decanted, rinsed with deionized water, suspended in
two volumes of 1\% alconox, and then autoclaved. Cellulose pellicules were
purified by dialysis for 12 hr in 1 L deionized water and dialysis was
repeated~10 times. Harvested pellicules were dried overnight (60$^{\circ}$C),
cut into pieces, and ground to 53 $\mu$m~-~250 $\mu$m using 5100 Mixer/Mill (SPEX
SamplePrep, Metuchen, NJ) and dry sieve. The particulate size range was
selected to be representative of particulate organic matter in soils (3).

% Fakesubsubsection: The purity of ground cellulose
The purity of ground cellulose was checked by biological assay, Benedict's
reducing sugars assay, Bradford assay, and isotopic analysis. \textit{E.coli}
is not able to use cellulose as a C source but is capable of growth on
a variety of nutrients available in the Heo and Son medium. The biological
assay consisted of \textit{E. coli} inoculated into minimal M9 media which
lacked a C source and was supplemented with either: (1) 0.01\% glucose, (2) 2.5
mg purified, ground cellulose, (3) 25 mg purified, ground cellulose, (4) 25 mg
purified, ground cellulose and 0.01\% glucose. Growth in media was checked by
spectrometer (OD$_{450}$). No measurable growth was observed with either~2 mg
or~25 mg cellulose, indicating absence of contaminating nutrients that can
support growth of \textit{E. coli}. In addition, the presence of 25 mg
cellulose did not inhibit the growth of \textit{E.coli} cultures provided with
glucose (relative to control), indicating the absence of compounds in the
purified cellulose that could inhibit microbial growth.

% Fakesubsubsection: Purified cellulose was also assayed
Purified cellulose was also assayed for residual proteins and sugars using
Bradford and Benedict's assays, respectively. Bradford assay was performed as
in \citep{Bradford_1976} with a standard curve ranging from 0 - 2000 $\mu$g ml
$^{-1}$ BSA. Ground, purified cellulose contained 6.92 $\mu$g protein mg
cellulose$^{-1} (\textit{i.e.}$ 99.31\% purity). Reducing sugars were not
detected in cellulose using Benedict's reducing sugar assay
\citep{benedict1909reagent} tested at 10 mg cellulose ml$^{-1}$. Finally,
$^{13}$C-cellulose had an average 96\% $\pm$
5 (s.d.) degree of $^{13}$C labeling as determined by isotopic analysis
  (UCDavis Stable Isotope Facility).           

\subsection{Soil microcosms}
% Fakesubsubsection: Microcosms were created
Microcosms were created by adding 10 g d.w. sieved soil to a 250 mL
Erlenmeyer flask capped with a butyl rubber stopper. The headspace was flushed
with air every 3 days which was sufficient to prevent anoxia (data not shown).
Microcosms were pre-incubated at room temperature for 2 weeks until the soil
respiration rate (determined by GCMS measurement of head space CO$_{2}$) had
stabilized. Sieving causes a transient increase in soil respiration rate
presumably due to the liberation of fresh labile soil organic matter
\citep{Datta_2014}. Pre-incubation ensures that this labile organic matter is
consumed and/or stabilized prior to the beginning of the experiment.
Respiration rate (CO$_{2}$) stabilized after 10 days (data not shown). 

% Fakesubsubsection: Three parallel treatments
Three parallel treatments were established. Each treatment received the same
amendment, where the only difference was the isotopically labeled component in
the amendment. Specifically, we made unlabeled control treatment and treatments
that substituted either $^{13}$C-cellulose (synthesized as described above) or
$^{13}$C$_{5}$-D-xylose, 98 atom \% $^{13}$C (Isotec) for their unlabeled
equivalents. The molecular composition of the amendment was designed to
approximate switchgrass biomass with hemicellulose replaced by its constituent
monomers \citep{Yan_2010,David_2010}. The amendment was added at 5.3 mg
g$^{-1}$ d.w. soil which is representative of natural concentrations in soil
during early phases of decomposition \citep{Schneckenberger_2008}. The
amendment contained by mass: 38\% cellulose, 23\% lignin, 20\% xylose, 3\%
arabinose, 1\% galactose, 1\% glucose, and 0.5\% mannose, with the remaining
13.5\% composed of amino acids (Teknova C0705) and a basal salt mixture
(Murashige and Skoog, Sigma Aldrich M5524). The amendment had a C:N ratio of
10. Cellulose (2 mg cellulose g$^{-1}$ d.w. soil) and lignin (1.2 mg lignin g-1
d.w. soil) were uniformly distributed over the soil surface as a powder and
the remaining constituents were added in solution in a volume of
0.12 ml g$^{-1}$ d.w. soil. The volume of liquid was determined in relation to
soil moisture to achieve 50\% water holding capacity. Water holding capacity
of 50\% was chosen, in relation to the texture for this soil, to achieve
approximately 70\% water filled pore space, which is the optimal water content
for respiration \citep{Linn_1984}. A total of 12 microcosms were established
per treatment. Microcosms were sampled destructively on days 1, 3, 7, 14, and
30 and soils were frozen at -80 $^{\circ}$C. The cellulose treatment was not
sampled on day 1 because it was not expected that significant cellulose
metabolism would have occurred within this time. The abbreviation “13CXPS”
refers to the 13C-xylose treatment (13C Xylose Plant Simulant), “13CCPS”
refers to the 13C-cellulose treatment and “12CCPS” refers to the unlabeled
control. A subset of soil from each sample was reserved for isotopic
analysis at the Cornell University Stable Isotope Laboratory to determine
the mass of $^{13}$C remaining in soil.

\subsection{Nucleic acid extraction}
% Fakesubsubsection: Nucleic acids were extracted
Nucleic acids were extracted from 0.25 g soil using a modified Griffiths
protocol \citep{Griffiths_2000}. Cells were lysed by bead beating for
1 min at 5.5 ms$^{-1}$ in 2mL lysis tubes containing 0.5 g of 0.1
mm diameter silica/zirconia beads (treated at 300°C for 4 hours to remove
RNases), 0.5 mL extraction buffer (240 mM Phosphate buffer 0.5\% N-lauryl
sarcosine), and 0.5 mL phenol-chloroform-isoamyl alcohol (25:24:1) for 1 min at
5.5 ms$^{-1}$. After lysis, 85 uL 5 M NaCl and 60 uL 10\%
hexadecyltriammonium bromide (CTAB)/0.7 M NaCl were added to lysis tube,
vortexed, chilled for 1 min on ice, and centrifuged at 16,000 x g for 5 min at
4°C. The aqueous layer was transferred to a new tube and reserved on ice. To
increase DNA recovery, the pellet was back extracted with 85 uL 5 M NaCl and
0.5 mL extraction buffer. The aqueous extract was washed with 0.5 mL
chloroform:isoamyl alcohol (24:1). Nucleic acids were precipitated by addition
of 2 volumes polyethylene glycol solution (30\% PEG 8000, 1.6 M NaCl) on ice
for 2 hrs, followed by centrifugation at 16,000 x g, 4°C for 30 min. The
supernatant was discarded and pellets were washed with 1 mL ice cold 70\% EtOH.
Pellets were air dried, resuspended in 50 uL TE and stored at -20°C. To prepare
nucleic acid extracts for isopycnic centrifugation as previously described
\citep{Buckley_2007}, DNA was size selected ($>$ 4kb) using 1\% low melt
agarose gel and $\beta$-agarase I enzyme extraction per manufacturers
protocol (New England Biolab, M0392S).  Final resuspension of DNA pellet
was in 50 $\mu$L TE.   

\subsection{Isopycnic centrifugation and fractionation} 
% Fakesubsubsection: We fractionated DNA on density gradients
We fractionated DNA on density gradients for $^{13}$C-xylose treatments (days
1, 3, 7, 14, 30), $^{13}$C-cellulose treatments (days 3, 7, 14, 30), and
control treatments (days 1, 3 ,7, 14, 30). A total of 5 $\mu$g DNA was added to
each~4.7 mL CsCl density gradient.  Density gradient were composed of 1.69
g mL$^{-1}$ CsCl ml$^{-1}$ in gradient buffer solution (pH 8.0 15 mM
Tris-HCl,~15 mM EDTA, 15 mM KCl). Centrifugation was performed at 55,000 rpm~20
$^{\circ}$C for 66 hr using a TLA-110 rotor in a Bechman Coulter Optima MAX-E
ultracentrifuge. Fractions of $\sim$100 $\mu$L were collected from below by
displacing the DNA-CsCl-gradient buffer solution in the centrifugation tube
with water using a syringe pump at a flow rate of 3.3 $\mu$L s$^{-1}$
\citep{Manefield_2002} into Acroprep 96 filter plate (part no. 5035, Pall Life
Sciences). The refractive index of each fraction was measured using a Reichart
AR200 digital refractometer modified as previously described to measure
a volume of 5 $\mu$L \citep{Buckley_2007}. Buoyant density was calculated from
the refractive index as previously described \citep{Buckley_2007} using the
equation $\rho$=\textit{a}$\eta$-\textit{b}, where $\rho$ is the density of the
CsCl (g ml$^{-1}$), $\eta$ is the measured refractive index, and \textit{a} and
\textit{b} are coefficient values of 10.9276 and 13.593, respectively, for CsCl
at 20°C \citep{9780408708036} and correcting for non-CsCl salts in the gradient
buffer. A total of 35 gractions were collected from each gradient and the
average density between fractions was~0.0040 g mL$^{-1}$. The DNA was desalted
by washing with TE (5X 200 $\mu$L) in the Acroprep filter wells. DNA was
resuspeneded in~50 $\mu$L TE. 

\subsection{DNA Sequencing}
\subsubsection{PCR amplification of SSU rRNA genes} 
% Fakesubsubsection: SSU rRNA genes
SSU rRNA genes were amplified from gradient fractions (n = 20 per
gradient) and from non-fractionated DNA from soil. Barcoded primers
consisted of: 454-specific adapter B, a 10 bp barcode (Reference
90), a 2 bp linker (5’-CA-3’), and 806R primer for reverse primer
(BA806R); and 454-specific adapter A, a 2 bp linker (5’-TC-3’), and 515F
primer for forward primer (BA515F). Each PCR contained 1.25 U μl-1
AmpliTaq Gold (Life Technologies, Grand Island, NY; N8080243), 1X Buffer
II (100 mM Tris-HCl, 500 mM KCl, pH
8.3), 2.5 mM MgCl2, 200 μM of each dNTP, 0.5 mg ml-1 BSA, 0.2 μM BA515F, 0.2 μM
BA806R, and 10 μL of 1:30 DNA template in 25 μl total volume). The PCR
conditions were 95°C for 5min followed by 22 cycles of 95°C for 10s, 53°C for
30s, and 72°C for 30s, followed by a final elongation at 72°C for 5 min.
Amplification products were checked by 1\% agarose gel. Reactions were
performed in triplicate and pooled. Amplified DNA was gel purified (1\% low
melt agarose) using the Wizard SV gel and PCR clean-up system (Promega,
Madison, WI; A9281) per manufacturer’s protocol. Samples were normalized by
SequalPrepTM normalization plates (Invitrogen, Carlsbad, CA; A10510) and pooled
in equimolar concentration. Amplicons were sequenced on Roche 454 FLX system
using titanium chemistry at Selah Genomics (Columbia, SC).

\subsubsection{DNA sequence quality control}
% Fakesubsubsection: SSU rRNA gene sequences
SSU rRNA gene sequences were initially screened by maximum expected errors
at a specific read length threshold \citep{edgar2013}. Reads that had more
than~0.5 expected errors at a length of 250 nt were discarded. The
remaining reads were aligned to the Silva Reference Alignment as provided
in the Mothur software package using the Mothur NAST aligner
\citep{DeSantis2005,schloss2009}. Reads that did not align to the expected
region of the SSU rRNA gene were discarded. After expected error and
alignment based quality control. The remaining quality controlled reads
were annotated using the “UClust” taxonomic annotation framework in
\citep{caporaso2010,edgar2010}. We used 97\% cluster seeds from the Silva
SSU rRNA database (release 111Ref) \citep{quast2013} as reference for
taxonomic annotation (provided on the QIIME website) \citep{quast2013}.
Quality control screening filtered out 344,472 or 1,720,480 raw sequencing
reads. Reads annotated as "Chlorloplast", "Eukaryota", "Archaea",
"Unassigned" or "mitochondria" were culled from the dataset. 

\subsubsection{OTU binning} 
% Fakesubsubsection: Sequences were distributed
Sequences were distributed into OTUs with a centroid based clustering
algorithm (i.e. UPARSE \citep{edgar2013}). The centroid selection also
included robust chimera screening \citep{edgar2013}. OTU centroids were
established at a threshold of 97\% sequence identity and non-centroid
sequences were mapped back to centroids. Reads that could not be mapped to
an OTU centroid at greater than or equal to 97\% sequence identity were
discarded. 

\subsubsection{Phylogenetic reconstruction}
% Fakesubsubsection: For phylogenetic
We used SSU-Align \citep{nawrocki2009,nawrocki2013} to align SSU rRNA gene
sequences. Columns in the alignment that were aligned with poor confidence
($<$ 95\% of characters had posterior probability $>$ 95\%) were not
considered when building the phylogenetic tree. Additionally, the
alignment was trimmed to coordinates such that all sequences in the
alignment began and ended at the same positions. FastTree
\citep{price2010} was used with default parameters to build the phylogeny.
NMDS ordination was performed on weighted Unifrac \citep{lozupone2005}
distances between samples. The Phyloseq \citep{mcmurdie2013} wrapper for
Vegan \citep{oksanen2015} (both R packages) was used to compute sample
values along NMDS axes. The 'adonis' function in Vegan was used to perform
Adonis tests (default parameters) \citep{Anderson2001a}.

\subsection{OTU characteristics}
%
\subsubsection{Identifying $^{13}$C responders} 
% Fakesubsubsection: Figures S11 and S12
Figures~S11 and~S12 demonstrate raw data for responder and non-responder OTUs,
respectively. Responders increased in relative abundance in the heavy fractions
due to $^{13}$C-labeling of their DNA. As our data is compositional, often OTUs
had consistent \textit{relative} abundance across the density gradients. If OTU
DNA is positioned in heavy or light fractions, however, due to G$+$C content
and/or $^{13}$C-labeling, it spikes in relative abundance near where it is
centered. Thus, we identified responders by finding OTUs enriched in heavy
fractions of $^{13}$C treatment gradients relative to control. This technique
accounts for the variation in OTU base abundance and the variation in OTU G$+$C
content (and therefore natural buoyant density) because $^{13}$C treatment
abundances are always compared to appropriate control abundances.  

\subsubsection{Estimating rrn copy number}
% Fakesubsubsection: We estimated the rrn
We estimated the \textit{rrn} copy number for each OTU as described in \citet{Kembel_2012}
(i.e. we used the code and reference information provided in \citet{Kembel_2012}
directly). In brief, OTU centroid sequences were inserted into a reference SSU
rRNA gene phylogeny \citep{matsen2010} from organisms of known \textit{rrn} copy number.
The \textit{rrn} copy number was then inferred from the phylogenetic placement
in the reference phylogeny. 

\subsubsection{NRI, NTI, and consenTRAIT}
% Fakesubsubsection: NRI and NTI were calculated using
NRI and NTI were calculated using the ``picante'' R package
\citep{kembel2010}. We used the ``independentswap'' null model for
phylogenetic distribution. The consenTRAIT clade depth for xylose and cellulose
responders was calculated using R code used to calculate the metric in
\citet{Martiny2013} which employs the R ``adephylo'' package
\citep{jombart2010}.

\subsubsection{Buoyant density shift estimates}
% Fakesubsubsection: Upon labeling, DNA from
Upon labeling, DNA from an organism that incorporates exclusively $^{13}$C will
increase in BD more than DNA from an organism that does not exclusively utilize
isotopically labeled C. Therefore, the magnitude DNA $\Delta\hat{BD}$ indicates
substrate specificity given our experimental design as only one substrate was
labeled in each amendment (assuming all members of an OTU behave similarly with
respect to $^{13}$C incorporation). We measured $\Delta\hat{BD}$ as the change in
an OTU's density profile center of mass between corresponding control and
labeled gradients (Figure~S11). Because all gradients did not span the same
density range and gradient fractions cannot be taken at specific density
positions, we limited our BD shift analysis to the density range for which
fractions were taken for all gradients. Within this density range we linearly
interpolated~20 evenly spaced relative abundance values. The center of mass for
an OTU along the density gradient was then the density weighted average where
weights were the linearly interpolated relative abundance values.
$\Delta\hat{BD}$ should not be evaluated on an individual OTU basis as a small
number of $\Delta\hat{BD}$ values are observed for each OTU and the variance of
the density shift metric at the level of individual OTUs is unknown. It is
therefore more informative to compare $\Delta\hat{BD}$ among substrate
responder groups. Further, $\Delta\hat{BD}$ values are based on relative
abundance profiles and would be distorted in comparison to $\Delta\hat{BD}$
based on absolute DNA concentration profiles and should be interpreted with
this transformation in mind. It should also be noted that there was overlap in
observed $\Delta\hat{BD}$ between $^{13}$C-cellulose and $^{13}$C-xylose
responder groups. 

\subsection{Sequencing and density fractionation statistics}\label{seq_stats}
% Fakesubsubsection: Microcosm DNA was density
Microcosm DNA was density fractionated on CsCl density gradients. We sequenced
SSU rRNA gene amplicons from a total of 277 CsCl gradient fractions from 14
CsCl gradients and 12 bulk microcosm DNA samples. The SSU rRNA gene data set
contained 1,102,685 total sequences. The average number of sequences per sample
was 3,816 (sd 3,629) and 265 samples had over 1,000 sequences. We sequenced SSU
rRNA gene amplicons from an average of 19.8 fractions per CsCl gradient (sd
0.57). The average density between fractions was  0.0040 g mL$^{-1}$ The
sequencing effort recovered a total of 5,940 OTUs. 2,943 of the total 5,940
OTUs were observed in bulk samples. We observed 33 unique phylum and 340 unique
genus annotations.

% Fakesubsubseciton: bib
\bibliography{bibliography/biblio}

\end{document}
