marine and atmospheric SOM represents more C than the marine and atmospheric
C reservoirs combined and approximately 80\% of SOM flux is mediated by
microorganisms CITE. SOM decomposition is more sensitive to temperature changes
than primary productivity CITE. Climate change can affect terrestrial microbial
communities on a global scale CITE Garcia-Pichel. Therefore, climate change has
the potential to influence SOM flux and storage. Knowing how temperature
changes the rates at which C decomposes in soil is essential to predict how
climate change will affect SOM flux and storage. Additionally we need to
observe how climate change alters the abundance and activity of key microbial
players. Bur first, we must identify which microbial phylogenetic types
decompose different SOM C components.


'Heavy' fraction amplicon pools from samples that received $^{13}$C-xylose
diverged from corresponding controls on days 1 through 7 . Furthermore,
amplicon pool composition varied across these days indicating dynamic changes
in $^{13}$C-xylose assimilation with time. At days 14 and 30 heavy fractions
from $^{13}$C-xylose labeled samples are no longer differentiated from
corresponding controls indicating that $^{13}$C is no longer detectable in DNA.
The decline in $^{13}$C-labelling of DNA is likely due to isotopic dilution
resulting from assimilation of unlabeled C and/or due to cell turnover
resulting from mortality. 

There were 6 shared responders among all unique responders identified in both
the xylose and cellulose treatments (n = 72); Stenotrophomonas, Planctomyces,
two Rhizobiaceae, Comamonadaceae, and Cellvibrio. Of these, Stenotrophomonas
and Comamonadaceae are the only taxa that are among the top ten l2fc responses
measured in both treatments. On the other hand, the only shared responder that
is not among the top ten responders for either the cellulose or xylose
treatment is Rhizobiaceae. Two of the shared responders corresponded in time
between the two treatments

many taxa important to cellulose cycling are present in the rarer fraction of the
overall microbial community.

\textbf{Patterns of carbon use vary dramatically within phylum.} Dynamic
patterns of $^{13}$C-assimilation from xylose and cellulose occur at discrete,
fine-scale taxonomic units . Responders for xylose and cellulose are widespread
across 6 and 7 phyla, respectively . There are 5 phyla containing responders
for both treatments; of all the responder OTUS detected within those phyla for
either xylose or cellulose, there are only six OTUs that respond to both xylose
and cellulose (discussed previously). This result suggests that phyla do not
represent coherent ecological units with respect to the soil C-cycle, that is,
taxa within phyla exhibit differences in substrate use, level of substrate
specialization, and dynamics of incorporation. 

In this study, we have identified Actinobacteria responders for both substrates. 
Although there were no shared Actinobacteria OTUs that responded to both
xylose (Microbacteriaceae, Micrococcaceae, Cellulomonadaceae, Nakamurellaceae,
Promicromonosporaceae, and Geodermatophilaceae) and cellulose
(Streptomycetaceae and Pseudonocardiaceae). This information may suggest that
while Actinobacteria exhibit an ability to utilize an array of carbon
substrates, substrate use may be more clade specific and not widespread
throughout the phylum . Similarly, Bacteroidetes responders were identified for
both substrates, yet, at a finer taxonomic resolution there is a clear
differential response for xylose (Flavobacteriaceae and Chitinophagaceae) and
cellulose (Cytophagaceae). 

Whole phylum responses were not detected for xylose or cellulose yet
utilization of these substrates spanned many phylogenetically diverse groups.
Within each phylum we observed substrate utilization at the clade or single
taxa level with each exhibiting a unique pattern of $^{13}$C-assimilation over
time. It has previously been suggested that all taxa within a phylum are
unlikely to share ecological characteristics \citep{Fierer_2007}, and
furthermore, within a species population
\citep{Choudoir_2012,Preheim_2011,Hunt_2008}. Habitat traits of coastal Vibrio
isolates were mapped onto microbial phylogeny revealing discrete ecological
populations based on seasonal occurrence and particulate size fractionation
\citep{Preheim_2011,Hunt_2008}. Yet, it has been proposed that the microbial
community functionality responsible for soil C cycling appear at the level of
phlya rather than species/genera \citep{Schimel_2012}. The traditional phylum
level assignment conventions could in part be due to limitations in finer scale
taxonomic identifications or methodological limitations (\textit{i.e.}
sequencing depth). Our data in concert with others
\citep{Goldfarb_2011,Fierer_2007,Choudoir_2012,Preheim_2011,Hunt_2008} would
suggest that assigning substrate utilization of a few OTUs or clades as a
phylum level response is not accurate.

\textbf{Conclusions.} We have demonstrated how next generation
sequencing-enabled SIP gives an OTU level resolution for substrate utilization.
Using this technique, we are able to resolve discrete OTUs that would otherwise
be missed using bulk community sequencing efforts. Additionally, this technique
provides greater taxonomic resolution than previous techniques (cloning, TRFLP,
ARISA) used to determine substrate utilizing community members. While we are
currently able to resolve highly responsive OTUs, there is still a need to
resolve taxa that are partially responsive which we cannot differentiate from
noise with confidence at this time. Although, if we could identify partially
responsive taxa, their contributions to the C-cycle would still be difficult to
discern. For example, a generalist utilizing many substrates including $^{12}$C
substrates and the $^{13}$C-labeled substrate may exhibit the same partial
labeling that a specialist utilizing both the $^{13}$C-substrate and the same
substrate (unlabeled) that is inherent in the soil. Additionally, partially
labeled taxa could be further down the trophic cascade including predators or
secondary consumers of waste products from primary consumer microbes that were
highly labeled.   

OTUs that assimilate xylose and those that assimilate cellulose are largely
mutually exclusive. Those OTUs that assimilate xylose are labeled within 1-7
days, while those that assimilate cellulose are labeled primarily after 2-4
weeks. The xylose responders demonstrate a smaller change in BD than the
cellulose responders suggesting that xylose responders assimilate multiple C
sources (labeled and unlabeled) consistent with a generalist response, while
cellulose responders are more heavily labeled suggesting that cellulose is
their main source of C, a response more consistent with a specialist lifestyle.
Xylose responders include many taxa, such as spore-fomers, known for the
ability to respond rapidly to an influx of new nutrients while cellulose
responders include many OTUs that are common uncultivated soil organisms.
Finally, xylose responders are more abundant in the community while cellulose
responders are, on average, more rare as indicated by their rank abundance
within the soil community. These results indicate that different bacteria in
soil have distinct physiological and ecological responses which govern their
interactions with soil C pools. 

We did not observe consistent C utilization at the phylum level although both
xylose and cellulose utilization were observed across 7 phyla each revealing a
high diversity of bacteria able to utilize these substrates. The high taxonomic
diversity may enable substrate metabolism under a broad range of environmental
conditions \citep{Goldfarb_2011}. Other studies of microbial communities have
observed a positive correlation with taxonomic or phylogenetic diversity and
functional diversity
\citep{Fierer_2012,Fierer_2013,Philippot_2010,Tringe_2005,Gilbert_2010,Bryant_2012}.
The data presented here supports that specific functional attributes can be
shared among diverse, yet distinct, taxa while closely related taxa may have
very different physiologies \citep{Fierer_2012,Philippot_2010}. This
information adds to the growing collection of data suggesting that community
membership is important to biogeochemical processes. Furthermore, it highlights
a need to examine substrate utilization by discrete microbial taxa within a
whole community context to better understand how specific community members
function within the whole. 

The sensitivity of SIP-NGS provides a means to elucidate substrate utilization
by discrete microbial taxa with the hope that we can begin to construct a
belowground C food web. We obtained enough information to conclusively
determine isotope incorporation for 61\% of the more than 6,000 OTUs detected.
For those OTUs with enough information (n = 3,825), approximately 2\% (n = 72)
significantly assimilated $^{13}$C from either xylose or cellulose. In the
future deeper sequencing will enable us to increase coverage and assess C use
by more community members. Using the informations we gain from SIP-NGS, we can
expand our knowledge of specific C-cycling OTUs by taking a targeted
metagenomic approach in the nucleic acid pools of 'heavy' fractions.
Furthermore, we can now expand our knowledge of soil C use dynamics to a wide
array of C substrates and increase our grasp on specific community member
contributions. Illuminating these microbial contributions associated with
decomposition in soil are important because as environments change, there are
measurable and functional changes in soil C \citep{Grandy_2008} which could
cumulatively have large impacts at a global scale.
